<!--
The {#rmd-basics} text after the chapter declaration will allow us to link throughout the document back to the beginning of Chapter 1.  These labels will automatically be generated (if not specified) by changing the spaces to hyphens and capital letters to lowercase.  Look for the reference to this label at the beginning of Chapter 2.
-->

# Characterization of Pacific oyster (*Crassostrea gigas*) proteomic response to natural environmental differences

## Abstract

Global climate change is rapidly altering coastal marine ecosystems important for food production. A comprehensive understanding of how organisms will respond to these complex environmental changes can come only from observing and studying species within their natural environment. To this end, the effects of environmental drivers — pH, dissolved oxygen content, salinity, and temperature — on Pacific oyster (*Crassostrea gigas*) physiology were evaluated in an outplant experiment. Sibling juvenile oysters were outplanted to eelgrass and unvegetated habitat at five different estuarine sites within the Acidification Nearshore Monitoring Network in Washington State, USA to evaluate how regional environmental drivers influence molecular physiology. Within each site, we also determined if eelgrass presence that buffered pH conditions changed the oysters’ expressed proteome. A novel, two-step, gel-free proteomic approach was used to identify differences in protein abundance in *C. gigas* ctenidia tissue after a 29 day outplant by 1) identifying proteins in a data independent acquisition survey step and 2) comparing relative quantities of targeted environmental response proteins using selected reaction monitoring. While there was no difference in protein abundance detected between habitats or among sites within Puget Sound, *C. gigas* outplanted at Willapa Bay had significantly higher abundances of antioxidant enzymes and molecular chaperones. Environmental factors at Willapa Bay, such as higher average temperature, may have driven this protein abundance pattern. These findings generate a suite of new hypotheses for lab and field experiments to compare the effects of regional conditions on physiological responses of marine invertebrates.

## Introduction

Global climate change will influence estuarine dynamics and impact the organisms that inhabit these environments. Estuaries are already variable across spatial and temporal scales in terms of phytoplankton production [@Pennock1986], nutrient availability [@Paerl2014], heavy metal contamination [@Liu2015], salinity [@Banas2004], and carbonate chemistry [@Feely2010; @Pelletier2018]. Since climate change will affect these parameters, it is important to consider how estuarine organisms will respond.

Proteomics, or the study of protein abundance and expression, can be used to shed light on physiological changes on a molecular level. Proteins direct all major cellular functions, thus examining protein abundance provides direct evidence of an organism’s physiological response to the estuarine environment [@Tomanek2014]. The proteome is dynamic, as it must rapidly respond to perturbation, providing mechanistic information that standard gene expression and mRNA quantification studies cannot [@Veldhoen2012; @Flores-Nunes2015]. As a result of the proteome’s dynamic nature, proteins analyzed at the time of collection represent an organism’s response to the environment in near real-time. Long-term exposure to environmental conditions, as well as natural organismal aging, are also reflected in the proteome [@Hercus2003]. Discovery-based proteomic methods can elucidate responses to environmental drivers [@Flores-Nunes2015]. Several studies have connected protein abundances with changes in laboratory-simulated environmental conditions, identifying key proteins and mechanisms involved in specific environmental responses [@Timmins-Schiffman2014; @Dineshram2016; @Meng2017]. While these studies provide insight into organismal adaptation and physiology, laboratory studies alone cannot fully encapsulate the effects of multiple environmental drivers within an ecosystem context [@Riebesell2014]. 

Although challenging, in situ field studies provide a necessary biological realism when considering variable environments [@Slattery2012; @Cornwall2016]. Such experiments can be leveraged to study the effects of multiple environmental drivers on organismal physiology and to incorporate realistic variability, as opposed to examining the effect of a single stressor on an organism [@Riebesell2014]. Through transcriptomics, @Chapman2011 demonstrated the power of an in situ experimental design for examining the impacts of regional environmental conditions on Eastern oyster (*Crassostrea virginica*) physiology. Transcript signatures from *C. virginica* sampled from various locations in southeastern United States revealed that temperature, pH, salinity, dissolved oxygen and pollutant load at each location impacted gene expression. Furthermore, they were able to disentangle the interactions of these environmental factors on gene expression. RNA and protein abundances can be influenced by several environmental factors, and in situ studies can determine which drivers will be more important to consider for organismal physiology. 

Marine invertebrates have proven to be informative bioindicators in proteomic studies to examine the effects of in situ conditions on organismal physiological responses to environmental change. When marine invertebrates have been exposed to varying environmental conditions, proteomics have demonstrated changes in cellular defense, immune responses, and genome function [@Veldhoen2012]. Changes in protein abundance in bivalves like the Pacific oyster (*Crassostrea gigas*) and blue mussels (*Mytilus edulis spp*.) have been used to develop biomarkers for environmental contaminants [@Slattery2012; @Beyer2017]. Proteomic responses to natural environmental drivers have also been evaluated in bivalves. For example, shotgun proteomic analysis of *M. edulis* ctenidia from Baltic Sea microcosms revealed that low salinity conditions lead to decreased abundance of cytoskeleton proteins, as well as calcium-binding messenger calmodulin, which plays an important role in signalling and intracellular membrane trafficking pathways [@Campos2016]. Using a growing wealth of genomic information to understand how these species fare under differential environmental conditions is critical for monitoring natural populations and commercial aquaculture.

Pacific oyster (*Crassostrea gigas*) rearing in estuarine environments in Washington State, USA (WA) provides an ideal system to examine the effect of in situ environmental conditions on the expressed proteome. *C. gigas* are extensively farmed in two different estuarine systems that show substantial regional variation: Puget Sound and Willapa Bay. Puget Sound is a complex estuarine system with interconnected sub-basins, each with different freshwater inputs, residence times, and stratification levels [@Feely2010; @Bianucci2018; @Pelletier2018]. Willapa Bay is a large shallow estuary on the Pacific coast that exchanges approximately half its water volume with the Pacific Ocean at each tide [@Banas2004; @Banas2007]. Seasonality and location within Puget Sound dictates temperature, dissolved oxygen, salinity, and pH conditions, while Willapa Bay conditions are influenced by diurnal fluctuations and proximity to either the ocean or rivers draining into the bay [@Banas2007; @Feely2010; @Ruesink2015]. 

Both Puget Sound and Willapa Bay also host eelgrass beds (*Zostera spp*.) that affect environmental conditions, such as oxygen concentrations, on diurnal time scales. The 2012 Washington State Blue Ribbon Panel on Ocean Acidification outlines key early actions, which include the examination of “vegetation-based systems of remediation” (Action 6.1.1) to improve local pH through photosynthetic drawdown of carbon dioxide. This experiment set out to test whether protein abundance patterns reflect reduced stress within vegetation. For example, eelgrass beds may reduce emersion stress relative to unvegetated areas through shading, the retention of water, and increased evaporative cooling at low tide. They can also ameliorate effects of ocean acidification through photosynthetic activity. Reduced pathogen prevalence has also been documented in seagrass beds, but not specifically in eelgrass [@Lamb2017]. In contrast, eelgrass beds may drive more extreme carbonate chemistry conditions [@Pacella2018]. @Lowe2018 also found that *C. gigas* shell strength and survival was significantly lower in eelgrass habitats in WA. Understanding how different aquaculture grow-out locations and habitats will affect the oyster’s ability to persist through environmental change is crucial for the industry and the ecosystem.

The purpose of this study was to use proteomic techniques to uncover the impacts of environmental drivers on Pacific oyster physiological outcomes in estuarine environments in WA. Naturally existing environmental variation was harnessed by outplanting *C. gigas* in different locations within Puget Sound and Willapa Bay, and habitat effects were taken into consideration by placing oysters in eelgrass and unvegetated habitats. Gel-free proteomic methods were used to examine the effects of outplant conditions on relative quantities of all expressed proteins in a series of in situ experiments in order to identify differentially abundant proteins. We predicted that differences in environmental drivers at each outplant location and within outplant habitats would yield unique protein abundance patterns. Oysters at outplant locations with warmer water temperatures, more variable water temperatures, lower dissolved oxygen content, lower salinity, or lower pH may have higher abundances of proteins related to environmental response. Eelgrass beds were expected to ameliorate stressful conditions, resulting in lower abundances of environmental stress response proteins than oysters in unvegetated habitats.

## Methods

### Shellfish Deployment

Sibling juvenile *C. gigas* (average shell length = 27.2 mm, age = 2 months) were outplanted for 29 days starting June 19, 2016 at five locations: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB) in Washington State, USA (Table \@ref(tab:deployment)). These sites were selected for differences in environmental parameters, as well as for the presence of unvegetated areas and eelgrass beds within each site. All sites were part of the Acidification Nearshore Monitoring Network, a network of sensors placed in various WA locations to monitor marine chemistry (ANeMoNe; Washington Department of Natural Resources). Prior to the outplant, oysters were reared in a controlled hatchery setting. At each site and habitat combination, custom-built Durafet-based sensors (Honeywell) were used to monitor pH. Commercially-available MiniDOT loggers (Precision Measurement Engineering) were used to measure dissolved oxygen, and Odyssey loggers were used to measure conductivity. All sensors recorded temperature measurements, and all sensors logged at 10-minute intervals across the outplant period, with the exception of SK, where sensors were installed two days into the outplant period. At each site, juvenile oysters were placed in bags of five oysters each directly onto the substrate at a tidal height of -1.5 MLLW, both inside and outside of eelgrass beds (n=15 per habitat type), for a total of thirty outplanted oysters per site. The animals were uniformly placed less than a lateral distance of 0.5 m from the sensors at the same tidal height as the instruments. Oysters were housed in exclusion cages to prevent predation. Juvenile oysters remained at each site for a 29-day exposure period. Because the ctenidia is the primary site where oysters interact with the environment, ctenidia samples were dissected at the end of the outplant and held on dry ice until storage at -80ºC [@Beyer2017; @Meng2017). 

Environmental data was treated as follows. Conductivity observations were removed when less than zero, which occurs when the instrument is dry at low tide. Remaining observations were converted to salinity measurements using the `swSCTp` function in the `oce` package in R [@Kelley2018; @R2018], with temperature at 25ºC and pressure at 10 dbar. For dissolved oxygen, pH, and salinity datasets, data were removed when collected by probes 1) during low tide or 2) when tidal depth was less than one foot to remove readings where the probes may have been exposed. Values collected during low tide or a depths less than one foot were retained for temperature datasets. Outliers were screened using the Tukey method for temperature, dissolved oxygen, pH, and salinity datasets [@Hoaglin1986]. Uniform outplant tidal heights were checked using `prop.test` in R [@R2018]. 

A non-metric multidimensional scaling analysis (NMDS) was used to evaluate differences in environmental parameters. First, mean and variance were calculated for each day of the outplant. Values were log-transformed, and a separate Gower’s distance matrix was calculated for daily mean and daily variances, accounting for missing data. The NMDS was conducted with the Gower’s distance matrix to visually compare means or variances between sites and habitats. Significant differences between site and habitat were identified using a One-way Analysis of Similarities (ANOSIM) for each environmental parameter. Pairwise ANOSIM tests for significant one-way ANOSIM results and two-way ANOSIM tests by site and habitat were not conducted due to lack of replicates within each site-habitat combination. R Scripts are available in the supplementary Github repository [@Venkataraman2018].

### Protein Discovery

To identify select protein targets for characterization across locations and environmental conditions, a subset of tissue samples were analyzed with data independent acquisition (DIA) mass spectrometry analysis. Two tissue samples were used from each site to make a peptide library and maximize the amount of protein abundance data collected from each sample. 

#### Protein Quantification

Tissue samples were homogenized in a solution of 50 mM NH4HCO3 with 6M Urea (500µl). Tissues were then sonicated 3 times (Fisher Scientific Sonic Dismembrator Model 100) for 10 seconds each and cooled between sonications in a bath of ethanol and dry ice. Protein quantities were measured with the Pierce BCA Protein Assay Kit microplate assay with a limited quantity of sonicated sample (11 µL). The protein concentration was measured via spectroscopy at 540 nm in a Labsystems (Waltham, MA) Multiskan MCC/340 and accompanying Ascent Software Version 2.6. Protein concentrations were calculated based on a standard curve with BSA (Pierce) per manufacturer’s instructions.

#### Protein Digestion

Protein digestion followed the protocol outlined in @Timmins-Schiffman2013. To each sample of 30 µg protein,1.5 M Tris pH 8.8 buffer (6.6 µL) and 200 mM TCEP (2.5 µL) were added. After solvent additions, each sample’s pH was verified to be basic (pH ≥ 8), and placed on a 37ºC heating block for one hour. Iodoacetamide (200 mM, 20 µL) was then added to each sample to maximize digestion enzyme access to protein cleavage sites. Samples were covered with aluminum foil to incubate in the dark for 1 hour at room temperature. Afterwards, diothiolthreitol (200 mM, 20 µL) was added and samples were incubated at room temperature for one hour. Lysyl Endopeptidase (Wako Chemicals) was then added to each sample in a 1 µg enzyme:30 µg oyster protein ratio, followed by one hour of incubation at room temperature. Urea was diluted with NH4HCO3 (25 mM, 800 µL) and HPLC grade methanol (200 µL). Trypsin (Promega) was added to each sample in a 1 µg trypsin: 30 µg oyster protein ratio for overnight digestion at room temperature.

#### Peptide Isolation

After overnight incubation, samples were evaporated to near dryness at 4ºC with a speedvac (CentriVap ® Refrigerated Centrifugal Concentrator Model 7310021). Samples were then reconstituted in 100 µL of a 5% Acetonitrile and 0.1% Trifluoroacetic Acid (Solvent A) to isolate peptides. If samples were not at pH ≤ 2, 10-20 µL aliquots of 10% Formic Acid were added until this pH was achieved.

Before desalting peptide samples, Macrospin C18 columns (The Nest Group) were prepared by adding 200 µL of a 60% Acetonitrile with 0.1% Trifluoroacetic Acid (Solvent B). The columns were spun for three minutes at 2000 rpm, and flow-through liquid from the column was discarded. The spinning and discarding process was completed a total of four times.To wash columns, 200 µL of Solvent A was added to each column. The columns were once again spun for three minutes at 2000 rpm and liquid was discarded afterwards; the solvent addition, spinning, and discarding process was completed a total of three times.

To bind peptides to the columns, digested peptides were added to prepared columns, then the columns were spun at 3000 rpm for three minutes. The filtrate was pipetted back onto the column and spun again at 3000 rpm for three minutes. Solvent A (200 µL) was added to each column three separate times, then the column was spun for three minutes at 3000 rpm to wash salts off the column. 

Peptides were eluted with two additions of 100 µL Solvent B to each column. Columns were spun at 3000 rpm for three minutes and the peptide fraction (filtrate) was reserved. Samples were placed in a speed vacuum at 4ºC until they were nearly dry (approximately two hours) to dry peptides. Peptides were reconstituted with 60 µL of 3% Acetonitrile + 0.1% Formic Acid, and stored at -80ºC.

#### Internal Standard Addition

Peptide Retention Time Calibration (PRTC; Pierce) is used as an internal standard to ensure consistency of peptides detected and measured throughout a mass spectrometry run. The stock solution of PRTC was diluted to 0.2 pmol/µL using 3% Acetonitrile with 0.1% Formic Acid. In a clean centrifuge tube, 6 µg of oyster protein and 0.376 pmol of PRTC were mixed together as per the PRTC user guide. Sample volume was brought up to 15 µL using a 3% acetonitrile and 0.1% formic acid solution. A quality control solution was also prepared (1 µL PRTC + BSA:3 µL 3% Acetonitrile and 0.1% Formic Acid solution).

### Data Independent Acquisition Mass Spectrometry

Peptides were analyzed on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) using Data Independent Acquisition Mass Spectrometry (DIA). DIA analyses were completed as a comprehensive, non-random analytical method for detecting peptide ions present within a sample to create a peptide library. The peptide library was then leveraged to develop a targeted proteomics assay for quantification (see *Selected Reaction Monitoring Assay*). A 30 cm analytical column and 3 cm pre-column were packed in-house with 3 µm C18 beads (Dr. Maisch). Samples were run in a randomized order. A blank injection followed each sample, with procedural blanks run at the very end. Every injection was 3 µL, which included 1 µg of oyster protein and 0.0752 pmol of PRTC. Peptides were analyzed in MS1 over the m/z range of 450-950 with 12 m/z wide windows with 5 m/z overlaps [@Egertson2013]. MS1 resolution was 60000 and AGC target was 400000 with a three second cycle time. The MS2 loop count was set to 20 and MS2 data was collected with a resolution of 15000 on charge state of 2 with an AGC target of 50000. No dynamic exclusion was used.  

### Peptide-Centric Proteomic Analyses

Unknown peptide spectra from mass spectrometry samples were matched with known peptides using Peptide-Centric Analysis in the PECAN software [@Ting2015]. Raw mass spectrometry files were converted to mzML files, then demultiplexed using MSConvert [@Chambers2012]. The C. gigas proteome was digested with in silico tryptic digest using Protein Digestion Simulator [@Riviere2015]. All known peptides from the mzML files were identified in comparison to the digested *C. gigas* proteome [@Riviere2015].

The PECAN-generated spectral library (.blib) file was used to detect peptides of interest in raw DIA files in Skyline [@MacLean2010]. Skyline identified peptides using chromatogram peak picking, where ions that elute at the same time and mass are detected as a peptide (file available at Panorama Public). All PRTC peptides and approximately 100 different oyster proteins and their peptide transitions were manually checked for retention time and peak area ratio consistency to determine a Skyline auto peak picker error rate (24.3% ± 25%, range: 0% to 100%).

Proteins had to satisfy four criteria to be considered appropriate targets for the study. 1) After an extensive literature search, functions related to oxidative stress, hypoxia, heat shock, immune resistance, shell formation, growth, and cellular maintenance were determined useful for evaluating environmental response. Proteins with annotations matching these functions were considered potential targets. 2) Protein data was then evaluated in Skyline to ensure there was no missing data for any peptide or sample. 3) Peaks with significant interference from other peptides were not considered. 4) Proteins needed at least two peptides with three transitions per peptide to quality as a potential target for downstream assays. The fifteen proteins (41 peptides and 123 transitions) that matched all of these criteria were selected as targets (Table \@ref(tab:targets)).

### Selected Reaction Monitoring Assay

Following the protein discovery phase (DIA), proteins were isolated as described above from an additional five randomly selected samples per site and habitat combination (for a total of 5 oysters per group) and analyzed with Selected Reaction Monitoring (SRM). Samples were prepared as described for DIA, except tissue samples were homogenized in 100 µL, and peptide samples were evaporated at 25ºC after peptide isolation.
Proteins of interest identified from the DIA analysis were used as targets in a SRM assay following the workflow and informatic pipeline of [@Timmins-Schiffman2016]. Target peptide transitions were monitored using SRM on a Thermo TSQ Vantage. SRM data were collected during a gradient of 2-60% acetonitrile over 40 minutes. All samples were run in technical duplicates in a randomized order with a 1 µg oyster peptide and 0.0752 pmol PRTC injection. A quality control injection and blank injection were run after every five sample injections, and PRTC peptides were monitored throughout the experiment.

#### Target Peptide Specificity

To ensure SRM assay specificity to oyster peptides of interest, oyster peptides were diluted in a background matrix of similar complexity (Pacific geoduck — *Panopea generosa* — peptides), then analyzed using the oyster SRM assay. An oyster-specific SRM target would decrease in abundance with a decreasing abundance of oyster peptides in a mixture. Non-specific peptides — more likely to be found in background matrix of similar complexity — or peptides susceptible to interference would not correlate with oyster peptide abundance, and therefore, would be uninformative. Five *C. gigas* samples used for SRM were randomly selected and pooled in equal quantities. A ten-sample oyster:geoduck dilution series was prepared and run using the same methods as other SRM samples.

#### Target Analysis

Raw SRM files, a background *C. gigas* proteome, and the PECAN spectral library file from DIA were used to create a Skyline document (file available at Panorama Public). Correct transition peaks were selected based on predicted retention times from DIA results by comparing the relative retention times between identical PRTC peptides in the DIA and SRM datasets (R^2^ = 0.99431). Based on peptide specificity analyses, heat shock protein 70 B2 and one constituent peptide of 
glucose-6-phosphate 1-dehydrogenase were removed from analyses.

Further filters were applied to the data to maintain only high quality peptides and transitions in the analysis. Coefficients of variation were calculated between technical replicates for each peptide transition. Peptides were removed from the dataset if CV > 20%. To maintain high sample quality, any sample missing data for more than 50% of peptide transitions was deemed poor quality for downstream analyses and excluded. Abundance data was normalized using total ion current (TIC) values from the mass spectrometer. Consistency between technical replicates was verified in remaining samples using a NMDS with TIC-normalized data and a euclidean dissimilarity matrix. Technical replicates were consistent if replicates lay closer together than to other samples in the NMDS. These replicates were then averaged for multivariate analytical methods.
	
Averaged technical replicate data was used to determine if peptides were differentially abundant between outplant sites and habitats. Before proceeding with analysis, peptide abundances were subjected to a Hellinger transformation to give low weights to any peptides with low counts. A NMDS was used to visually compare relative peptide abundance. One-way ANOSIM tests by site, region (Puget Sound vs. Willapa Bay), and habitat, as well as a two-way ANOSIM test by site and habitat, were used to determine significant differences. Pairwise ANOSIM tests and post-hoc similarity percentage (SIMPER) analyses were conducted for each one- or two-way ANOSIM result significant at the 0.05 level. The first ten SIMPER entries were deemed influential peptides for each significant comparison.

The importance of environmental variables for explaining peptide abundance was evaluated with a redundancy analysis (RDA). For each site and habitat combination, mean and variance were calculated for pH, dissolved oxygen, salinity, and temperature over the course of the entire outplant. Environmental variables were then used as predictors to constrain peptide abundance. Predictors with missing values were not included. A triplot was used to visually assess differences in peptide abundance by site and habitat and the influence of individual peptides and environmental parameters. Analysis of Variance (ANOVA) was used to calculate significance of the RDA and environmental variables, with predictors deemed significant at p < 0.05. Since estuarine sites are highly variable, a second RDA was conducted constraining peptide abundance by environmental conditions on the day of collection to evaluate robustness of proteomic methods. R Scripts are available in the supplementary Github repository [@Venkataraman2018].






##Results

Bulk soil stable isotope analysis indicated that salmon carcasses enriched the N isotope pools (Table 1). $\delta^{15}N$ values peaked between 3 and 6 m from the stream edge, which was the distance salmon were typically relocated to during the experiment and declined at distances greater than 6 m. Maximum $\delta^{15}N$ of bulk soils was 11.8‰ for the salmon enhanced bank and 11.6‰ for the salmon depleted bank and no observations exceeded the sockeye salmon end-member value of 12.6‰ (Figure \@ref(fig:modIsotope)a). $\delta^{13}C$ was more enriched at greater distances from the bank and on average was highest at 20 m (Figure \@ref(fig:modIsotope)b). $\delta^{13}C$ was primarily governed by distance, with some evidence [N~tot~] and bank also had an effect (\@ref(tab:suppmod1)). 

Salmon carcass manipulation also enriched $\delta^{15}N$ of soil NH~4~^+^. Stable isotope values were enriched at 3 m from the stream edge on the salmon enhanced bank, and declined at distances > 3 m. On the salmon depleted bank, $\delta^{15}N$ of soil NH~4~^+^ was most enriched at 1 m and declined with distance (Figure \@ref(fig:modsupp1.2)C). The only model with support contained a quadratic interaction of distance and bank, which provides strong evidence that $\delta^{15}N$ of NH4+ was affected by salmon (Table \@ref(tab:suppmod1)). In contrast to bulk soil N, $\delta^{15}N$ values of NH~4~^+^ exceeded the salmon endmember of 12.6‰ for 23% of all observations (n=21).

Inorganic nitrogen concentrations were primarily governed by bank and GW (Table \@ref(tab:suppmod1)). The salmon enhanced bank had a higher mean [NH~4~^+^] and [NO~3~^-^] compared to the salmon depleted bank (Figure \@ref(fig:modConc)d, e). The most supported models for both [NH~4~^+^] and [NO~3~^-^] showed evidence for H1, that observed differences were not caused by salmon. For [NH~4~^+^] there was substantial model uncertainty, with six competing models receiving relative support (ΔAIC < 2) (Table \@ref(tab:suppmod1)) but none of the competing models supported a salmon effect. Two competing models for [NO~3~^-^] supported a site variability effect and one competing model supported a salmon effect (Tabe \@ref(tab:suppmod1)) and all three contained gravimetric water content as a covariate. This indicates [NH~4~^+^] was driven by site factors unrelated to salmon while [NO~3~^-^] was driven by gravimetric water content and with some support for salmon enhancement.

Nitrogen transformation rates were unaffected by salmon carcass manipulation. Both net nitrification and net mineralization models with relative support contained N substrate ([NH~4~^+^] and [N~org~] respectively), and the models with the most support did not include distance or bank. Net mineralization had some model uncertainty, with four models receiving relative support; however, all of the competing models supported either H1 or H3 with no support for a salmon effect. [N~org~] was the only covariate included in all of the competing models, indicating [N~org~] was the most important covariate tested for determining net mineralization. Net nitrification had greater model certainty and both models that received relative support contained [NH~4~^+^] and gravimetric water content. Similar to net mineralization, these models supported H1 and H3 with no support for H2, the salmon effect, though net nitrification was slightly higher on average between 3 – 6 m on the salmon enhanced bank (Table \@ref(tab:suppmod1)). Overall, these results demonstrated the manipulation of salmon carcasses did not have clearly detectable effects on N transformation rates.

Both [N~org~] and GW indicated there are site differences caused by distance and bank unrelated to salmon carcass manipulation. On average [N~org~] was higher on the salmon depleted bank than the salmon enhanced bank. There was model support of H1 for both GW and [N~org~], indicating these variables decrease with distance (Table \@ref(tab:suppmod1), Figure \@ref(fig:modConc) h, i). While there was some evidence that there was both a distance and bank effect on GW, it was not caused by salmon as the salmon enhanced bank does not show a peak GW at 3 - 6 m from the stream, which was where there was the highest observed isotopic enrichment and expected MDN. However, one competing model for [N~org~] did support H2, indicating site factors and salmon may both affect [N~org~]. However, the mean [N~org~] for the salmon enhanced bank was 18.42 mg/g and 18.97 mg/g for the salmon depleted bank indicating salmon decrease [N~org~], if they affect it at all.\newline 

C:N, percent nitrification, and percent carbon indicate relatively high nitrogen availability across sampling sites in the Hansen Creek system. Mean percent carbon was 24.2 and 24.9 on the enhanced and depleted banks respectively (S3). Soil C:N of bulk isotopes was less than 20 for all sites, with a mean of 15.8 (enhanced) and 14.2 (depleted). These values are well below the critical microbial C:N threshold of 29, demonstrating N is more available to meet microbial metabolic demands relative to C (Figure \@ref(fig:modConc)j). In contrast, percent nitrification was relatively high with a mean of 64% and 62% on the enhanced and depleted banks (S3).  


##Discussion

This study confirmed that MDN was both present in soils and increased on the bank enhanced with salmon carcasses for 20 years. However, plant-available inorganic N pools and N transformation rates measured in soil during the peak growing season immediately prior to the annual return of salmon were largely unaffected by salmon enhancement. Even though the salmon enhanced bank had increased net nitrification compared to the salmon depleted bank, our analysis found no pattern with distance from the stream, suggesting that elevated nitrification was caused by bank characteristics unrelated to salmon carcass density. Given numerous conventional long-term fertilization experiments worldwide have shown a consistent pattern of elevated soil inorganic N pools and N transformations, [@Hogberg2006; @Lu2011], it was surprising that 20 years of MDN inputs did not clearly accelerate soil N cycling in our study. Soils are the dominant (> 70%) sink for added N in forests worldwide [@Templer2012] and tree growth in high latitude conifer forests is often strongly N-limited [@Nordin2001], both of which should have fostered retention of salmon N inputs to our site. Indeed, the 20 years of cumulative salmon N additions in the zone near the stream in our study (~ 6,690 kg N/ha) greatly exceeded typical riparian surface soil N pools (500 to 2500 kg N/ha) [@Walker1989; @Morris2011; @Perry2017], suggesting that even partial retention of salmon N inputs in soils should have increased soil [N~org~]. The lack of increase in soil [N~org~] due to salmon that we observed is consistent with the lack of increase in N availability, because soil [N~org~] fuels long-term changes in N availability and recycling via plant uptake, litterfall, and decomposition [@Chapell1999; @Perakis2011; @Perakis2012].  Combined with observations of low C:N and high percent nitrification, this indicates N from salmon subsidies is not being retained in this system. Overall, the lack of increase in soil organic and inorganic N concentrations and N transformations that we observed following 20-year salmon manipulation raises questions of whether plant growth responses should be expected at our site.

Prior work at Hansen Creek inferred that MDN stimulated white spruce growth based on tree ring analyses [@Quinn2018]. However, substantial salmon enhancement corresponding to approximately 669 g/m^2^ (6,690 kg/ ha) of N and 113 g/m^2^ (1,130 kg/ha) of P over the past 20 years was unable to overcome pre-treatment differences in forest growth between banks. For reference, it is estimated white spruce in floodplain stands require approximately 1.35 g/m^2^/y of N [@Chapin2006], which was far exceeded by the mean change of 33.45 g/m^2^/y of N added from this manipulation. Additionally, fertilization experiments apply N on the order of 100 - 1,000 kg/ha with clear results [@Chapell1999], a much lower application rate than in this study. Factors such as climate, stand demography, and site and landscape variability also affect tree growth in this system. Indeed, white spruce growth response to recent warming across southwest Alaska depends strongly on tree density [@Wright2018]. Basal area density is highly variable across our site, differing on average 40% between salmon-enhanced and salmon-depleted banks, although the difference was not statistically significant [@Quinn2018]. Ultimately, the hierarchy of drivers of tree growth in this ecosystem appears to be landscape position (and associated forest demography) followed by climate and thirdly, nutrients. All told, a lack of long-term changes in soil nutrient dynamics and only marginal response in tree growth indicates that salmon nutrients are not a strong bottom-up force in northern riparian forest dynamics. 

Our ^15^N/^14^N stable isotope data raise further questions of assessing MDN subsidies to tree growth. Vegetation typically takes up only 17% of added N to forests, with soils instead being the dominant N sink [@Templer2012]. Thus, elevated bulk soil ^15^N/^14^N in our study suggests a potentially significant MDN sink in soil. On the other hand, elevated bulk soil ^15^N/^14^N may also reflect increases in soil N fractionation during N cycling and loss under salmon. Highly localized N pulses (as occur with MDN and other N subsidies) temporarily exceed plant and soil N sinks, leading to accelerated N loss via ammonia volatilization, nitrification and nitrate leaching, and/or denitrification [@Perakis2002]. All of these N loss pathways favor 14N and discriminate against 15N (in some cases with a fractionation up to 30‰), and effects are strongest at high N availability, leading to high values of residual soil 15N [@Hogberg1998]. Prior work has shown that MDN inputs accelerate N losses from soil, particularly gaseous N losses [@Holtgrieve2009] that are associated with large isotope fractionation [@Hogberg1998]. Our finding that $\delta^{15}N$ of soil NH~4~^+^ was greater than bulk soil $\delta^{15}N$ for 95% of observations on the salmon enhanced bank and 84% of observations on the salmon depleted bank, further confirms that isotopic fractionation is important at Hansen Creek and likely elsewhere. 

There is a global trend for higher foliar $\delta^{15}N$ with increased soil N supply [@Craine2009] indicating accelerated soil N cycling and $\delta^{15}N$ fractionation due to exogenous N (from salmon or elsewhere) will alter plant foliar $\delta^{15}N$. This has important implications for using two-source mixing models to assess salmon N subsidies to riparian forests. Typical MDN mixing models assume 1) the isotopic signature of salmon is unchanged in the soils prior to plant uptake, and 2) reference sites are biogeochemically similar to salmon sites. However, our data suggest that both of these assumptions are violated at Hansen Creek, and are likely violated at all salmon-influenced riparian ecosystems. First, we observed that $\delta^{15}N$ of NH~4~^+^, the dominant form of inorganic N in our soils, exceeded the 12.6‰ salmon end-member for 26% of our observations from the salmon enriched bank and 9% of observations from the salmon depleted bank, thus violating assumption (1) above. Our soil N data indicate Hansen Creek is a site of intermediate fertility relative to other boreal forests, so that soil NH~4~^+^ (rather than organic N or NO~3~^-^) is most likely the dominant N source taken up by plants [@Chapin2011]. Second, [N~Org~], C:N, $\delta^{13}C$, and GW varied with distance from the stream independent of salmon enhancement indicating site variability is a dominant driver of N cycling in this system. This presents a challenge for selecting control sites to calculate terrestrial end members, as key N cycling factors vary longitudinally away from streams and simply selecting reference sites that are beyond the reach of salmon would likely violate the mixing model assumption of biogeochemical similarity. Additionally, observations of $\delta^{13}C$ increasing and GW decreasing from the creek edge are consistent with higher water use efficiency and less 13C discrimination by vegetation, resulting in higher $\delta^{13}C$ in soil due to litterfall [@Oltean2016]. These data identify systematic differences between salmon-enhanced vs. salmon-depleted banks that cannot be attributed to salmon, and which likely reflect landscape or soil differences. Previous studies examining contributions of MDN to riparian vegetation have not tested biogeochemical similarity across sites, an assumption that is likely violated beyond Hansen Creek specifically.  

Violation of mixing model assumptions can lead to significant bias in calculations of MDN sources. To illustrate this point, we applied a typical mixing model framework to our maximum observed $\delta^{15}N$ of NH~4~^+^ values to calculate the percent MDN contribution of salmon to NH~4~^+^ for the most extreme observation, representing the greatest possible bias in calculations. Assuming soil processes have no effect on the isotopic signature yielded impossible result of 298% MDN contribution. To account for isotopic fractionation in soils, we applied our mean observed $\delta^{15}N$ of soil NH~4~^+^ at the 3 m distance (19.25‰) as the marine endmember to mean foliar 15N data at the same site from @Quinn2018 and estimate 59.24% MDN on the salmon bank, which is 27.6% lower than the original estimate of 86.8% using salmon $\delta^{15}N$ as the marine endmember. Repeating this with our maximum observed value for $\delta^{15}N$ of NH~4~^+^ (41.2‰), we estimate only 28.9% of foliar N on salmon enhanced bank was MDN (a 57.9% reduction from @Quinn2018 estimates). Thus, failure to account for isotopic enrichment associated with soil N transformations can lead to overestimates of MDN contributions to plants, and observed variability in $\delta^{15}N$ of NH~4~^+^ can produce a wide range of MDN estimates not previously considered. Given that our elevated $\delta^{15}N$ of NH~4~^+^ values are consistent with expected changes during soil N transformation [@Hogberg1998], there is a distinct possibility that previous MDN studies have overestimated the amount of MDN by not considering the effects of ^15^N/^14^N fractionation in mixing model calculations. @Wheeler2017 found similar results in a semi-arid ecosystem of central Idaho, where accounting for fractionation from decomposition resulted in a 16% reduction in estimated N deposition rates from salmon carcasses. The effects of fractionation on soil N pools is occurring in both of these systems, and likely elsewhere, and needs to be considered when applying mixing models to MDN data to avoid overestimations of salmon N contributions to riparian systems. 

Our study is comprehensive in terms of the number of ecosystems factors considered but limited in that it includes only one seasonal timeframe. As much as 40% of the annual inorganic N flux is released during the eight-month dormant season (September-May) and it has been posited spring and fall may be important for many biogeochemical processes in boreal forests [@Hobbie1996; @Chapin2006; @Drake2006]. While MDN inputs do not affect the N pools and transformation rates during the summer growth period based on our results, N concentrations and transformations may be elevated in this system on shorter timescales (weeks to months after salmon return). The objective of this study was to identify the long-term legacy of salmon subsidies; short-term effects were both beyond the scope of this study and have been previously investigated in this system [@Holtgrieve2009]. Considering long-term effect of N subsidies as opposed to short-term provides new information on sustained N use and retention in the ecosystem and whether these salmon nutrients have lasting impacts on ecosystem function meaningful in a restoration context. 

While this study is limited to one system, the results that N transformations cause a fractionation that can bias MDN mixing model estimates and that landscape factors are the primary driver of long-term N retention and use, are relevant to other systems where anadromous, semelparous salmon are abundant (Pacific, Atlantic, Great Lakes) [@Quinn2018]. This result also agrees with related research examining fractionation of mineralization and nitrification [@Hogberg1998], and fertilization studies [@Lu2011]. Additionally, it demonstrates salmon N subsidies may have a short-term and likely small spatial scale [@Drake2006] legacy in soils. While the importance of site variability relative to salmon subsidies may vary by system, this work demonstrates the importance of considering site variability and demonstrating biogeochemical similarity when selecting control sites for riparian MDN studies. 

Salmon provide critical food resources to many of terrestrial and aquatic consumers [@Cederholm1999; @Gende2002; @Schindler2003], but the evidence that MDN stimulate terrestrial primary production is less certain. The salmon carcass manipulation experiment described here and in @Quinn2018 represents an extreme case of carcass addition and depletion to riparian areas, as measured by bulk $\delta^{15}N$ and estimated percent contribution was approximately twice previous studies for both trees and soils [@Helfield2002; @Bartz2005]. Generally, results of this manipulation were equivocal for soils and a statistically significant but ecologically small effect on trees [@Quinn2018]. Simultaneously, other recent changes to boreal forest systems, such as moisture and temperature, appear to have a greater potential than MDN to alter biogeochemical pathways and primary production in these systems [@Chapin2006; @Yarie2008; @Lloyd2013; @Wright2018]. This study also demonstrates the importance of testing biogeochemical and site similarity between experimental and control sites in nutrient subsidy studies, as even banks on the same creek can have landscape and soil variability that alter N concentration, transformations, and vegetative growth. Altogether, while salmon have clear benefits for consumers, management of salmon populations or application of compensatory restoration strategies based on terrestrial productivity response to salmon inputs may be unfounded for some systems, and at least, hard to predict. 

## Tables
**Table** \@ref(tab:deployment):  Latitude and longitude of *C. gigas* outplants, as well as time of day and tidal height at collection. Oysters were placed at five locations sites: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB).

```{r deployment,echo=FALSE}
deployment <- read.csv("data/Ch1/Table1.1.csv")
  kableExtra:::kable_styling(knitr::kable(deployment, "latex",col.names = gsub("[.]", " ", names(deployment)), 
                  caption = "Outplant locations and time of day and tidal height at collection",
                  longtable = TRUE, booktabs = TRUE), font_size = 10, bootstrap_options = c("striped"))
```
\clearpage

**Table** \@ref(tab:targets): Proteins used as targets for a Selected Reaction Monitoring Assay (SRM). Targets were identified based on differential abundance and stress-related annotations. At least two peptides and six transitions were included in the assy for each protein. The protein Catalase had two isoforms under separate proteome IDs — CHOYP_CATA.1.3|m.11120 and CHOYP_CATA.3.3|m.21642 — and target peptides were chosen from both IDs. A total of four peptides and twelve associated transitions were used as SRM targets.

```{r targets,echo=FALSE}
targets <- read.csv("data/Ch1/Table1.2.csv")
  kableExtra:::kable_styling(knitr::kable(supp1, "latex",col.names = gsub("[.]", " ", names(targets)), 
                  caption = "Target peptides used for Selected Reaction Monitoring Assay (SRM)",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:outplantenv): Environmental conditions (mean ± standard deviation) at outplant locations (Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), Willapa Bay (WB).

```{r outplantenv, echo=FALSE}
outplantenv <- read.csv("data/Ch1/Table1.3.csv")
  kableExtra:::kable_styling(knitr::kable(outplantenv, "latex",col.names = gsub("[.]", " ", names(outplantenv)), 
                  caption = "Outplant conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:envmeans): Analysis of Similarities (ANOSIM) R statistic and p-value for mean values of environmental variables by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r envmeans, echo=FALSE}
envmeans <- read.csv("data/Ch1/Table1.4.csv")
  kableExtra:::kable_styling(knitr::kable(envmeans, "latex",col.names = gsub("[.]", " ", names(envmeans)), 
                  caption = "ANOSIM for mean environmental conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:envvar): Analysis of Similarities (ANOSIM) R statistic and p-value for environmental variable variances by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r envvar, echo=FALSE}
envvar <- read.csv("data/Ch1/Table1.5csv")
  kableExtra:::kable_styling(knitr::kable(envvar, "latex",col.names = gsub("[.]", " ", names(envvar)), 
                  caption = "ANOSIM for variance of environmental conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:protANOSIM): Results of pairwise ANOSIM tests between sites for protein abundance data. R values are listed above the diagonal, and p values are listed below the diagonal. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r protANOSIM, echo=FALSE}
protANOSIM <- read.csv("data/Ch1/Table1.6.csv")
  kableExtra:::kable_styling(knitr::kable(protANOSIM, "latex",col.names = gsub("[.]", " ", names(protANOSIM)), 
                  caption = "Pairwise ANOSIM for protein abundance",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:RDA): Significance of each predictor in the RDA. Salinity mean and variance were not included due to missing values. Predictors significant at the 0.10 level are marked by a single asterisk.

```{r RDA, echo=FALSE}
RDA <- read.csv("data/Ch1/Table1.7.csv")
  kableExtra:::kable_styling(knitr::kable(RDA, "latex",col.names = gsub("[.]", " ", names(RDA)), 
                  caption = "RDA Results",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:protfunctions): Functions of differentially abundant proteins.

```{r protfunctions, echo=FALSE}
protfunctions <- read.csv("data/Ch1/Table1.8.csv")
  kableExtra:::kable_styling(knitr::kable(protfunctions, "latex",col.names = gsub("[.]", " ", names(protfunctions)), 
                  caption = "Protein functions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

##Figures

**Figure** \@ref(fig:npathways): Nitrogen pathways in soil where MDN enters terrestrial systems via decay of salmon organic tissues or excretion from direct salmon consumers such as bears. Arrows represent conversion pathways with the potential to impart isotopic fractionations on plant available nitrogen (NH~4~^+^ or NO~3~^-^).\newline
  
\begin{figure}[h]
  \includegraphics{figure/Ch1/fig1.1.pdf}
  \caption{Nitrogen pathways in soil}
  (\#fig:npathways)
\end{figure}

\clearpage

**Figure** \@ref(fig:modIsotope): Data (closed circles) and predicted values (open circles) for the model with the most support (Table \@ref(tab:suppmod1)) for soil organic  $\delta^{15}N$ and  $\delta^{13}C$,  $\delta^{15}N$ of NH~4~^+^, and C:N for both the salmon-enhanced and the salmondepleted banks of Hansen Creek at 1, 3, 6, 10, and 20 m from the edge of the creek bed with 95% confidence intervals (dashed line) for predicted values. Blue (a and c) denotes measures of marine-derived nitrogen, and green (b and d) denotes site variable factors.\newline

\begin{figure}[h]
  \includegraphics[width=0.85\textwidth]{figure/Ch1/Figure2_Feddernetal.pdf}
  \caption{Data and predicted values for the model with the most support: Stable Isotopes}
  (\#fig:modIsotope)
\end{figure} 

\clearpage

**Figure** \@ref(fig:modConc): Data (closed circles) and predicted values (open circles) for the model with the most support (Table \@ref(tab:suppmod1)) for NH~4~^+^ and NO~3~^-^, net mineralization and nitrification, [N~org~], and gravimetric water content for both the salmon-enhanced and the salmon-depleted banks of Hansen Creek at 1, 3, 6, 10, and 20 m from the edge of the creek bed with 95% confidence intervals (dashed line) for predicted values. Red (a, b, c, d) denotes measures of soil productivity, and green (e and f) denotes site variable factors.\newline 

\begin{figure}[h]
  \includegraphics[width=0.85\textwidth]{figure/Ch1/Figure3_Feddernetal.pdf}
  \caption{Data and predicted values for the model with the most support: Concentrations and Transformations}
  (\#fig:modConc)
\end{figure}
\clearpage

**Figure** \@ref(fig:ch1resid): Predicted verse observed values and predicted verse residuals for the model with the most support (Table \@ref(tab:suppmod1), Figure \@ref(fig:modIsotope), \@ref(fig:modConc)) for each the response variables. \newline 
\begin{figure}[h]
  \includegraphics[width=1\textwidth]{figure/Ch1/ch1residuals.pdf}
  \caption{Residual Plots for Best Models}
  (\#fig:ch1resid)
\end{figure} 

