<!--
The {#rmd-basics} text after the chapter declaration will allow us to link throughout the document back to the beginning of Chapter 1.  These labels will automatically be generated (if not specified) by changing the spaces to hyphens and capital letters to lowercase.  Look for the reference to this label at the beginning of Chapter 2.
-->

# Characterization of Pacific oyster (*Crassostrea gigas*) proteomic response to natural environmental differences

## Abstract

Global climate change is rapidly altering coastal marine ecosystems important for food production. A comprehensive understanding of how organisms will respond to these complex environmental changes can come only from observing and studying species within their natural environment. To this end, the effects of environmental drivers — pH, dissolved oxygen content, salinity, and temperature — on Pacific oyster (*Crassostrea gigas*) physiology were evaluated in an outplant experiment. Sibling juvenile oysters were outplanted to eelgrass and unvegetated habitat at five different estuarine sites within the Acidification Nearshore Monitoring Network in Washington State, USA to evaluate how regional environmental drivers influence molecular physiology. Within each site, we also determined if eelgrass presence that buffered pH conditions changed the oysters’ expressed proteome. A novel, two-step, gel-free proteomic approach was used to identify differences in protein abundance in *C. gigas* ctenidia tissue after a 29 day outplant by 1) identifying proteins in a data independent acquisition survey step and 2) comparing relative quantities of targeted environmental response proteins using selected reaction monitoring. While there was no difference in protein abundance detected between habitats or among sites within Puget Sound, *C. gigas* outplanted at Willapa Bay had significantly higher abundances of antioxidant enzymes and molecular chaperones. Environmental factors at Willapa Bay, such as higher average temperature, may have driven this protein abundance pattern. These findings generate a suite of new hypotheses for lab and field experiments to compare the effects of regional conditions on physiological responses of marine invertebrates.

## Introduction

Global climate change will influence estuarine dynamics and impact the organisms that inhabit these environments. Estuaries are already variable across spatial and temporal scales in terms of phytoplankton production [@Pennock1986], nutrient availability [@Paerl2014], heavy metal contamination [@Liu2015], salinity [@Banas2004], and carbonate chemistry [@Feely2010; @Pelletier2018]. Since climate change will affect these parameters, it is important to consider how estuarine organisms will respond.

Proteomics, or the study of protein abundance and expression, can be used to shed light on physiological changes on a molecular level. Proteins direct all major cellular functions, thus examining protein abundance provides direct evidence of an organism’s physiological response to the estuarine environment [@Tomanek2014]. The proteome is dynamic, as it must rapidly respond to perturbation, providing mechanistic information that standard gene expression and mRNA quantification studies cannot [@Veldhoen2012; @Flores-Nunes2015]. As a result of the proteome’s dynamic nature, proteins analyzed at the time of collection represent an organism’s response to the environment in near real-time. Long-term exposure to environmental conditions, as well as natural organismal aging, are also reflected in the proteome [@Hercus2003]. Discovery-based proteomic methods can elucidate responses to environmental drivers [@Flores-Nunes2015]. Several studies have connected protein abundances with changes in laboratory-simulated environmental conditions, identifying key proteins and mechanisms involved in specific environmental responses [@Timmins-Schiffman2014; @Dineshram2016; @Meng2017]. While these studies provide insight into organismal adaptation and physiology, laboratory studies alone cannot fully encapsulate the effects of multiple environmental drivers within an ecosystem context [@Riebesell2014]. 

Although challenging, in situ field studies provide a necessary biological realism when considering variable environments [@Slattery2012; @Cornwall2016]. Such experiments can be leveraged to study the effects of multiple environmental drivers on organismal physiology and to incorporate realistic variability, as opposed to examining the effect of a single stressor on an organism [@Riebesell2014]. Through transcriptomics, @Chapman2011 demonstrated the power of an in situ experimental design for examining the impacts of regional environmental conditions on Eastern oyster (*Crassostrea virginica*) physiology. Transcript signatures from *C. virginica* sampled from various locations in southeastern United States revealed that temperature, pH, salinity, dissolved oxygen and pollutant load at each location impacted gene expression. Furthermore, they were able to disentangle the interactions of these environmental factors on gene expression. RNA and protein abundances can be influenced by several environmental factors, and in situ studies can determine which drivers will be more important to consider for organismal physiology. 

Marine invertebrates have proven to be informative bioindicators in proteomic studies to examine the effects of in situ conditions on organismal physiological responses to environmental change. When marine invertebrates have been exposed to varying environmental conditions, proteomics have demonstrated changes in cellular defense, immune responses, and genome function [@Veldhoen2012]. Changes in protein abundance in bivalves like the Pacific oyster (*Crassostrea gigas*) and blue mussels (*Mytilus edulis spp*.) have been used to develop biomarkers for environmental contaminants [@Slattery2012; @Beyer2017]. Proteomic responses to natural environmental drivers have also been evaluated in bivalves. For example, shotgun proteomic analysis of *M. edulis* ctenidia from Baltic Sea microcosms revealed that low salinity conditions lead to decreased abundance of cytoskeleton proteins, as well as calcium-binding messenger calmodulin, which plays an important role in signalling and intracellular membrane trafficking pathways [@Campos2016]. Using a growing wealth of genomic information to understand how these species fare under differential environmental conditions is critical for monitoring natural populations and commercial aquaculture.

Pacific oyster (*Crassostrea gigas*) rearing in estuarine environments in Washington State, USA (WA) provides an ideal system to examine the effect of in situ environmental conditions on the expressed proteome. *C. gigas* are extensively farmed in two different estuarine systems that show substantial regional variation: Puget Sound and Willapa Bay. Puget Sound is a complex estuarine system with interconnected sub-basins, each with different freshwater inputs, residence times, and stratification levels [@Feely2010; @Bianucci2018; @Pelletier2018]. Willapa Bay is a large shallow estuary on the Pacific coast that exchanges approximately half its water volume with the Pacific Ocean at each tide [@Banas2004; @Banas2007]. Seasonality and location within Puget Sound dictates temperature, dissolved oxygen, salinity, and pH conditions, while Willapa Bay conditions are influenced by diurnal fluctuations and proximity to either the ocean or rivers draining into the bay [@Banas2007; @Feely2010; @Ruesink2015]. 

Both Puget Sound and Willapa Bay also host eelgrass beds (*Zostera spp*.) that affect environmental conditions, such as oxygen concentrations, on diurnal time scales. The 2012 Washington State Blue Ribbon Panel on Ocean Acidification outlines key early actions, which include the examination of “vegetation-based systems of remediation” (Action 6.1.1) to improve local pH through photosynthetic drawdown of carbon dioxide. This experiment set out to test whether protein abundance patterns reflect reduced stress within vegetation. For example, eelgrass beds may reduce emersion stress relative to unvegetated areas through shading, the retention of water, and increased evaporative cooling at low tide. They can also ameliorate effects of ocean acidification through photosynthetic activity. Reduced pathogen prevalence has also been documented in seagrass beds, but not specifically in eelgrass [@Lamb2017]. In contrast, eelgrass beds may drive more extreme carbonate chemistry conditions [@Pacella2018]. @Lowe2018 also found that *C. gigas* shell strength and survival was significantly lower in eelgrass habitats in WA. Understanding how different aquaculture grow-out locations and habitats will affect the oyster’s ability to persist through environmental change is crucial for the industry and the ecosystem.

The purpose of this study was to use proteomic techniques to uncover the impacts of environmental drivers on Pacific oyster physiological outcomes in estuarine environments in WA. Naturally existing environmental variation was harnessed by outplanting *C. gigas* in different locations within Puget Sound and Willapa Bay, and habitat effects were taken into consideration by placing oysters in eelgrass and unvegetated habitats. Gel-free proteomic methods were used to examine the effects of outplant conditions on relative quantities of all expressed proteins in a series of in situ experiments in order to identify differentially abundant proteins. We predicted that differences in environmental drivers at each outplant location and within outplant habitats would yield unique protein abundance patterns. Oysters at outplant locations with warmer water temperatures, more variable water temperatures, lower dissolved oxygen content, lower salinity, or lower pH may have higher abundances of proteins related to environmental response. Eelgrass beds were expected to ameliorate stressful conditions, resulting in lower abundances of environmental stress response proteins than oysters in unvegetated habitats.

## Methods

### Shellfish Deployment

Sibling juvenile *C. gigas* (average shell length = 27.2 mm, age = 2 months) were outplanted for 29 days starting June 19, 2016 at five locations: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB) in Washington State, USA (Table \@ref(tab:deployment); Figure \@ref(fig:sitemap)). These sites were selected for differences in environmental parameters, as well as for the presence of unvegetated areas and eelgrass beds within each site. All sites were part of the Acidification Nearshore Monitoring Network, a network of sensors placed in various WA locations to monitor marine chemistry (ANeMoNe; Washington Department of Natural Resources). Prior to the outplant, oysters were reared in a controlled hatchery setting. At each site and habitat combination, custom-built Durafet-based sensors (Honeywell) were used to monitor pH. Commercially-available MiniDOT loggers (Precision Measurement Engineering) were used to measure dissolved oxygen, and Odyssey loggers were used to measure conductivity. All sensors recorded temperature measurements, and all sensors logged at 10-minute intervals across the outplant period, with the exception of SK, where sensors were installed two days into the outplant period. At each site, juvenile oysters were placed in bags of five oysters each directly onto the substrate at a tidal height of -1.5 MLLW, both inside and outside of eelgrass beds (n=15 per habitat type), for a total of thirty outplanted oysters per site. The animals were uniformly placed less than a lateral distance of 0.5 m from the sensors at the same tidal height as the instruments. Oysters were housed in exclusion cages to prevent predation. Juvenile oysters remained at each site for a 29-day exposure period. Because the ctenidia is the primary site where oysters interact with the environment, ctenidia samples were dissected at the end of the outplant and held on dry ice until storage at -80ºC [@Beyer2017; @Meng2017). 

Environmental data was treated as follows. Conductivity observations were removed when less than zero, which occurs when the instrument is dry at low tide. Remaining observations were converted to salinity measurements using the `swSCTp` function in the `oce` package in R [@Kelley2018; @R2018], with temperature at 25ºC and pressure at 10 dbar. For dissolved oxygen, pH, and salinity datasets, data were removed when collected by probes 1) during low tide or 2) when tidal depth was less than one foot to remove readings where the probes may have been exposed. Values collected during low tide or a depths less than one foot were retained for temperature datasets. Outliers were screened using the Tukey method for temperature, dissolved oxygen, pH, and salinity datasets [@Hoaglin1986]. Uniform outplant tidal heights were checked using `prop.test` in R [@R2018]. 

A non-metric multidimensional scaling analysis (NMDS) was used to evaluate differences in environmental parameters. First, mean and variance were calculated for each day of the outplant. Values were log-transformed, and a separate Gower’s distance matrix was calculated for daily mean and daily variances, accounting for missing data. The NMDS was conducted with the Gower’s distance matrix to visually compare means or variances between sites and habitats. Significant differences between site and habitat were identified using a One-way Analysis of Similarities (ANOSIM) for each environmental parameter. Pairwise ANOSIM tests for significant one-way ANOSIM results and two-way ANOSIM tests by site and habitat were not conducted due to lack of replicates within each site-habitat combination. R Scripts are available in the supplementary [Github repository](https://github.com/RobertsLab/paper-gigas-DNR-proteomics).

### Protein Discovery

To identify select protein targets for characterization across locations and environmental conditions, a subset of tissue samples were analyzed with data independent acquisition (DIA) mass spectrometry analysis. Two tissue samples were used from each site to make a peptide library and maximize the amount of protein abundance data collected from each sample. 

#### Protein Quantification

Tissue samples were homogenized in a solution of 50 mM NH4HCO3 with 6M Urea (500µl). Tissues were then sonicated 3 times (Fisher Scientific Sonic Dismembrator Model 100) for 10 seconds each and cooled between sonications in a bath of ethanol and dry ice. Protein quantities were measured with the Pierce BCA Protein Assay Kit microplate assay with a limited quantity of sonicated sample (11 µL). The protein concentration was measured via spectroscopy at 540 nm in a Labsystems (Waltham, MA) Multiskan MCC/340 and accompanying Ascent Software Version 2.6. Protein concentrations were calculated based on a standard curve with BSA (Pierce) per manufacturer’s instructions.

#### Protein Digestion

Protein digestion followed the protocol outlined in @Timmins-Schiffman2013. To each sample of 30 µg protein,1.5 M Tris pH 8.8 buffer (6.6 µL) and 200 mM TCEP (2.5 µL) were added. After solvent additions, each sample’s pH was verified to be basic (pH ≥ 8), and placed on a 37ºC heating block for one hour. Iodoacetamide (200 mM, 20 µL) was then added to each sample to maximize digestion enzyme access to protein cleavage sites. Samples were covered with aluminum foil to incubate in the dark for 1 hour at room temperature. Afterwards, diothiolthreitol (200 mM, 20 µL) was added and samples were incubated at room temperature for one hour. Lysyl Endopeptidase (Wako Chemicals) was then added to each sample in a 1 µg enzyme:30 µg oyster protein ratio, followed by one hour of incubation at room temperature. Urea was diluted with NH~4~HCO~3~ (25 mM, 800 µL) and HPLC grade methanol (200 µL). Trypsin (Promega) was added to each sample in a 1 µg trypsin: 30 µg oyster protein ratio for overnight digestion at room temperature.

#### Peptide Isolation

After overnight incubation, samples were evaporated to near dryness at 4ºC with a speedvac (CentriVap ® Refrigerated Centrifugal Concentrator Model 7310021). Samples were then reconstituted in 100 µL of a 5% Acetonitrile and 0.1% Trifluoroacetic Acid (Solvent A) to isolate peptides. If samples were not at pH ≤ 2, 10-20 µL aliquots of 10% Formic Acid were added until this pH was achieved.

Before desalting peptide samples, Macrospin C18 columns (The Nest Group) were prepared by adding 200 µL of a 60% Acetonitrile with 0.1% Trifluoroacetic Acid (Solvent B). The columns were spun for three minutes at 2000 rpm, and flow-through liquid from the column was discarded. The spinning and discarding process was completed a total of four times.To wash columns, 200 µL of Solvent A was added to each column. The columns were once again spun for three minutes at 2000 rpm and liquid was discarded afterwards; the solvent addition, spinning, and discarding process was completed a total of three times.

To bind peptides to the columns, digested peptides were added to prepared columns, then the columns were spun at 3000 rpm for three minutes. The filtrate was pipetted back onto the column and spun again at 3000 rpm for three minutes. Solvent A (200 µL) was added to each column three separate times, then the column was spun for three minutes at 3000 rpm to wash salts off the column. 

Peptides were eluted with two additions of 100 µL Solvent B to each column. Columns were spun at 3000 rpm for three minutes and the peptide fraction (filtrate) was reserved. Samples were placed in a speed vacuum at 4ºC until they were nearly dry (approximately two hours) to dry peptides. Peptides were reconstituted with 60 µL of 3% Acetonitrile + 0.1% Formic Acid, and stored at -80ºC.

#### Internal Standard Addition

Peptide Retention Time Calibration (PRTC; Pierce) is used as an internal standard to ensure consistency of peptides detected and measured throughout a mass spectrometry run. The stock solution of PRTC was diluted to 0.2 pmol/µL using 3% Acetonitrile with 0.1% Formic Acid. In a clean centrifuge tube, 6 µg of oyster protein and 0.376 pmol of PRTC were mixed together as per the PRTC user guide. Sample volume was brought up to 15 µL using a 3% acetonitrile and 0.1% formic acid solution. A quality control solution was also prepared (1 µL PRTC + BSA:3 µL 3% Acetonitrile and 0.1% Formic Acid solution).

### Data Independent Acquisition Mass Spectrometry

Peptides were analyzed on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) using Data Independent Acquisition Mass Spectrometry (DIA). DIA analyses were completed as a comprehensive, non-random analytical method for detecting peptide ions present within a sample to create a peptide library. The peptide library was then leveraged to develop a targeted proteomics assay for quantification (see *Selected Reaction Monitoring Assay*). A 30 cm analytical column and 3 cm pre-column were packed in-house with 3 µm C18 beads (Dr. Maisch). Samples were run in a randomized order. A blank injection followed each sample, with procedural blanks run at the very end. Every injection was 3 µL, which included 1 µg of oyster protein and 0.0752 pmol of PRTC. Peptides were analyzed in MS1 over the m/z range of 450-950 with 12 m/z wide windows with 5 m/z overlaps [@Egertson2013]. MS1 resolution was 60000 and AGC target was 400000 with a three second cycle time. The MS2 loop count was set to 20 and MS2 data was collected with a resolution of 15000 on charge state of 2 with an AGC target of 50000. No dynamic exclusion was used.  

### Peptide-Centric Proteomic Analyses

Unknown peptide spectra from mass spectrometry samples were matched with known peptides using Peptide-Centric Analysis in the PECAN software [@Ting2015]. Raw mass spectrometry files were converted to mzML files, then demultiplexed using MSConvert [@Chambers2012]. The C. gigas proteome was digested with in silico tryptic digest using Protein Digestion Simulator [@Riviere2015]. All known peptides from the mzML files were identified in comparison to the digested *C. gigas* proteome [@Riviere2015].

The PECAN-generated spectral library (.blib) file was used to detect peptides of interest in raw DIA files in Skyline [@MacLean2010]. Skyline identified peptides using chromatogram peak picking, where ions that elute at the same time and mass are detected as a peptide (file available at Panorama Public). All PRTC peptides and approximately 100 different oyster proteins and their peptide transitions were manually checked for retention time and peak area ratio consistency to determine a Skyline auto peak picker error rate (24.3% ± 25%, range: 0% to 100%).

Proteins had to satisfy four criteria to be considered appropriate targets for the study. 1) After an extensive literature search, functions related to oxidative stress, hypoxia, heat shock, immune resistance, shell formation, growth, and cellular maintenance were determined useful for evaluating environmental response. Proteins with annotations matching these functions were considered potential targets. 2) Protein data was then evaluated in Skyline to ensure there was no missing data for any peptide or sample. 3) Peaks with significant interference from other peptides were not considered. 4) Proteins needed at least two peptides with three transitions per peptide to quality as a potential target for downstream assays. The fifteen proteins (41 peptides and 123 transitions) that matched all of these criteria were selected as targets (Table 1.2).

### Selected Reaction Monitoring Assay

Following the protein discovery phase (DIA), proteins were isolated as described above from an additional five randomly selected samples per site and habitat combination (for a total of 5 oysters per group) and analyzed with Selected Reaction Monitoring (SRM). Samples were prepared as described for DIA, except tissue samples were homogenized in 100 µL, and peptide samples were evaporated at 25ºC after peptide isolation.
Proteins of interest identified from the DIA analysis were used as targets in a SRM assay following the workflow and informatic pipeline of [@Timmins-Schiffman2016]. Target peptide transitions were monitored using SRM on a Thermo TSQ Vantage. SRM data were collected during a gradient of 2-60% acetonitrile over 40 minutes. All samples were run in technical duplicates in a randomized order with a 1 µg oyster peptide and 0.0752 pmol PRTC injection. A quality control injection and blank injection were run after every five sample injections, and PRTC peptides were monitored throughout the experiment.

#### Target Peptide Specificity

To ensure SRM assay specificity to oyster peptides of interest, oyster peptides were diluted in a background matrix of similar complexity (Pacific geoduck — *Panopea generosa* — peptides), then analyzed using the oyster SRM assay. An oyster-specific SRM target would decrease in abundance with a decreasing abundance of oyster peptides in a mixture. Non-specific peptides — more likely to be found in background matrix of similar complexity — or peptides susceptible to interference would not correlate with oyster peptide abundance, and therefore, would be uninformative. Five *C. gigas* samples used for SRM were randomly selected and pooled in equal quantities. A ten-sample oyster:geoduck dilution series was prepared and run using the same methods as other SRM samples.

#### Target Analysis

Raw SRM files, a background *C. gigas* proteome, and the PECAN spectral library file from DIA were used to create a Skyline document (file available at Panorama Public). Correct transition peaks were selected based on predicted retention times from DIA results by comparing the relative retention times between identical PRTC peptides in the DIA and SRM datasets (R^2^ = 0.99431). Based on peptide specificity analyses, heat shock protein 70 B2 and one constituent peptide of 
glucose-6-phosphate 1-dehydrogenase were removed from analyses.

Further filters were applied to the data to maintain only high quality peptides and transitions in the analysis. Coefficients of variation were calculated between technical replicates for each peptide transition. Peptides were removed from the dataset if CV > 20%. To maintain high sample quality, any sample missing data for more than 50% of peptide transitions was deemed poor quality for downstream analyses and excluded. Abundance data was normalized using total ion current (TIC) values from the mass spectrometer. Consistency between technical replicates was verified in remaining samples using a NMDS with TIC-normalized data and a euclidean dissimilarity matrix. Technical replicates were consistent if replicates lay closer together than to other samples in the NMDS. These replicates were then averaged for multivariate analytical methods.
	
Averaged technical replicate data was used to determine if peptides were differentially abundant between outplant sites and habitats. Before proceeding with analysis, peptide abundances were subjected to a Hellinger transformation to give low weights to any peptides with low counts. A NMDS was used to visually compare relative peptide abundance. One-way ANOSIM tests by site, region (Puget Sound vs. Willapa Bay), and habitat, as well as a two-way ANOSIM test by site and habitat, were used to determine significant differences. Pairwise ANOSIM tests and post-hoc similarity percentage (SIMPER) analyses were conducted for each one- or two-way ANOSIM result significant at the 0.05 level. The first ten SIMPER entries were deemed influential peptides for each significant comparison.

The importance of environmental variables for explaining peptide abundance was evaluated with a redundancy analysis (RDA). For each site and habitat combination, mean and variance were calculated for pH, dissolved oxygen, salinity, and temperature over the course of the entire outplant. Environmental variables were then used as predictors to constrain peptide abundance. Predictors with missing values were not included. A triplot was used to visually assess differences in peptide abundance by site and habitat and the influence of individual peptides and environmental parameters. Analysis of Variance (ANOVA) was used to calculate significance of the RDA and environmental variables, with predictors deemed significant at p < 0.05. Since estuarine sites are highly variable, a second RDA was conducted constraining peptide abundance by environmental conditions on the day of collection to evaluate robustness of proteomic methods. R Scripts are available in the supplementary [Github repository](https://github.com/RobertsLab/paper-gigas-DNR-proteomics).

## Results

### Conditions at Outplant Locations

Outplanted oysters experienced environmental variables representative of standard summer conditions in Puget Sound (PS) and WB. Wild or cultured oysters were present at the same tidal elevation as sensors, so outplant conditions represent experiences of non-experimental organisms. NMDS plots revealed mean environmental conditions were more similar among sites than variances. Both NMDS had stress values less than 0.20 and were considered appropriate multivariate representations of environmental data (Mean NMDS: stress = 0.0170, p = 0.0100; Variance NMDS: stress = 0.0340, p = 0.0100). Mean dissolved oxygen and temperature were significantly different between sites (Table \@ref(tab:envmeans); ANOSIM; dissolved oxygen: R = 0.4063, p = 0.0530; temperature: R = 1.0000, p = 0.0020), but not between habitats. Variances of all environmental parameters were significantly different between sites (Table 5; ANOSIM; pH: R = 0.5313, p = 0.0180; dissolved oxygen: R = 0.6800, p = 0.0030; salinity: R = 0.8400, p = 0.0130; temperature: R = 0.9400, p = 0.0010). WB had the warmest, but least variable, temperature of 18.0ºC ± 1.3ºC compared to averages ranging from 14.8ºC ± 1.8ºC to 16.1ºC ± 1.7ºC in PS (Table \@ref(tab:outplantenv); Figure \@ref(fig:envdatalines)). All variance values, mean pH, and mean salinity were not significantly different between habitats (Table \@ref(tab:envmeans); Table \@ref(tab:envvariance)). 

Although oyster position was standardized by tidal height, outplanted oysters experienced different amounts of exposure at low tide at each site due to differences in tidal amplitude (χ~4~^2^ = 25.29, p < 0.0001). Oysters at FB spent the highest percent of time emersed at tides less than 1 foot MLLW (13.99%; 188 hours), followed by PG (10.93%; 146 hours), SK (10.29%; 138 hours), CI (9.35%; 125 hours), and WB (8.48%; 113 hours). CI and FB outplants (p = 0.0021), FB and SK outplants (p = 0.0324), and FB and WB outplants (p < 0.0001) spent significantly different amounts of time emersed during low tide conditions.

### Data Independent Acquisition Mass Spectrometry

Out of 39816 predicted proteins in the *C. gigas* FASTA proteome, 9047 proteins were detected in C. gigas across five sites and two habitats using DIA (Skyline auto peak picker error rate 24.3% ± 25%, range: 0% to 100%). Proteins detected included, but were not limited to, those annotated from processes such as responses to hypoxia and oxidative stress, removal of superoxide radicals, protein folding, muscle organ development, and negative regulation of apoptosis (Raw data: PeptideAtlas accession no. PASS01305).

### Selected Reaction Monitoring Assay

Differential abundance of protein targets was evaluated at the peptide level after combining technical replicates. Proteins were considered differentially abundant if at least one monitored peptide was differentially abundant. There was no significant difference in SRM peptide abundance between unvegetated and eelgrass habitats across sites (One-way ANOSIM; R = 0.0440, p = 0.1220). Abundance data from both habitats were pooled for downstream analyses comparing protein abundances among sites (Raw data: PeptideAtlas accession no. PASS01304).

Rank distances between peptide abundances in multivariate space revealed differences in peptide abundance between WB versus the other four sites (NMDS: stress = 0.0750, p = 0.0099). One-way ANOSIM by site demonstrated no significant differences in peptide abundance (R = 0.0640, p = 0.065), but a one-way ANOSIM by region (PS vs. WB) revealed a trend towards peptide abundance differences (R = 0.2260, p = 0.0530). Peptide abundance was significantly different between FB and Willapa Bay (Table \@ref(tab:protANOSIM); Pairwise ANOSIM; R = 0.2568, p = 0.0350). Environmental variables explained 29% of variance in peptide abundance, but the proportion of variation explained was not significant (ANOVA for RDA; F6,19 = 1.3064; p = 0.1950). Temperature mean and variance were the two most influential environmental predictors of peptide abundance, but were not significant at the 0.05 level (ANOVA on RDA; Table \@ref(tab:RDA); temperature mean: F~1,19~ = 2.2375, p = 0.0650; temperature variance: F~1,19~ = 2.1627, p = 0.0670). Peptides differentially abundant between FB and WB are primarily positively loaded onto temperature mean, with two negatively loaded on temperature variance. Robustness of analysis was evaluated by performing a second RDA to predict peptide abundance based on mean and variance of temperature and pH at time of collection. Due to missing values, dissolved oxygen and salinity could not be included in analysis. Temperature variance was the most influential predictor of peptide abundance at time of collection, but was not significant at the 0.05 level (ANOVA on RDA;  F~1,13~ = 2.2312, p = 0.0800).

Peroxiredoxin-5 (PRX), catalase (EC 1.11.1.6) (CAT), NAD(P) transhydrogenase (NADPt), glucose-6-phosphate 1-dehydrogenase (G6PD), carbonic anhydrase (CA), protein disulfide-isomerase 1 and 2 (PDI) had peptides that were identified as differentially abundant between Willapa Bay and Fidalgo Bay by SIMPER analysis (Figure \@ref(fig:pepordination)). Fidalgo Bay peptide composition was differentiated by CA abundance, while all other significantly abundant proteins differentiated Willapa Bay. These proteins are involved in general cellular stress responses like reactive oxygen species neutralization or protein folding (Table 1.8). All differentially abundant proteins were detected at higher levels in the WB oysters than in oysters from the other four PS locations (CI, FB, PG, and SK), regardless of protein function (Figure \@ref(fig:protheatmap)). No differences in protein abundance were detected among the PS sites (Table \@ref(tab:protANOSIM)).

## Discussion

### Antioxidant enzymes and Acid-Base Regulation

Higher antioxidant enzyme abundances can be a direct response to an increase in reactive oxygen species (ROS) [@Limón-Pacheco2009; @Zhang2015]. During electron transport, oysters can produce ROS that induce oxidative stress if not neutralized [@Abele2007; @Limón-Pacheco2009]. Peroxiredoxin-5 (PRX), and catalase (CAT) scavenge these ROS and degrade them before they cause cellular harm, while NAD(P) transhydrogenase (NADPt) maintains the cellular redox state [@Limón-Pacheco2009; @Sussarellu2012; @Flores-Nunes2015]. Higher abundances of antioxidant enzymes in WB oysters suggests the need for ROS scavenging to acclimatize to local conditions. 

Elevated antioxidant enzyme abundance at WB may be a response to higher levels of ROS brought on by heat stress. Mollusc species, like *C. gigas*, have been known to increase ROS production at higher temperatures [@Abele2007; @Tomanek2014]. Warmer and more variable temperature conditions at WB could be driving ROS production and in turn, higher abundances of PRX, CAT, and NADPt to ameliorate ROS-associated stress [@Zhang2015]. The shallow bathymetry of WB may have also contributed to elevated ROS presence and the need for antioxidant enzymes. At low tide, shallow waters would warm faster, leading to the higher temperatures observed at WB (Table \@ref(tab:deployment)). Warmer waters at low tide may prompt oysters to spend more time with their shells closed, inducing hypoxia and hypercapnia. Low oxygen concentrations within the shell could then augment ROS levels [@Guzy2006]. Oysters could respond through increased abundance of antioxidant enzymes [@Sussarellu2012].

The need for WB outplants to regulate internal acid-base conditions is demonstrated by higher abundance of carbonic anhydrase (CA) in these oysters. *C. gigas* can regulate hemolymph pH by increasing HCO~3~- concentration via the conversion of CO~2~ to HCO~3~-, catalyzed by CA [@Michaelidis2005; @Wei2015]. If warmer water conditions at WB prompted shell closure, *C. gigas* would need to switch to anaerobic metabolism, inducing a significant reduction in hemolymph pH [@Michaelidis2005]. Buffering of hemolymph pH could be accomplished by elevated CA abundance. Although oysters at FB spent more time in low tide conditions that would also prompt shell closure and similar molecular responses, upregulated protein abundance at WB implies conditions at this site required a greater proteomic response in these specific biomarkers for acclimatization.

Higher abundance of glucose-6-phosphate 1-dehydrogenase (G6PD) in WB oysters indicates these oysters maintained metabolic activity in warmer temperature conditions. G6PD catalyzes the oxidative portion of the pentose phosphate pathway, and products from this pathway are often the precursors for nucleic and aromatic amino acids [@Livingstone1981]. Additionally, G6PD activity generates NADPH and can indirectly regulate the redox environment and ameliorate effects of ROS [@Livingstone1981]. For example, exposure to ROS-inducing pollutants led to increased G6PD activity in *C. gigas* ctenidia [@Flores-Nunes2015]. Increased abundance of G6PD in WB not only could have maintained transcription and translation processes, but also levels of cellular and metabolic activity by indirectly dealing with ROS.

ROS are produced in response to many environmental changes, thus biomarkers of ROS scavenging are difficult to link to a single environmental difference in a variable and complex setting. For example, reduction of ROS was found to be a common response to both increased temperatures and aerial exposure in *C. gigas* [@Zhang2015]. Since ROS mediation is a conserved response to several stressors, it is possible that environmental parameters we did not measure (e.g., contaminants, microbiota abundance, trace metals), or a combination of environmental parameters, could explain the observed variation in antioxidant enzyme abundance. Future work at these locations should take these variables into account.

### Molecular Chaperones Involved in Protein Folding

Much like cellular response to ROS, increased levels of protein disulfide isomerase 1 and 2 (PDI) demonstrate a generalist response to conditions at WB. PDI is a general chaperone protein that forms disulfide bonds and assists with protein folding [@Vargas-Albores2009]. Higher PDI abundance would reflect a need to refold misshapen proteins. In this regard, PDI would function similarly to molecular chaperones like heat shock proteins. Several invertebrate taxa demonstrate higher PDI abundance when faced with an immune challenge or metal contamination. When faced with an immune challenge, Pacific white shrimp (*Litopenaeus vannamei*) hemocytes synthesized higher abundances of immune response proteins, followed by elevated abundance of PDI to correct disulfide bonds in these proteins [@Vargas-Albores2009]. An immune challenge could also lead to more misfolded proteins that PDI would need to refold to avoid cellular damage [@Zhang2014]. In flat oysters (*Ostrea edulis*), PDI expression increased in response to disseminated neoplasia [@Silvestre2006; @Martín-Gómez2013]. Metal contamination at WB could also elevate PDI abundance: Chinese mitten crabs (*Eriocheir sinensis*) chronically exposed to cadmium over-expressed PDI [@Silvestre2006; @Martín-Gómez2013]. Since neither disease burden or environmental contamination was measured in this study, it is impossible to know if either triggered the PDI response. Examination of these factors in future studies may elucidate the specifics of elevated PDI abundance in WB.

### Proteomic Responses in Puget Sound and Between Habitats

Due to the observed differences across environmental parameters between Willapa Bay and Puget Sound locations, similar abundances for proteins involved in various environmental responses may be evidence of physiological plasticity. One particular protein that we expected to be differentially abundant was heat shock 70 kDA protein 12A (HSP12A), since Willapa Bay had higher average temperatures (Table \@ref(tab:outplantenv)). However, this trend was not observed. Higher abundances of heat shock proteins (HSPs) are generally induced when organisms are exposed to thermal stress [@Hamdoun2003; @Zhang2015]. In our experiment, average temperatures *C. gigas* experienced were lower than the 30ºC threshold to induce elevated HSP expression in a controlled setting (Figure \@ref(fig:envdatalines); Table \@ref(tab:outplantenv)), which would explain why we did not see elevated HSP12A expression in WB [@Hamdoun2003].

The lack of differential abundance for protein targets — both among Puget Sound sites and between unvegetated and eelgrass habitats — was unexpected. These similar proteomic profiles could be due to four factors. First, it is possible that a different suite of environmental response proteins in the SRM assay could have yielded a different view of acclimatization to the various outplant sites; however, the targets we chose have proven to yield insight into a range of environmental responses in previous studies (ex. Table 1.8). Second, outplant duration could have been too short to capture varied physiological response within Puget Sound, or oysters could have also acclimatized to conditions in their outplant locations. Third, post-translational modifications may have influenced how we detected proteins. Abundance between sites or habitats may have been similar, but addition of post-translational modifications may have varied. Finally, it is possible that the proteomic response was not different because the environmental conditions that would elicit up- or down-regulation of monitored proteins were similar across these five locations. For example, we found no differences in environmental conditions between eelgrass and unvegetated areas, nor any differences in protein responses between these habitats (Table \@ref(tab:envmeans); Table \@ref(tab:envvariance)). Our results suggest that the potential vegetation-based systems of remediation could be limited under current conditions in the field. A recent comparison of stable isotopes and fatty acid composition in *C. gigas* at eelgrass and unvegetated habitats found no differences in δ13C, δ15N, total fatty acids, or proportional fatty composition in WB outplants, providing another line of evidence that suggests habitat may not affect *C. gigas* physiological performance [@Lowe2018]. 

## Conclusion

Differential abundance of target proteins observed between sibling oysters placed for 29 days in WB or PS indicates that environmental factors at WB tended to increase antioxidant enzyme and molecular chaperone protein abundance in Pacific oysters. This study is one the first to link regional environmental conditions to physiological responses in Puget Sound and Willapa Bay, as well as compare responses between Puget Sound and Willapa Bay. Understanding the difference between these two estuaries is important for the persistence of oyster reefs and aquaculture in the face of climate change. Of the environmental parameters measured, higher mean temperature, as well as less variable temperature conditions, at Willapa Bay may have influenced protein abundance. The lack of protein abundance differences between PS sites may imply that 2016 conditions were well-within the tolerances of outplanted *C. gigas*, so patterns of stress response witnessed under laboratory conditions were not apparent in the field. Together, the results generate a suite of new hypotheses for lab and field experiments comparing the effects of environmental conditions on physiological responses of marine invertebrates.

As global climate change continues to rapidly influence estuarine dynamics, this study illustrates the importance of investigating the effect of multiple modes of change on organismal physiology in situ. Our finding that oysters used generalist proteins to ameliorate stressors implies that assays for elucidating responses to multiple drivers in situ should include these conserved responses in addition to specific, stressor-related proteins identified in laboratory experiments. Pacific oysters are grown commercially throughout the Pacific Northwest region of the United States, so it is possible the population used for this study has a broad environmental tolerance. Proteomic assays allow for quantification of that tolerance, which is crucial for aquaculture industry and natural resource management. Future in situ explorations of environmental drivers on physiology should include a longer outplant duration, additional environmental variables, and multiple sampling points to provide a clearer connection between ecosystem dynamics and physiological performance.

## Tables

**Table** \@ref(tab:deployment):  Latitude and longitude of *C. gigas* outplants, as well as time of day and tidal height at collection. Oysters were placed at five locations sites: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB).

```{r deployment,echo=FALSE}
deployment <- read.csv("data/Ch1/Table1.1.csv")
  kableExtra:::kable_styling(knitr::kable(deployment, "latex",col.names = gsub("[.]", " ", names(deployment)), 
                  caption = "Outplant locations and time of day and tidal height at collection",
                  longtable = TRUE, booktabs = TRUE), font_size = 10, bootstrap_options = c("striped"))
```
\clearpage

\begin{landscape}

Table 1.2: Proteins used as targets for a Selected Reaction Monitoring Assay (SRM). Targets were identified based on differential abundance and stress-related annotations. At least two peptides and six transitions were included in the assay for each protein. The protein Catalase had two isoforms under separate proteome IDs and target peptides were chosen from both IDs. A total of four peptides and twelve associated transitions were used as SRM targets.

```{r targets,echo=FALSE}
targets <- read.csv("data/Ch1/Table1.2.csv")
  kableExtra:::kable_styling(knitr::kable(targets, "latex",col.names = gsub("[.]", " ", names(targets)), 
                  caption = "Target peptides used for Selected Reaction Monitoring Assay (SRM)",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```

\end{landscape}

\clearpage

**Table** \@ref(tab:outplantenv): Environmental conditions (mean ± standard deviation) at outplant locations (Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), Willapa Bay (WB).

```{r outplantenv, echo=FALSE}
outplantenv <- read.csv("data/Ch1/Table1.3.csv")
  kableExtra:::kable_styling(knitr::kable(outplantenv, "latex",col.names = gsub("[.]", " ", names(outplantenv)), 
                  caption = "Outplant conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:envmeans): Analysis of Similarities (ANOSIM) R statistic and p-value for mean values of environmental variables by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r envmeans, echo=FALSE}
envmeans <- read.csv("data/Ch1/Table1.4.csv")
  kableExtra:::kable_styling(knitr::kable(envmeans, "latex",col.names = gsub("[.]", " ", names(envmeans)), 
                  caption = "ANOSIM for mean environmental conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:envvariance): Analysis of Similarities (ANOSIM) R statistic and p-value for environmental variable variances by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r envvariance, echo=FALSE}
envvariance <- read.csv("data/Ch1/Table1.5.csv")
  kableExtra:::kable_styling(knitr::kable(envvariance, "latex",col.names = gsub("[.]", " ", names(envvariance)), 
                  caption = "ANOSIM for variance of environmental conditions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:protANOSIM): Results of pairwise ANOSIM tests between sites for protein abundance data. R values are listed above the diagonal, and p values are listed below the diagonal. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

```{r protANOSIM, echo=FALSE}
protANOSIM <- read.csv("data/Ch1/Table1.6.csv")
  kableExtra:::kable_styling(knitr::kable(protANOSIM, "latex",col.names = gsub("[.]", " ", names(protANOSIM)), 
                  caption = "Pairwise ANOSIM for protein abundance",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

**Table** \@ref(tab:RDA): Significance of each predictor in the RDA. Salinity mean and variance were not included due to missing values. Predictors significant at the 0.10 level are marked by a single asterisk.

```{r RDA, echo=FALSE}
RDA <- read.csv("data/Ch1/Table1.7.csv")
  kableExtra:::kable_styling(knitr::kable(RDA, "latex",col.names = gsub("[.]", " ", names(RDA)), 
                  caption = "RDA Results",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\clearpage

\begin{landscape}

Table 1.8: Functions of differentially abundant proteins.

```{r protfunctions, echo=FALSE}
protfunctions <- read.csv("data/Ch1/Table1.8.csv")
  kableExtra:::kable_styling(knitr::kable(protfunctions, "latex",col.names = gsub("[.]", " ", names(protfunctions)), 
                  caption = "Protein functions",
                  longtable = TRUE, booktabs = TRUE), font_size = 8)
```
\end{landscape}

\clearpage

## Figures

**Figure** \@ref(fig:sitemap): Map of outplant locations. Oysters were placed at five sites: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB). Sites in Puget Sound are outlined in a grey box.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=0.65\textwidth]{figure/Ch1/fig1.1.pdf}
  \caption{Map of outplant sites}
  (\#fig:sitemap)
\end{figure}

\clearpage

**Figure** \@ref(fig:envdatalines): Environmental variables for each site (Case Inlet, Fidalgo Bay, Port Gamble Bay, Skokomish River Delta, and Willapa Bay) and habitat (unvegetated and eelgrass) over the course of the 29 day outplant.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch1/fig1.2.pdf}
  \caption{Environmental data at each site}
  (\#fig:envdatalines)
\end{figure}

\clearpage

**Figure** \@ref(fig:pepordination): Ordination results for peptide abundance and environmental data. a) Environmental variables (pH, dissolved oxygen, salinity, and temperature) explained 30% of variance in peptide abundance data; however, redundancy analysis (RDA) for peptide abundance constrained by environmental variable was not significant (ANOVA; F~6,19~ = 1.306, p = 0.195). Temperature mean and variance were the most influential, yet nonsignificant, environmental predictors (ANOVA; temperature mean: F~1,25~ = 2.3275, p = 0.065; temperature variance: F~1,25~ = 2.1527, p = 0.067). Peptides differentially abundant between Fidalgo Bay (FB) and Willapa Bay (WB) are primarily positively loaded onto temperature mean, with two negatively loaded on temperature variance. b) Non-metric multidimensional scaling plot depicting peptide abundance by site and habitat, with 95% confidence ellipses around each site (stress = 0.075, p = 0.0099). c) Peptides that contributed to significant peptide abundance differences between FB and WB, as determined by post-hoc similarity percentage analysis (SIMPER). Peptides denoted 1 correspond to peroxiredoxin-5 (PRX), 2 for carbonic anhydrase (CA), 3 for catalase (CAT), 4 for glucose-6-phosphate 1-dehydrogenase (G6PD), 5 for NADP transhydrogenase (NADPt), and 6 for protein disulfide isomerases 1 and 2 (PDI). FB peptide composition was differentiated by CA abundance, while all other proteins influenced peptide abundances at WB.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch1/fig1.3.pdf}
  \caption{Peptide abundance ordination results}
  (\#fig:pepordination)
\end{figure}

\clearpage

**Figure** \@ref(fig:protheatmap): Average protein abundance by constituent peptides across experimental sites from SRM. Peptide abundance data at each site was averaged, then log transformed. Proteins were considered differentially abundant if at least one constituent peptide was significantly different (indicated by an asterisk). There were no significant differences in protein abundance among the Puget Sound locations, or between unvegetated and eelgrass habitats. Proteins were only significantly different between Fidalgo Bay and Willapa Bay or Skokomish River Delta and Willapa Bay.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch1/fig1.4.png}
  \caption{Average protein abundance by site}
  (\#fig:protheatmap)
\end{figure}
