# Low pH influences methylation patterns of gonad growth genes in the Pacific oyster (*Crassostrea gigas*)

## Abstract

There is a need to investigate mechanisms of phenotypic plasticity in marine invertebrates as negative effects of climate change are experienced by coastal ecosystems. Environmentally-induced changes to the methylome can confer plasticity through regulation of gene expression, but methylome responses may be tissue- and species-specific. To assess how ocean acidification affects the Pacific oyster (*Crassostrea gigas*) epigenome, we exposed oysters to either low pH (7.31 ± 0.02) or ambient pH (7.82 ± 0.02) conditions for seven weeks. Whole genome bisulfite sequencing was used to identify methylated regions in female oyster gonad samples. Analysis of gonad methylomes revealed a total of 1,599 differentially methylated loci (DML) were identified and found primarily in genes, with several genes containing multiple DML. Approximately 20% of DML overlapped with C/T SNPs. Overrepresented biological processes and cellular components in genes containing DML included cilium movement, development, and cytoskeletal components, suggesting that pH-responsive methylation may target gonad growth processes. Since enriched processes were associated with genic DML in multiple transcripts, it is possible DML in these genes could regulate alternative splicing. Our findings provide a foundation for understanding methylation-induced plasticity in *C. gigas*.

## Introduction

There is great interest in elucidating how changes in the environment can impact marine invertebrate stress responses, and determining how plastic that phenotype is by examining DNA methylation in marine invertebrates [@Eirin-Lopez2018; @Hofmann2017]. Environmental variation can alter the positions of methyl groups on CpG dinucleotides [@Eirin-Lopez2018]. Methylation does not alter the DNA sequence itself, but can regulate gene expression by fine-tuning expression of housekeeping genes or providing additional transcriptional opportunities for environmental response genes [@Gavery2014]. Methylation regulation of gene expression could then influence plasticity in response to environmental stressors [@Eirin-Lopez2018]. Ocean acidification [@ChandraRajan2021; @Bogan2020; @Downey-Wall020; @Lim2020; @Venkataraman2020; @Liew2018], heat stress [@Arredondo-Espinoza2021], upwelling conditions [@Strader2019], disease [@Johnson2020], pesticide exposure [@Rondon2017], and temperature and salinity [@Liew2018] have all elicited changes in marine invertebrate methylomes, demonstrating a potential for methylation to regulate organismal responses to environmental conditions.

Given the impact of ocean acidification on calcifying species like oysters [@Melzner2019; @Parker2013; @Kroeker2010], several environmental epigenetic studies have examined the influence of ocean acidification on molluscan methylomes (Table 1). For example, eastern oyster (*Crassostrea virginica*) mantle [@Downey-Wall2020] and reproductive tissue [@Venkataraman2020], and Hong kong oyster (*Crassostrea hongkongensis*) mantle [@ChandraRajan2021], and larval methylomes [@Lim2020] all display changes in DNA methylation after experimental ocean acidification exposure. Between all four studies, only @Venkataraman2020 reports the percent of CpGs methylated: the stated 22% is consistent with the baseline 15% methylation observed in initial Pacific oyster (*Crassostrea gigas*) methylation studies [@Gavery2013], as well as examination of methylome responses to heat stress (14.4%; [@Arredondo-Espinoza2021]) and diuron exposure (16.6%; [@Rondon2017]). In *C. virginica*, the mantle methylome was hypomethylated after ocean acidification exposure [@Downey-Wall2020], while there were no differences in global gonad methylation patterns [@Venkataraman2020]. Experimental ocean acidification exposure yielded differential methylation predominantly in gene bodies, but concentration in exons [@Lim2020; @Venkataraman2020] versus introns [@ChandraRajan2021] varies. Hyper- and hypomethylation occurs with equal frequency in response to ocean acidification. This is consistent with diuron exposure [@Rondon2017], but not with heat stress [@Arredondo-Espinoza2021]. Gene functions enriched in differential methylation datasets were inconsistent between species and tissue types (Table 1). While aminotransferase complex and biosynthetic processes were found to be enriched in hypomethylated DML in *C. virginica* mantle tissue [@Downey-Wall2020], there were no enriched GOterms associated with genic DML in *C. virginica* gonad samples [@Venkataraman2020]. In contrast, several processes were enriched in methylation datasets for *C. hongkongensis*, including acetoacetylco-A reductase activity, dehydrogenase activity, cellular response to pH, protein xylosyltransferase activity, translation factor activity, RNA binding, and diacylglycerol kinase activity in the mantle [@ChandraRajan2021] and cytoskeletal and signal transduction, oxidative stress, metabolic processes, and larval metamorphosis in larvae [@Lim2020]. While considerable effort has been made to understand molluscan methylation responses to ocean acidification, it is clear that there may be species- and tissue-specific responses. Additionally, use of enrichment methods prior to bisulfite conversion and sequencing in previous studies may impact functional interpretation differential methylation data [@Trigg2021].

In addition to being an important global aquaculture species, *C. gigas* has been used to further our understanding of methylation in marine invertebrates [@Roberts2012; @Gavery2013; @Riviere2013; @Gavery2014; @Olson2014; @Olson2015; @Saint-Carlier2015; @Gavery2017; @Riviere2017; @Song2017]. Examination of diuron exposure [@Rondon2017] and heat stress on different *C. gigas* phenotypes [@Arredondo-Espinoza2021] reveal that environmental conditions do influence the Pacific oyster methylome, and other oyster methylomes are responsive to ocean acidification (Table 1). However, studies examining the influence of ocean acidification on the *C. gigas* methylome are currently absent. Additionally, a recent study found that female *C. gigas* exposure to low pH conditions, followed by four months in ambient pH conditions prior to spawning, can still negatively impact larval survival 18 hours post-fertilization [@Venkataraman2019]. Since maturation stage was not different between female *C. gigas* after low pH exposure, this finding suggests environmental “memory” may be mediating the carryover effect. 

We sought to understand impacts of ocean acidification on the Pacific oyster gonad methylome in order to not only understand DNA methylation as a potential mechanism for the observed negative maternal carryover effect previously reported in @Venkataraman2019, but to also examine the functional role of methylation in regulating biological responses to ocean acidification. Female oyster gonad methylome sequencing revealed distinct methylation changes associated with low pH exposure. These results provide the foundation for examining phenotypic plasticity within a generation, as well as larval methylomes and potential intergenerational effects.

## Methods

### Experimental Design

The experimental design and seawater chemistry analysis with carbon chemistry parameters are described in @Venkataraman2019. Briefly, adult hatchery-raised *C. gigas* (average shell length = 117.46 ± 19.16 cm) were exposed to either low pH (7.31 ± 0.02) or ambient pH (7.82 ± 0.02) conditions from February 15, 2017 to April 8, 2017 at the Kenneth K. Chew Center for Shellfish Research and Restoration at the National Oceanic and Atmospheric Administration Manchester Field Station (47º 34’ 09.1” N 122º 33’ 19.0” W, Manchester, WA). Shell length, width, and oyster wet weight were obtained before and after pH exposure, and gonad tissue was extracted for histological analysis at each sampling point. Prior to exposure, twenty *C. gigas* were randomly selected for histology, while ten oysters from each treatment were randomly selected from tanks to assess maturation. Gonad tissue in cassettes was fixed for histology using PAXgene Tissue Fix and Stabilizer and sent to Diagnostic Pathology Medical Group, Inc. (Sacramento, CA) for hematoxylin and eosin staining and slide preparation. Tissues exposed to ambient pH were confounded during processing, preventing any tank identification. Maturation state and organism sex was evaluated histologically at 40x magnification [@Enríquez-Díaz2008; @Fabioux2005].

### DNA Extraction and Library Preparation

To determine how pH exposure altered maternal gonad methylation, DNA was extracted from histology blocks for four female (or presumed female) oysters for each treatment using the PAXgene Tissue DNA Kit (Qiagen) with modifications specified below at the end of the seven-week exposure. For each sample, up to 0.02 g of histology tissue embedded in paraffin was cut from the histology block and placed in a 2 mL round-bottomed tube. The tissue was then further homogenized within the tube. After adding 1 mL of xylene, each sample was vortexed for 20 seconds, incubated at room temperature for three minutes, then centrifuged at maximum speed for three minutes. To evaporate ethanol from samples after ethanol addition and vortexing steps, samples were placed on a heat block at 37ºC for 10-20 minutes to evaporate any residual ethanol. 

The resuspended pellet was lysed using a TissueTearor at setting 1. Prior to lysis and between samples, the TissueTearor was run in a 10% bleach solution for 20 seconds, followed by two consecutive 20 second runs in DI water to clean the instrument. Lysates were transferred to clean, labelled 1.5 mL microcentrifuge-safelock tubes. Proteinase K (20 µL) was added to each sample, and pulse-vortexed for 15 seconds to mix. The sample-Proteinase K solution was incubated 56ºC for 60 minutes. Every ten minutes, the samples were briefly removed from the heat block to vortex at maximum speed for one minute. After 60 minutes, RNase A (4 µL, 100 mg/mL) was added to each sample. Samples were vortexed at maximum speed for 20 seconds to mix and incubated for two minutes at room temperature to obtain RNA-free genomic DNA and reduce possible interference for downstream enzyme reactions. Samples were then kept at 80ºC for 60 minutes, and vortexed at maximum speed for one minute in ten-minute intervals. After the incubation, Buffer TD2 and 200 µL ethanol were added to each sample, vortexing thoroughly to mix. 

DNA was isolated for each sample after lysis and incubation following manufacturer’s instructions. To elute DNA, samples were loaded onto the spin column with Buffer TD5 (50 µL) and incubated at room temperature for five minutes to increase yield, then centrifuged at maximum speed for one minute. A Qubit™ 3.0 and dsDNA BR Assay (Invitrogen) was used to quantify sample yield using 1 µL of DNA for each sample. For two samples, DNA was further concentrated using an ethanol precipitation. 

Samples were sent to ZymoResearch for bisulfite conversion and library preparation. Bisulfite conversion was performed with the Zymo-Seq WGBS Library Kit (Cat. #D5465) using 100 ng of genomic DNA following manufacturer’s instructions. Per ZymoResearch protocol, samples were spiked with a mixture of six unique double-stranded synthetic amplicons (180-200 bp) from the Escherichia coli K12 genome. Since each amplicon represents a different percent methylation, the spike-in was used to determine bisulfite conversion efficiency. Libraries were barcoded and pooled into a single lane to generate 150 bp paired-end reads in a single Illumina NovaSeq flowcell. 

### Genome Information and Feature Tracks

Nuclear [@Peñaloza2021] (NCBI GenBank: GCA_902806645.1, RefSeq: GCF_902806645.1) and mitochondrial genome sequences (NCBI Reference Sequence: NC_001276.1) were used in downstream analysis. These sequences were combined for downstream alignment. The `fuzznuc` function within EMBOSS was used to identify CG motifs in the combined genomes.

The RefSeq annotation was used to obtain genome feature tracks, and create additional tracks. The annotation included a combination of Gnomon, RefSeq, cmsearch, and tRNAscan-SE predictions. Gene, coding sequence, exon, and lncRNA tracks were extracted from the RefSeq annotation. These tracks were then used to obtain additional genome feature information. To create an intron track, the complement of the exon track was generated with BEDtools `complementBed` v.2.26.0 to create a non-coding sequence track [@Quinlan2010]. The overlap between the genes and the non-coding sequences was found with `intersectBed` and defined as introns. Similarly, untranslated regions of exons were obtained by subtracting coding sequences from exon information with `subtractBed`. Flanking regions were defined as the 1000 bp upstream or downstream of genes. Upstream flanks were generated by adding 1000 bp upstream of genes, taking into account strand, with `flankBed`. Existing genes were removed from flanking sequences using `subtractBed`. A similar process was performed to generate downstream flanking regions. Intergenic regions were isolated by finding the complement of genes with `complementBed`, then using subtractBed to remove any flanks. Transposable element locations were obtained from RepeatMasker using the NCBI RefSeq annotation (RefSeq: GCF_902806645.1) (Smit et al., 2013; Tarailo-Graovac & Chen, 2009). The number of CG motifs in a given feature track was obtained using `intersectBed`. 

### Sequence Alignment

Reads were trimmed three times prior to alignment using TrimGalore! v.0.6.6 [@Martin2011]. Known Illumina adapter sequences were removed (`--illumina`), along with 10 bp from both the 5’ (`--clip_R1 10` and `--clip_R2 10`) and 3’ ends of 150 bp paired-end reads  (`--three_prime_clip_R1 10` and `--three_prime_clip_R2 10`). Any remaining adapters were removed in a second round of trimming. Finally, poly-G tails were trimmed out of samples by manually specifying adapter sequences (`--adapter GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG` and `--adapter2 GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG`). Sequence quality was assessed with FastQC v.0.11.9 [@Andrews2010] and MultiQC [@Ewels2016] after each round of trimming.

Trimmed samples were then aligned to the combined nuclear and mitochondrial genomes for *C. gigas* [@Peñaloza2021]. The genome was bisulfite converted with Bowtie 2-2.3.4 (Linux x84_64 version [@Langmead2012]) using the bismark_genome_preparation function within Bismark v.0.22.3 [@Krueger2011]. Reads were aligned to the bisulfite-converted genome. Non-directional input was specified (`--non_directional`), and alignment scores could be no lower than -90 (`-score_min L,0,-0.6`). Resultant BAM files were deduplicated (`--deduplicate_bismark`), and methylation calls were extracted from deduplicated BAM files (`bismark_methylation_extractor`). Deduplicated BAM files were also sorted and indexed for downstream applications using SAMtools v.1.10 [@Li2009]. Using coverage files generated from methylation extraction, CpG information was consolidated between strands to report 1-based chromosomal coordinates (`coverage2cystosine`). Using coverage2cytosine coverage file output, genomic coordinates for CpGs with at least 5x coverage across all samples were written to BEDgraphs. Summary reports with alignment information for each sample were generated (`bismark2report` and `bismark2summary`) and concatenated with MultiQC. Code for all sequence alignment steps can be found in the [associated Github repository](https://github.com/RobertsLab/project-gigas-oa-meth).

### SNP Identification

As genomic differences can play a role in methylation, single nucleotide polymorphisms (SNPs) were identified from WGBS data to determine if they overlapped with differentially methylated loci. SNP variants across all samples and in individuals were identified with `BS-Snper` [@Gao2015]. As `BS-Snper` only takes one input file, deduplicated and sorted BAM alignments generated with Bismark v.0.22.3 and SAMtools v.1.10 [@Krueger2011; @Li2009] were merged (`samtools merge`). Variants were then identified in the merged BAM file or individual deduplicated and sorted BAM files. Default `BS-Snper` settings were used, except for a minimum coverage of 5x to be consistent with methylation analyses. C/T SNPs were filtered by identifying overlaps between SNPs and CG motifs with intersectBed. The C/T SNPs were used in downstream analyses to identify differentially methylated loci that overlapped with SNPs.

### Global Methylation Characterization

Global methylation patterns were characterized averaging DNA methylation across all samples (using a 5x coverage threshold). A CpG locus was considered highly methylated if average percent methylation was greater than or equal to 50%, moderately methylated if percent methylation was between 10-50%, and lowly methylated if percent methylation was less than or equal to 10%. The genomic location of highly, moderately, and lowly methylated CpGs in relation to UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions were determined by using intersectBed. We tested the null hypothesis that there was no association between the genomic location of CpG loci and methylation status (all CpGs versus highly methylated CpGs) with a chi-squared contingency test (`chisq.test` in R Version 3.5.3).

### Identification of Differentially Methylated Loci

Identification of differentially methylated loci, or DML, was conducted with `methylKit` v.1.17.4 [@Akalin2012] in R Version 3.5.3. Prior to DML identification, data were imported and processed. Data with at least 2x coverage were imported using `methRead` from merged CpG coverage files produced by `coverage2cytosine`. Imported data were filtered again (`filterByCoverage`) to require 5x coverage for each CpG locus (`lo.count = 5`) in each sample. Potential PCR duplicates in each sample were also removed by excluding CpG loci in the 99.9th percentile of coverage (`high.perc = 99.9`). Once filtered, data were normalized across samples (`normalizeCoverage`) to avoid over-sampling reads from one sample during downstream statistical analysis. Histograms of percent methylation (`getMethylationStats`) and CpG coverage (`getCoverageStats`) were used to confirm normalization. After filtering and normalization, data at each CpG locus with 5x were combined (`unite`) to create a CpG background, ensuring that each locus had at least 5x coverage in each sample. 

Methylation differences were then identified for *C. gigas* gonad samples. A logistic regression (`calculateDiffMeth`) modeled the log odds ratio based on the proportion of methylation at each CpG locus. A covariate matrix with gonad stage information (`covariates`) were applied to the model for the *C. gigas* data to account for varying maturation stages in samples. Additionally, an overdispersion correction with a chi-squared test (`overdispersion = "MN", test = "Chisq"`) was applied.

Differentially methylated loci, or DML, were defined as CpG dinucleotides with at least a 50% methylation difference between pH treatments, and a q-value < 0.01 based on correction for false discovery rate with the SLIM method [@Wang2011]. Hypermethylated DML were defined as those with significantly higher percent methylation in low pH samples, and hypomethylated DML with significantly lower percent methylation in low pH samples. BEDfiles with DML locations were created and viewed with the Integrative Genomics Viewer (IGV) version 2.9.2 [@Thorvaldsdóttir2013].

### DML Characterization

The location of DML was characterized in relation to various genome feature tracks. Presence of DML in UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions was discerned using intersectBed. A chi-squared contingency test was used to test the null hypothesis of no association between genomic location and differential methylation by comparing the genomic location of all CpGs with 5x data and DML. Additionally, the number of DML present in each chromosome was quantified to see if DML were distributed uniformly across the genome. Overlaps between DML and SNPs were identified to determine if DML were solely attributed to treatment differences, or if genetic differences contributed to differential methylation.

### Enrichment Analysis

To determine if genes containing DML were associated with overrepresented biological processes, cellular component, or molecular function gene ontologies (GO), we conducted functional enrichment analysis. Prior to this analysis, the *C. gigas* genome was annotated. A `blastx` was performed against the Uniprot-SwissProt database (accessed June 01, 2021) to get Uniprot Accession information for each transcript [@UniProt2019]. The transcript nucleotide sequences were extracted from the genome (GCF_902806645.1_cgigas_uk_roslin_v1_rna_from_genomic.fna available in the NCBI Annotation Release 102). Transcript IDs from the `blastx` output were matched with GO terms from the Uniprot-Swissprot database using Uniprot Accession codes. These transcript IDs and corresponding GO terms were then used to create a gene ID-to-GO term database (`geneID2GO`) for manual GO term annotation. The transcript IDs were filtered for genes that contained CpGs with at least 5x coverage in all samples, as the same parameters were used to generate the CpG background for differential methylation analysis. Each line of the database contained transcript ID in one column, and all corresponding GO terms in another column.

Gene enrichment analysis was conducted with `topGO` v.2.34.0 in R [@Alexa2010]. The `geneID2GO` database was used as the gene universe for enrichment, while transcript IDs for genes with DML were used as the genes of interest. These transcript IDs were filtered such that they did not include any transcripts associated with DML-SNP overlaps. A `topGO` object was generated for each DML list and GO category (biological process, cellular component, or molecular function), with GO term annotation performed using the `geneID2GO` database. A Fisher’s exact test was used to identify GO terms in each DML list significantly enriched with respect to the gene background (P-value < 0.01). Keeping with `topGO` default settings, we did not correct for multiple comparisons [@Alexa2010]. Visualization of enriched GO terms was conducted using `simplifyEnrichment` v.1.2.0 default settings [@Gu2021]. Briefly, a semantic similarity matrix was created from enriched GO IDs using the “Relevance” similarity measure and GO tree topology (`GO_similarity`). The semantic similarity matrix was clustered using the binary cut method, and visualized as a word cloud alongside a heatmap of semantic similarity (`simplifyGO`).

## Results

### Sequence Alignment

Whole genome bisulfite sequencing produced 1.38 x 10^9^ total 150 bp paired-end reads. After quality trimming, 1.35 x 10^9^ total paired reads remained. Of the trimmed paired-end reads, 8.31 x 108 (61.7%) were uniquely aligned to the bisulfite-converted *C. gigas* genome with appended mitochondrial sequence.

### SNP Identification

C/T SNP variants were identified from WGBS data to determine if genomic differences contributed to differential methylation. A total of 13,234,183 unique SNPs were found in individual or merged BAM files, including 300,278 C/T SNPs. These C/T SNPs were used in downstream analyses (see *DML Characterization and Enrichment Analysis*).

### Global Methylation Patterns

When data were combined, using a 5x coverage threshold 11,238,223 CpG loci (84.8% of 13,246,547 total CpGs in *C. gigas* genome) were characterized. The majority of CpGs, 9,201,922 loci (81.9%) were lowly methylated, followed by 1,029,894 highly methylated (9.2%) and 1,006,406 moderately methylated (9.0%) CpGs. Highly methylated CpGs were found primarily in genic regions (964,976 CpGs, 93.7%), with 57,896 (5.6%) in exon UTR, 355,575 (34.5%) in CDS, and 554,093 (53.8%) in introns (Figure 1). The distribution of highly methylated CpGs was significantly different than all 5x CpGs detected by WGBS. A Principal Component Analysis (PCA) did not demonstrate any clear separation of samples by treatment, and consistently high Pairwise Pearson’s correlation coefficients support the lack of global percent methylation differences between pH treatments.

### DML Characterization

A total of 1,599 DML were identified using a 50% methylation difference threshold (Figure 2), which were evenly split between 793 (49.6%) hypomethylated and 806 (50.4%) hypermethylated DML (Figure 2). Of these DML, 315 (19.7%) overlapped with C/T SNPs. The DML were distributed amongst the ten main chromosomes, as well as scaffolds not placed in any chromosome (Figure 3).

The majority of DML (1,478; 92.4%) were located in genic regions (Figure 4), consisting of 1080 unique genes. Most genes only contained one DML; however, several genes contained multiple DML, ranging from 2-104 DML in one gene. The genic DML were primarily located in introns (941; 58.8%), followed by CDS (442; 27.6%) and exon UTR (95; 5.9%). Putative promoter regions upstream of transcription start sites contained DML (12; 0.8%), but more were located in downstream flanking regions (73; 4.6%). Additionally, 297 of the 1,478 genic DML overlapped with C/T SNPs in 283 unique genes. The DML were also found in transposable elements (496; 31.0%), lncRNA (9; 0.6%), and intergenic regions (42; 2.6%). The number of DML in all genome features differed significantly from all 5x CpGs. 

### Enrichment Analysis

We identified 10 biological processes and 4 molecular function GOterms significantly enriched in genes containing DML that did not overlap with C/T SNPs (Figure 5). A total of 339 genes with GOterm annotations containing 437 DML were used for this analysis. The enriched biological process GOterms were involved in motility (ex. cilium-dependent cell motility, cilium movement) and development (insta larval or pupal development, post-embryonic development), while the cellular component GOterms were involved in microtubule-associated and histone acetyltransferase complexes. A total of 7 unique genes and 34 unique transcripts were associated with the biological process terms, and 2 unique genes and 15 unique transcripts were associated with cellular component terms.

## Discussion

Our work examined the potential role of environmentally-induced changes to the reproductive tissue methylome to mediate carryover effects in *C. gigas*. We identified a total of 1,599 differentially methylated loci, showing that the methylome is responsive to ocean acidification. We also found significantly enriched biological processes associated with genic DML, implying that low pH may impact regulation of distinct processes, such as gonad development. We also highlight the importance of understanding genomic contributions to differential methylation through the identification of SNPs that overlap with DML. Our findings contribute to the growing body of work examining epigenetic responses in molluscan species, and provide the foundation for future research examining intergenerational epigenetic inheritance and its potential to mediate plastic responses to environmental stressors.

The enriched GOterms associated with genes containing DML suggest environmentally-responsive methylation can regulate gonad growth processes. Enrichment of cilium movement and overrepresentation of microtubule-associated complexes could be related to changes in cytoskeletal structure. These changes may signify reorganization of female reproductive tissue to accommodate growing oocytes, or a need to regulate protein transport and cytoskeletal activity. Differentially methylated genes in *C. hongkongensis* larvae exposed to ocean acidification also were enriched in cytoskeletal processes [@Lim2020], suggesting that this particular process may be targeted by methylation across life stages. In female oysters, expression of microtubule-associated genes increases from stage 0 to 2, but decreases after stage 3 [@Dheilly2012]. As @Venkataraman2019 did not identify any difference in maturation stage between oysters in low and ambient pH conditions, methylation regulation of these cytoskeletal genes may have allowed oysters in low pH to maintain gonad growth at the same rate as those in ambient pH conditions. It is also possible that the mixed cell types used in samples could be influencing enrichment results. Microtubule-associated genes are also present in somatic tissue [@Dheilly2012], so changes to the somatic tissue methylation may have also been captured in our analyses as we did not separate cell types. A similar study of *C. virginica* reproductive tissue did not find enrichment of growth processes, which may be due to confounding factors such as mixed cell types, as well as mixed sex and maturation stage in samples [@Venkataraman2020]. By using females of known maturation stage, we were able to identify processes influenced by low pH in gonad tissue. Future work should separate different cell types to understand environmentally-responsive methylation specifically in gonad cells.

Additional functions associated with genes containing DML provide insight into potential mechanisms mediating the negative maternal carryover effect observed in @Venkataraman2019]. Enriched biological processes associated with development were involved in larval or post-embryonic development, suggesting methylation regulation of genes that may be important for early life-stages. The most common functions associated with genes containing DML were related to hormone signaling, secretion, and metabolism. Whether acidification alters invertebrate reproductive hormones is not yet known, but alterations to hormone pathways through DNA methylation may impact maturation. While there was no difference in female maturation status after the seven-week pH exposure [@Venkataraman2019], persistent changes to expression of genes related to hormonal processes could impact oyster gametogenesis. Several genes involved in lipid metabolism and biomineralization also contained DML. Reduction in lipid content has been observed in the Sydney rock oyster *S. glomerata* exposed to experimental acidification conditions, but this lipid content reduction did not yield negative effects on embryonic development [@Scanes2018], so the connection between adult *C. gigas* low pH exposure, maternal provisioning, and poor larval survival may not be supported. Several studies note impacts of adult pH exposure on larval calcification process. For example, *S. glomerata* larvae exhibit faster shell growth in high pCO~2~ conditions when parents mature in those same conditions [@Parker2012; @Parker2015]. In contrast, larvae from other species found in the North Atlantic such as northern quahog (= hard clam; *Mercenaria mercenaria*) and bay scallops (*Argopecten irradians*) developed slower when parents were reproductively conditioned in low pH conditions [@Griffith2017]. It is possible that *C. gigas* larvae experienced reductions in calcification which lead to lower survival after adult low pH exposure. Additional research is required to discern if methylation changes to hormone signaling, lipid metabolism, and biomineralization genes persist in larvae, and how those changes may influence larval gene expression or survival.

In *C. gigas*, low pH may elicit methylation changes in genes to improve transcriptional control and maintain homeostasis. Differential methylation was found primarily in introns and exons, which suggests that these loci may influence gene expression. This finding is consistent with the literature examining molluscan methylome responses to ocean acidification [@ChandraRajan2021; @Downey-Wall2020; @Lim2020; @Venkataraman2020], disease [@Johnson020], pesticide exposure [@Rondon2017], and heat stress [@Arredondo-Espinoza2021], as well as environmental responses in corals [@Liew2018a; @Liew2018b], urchins [@Strader2019], and pteropods [@Bogan2020]. The relationship between methylation and gene expression has been investigated in *C. gigas*. High levels of methylation in genes were associated with increased gene expression and low expression variability, implying that methylation leads to transcriptional stability [@Gavery2013; @Olson2014]. Methylation was also found to reduce transcriptional noise in response to ocean acidification in corals [@Liew2018]. While pH-influenced differential methylation was not found to impact gene expression directly in *C. virginica* [@Downey-Wall2020] or *C. hongkongensis* [@ChandraRajan2021], neither study investigated methylation connections with alternative splicing. As enriched biological process GOterms were found in various transcripts within genes, there may be a role for methylation regulation of expression regulation via alternative splicing. Additionally enrichment of GOterms related to histone acetyltransferases could be indicative of chromatin regulation, which would then impact methylation and gene expression (ex. @Gatzmann2018). Both elevated temperature and low pH conditions have been found to influence methylation of histone-related genes in *C. gigas* [@Arredondo-Espinoza2021] and *C. hongkongensis* [@Lim2020; @ChandraRajan2021]. Our findings suggest that the connection between methylation and gene expression requires additional investigation when considering environmental response, with specific inclusion of mechanisms like alternative splicing and chromatin regulation.

Confounding elements may have influenced gonad methylome responses to ocean acidification, and our interpretation of those responses. Although specimens used were sibling oysters in an effort to reduce genomic variation, we identified several SNPs that overlapped with DML. It is possible that the DML algorithm used is biased towards identification of DML that overlap with SNP variants, as it removes individual variation [@Feng2014]. With a limited sample size, SNP identification from WGBS is useful for evaluating if DML are solely environmentally-induced, or if genomic variation influences methylation differences. Inclusion of DML that overlap SNPs during enrichment analysis may also confound results, leading to a different biological interpretation. Additionally, the lack of annotated genes used in the enrichment analysis limits our interpretation of which processes are most impacted by pH-sensitive methylation. 

Maturation stage is known to produce distinct baseline methylation patterns in *C. gigas* [@Zhang2018]. While the overall maturation stage was used as a covariate when identifying DML, it is possible that each sample methylome contains information from cells with various maturation stages or cell types. Even though methylation differences in genic regions support our conclusion that low pH does impact the *C. gigas* gonad methylome, single-cell sequencing efforts or investigations within a single maturation stage would clarify how these sources of variation influenced our findings.

## Conclusion

Our work — which is the first to explore methylation responses to ocean acidification in Pacific oysters — found that low pH treatment altered the *C. gigas* female gonad methylome after a seven-week exposure. We found that differential methylation occurs primarily in genic regions, which is consistent with other studies examining oyster methylome responses to ocean acidification. In conjunction with histology data from @Venkataraman2019, enriched biological processes and cellular components in genes containing DML suggest that methylation may be used to maintain gonad growth in low pH conditions. Our work expands on previous molluscan methylation research by not only demonstrating a potential role for methylation in plastic responses to ocean acidification, but also incorporating removing genomic variation from analysis to strengthen biological interpretation of data.

As inheritance of environmentally-induced methylation — and any resulting phenotypic plasticity — is contingent on epigenetic marks being present in the germline, withstanding chromatin reorganization, and being packaged into gametes [@Eirin-Lopez2018], the next step in this work is correlating germline methylation with offspring methylation, and examining gene expression, protein abundance, and phenotype. Comparing parental and offspring phenotype will allow researchers to assess whether parental experience and resulting epigenetic responses are accurate, reliable cues for adaptive plasticity in offspring.

\clearpage

## Tables

\begin{landscape}

Table 3.1: Summary of published studies examining the impact of environmental stressors on oyster methylation (NR = not reported).

```{r oystermethstudies,echo=FALSE}
oystermethstudies <- read.csv("data/Ch3/Table3.1.csv")
  kableExtra:::kable_styling(knitr::kable(oystermethstudies, "latex",col.names = gsub("[.]", " ", names(oystermethstudies)), 
                  caption = "Previous oyster methylation studies",
                  longtable = TRUE, booktabs = TRUE), font_size = 5)
```

\end{landscape}

\clearpage

## Figures

\begin{landscape}

Figure 2.1: Pacific oysters (n = 140) were acclimated for 15 days, then twenty were randomly sampled for histological analyses. Remaining oysters were divided into ambient pH or low pH treatments for seven weeks. Three experimental tanks for each treatment were used with 20 oysters per tank for a total of 60 oysters per treatment. At the end of the pH exposure, a total of ten oysters were randomly selected from each treatment and sampled for histological analyses. All remaining oysters were then held in ambient pH conditions for 3 months. Finally, oysters were reproductively conditioned and strip spawned. Larvae were counted eighteen hours post-fertilization.\newline 

\begin{figure}[h]
\centering
  \includegraphics[width=1.3\textwidth]{figure/Ch2/Figure2.1.jpg}
  \caption{Experimental timeline}
  (\#fig:expdesign)
\end{figure}
\end{landscape}
\clearpage

**Figure** \@ref(fig:carryoverboxplot): Proportion of live D-hinge larvae eighteen hours post-fertilization by female treatment. Each box represents proportions of live larvae between the first and third quartiles for half-sibling families where the female was exposed to either ambient or low pH conditions. Horizontal lines outside the box indicate the minimum value before the lower fence and the maximum value before the upper fence, with the solid line indicating the median. Circles represent outliers. A proportion of 1.0 indicates that all eggs in a cross were successfully fertilized and developed into D-hinge larvae. A linear mixed model, with sire as a random effect, indicated significantly fewer D-hinge larvae were present in half-sibling families where females were exposed to low pH conditions (t = -2.999; p = 0.0119). Significantly different proportions are indicated by a letter.\newline 

\begin{figure}[h]
\centering
  \includegraphics[width=0.85\textwidth]{figure/Ch2/Figure2.2.pdf}
  \caption{Larval survival 18 hours post-fertilization}
  (\#fig:carryoverboxplot)
\end{figure}
\clearpage
