# Low pH influences methylation patterns of gonad growth genes in the Pacific oyster (*Crassostrea gigas*)

## Abstract

There is a need to investigate mechanisms of phenotypic plasticity in marine invertebrates as negative effects of climate change are experienced by coastal ecosystems. Environmentally-induced changes to the methylome can confer plasticity through regulation of gene expression, but methylome responses may be tissue- and species-specific. To assess how ocean acidification affects the Pacific oyster (*Crassostrea gigas*) epigenome, we exposed oysters to either low pH (7.31 ± 0.02) or ambient pH (7.82 ± 0.02) conditions for seven weeks. Whole genome bisulfite sequencing was used to identify methylated regions in female oyster gonad samples. Analysis of gonad methylomes revealed a total of 1,599 differentially methylated loci (DML) were identified and found primarily in genes, with several genes containing multiple DML. Approximately 20% of DML overlapped with C/T SNPs. Overrepresented biological processes and cellular components in genes containing DML included cilium movement, development, and cytoskeletal components, suggesting that pH-responsive methylation may target gonad growth processes. Since enriched processes were associated with genic DML in multiple transcripts, it is possible DML in these genes could regulate alternative splicing. Our findings provide a foundation for understanding methylation-induced plasticity in *C. gigas*.

## Introduction

There is great interest in elucidating how changes in the environment can impact marine invertebrate stress responses, and determining how plastic that phenotype is by examining DNA methylation in marine invertebrates [@Eirin-Lopez2018; @Hofmann2017]. Environmental variation can alter the positions of methyl groups on CpG dinucleotides [@Eirin-Lopez2018]. Methylation does not alter the DNA sequence itself, but can regulate gene expression by fine-tuning expression of housekeeping genes or providing additional transcriptional opportunities for environmental response genes [@Gavery2014]. Methylation regulation of gene expression could then influence plasticity in response to environmental stressors [@Eirin-Lopez2018]. Ocean acidification [@ChandraRajan2021; @Bogan2020; @Downey-Wall020; @Lim2020; @Venkataraman2020; @Liew2018], heat stress [@Arredondo-Espinoza2021], upwelling conditions [@Strader2019], disease [@Johnson2020], pesticide exposure [@Rondon2017], and temperature and salinity [@Liew2018] have all elicited changes in marine invertebrate methylomes, demonstrating a potential for methylation to regulate organismal responses to environmental conditions.

Given the impact of ocean acidification on calcifying species like oysters [@Melzner2019; @Parker2013; @Kroeker2010], several environmental epigenetic studies have examined the influence of ocean acidification on molluscan methylomes (Table 1). For example, eastern oyster (*Crassostrea virginica*) mantle [@Downey-Wall2020] and reproductive tissue [@Venkataraman2020], and Hong kong oyster (*Crassostrea hongkongensis*) mantle [@ChandraRajan2021], and larval methylomes [@Lim2020] all display changes in DNA methylation after experimental ocean acidification exposure. Between all four studies, only @Venkataraman2020 reports the percent of CpGs methylated: the stated 22% is consistent with the baseline 15% methylation observed in initial Pacific oyster (*Crassostrea gigas*) methylation studies [@Gavery2013], as well as examination of methylome responses to heat stress (14.4%; [@Arredondo-Espinoza2021]) and diuron exposure (16.6%; [@Rondon2017]). In *C. virginica*, the mantle methylome was hypomethylated after ocean acidification exposure [@Downey-Wall2020], while there were no differences in global gonad methylation patterns [@Venkataraman2020]. Experimental ocean acidification exposure yielded differential methylation predominantly in gene bodies, but concentration in exons [@Lim2020; @Venkataraman2020] versus introns [@ChandraRajan2021] varies. Hyper- and hypomethylation occurs with equal frequency in response to ocean acidification. This is consistent with diuron exposure [@Rondon2017], but not with heat stress [@Arredondo-Espinoza2021]. Gene functions enriched in differential methylation datasets were inconsistent between species and tissue types (Table 1). While aminotransferase complex and biosynthetic processes were found to be enriched in hypomethylated DML in *C. virginica* mantle tissue [@Downey-Wall2020], there were no enriched GOterms associated with genic DML in *C. virginica* gonad samples [@Venkataraman2020]. In contrast, several processes were enriched in methylation datasets for *C. hongkongensis*, including acetoacetylco-A reductase activity, dehydrogenase activity, cellular response to pH, protein xylosyltransferase activity, translation factor activity, RNA binding, and diacylglycerol kinase activity in the mantle [@ChandraRajan2021] and cytoskeletal and signal transduction, oxidative stress, metabolic processes, and larval metamorphosis in larvae [@Lim2020]. While considerable effort has been made to understand molluscan methylation responses to ocean acidification, it is clear that there may be species- and tissue-specific responses. Additionally, use of enrichment methods prior to bisulfite conversion and sequencing in previous studies may impact functional interpretation differential methylation data [@Trigg2021].

In addition to being an important global aquaculture species, *C. gigas* has been used to further our understanding of methylation in marine invertebrates [@Roberts2012; @Gavery2013; @Riviere2013; @Gavery2014; @Olson2014; @Olson2015; @Saint-Carlier2015; @Gavery2017; @Riviere2017; @Song2017]. Examination of diuron exposure [@Rondon2017] and heat stress on different *C. gigas* phenotypes [@Arredondo-Espinoza2021] reveal that environmental conditions do influence the Pacific oyster methylome, and other oyster methylomes are responsive to ocean acidification (Table 1). However, studies examining the influence of ocean acidification on the *C. gigas* methylome are currently absent. Additionally, a recent study found that female *C. gigas* exposure to low pH conditions, followed by four months in ambient pH conditions prior to spawning, can still negatively impact larval survival 18 hours post-fertilization [@Venkataraman2019]. Since maturation stage was not different between female *C. gigas* after low pH exposure, this finding suggests environmental “memory” may be mediating the carryover effect. 

We sought to understand impacts of ocean acidification on the Pacific oyster gonad methylome in order to not only understand DNA methylation as a potential mechanism for the observed negative maternal carryover effect previously reported in @Venkataraman2019, but to also examine the functional role of methylation in regulating biological responses to ocean acidification. Female oyster gonad methylome sequencing revealed distinct methylation changes associated with low pH exposure. These results provide the foundation for examining phenotypic plasticity within a generation, as well as larval methylomes and potential intergenerational effects.

## Methods

### Experimental Design

The experimental design and seawater chemistry analysis with carbon chemistry parameters are described in @Venkataraman2019. Briefly, adult hatchery-raised *C. gigas* (average shell length = 117.46 ± 19.16 cm) were exposed to either low pH (7.31 ± 0.02) or ambient pH (7.82 ± 0.02) conditions from February 15, 2017 to April 8, 2017 at the Kenneth K. Chew Center for Shellfish Research and Restoration at the National Oceanic and Atmospheric Administration Manchester Field Station (47º 34’ 09.1” N 122º 33’ 19.0” W, Manchester, WA). Shell length, width, and oyster wet weight were obtained before and after pH exposure, and gonad tissue was extracted for histological analysis at each sampling point. Prior to exposure, twenty *C. gigas* were randomly selected for histology, while ten oysters from each treatment were randomly selected from tanks to assess maturation. Gonad tissue in cassettes was fixed for histology using PAXgene Tissue Fix and Stabilizer and sent to Diagnostic Pathology Medical Group, Inc. (Sacramento, CA) for hematoxylin and eosin staining and slide preparation. Tissues exposed to ambient pH were confounded during processing, preventing any tank identification. Maturation state and organism sex was evaluated histologically at 40x magnification [@Enríquez-Díaz2008; @Fabioux2005].

### DNA Extraction and Library Preparation

To determine how pH exposure altered maternal gonad methylation, DNA was extracted from histology blocks for four female (or presumed female) oysters for each treatment using the PAXgene Tissue DNA Kit (Qiagen) with modifications specified below at the end of the seven-week exposure. For each sample, up to 0.02 g of histology tissue embedded in paraffin was cut from the histology block and placed in a 2 mL round-bottomed tube. The tissue was then further homogenized within the tube. After adding 1 mL of xylene, each sample was vortexed for 20 seconds, incubated at room temperature for three minutes, then centrifuged at maximum speed for three minutes. To evaporate ethanol from samples after ethanol addition and vortexing steps, samples were placed on a heat block at 37ºC for 10-20 minutes to evaporate any residual ethanol. 

The resuspended pellet was lysed using a TissueTearor at setting 1. Prior to lysis and between samples, the TissueTearor was run in a 10% bleach solution for 20 seconds, followed by two consecutive 20 second runs in DI water to clean the instrument. Lysates were transferred to clean, labelled 1.5 mL microcentrifuge-safelock tubes. Proteinase K (20 µL) was added to each sample, and pulse-vortexed for 15 seconds to mix. The sample-Proteinase K solution was incubated 56ºC for 60 minutes. Every ten minutes, the samples were briefly removed from the heat block to vortex at maximum speed for one minute. After 60 minutes, RNase A (4 µL, 100 mg/mL) was added to each sample. Samples were vortexed at maximum speed for 20 seconds to mix and incubated for two minutes at room temperature to obtain RNA-free genomic DNA and reduce possible interference for downstream enzyme reactions. Samples were then kept at 80ºC for 60 minutes, and vortexed at maximum speed for one minute in ten-minute intervals. After the incubation, Buffer TD2 and 200 µL ethanol were added to each sample, vortexing thoroughly to mix. 

DNA was isolated for each sample after lysis and incubation following manufacturer’s instructions. To elute DNA, samples were loaded onto the spin column with Buffer TD5 (50 µL) and incubated at room temperature for five minutes to increase yield, then centrifuged at maximum speed for one minute. A Qubit™ 3.0 and dsDNA BR Assay (Invitrogen) was used to quantify sample yield using 1 µL of DNA for each sample. For two samples, DNA was further concentrated using an ethanol precipitation. 

Samples were sent to ZymoResearch for bisulfite conversion and library preparation. Bisulfite conversion was performed with the Zymo-Seq WGBS Library Kit (Cat. #D5465) using 100 ng of genomic DNA following manufacturer’s instructions. Per ZymoResearch protocol, samples were spiked with a mixture of six unique double-stranded synthetic amplicons (180-200 bp) from the Escherichia coli K12 genome. Since each amplicon represents a different percent methylation, the spike-in was used to determine bisulfite conversion efficiency. Libraries were barcoded and pooled into a single lane to generate 150 bp paired-end reads in a single Illumina NovaSeq flowcell. 

### Genome Information and Feature Tracks

Nuclear [@Peñaloza2021] (NCBI GenBank: GCA_902806645.1, RefSeq: GCF_902806645.1) and mitochondrial genome sequences (NCBI Reference Sequence: NC_001276.1) were used in downstream analysis. These sequences were combined for downstream alignment. The `fuzznuc` function within EMBOSS was used to identify CG motifs in the combined genomes.

The RefSeq annotation was used to obtain genome feature tracks, and create additional tracks. The annotation included a combination of Gnomon, RefSeq, cmsearch, and tRNAscan-SE predictions. Gene, coding sequence, exon, and lncRNA tracks were extracted from the RefSeq annotation. These tracks were then used to obtain additional genome feature information. To create an intron track, the complement of the exon track was generated with BEDtools `complementBed` v.2.26.0 to create a non-coding sequence track [@Quinlan2010]. The overlap between the genes and the non-coding sequences was found with `intersectBed` and defined as introns. Similarly, untranslated regions of exons were obtained by subtracting coding sequences from exon information with `subtractBed`. Flanking regions were defined as the 1000 bp upstream or downstream of genes. Upstream flanks were generated by adding 1000 bp upstream of genes, taking into account strand, with `flankBed`. Existing genes were removed from flanking sequences using `subtractBed`. A similar process was performed to generate downstream flanking regions. Intergenic regions were isolated by finding the complement of genes with `complementBed`, then using subtractBed to remove any flanks. Transposable element locations were obtained from RepeatMasker using the NCBI RefSeq annotation (RefSeq: GCF_902806645.1) (Smit et al., 2013; Tarailo-Graovac & Chen, 2009). The number of CG motifs in a given feature track was obtained using `intersectBed`. 

### Sequence Alignment

Reads were trimmed three times prior to alignment using TrimGalore! v.0.6.6 [@Martin2011]. Known Illumina adapter sequences were removed (`--illumina`), along with 10 bp from both the 5’ (`--clip_R1 10` and `--clip_R2 10`) and 3’ ends of 150 bp paired-end reads  (`--three_prime_clip_R1 10` and `--three_prime_clip_R2 10`). Any remaining adapters were removed in a second round of trimming. Finally, poly-G tails were trimmed out of samples by manually specifying adapter sequences (`--adapter GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG` and `--adapter2 GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG`). Sequence quality was assessed with FastQC v.0.11.9 [@Andrews2010] and MultiQC [@Ewels2016] after each round of trimming.

Trimmed samples were then aligned to the combined nuclear and mitochondrial genomes for *C. gigas* [@Peñaloza2021]. The genome was bisulfite converted with Bowtie 2-2.3.4 (Linux x84_64 version [@Langmead2012]) using the bismark_genome_preparation function within Bismark v.0.22.3 [@Krueger2011]. Reads were aligned to the bisulfite-converted genome. Non-directional input was specified (`--non_directional`), and alignment scores could be no lower than -90 (`-score_min L,0,-0.6`). Resultant BAM files were deduplicated (`--deduplicate_bismark`), and methylation calls were extracted from deduplicated BAM files (`bismark_methylation_extractor`). Deduplicated BAM files were also sorted and indexed for downstream applications using SAMtools v.1.10 [@Li2009]. Using coverage files generated from methylation extraction, CpG information was consolidated between strands to report 1-based chromosomal coordinates (`coverage2cystosine`). Using coverage2cytosine coverage file output, genomic coordinates for CpGs with at least 5x coverage across all samples were written to BEDgraphs. Summary reports with alignment information for each sample were generated (`bismark2report` and `bismark2summary`) and concatenated with MultiQC. Code for all sequence alignment steps can be found in the [associated Github repository](https://github.com/RobertsLab/project-gigas-oa-meth).

### SNP Identification

As genomic differences can play a role in methylation, single nucleotide polymorphisms (SNPs) were identified from WGBS data to determine if they overlapped with differentially methylated loci. SNP variants across all samples and in individuals were identified with `BS-Snper` [@Gao2015]. As `BS-Snper` only takes one input file, deduplicated and sorted BAM alignments generated with Bismark v.0.22.3 and SAMtools v.1.10 [@Krueger2011; @Li2009] were merged (`samtools merge`). Variants were then identified in the merged BAM file or individual deduplicated and sorted BAM files. Default `BS-Snper` settings were used, except for a minimum coverage of 5x to be consistent with methylation analyses. C/T SNPs were filtered by identifying overlaps between SNPs and CG motifs with intersectBed. The C/T SNPs were used in downstream analyses to identify differentially methylated loci that overlapped with SNPs.

### Global Methylation Characterization

Global methylation patterns were characterized averaging DNA methylation across all samples (using a 5x coverage threshold). A CpG locus was considered highly methylated if average percent methylation was greater than or equal to 50%, moderately methylated if percent methylation was between 10-50%, and lowly methylated if percent methylation was less than or equal to 10%. The genomic location of highly, moderately, and lowly methylated CpGs in relation to UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions were determined by using intersectBed. We tested the null hypothesis that there was no association between the genomic location of CpG loci and methylation status (all CpGs versus highly methylated CpGs) with a chi-squared contingency test (`chisq.test` in R Version 3.5.3).

### Identification of Differentially Methylated Loci

Identification of differentially methylated loci, or DML, was conducted with `methylKit` v.1.17.4 [@Akalin2012] in R Version 3.5.3. Prior to DML identification, data were imported and processed. Data with at least 2x coverage were imported using `methRead` from merged CpG coverage files produced by `coverage2cytosine`. Imported data were filtered again (`filterByCoverage`) to require 5x coverage for each CpG locus (`lo.count = 5`) in each sample. Potential PCR duplicates in each sample were also removed by excluding CpG loci in the 99.9th percentile of coverage (`high.perc = 99.9`). Once filtered, data were normalized across samples (`normalizeCoverage`) to avoid over-sampling reads from one sample during downstream statistical analysis. Histograms of percent methylation (`getMethylationStats`) and CpG coverage (`getCoverageStats`) were used to confirm normalization. After filtering and normalization, data at each CpG locus with 5x were combined (`unite`) to create a CpG background, ensuring that each locus had at least 5x coverage in each sample. 

Methylation differences were then identified for *C. gigas* gonad samples. A logistic regression (`calculateDiffMeth`) modeled the log odds ratio based on the proportion of methylation at each CpG locus. A covariate matrix with gonad stage information (`covariates`) were applied to the model for the *C. gigas* data to account for varying maturation stages in samples. Additionally, an overdispersion correction with a chi-squared test (`overdispersion = "MN", test = "Chisq"`) was applied.

Differentially methylated loci, or DML, were defined as CpG dinucleotides with at least a 50% methylation difference between pH treatments, and a q-value < 0.01 based on correction for false discovery rate with the SLIM method [@Wang2011]. Hypermethylated DML were defined as those with significantly higher percent methylation in low pH samples, and hypomethylated DML with significantly lower percent methylation in low pH samples. BEDfiles with DML locations were created and viewed with the Integrative Genomics Viewer (IGV) version 2.9.2 [@Thorvaldsdóttir2013].

### DML Characterization

The location of DML was characterized in relation to various genome feature tracks. Presence of DML in UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions was discerned using intersectBed. A chi-squared contingency test was used to test the null hypothesis of no association between genomic location and differential methylation by comparing the genomic location of all CpGs with 5x data and DML. Additionally, the number of DML present in each chromosome was quantified to see if DML were distributed uniformly across the genome. Overlaps between DML and SNPs were identified to determine if DML were solely attributed to treatment differences, or if genetic differences contributed to differential methylation.

### Enrichment Analysis

To determine if genes containing DML were associated with overrepresented biological processes, cellular component, or molecular function gene ontologies (GO), we conducted functional enrichment analysis. Prior to this analysis, the *C. gigas* genome was annotated. A `blastx` was performed against the Uniprot-SwissProt database (accessed June 01, 2021) to get Uniprot Accession information for each transcript [@UniProt2019]. The transcript nucleotide sequences were extracted from the genome (GCF_902806645.1_cgigas_uk_roslin_v1_rna_from_genomic.fna available in the NCBI Annotation Release 102). Transcript IDs from the `blastx` output were matched with GO terms from the Uniprot-Swissprot database using Uniprot Accession codes. These transcript IDs and corresponding GO terms were then used to create a gene ID-to-GO term database (`geneID2GO`) for manual GO term annotation. The transcript IDs were filtered for genes that contained CpGs with at least 5x coverage in all samples, as the same parameters were used to generate the CpG background for differential methylation analysis. Each line of the database contained transcript ID in one column, and all corresponding GO terms in another column.

Gene enrichment analysis was conducted with `topGO` v.2.34.0 in R [@Alexa2010]. The `geneID2GO` database was used as the gene universe for enrichment, while transcript IDs for genes with DML were used as the genes of interest. These transcript IDs were filtered such that they did not include any transcripts associated with DML-SNP overlaps. A `topGO` object was generated for each DML list and GO category (biological process, cellular component, or molecular function), with GO term annotation performed using the `geneID2GO` database. A Fisher’s exact test was used to identify GO terms in each DML list significantly enriched with respect to the gene background (P-value < 0.01). Keeping with `topGO` default settings, we did not correct for multiple comparisons [@Alexa2010]. Visualization of enriched GO terms was conducted using `simplifyEnrichment` v.1.2.0 default settings [@Gu2021]. Briefly, a semantic similarity matrix was created from enriched GO IDs using the “Relevance” similarity measure and GO tree topology (`GO_similarity`). The semantic similarity matrix was clustered using the binary cut method, and visualized as a word cloud alongside a heatmap of semantic similarity (`simplifyGO`).

## Results

### Water chemistry

Pacific oysters exposed to low pH experienced different water chemistry parameters than those in the ambient pH treatment (Table 2.1). Using water samples from days 5, 33, and 48, pH (One-way ANOVA; F1, 16 = 5838.7810, p = 6.1165e-22), pCO~2~ (One-way ANOVA; F~1, 16~ = 235.4018, p = 5.4421e-11), DIC (One-way ANOVA; F~1, 16~ = 7.1222, p = 0.0168), Ω~calcite~ (One-way ANOVA; F~1, 16~ = 528.9468, p = 1.0989e-13), Ω~aragonite~ (One-way ANOVA; F~1, 16~ = 526.5207, p =1.1389e-13) were significantly lower in the low pH treatment. Total alkalinity, however, was not significantly different between pH treatments (One-way ANOVA; F~1, 16~ = F = 1.382, p = 0.2570).

### Gonad maturation

A binomial GLM was used to compare gonad maturation of individuals sampled before and immediately after pH exposure, but before reproductive conditioning. The most parsimonious model included only sampling time (before or after pH treatment). Gonad maturation status was not significantly different between *C. gigas* sampled before and after pH treatment (binomial GLM; F~2, 37~ = 0.7973, p = 0.3442). Additionally, maturation status was not different between pH treatments (binomial GLM; F~3, 36~ = 2.2675, p = 0.1408). No sampled oysters possessed fully mature gametes, but some males sampled appeared to be undergoing resorption (supplemental information available in the [Github repository](https://github.com/RobertsLab/paper-gigas-early-gametogenic-exposure)). Sex ratios were also similar between low and ambient pH treatments (Chi-squared test for homogeneity; X^2^~2~ = 3.2279; p = 0.1942).

### Larval Survival 

A linear mixed effect model, with female pool and sire as random effects, demonstrated no significant difference in the number of D-hinge larvae counted eighteen hours post-fertilization between all four parental families (Linear mixed effect model; X^2^~3~ = 3.1325; p = 0.1066). Sire and female egg pools accounted for 0.8530% and 3.1623% of total variance, respectively. Significantly fewer D-hinge larvae were present in half-sibling families where females were exposed to low pH conditions (Figure \@ref(fig:carryoverboxplot); Linear mixed effect model; X^2^~1~ = 8.1781; p = 0.0042), with sire accounting for 0.3116053% of total variance.

## Discussion

The present study is the first to document the transgenerational influence of ocean acidification on Pacific oysters. Larval *C. gigas* was negatively impacted when maternal broodstock were exposed to low pH (pH = 7.31), suggesting a maternal carryover effect. The experimental design of this study is also unique — adult *C. gigas* experienced low pH conditions three months prior to reproductive conditioning, then were kept solely in ambient pH conditions through strip spawning and larval rearing. Since environmental perturbation experienced before Pacific oysters were mature still affected larval oysters, the results indicate a role for environmental memory in *C. gigas* response to ocean acidification. Mechanisms for transgenerational environmental memory have been explored in response to acute stressors in other species. For example, *Daphnia magna* exposed to high salinity conditions had altered DNA methylation patterns, and these patterns were inherited by the following three non-exposed generations [@Jeremias2018]. Significant carryover effects observed in *C. gigas* — solely exposed to low pH when immature — broaden the current understanding of stressor timing and its effect on organismal physiology.

While it is evident that acute exposure to low pH experienced by adult *C. gigas* resulted in detrimental effects for larvae, the fact that larvae were not reared in acidified conditions makes cross-study comparison difficult. If *C. gigas* larvae were also reared in acidified conditions, it is possible that larvae with a history of parental exposure to experimental ocean acidification may have exhibited a negative carryover effect on larval growth and performance. Negative carryover effects have been found in other marine invertebrate taxa, but all studies involved exposure to experimental ocean acidification during reproductive conditioning and larval rearing in acidified conditions. Tanner crabs (*Chionoecetes bairdi*) solely exposed to acidified water (pH 7.5 or 7.8) as larvae did not exhibit significant changes in morphology, size, Ca/Mg content, or metabolic rate, yet substantial effects on physiology was observed when larvae had a history of maternal exposure during oogenesis [@Long2016]. Larvae from adult northern quahog (= hard clam; *M. mercenaria*) and bay scallops (*A. irradians*) developed slower when parents were reproductively conditioned in low pH conditions (pH~T~ = 7.4) [@Griffith2017]. Additionally, larvae with a history of parental low pH exposure were more vulnerable to additional stressors like thermal stress, limited food, and harmful algae exposure [@Griffith2017]. Although *C. gigas* were not reproductively conditioned in acidified water, and the present study cannot distinguish between hatching success and early mortality, identifying a similar negative larval carryover effect four months after an acute environmental perturbation is arguably more surprising and significant, particularly in terms of efforts to understand the mechanism of environmental memory.

The severity of conditions experienced by organisms may also explain whether or not offspring demonstrate transgenerational acclimatization to stressors. For example, the negative carryover effect observed in *C. gigas* is different from the positive carryover effects observed in ocean acidification experiments conducted with Sydney rock oysters. When adult *S. glomerata* were exposed to acidified seawater (pCO~2~ = 856 µatm; pH~NBS~ = 7.89-7.90) during reproductive conditioning, resultant larvae were larger and developed faster in acidified conditions when compared to larvae from parents exposed to ambient conditions [@Parker2012]. This positive carryover effect was found to persist in the F2 generation. In acidified conditions, F2 offspring with a history of transgenerational (F0 and F1) pCO~2~ exposure grew faster and demonstrated fewer shell abnormalities [@Parker2015]. While species-specific responses can certainly explain the observed differences in larval phenotypes, it is also likely that inconsistencies in treatment conditions between experiments resulted in dose-dependent effects. @Parker2012, @Parker2015, and @Parker2017 used a high pCO~2~ treatment of 856 µatm (pH = 7.89-7.90), with a control of 380-385 µatm (pH = 8.19-8.20). Therefore, the elevated pCO~2~ treatment used in those studies is similar to the ambient pH treatment (7.82; pCO~2~ = 747.51-912.22) in the present study. Sydney rock oyster larvae with a history of transgenerational exposure exhibited faster development, but exhibited similar survival and were only 10% larger in acidified conditions when compared to larvae with no transgenerational exposure history [@Parker2012]. With a relatively smaller effect size and a milder treatment than used in this study, it is possible these studies are not at odds but reflect dose-dependent effects on larval phenotypes. Negative carryover effects demonstrated in this study and in @Griffith2017] can also be attributed to similar treatment pH levels [@Griffith2017: pHT = 7.4, this study: pH = 7.31]. Both of these studies used treatment levels more extreme than International Panel on Climate Change projections for open ocean acidification, but consistent with coastal and estuarine acidification scenarios experienced at study locations [@Feely2010; @Griffith2017; @Pelletier2018]. More research is required to understand how location-specific conditions will affect multiple generations in a single species.

Although the effect of water chemistry on gametogenesis has been recorded in other taxa, it is unlikely that a low pH exposure occurring three months prior to reproductive conditioning could have affected gonad maturation. Studies in which reproductive conditioning and experimental ocean acidification occur concurrently have demonstrated negative effects on maturation and fecundity. Gametogenesis, especially oogenesis, was disrupted in eastern oysters (*Crassostrea virginica*) that experienced severe ocean acidification conditions during reproductive conditioning (pH = 7.71, 5584 µatm) [@Boulais2017]. Green sea urchins (*Stronglyocentrotus droebachiensis*) exposed to high pCO~2~ (1200 µatm) conditions for four months demonstrated low fecundity [@Dupont2013], and *S. glomerata* conditioned in high pCO~2~ (856 µatm) conditions exhibited reduced rates of gametogenesis, smaller gonad area, and reduced fecundity [@Parker2018]. Gonad histology from *C. gigas* taken immediately after low or ambient pH exposure did not indicate any differences in maturation state, or interaction between sex and maturation state, between treatments. Even if fecundity or rates of gametogenesis differed between treatments, a return to ambient conditions for three months may have reversed any detrimental effects.

Reduced *C. gigas* larval abundance could have been a result of altered maternal provisioning in female oysters exposed to low pH conditions. In the face of stressors, females can either increase maternal provisioning [@Allen2008; @Sunday2011] — diverting more resources, like lipids or proteins, into eggs — or decrease provisioning due to energetic constraints [@Liu2010; @Uthicke2013]. For example, changes in fatty acid provisioning from maternal exposure to high pCO~2~ conditions (2300 µatm) in Atlantic silverside (*Menidia menidia*) resulted in lower embryo survival when eggs lacked certain fatty acids [@Snyder2018]. This phenomenon, however, was not documented in the Sydney rock oyster: while elevated pCO~2~ conditions (856 µatm) reduced the amount of energy invested in maternal gonads, these conditions did not impact *S. glomerata* egg size or total lipid content [@Parker2018]. Since adult *C. gigas* did not experience environmental perturbation after low pH exposure, and received enough food to spawn well, any impact on maternal provisioning and subsequent larval abundance was likely a result of low pH three months prior to reproductive conditioning. 

The documented effect on Pacific oyster larval abundance four months after low pH exposure indicates an important role for environmental memory in *C. gigas* response to ocean acidification. Low pH exposure may have induced epigenetic modifications (eg. changes in DNA methylation) in adult *C. gigas*. Studies of finfish and shellfish aquaculture species have demonstrated environmentally-induced epigenetic modifications that modify phenotypic responses in organisms [@Gavery2017]. One notable study on *C. gigas* examined parental effects of adult pollutant exposure on offspring [@Rondon2017]. Spat from parents exposed to the herbicide diuron had differential methylation in coding regions, with some changes leading to differential gene expression [@Rondon2017]. This research indicates that a mechanism crucial for phenotypic plasticity and acclimation across generations exists, and this knowledge can be analyzed in the context of climate-related environmental stressors. Epigenetic modifications in response to ocean acidification have been documented in coral species [@Putnam2016], but not in molluscs. Several experimental ocean acidification studies, however, hint at the role of epigenetic memory. Olympia oysters (*Ostrea lurida*) exposed to high pCO~2~ (1000 µatm) conditions still grew less in the juvenile life stage than counterparts reared in ambient pCO~2~, even after the stressor had been removed [@Hettinger2013]. Similarly, transgenerational acclimation of *S. glomerata* larvae with a history of exposure to acidified conditions could be explained by changes in epigenome that affect organismal performance [@Parker2012; @Parker2015; @Parker2017]. Methylation levels are known to increase over the course of gametogenesis, with male and female *C. gigas* exhibiting significantly different methylation patterns [@Zhang2018]. If epigenetic modifications were acquired by female oysters during low pH exposure, it could explain why a significant effect on larval abundance was detected four months after the exposure ended. Epigenetic mechanisms and altered maternal provisioning are not necessarily mutually exclusive — changes in the methylome could influence maternal provisioning — and both could contribute to the results observed in this study.

The results of this study emphasize the need to broaden the scope of when environmental perturbation experienced by an organism is considered stressful, and when an effect can be detected. Although there was no observable effect on adult gonad maturation right after low pH exposure, significant differences in larval abundance were detected four months after the exposure ended. Stressor timing and duration can impact transgenerational responses between mature parents and offspring [@Donelson2018]. While experimental ocean acidification (pH 7.7; pCO~2~ = 800 µatm) increased female investment in amphipods (*Gammarus locusta*), the subsequent generation exhibited fewer eggs and lower fecundity in the same conditions [@Borges2018]. Transgenerational benefits of maternal exposure to different temperatures (17ºC or 21ºC) in threespine stickleback (*Gasterosteus aculeatus*) differed based on exposure duration [@Shama2014]. Grandparents (F0) were only exposed to treatment temperatures during reproductive conditioning, while parents (F1) experienced either temperature over the course of development. The F1 generation exhibited temperature tolerances similar to the F0 maternal rearing environment, but the F2 generation tolerance was more similar to the F0 generation than the F1 generation [@Shama2014]. The present study demonstrates that length and timing of environmental perturbation experienced by immature individuals can still affect offspring. @Massamba-N’Siala2014 elucidated a similar phenomenon with marine polychaetes (*Ophryotocha labronica*): offspring experienced positive carryover effects of female exposure to temperature conditioning only when mothers were exposed to these conditions during late oogenesis; exposure during early oogenesis lead to negative carryover effects. More research should be conducted to understand how stressor timing, specifically before reproductive maturity, can impact carryover effects.

Most other experiments investigating stressor timing are conducted in a multiple stressor framework [@Gunderson2016]. For example, elevated temperatures and low salinity had synergistic effects on *O. lurida* when they were co-occurring stressors, but two to four weeks of recovery in between stressors negated these effects [@Bible2017]. Incorporating recovery time in a single-stressor experimental design is also crucial for accurately understanding how environmental perturbation impacts organism physiology. Exposure at one point in time may elicit a response much later in time, in a different environmental setting, or in a different generation, as evidenced by the present study and @Hettinger2013. The experimental design in the present study is unique, featuring a significant recovery time between low pH exposure and spawning. More single-stressor experiments should incorporate lag times between exposure to stress and measuring response variables to understand if these responses change over time. Adding a multigenerational component to such experiments can elucidate if acute exposures generate carryover effects.

Significant decreases in larval abundance four months after broodstock were exposed to acidified seawater has implications for both aquaculture and natural *C. gigas* populations. Parents and offspring — or even different offspring life stages — may not experience the same environmental chemistry. For example, upwelling conditions affecting adult *C. gigas* may subside once spawning occurs. Long-term monitoring of wild Pacific oyster populations, with detailed environmental chemistry reporting, will be crucial for understanding how brief exposures to adverse conditions affect reproductive success and larval abundance in the field. Responses to stressors should not only be documented during and after the perturbation occurs, but also for an extended time afterward. Hatchery-reared *C. gigas* larvae can also experience different conditions than broodstock. Facilities unable to control water chemistry conditions may be exposing immature individuals to environmental perturbations that could affect larvae once spawned. The success of “priming” — exposing *C. gigas* to stressful conditions to induce environmental memory and increase fitness — hinges on the identification of “programming windows” [@Gavery2017]. The present study shows that the period of time before reproductive conditioning can be important for transferring environmental memory, although only negative carryover effects have been demonstrated in *C. gigas*.

\clearpage

## Tables

\begin{landscape}

Table 2.1: Average (± SE) pH, total alkalinity (µmol/kg), pCO\textsubscript{2} (µatm), dissolved organic carbon (DIC; µmol/kg), calcite saturation state (Ω\textsubscript{calcite}), and aragonite saturation state (Ω\textsubscript{aragonite}) for three timepoints during low pH exposure (Day). The `seacarb` library in R was used to calculate total alkalinity, and in situ pCO\textsubscript{2}, Dissolved Inorganic Carbon (DIC), calcite saturation (Ω\textsubscript{calcite}), and aragonite saturation (Ω\textsubscript{aragonite}) for each oyster tank. Averages for both control (ambient pH) and experimental (low pH) values were calculated from three replicate tanks each. Between all three days, pH (One-way ANOVA; F\textsubscript{1, 16} = 5838.7810, p = 6.1165e-22), pCO\textsubscript{2} (One-way ANOVA; F\textsubscript{1, 16} = 235.4018, p = 5.4421e-11), DIC (One-way ANOVA; F\textsubscript{1, 16} = 7.1222, p = 0.0168), Ω\textsubscript{calcite} (One-way ANOVA; F\textsubscript{1, 16} = 528.9468, p = 1.0989e-13), Ω\textsubscript{aragonite} (One-way ANOVA; F~1, 16~ = 526.5207, p =1.1389e-13) were significantly lower experimental treatment. Total alkalinity, however, was not significantly different between treatments (One-way ANOVA; F\textsubscript{1, 16} = 1.382, p = 0.2570).

```{r carbchem,echo=FALSE}
carbchem <- read.csv("data/Ch2/Table2.1.csv")
  kableExtra:::kable_styling(knitr::kable(carbchem, "latex",col.names = gsub("[.]", " ", names(carbchem)), 
                  caption = "Carbonate chemistry parameters for control (C) and experimental (E) tanks",
                  longtable = TRUE, booktabs = TRUE), font_size = 5)
```

\end{landscape}

\clearpage

## Figures

\begin{landscape}

Figure 2.1: Pacific oysters (n = 140) were acclimated for 15 days, then twenty were randomly sampled for histological analyses. Remaining oysters were divided into ambient pH or low pH treatments for seven weeks. Three experimental tanks for each treatment were used with 20 oysters per tank for a total of 60 oysters per treatment. At the end of the pH exposure, a total of ten oysters were randomly selected from each treatment and sampled for histological analyses. All remaining oysters were then held in ambient pH conditions for 3 months. Finally, oysters were reproductively conditioned and strip spawned. Larvae were counted eighteen hours post-fertilization.\newline 

\begin{figure}[h]
\centering
  \includegraphics[width=1.3\textwidth]{figure/Ch2/Figure2.1.jpg}
  \caption{Experimental timeline}
  (\#fig:expdesign)
\end{figure}
\end{landscape}
\clearpage

**Figure** \@ref(fig:carryoverboxplot): Proportion of live D-hinge larvae eighteen hours post-fertilization by female treatment. Each box represents proportions of live larvae between the first and third quartiles for half-sibling families where the female was exposed to either ambient or low pH conditions. Horizontal lines outside the box indicate the minimum value before the lower fence and the maximum value before the upper fence, with the solid line indicating the median. Circles represent outliers. A proportion of 1.0 indicates that all eggs in a cross were successfully fertilized and developed into D-hinge larvae. A linear mixed model, with sire as a random effect, indicated significantly fewer D-hinge larvae were present in half-sibling families where females were exposed to low pH conditions (t = -2.999; p = 0.0119). Significantly different proportions are indicated by a letter.\newline 

\begin{figure}[h]
\centering
  \includegraphics[width=0.85\textwidth]{figure/Ch2/Figure2.2.pdf}
  \caption{Larval survival 18 hours post-fertilization}
  (\#fig:carryoverboxplot)
\end{figure}
\clearpage
