# Polyploidy and environmental stress have distinct impacts on Pacific oyster (*Crassostrea gigas*) ctenidia methylomes

## Abstract

Production of polyploid organisms resilient to environmental change requires a molecular understanding of stress response mechanisms. This is especially important when considering the use of triploid oysters in commercial aquaculture settings. Examination of DNA methylation can elucidate gene expression regulation mechanisms employed by triploid oysters, and how they may differ from their diploid counterparts. In other cultured marine organisms, polyploid induction does not yield methylation variation similar to organisms that undergo natural genome duplication. We examined how artificial genome duplication and environmental variation contribute to stress response phenotypes in the Pacific oyster (*Crassostrea gigas*). We exposed diploid (n = 12) and triploid (n = 12) *C. gigas* to low or high pH stress for six weeks. Ctenidia methylomes were then obtained using whole genome bisulfite sequencing. While there were no global methylation differences between samples, distinct differentially methylated loci (DML) were associated with ploidy status (154 ploidy-DML) and pH condition (178 pH-DML). Functions of genes containing ploidy-DML were not similar to those observed in organisms that undergo natural polyploidization. However, genes with pH-DML were associated with apoptosis, protein ubiquitination, zinc ion binding, and cytoskeletal processes commonly observed in oyster response to ocean acidification. Given the enrichment of both ploidy- and pH-DML in genes with multiple transcripts, it is possible that these methylation changes can influence gene expression and physiological phenotype. Our study — which is the first to provide single base-pair resolution assessment of diploid and triploid methylomes in a marine invertebrate — suggests that while methylation may play different regulatory roles during artificial and natural genome duplication events, environmentally-sensitive methylome responses may be uniform between diploid and triploid *C. gigas*.

## Introduction

As molluscan breeding programs focus on developing oyster lines resilient to environmental stressors associated with climate change, several programs are focusing on culturing triploid organisms. Triploid oysters, which may be generated through chemical induction or by mating diploid and tetraploid parents, contain three copies of the genome [@Nell2002]. This third genome copy renders these oysters partially sterile, with triploids diverting resources that would be for reproduction towards growth [@Nell2002]. As a result, triploids exhibit significantly faster growth rates, improved meat quality, and increased survival, with mated triploids having an additional advantage over chemically-induced oysters [@Wadsworth2019]. The potential advantages of triploidy have been examined with respect to heat stress, salinity, hypoxia, and disease pressure in *Crassostrea spp*. While triploid Pacific oysters (*Crassostrea gigas*) may have an energetic advantage over their diploid counterparts at elevated temperatures due to reduced gonadal investment and corresponding increases in tissue weight, condition index, protein, and carbohydrate levels (Shpigel et al., 1992), triploid eastern oysters (*C. virginica*) are more sensitive to desiccation stress associated with farm rearing, and display higher mortality than their diploid counterparts [@Bodenstein2021]. Triploid hybrids of the edible oyster *Crassostrea hongkongensis* and Suminoe oyster *Crassostrea ariakensis* grew faster as larvae in the hatchery and as juveniles in environmentally-distinct outplant locations, but displayed less salinity tolerance than their diploid hybrid counterparts [@Qin et al2020]. Low salinity conditions in the Chesapeake Bay removed growth advantages associated with *C. virginica* triploids [@Callam2016]. While triploid *C. virginica* juveniles did not display any survival differences in hypoxic conditions, time to mortality in *C. ariakensis* triploids was three times earlier than diploid oysters [@Lombardi2013]. Active gametogenesis in *C. gigas* lead to increased *Vibrio* infection, irrespective of ploidy status, but triploid oysters had poorer survival after disease challenges in winter conditions [@DeDecker2011]. In the Chesapeake Bay, triploid *C. virginica* had lower mortality and greater shell height, weight, yield, and a higher proportion of market-sized oysters than diploids, even when faced with similar *Perkinsus marinus* disease pressure [@Dégremont2012]. However, triploid *C. virginica* were susceptible to mortality events that are not associated with disease or environmental conditions, suggesting that there are elements of triploid physiology and stress response still unknown [@Matt2020]. To date, there is no documentation of triploid oyster responses to ocean acidification, and how this stress response differs from diploid organisms. Even though triploids are more sensitive to a variety of environmental stressors, without a mechanistic understanding of triploid sensitivity to ocean acidification, it is difficult to implement hatchery techniques such as environmental manipulation to produce more stress-tolerant triploid phenotypes [@Gavery2017].

Epigenetic mechanisms, like DNA methylation, can mediate phenotypic plasticity and responses to environmental stressors, which makes them useful for understanding triploid stress tolerance. Epigenetics refers to changes in gene expression that do not arise from changes in the DNA sequence, with methylation of cytosine bases being the most studied mechanism [@Bird2002; @Deans2015]. Baseline variation in methylation patterns, or standing epigenetic variation, can be associated with genomic structure. Currently, polyploidization impacts on DNA methylation is well-understood in organisms that demonstrate natural ploidy variation. Genome duplication events in plants are subsequently followed by sequence loss, gene duplication, transposon silencing, and changes to genome regulation mediated by epigenetic mechanisms [@Adams2005; @Chen2006; @Henry2005; @Matzke1999; @Salmon2005; @Verhoeven2010; @Wang2004]. These methylation changes promote normal genomic function after polyploidization, and can persist for a few generations after the duplication event [@Wang2004]. Social insects with natural genome duplication also display distinct methylation variation associated predominantly with ploidy status, which may be responsible for reducing transcriptional noise in haploid organisms [@Glastad2014]. Organisms that experience induced triploidy may not display methylation variation associated with genome duplication. For example, examination of diploid and artificially-induced *C. gigas* and brown trout (*Salmo trutta* L.) did not find any global methylation differences [@Covelo-Soto2015; @Jiang2016]. Even so, promoter sequences in triploid *Apostichopus japonicus* sea cucumbers exhibited correlation between methylation and expression [@Han2021]. As standing methylation can still influence gene expression in triploid organisms, investigating these mechanisms can provide insight into triploid oyster gene expression regulation.

Ocean acidification, can also influence methylomes [@Eirin-Lopez2018], with pteropods [@Bogan2020], corals [@Liew2018; @Putnam2016], and urchins [@Strader2020; @Strader2019] demonstrating distinct methylation patterns in response to low pH. Diploid oysters have been well-studied in this respect. Eastern oyster (*C. virginica*) mantle [@Downey-Wall2020] and gonad [@Venkataraman2020], and *C. hongkongensis* mantle [@ChandraRajan2021], and larvae [@Lim2020] all display changes in DNA methylation after experimental ocean acidification exposure. However, no studies to date have examined environmentally-sensitive methylation in triploids. In plants, polyploidization can influence the standing methylation variation that evolves over time and interacts with environmental stressors [@Verhoeven2010]. Transcriptomic investigations have started to unveil differences in diploid and triploid stress responses in molluscan species. For example, triploid abalone exposed to heat stress and hypoxia conditions exhibited significantly different transcriptional changes when compared to diploids [@Kim2021]. Since DNA methylation could influence gene activity and impact stress responses, elucidating genomic and environmental variation contribute to phenotype is crucial for understanding potential acclimatization mechanisms and their application to triploid aquaculture [@Gavery2017].

We present the first single-base pair resolution examination of diploid and triploid *C. gigas* methylomes in response to an environmental stressor. Our comparison of DNA methylation by ploidy status allows us to discern how epigenetic variation differs with genome duplication. By comparing oyster methylomes exposed to low and high pH conditions, we can elucidate how ocean acidification can exert an influence over ctenidia methylation. Finally, we determine if genetic and environmental variation interact to produce a unique methylation phenotype. Our work provides the foundation for understanding the mechanisms behind triploid *C. gigas* stress responses, and the application of knowledge for aquaculture practices that select for oysters resilient to climate stressors.

## Methods

### Experimental Design

Pacific oysters (n = 24) were obtained from the Hawaiian Shellfish LLC hatchery located in Kea`au, Hawaiʻi Island and held at the University of Hawai’i, Hilo. The broodstock used to produce these oysters originated from the Molluscan Broodstock Program at Oregon State University. Triploids were produced by mating diploid female and tetraploid male oysters. The diploids used in the experiment were not used to produce the triploids. Diploid (n = 12) and triploid (n = 12) *C. gigas* were randomly assigned to each of two treatments (high pH and low pH). 

The treatment tanks were 150 gallon conical bottom tanks with flow-through seawater from a deep well. Carbon dioxide was dispersed into both treatment tanks to achieve “non-acidification” conditions (pCO~2~ ranged from 67.37 to 98.61 µatm) to “ocean acidification” conditions (pCO~2~ ranged from 287.88 to 418.20 µatm). Water quality parameters were checked daily. pH probes were placed about 2.5 cm under the surface of the water and connected to pH controllers (American Marine). Dissolved oxygen, salinity, and temperature were monitored daily with an YSI Pro 2030. Ammonia concentration was monitored daily using an aquarium test kit (API). Full water exchanges were performed daily. Since the ctenidia is the primary tissue that interacts with the environment, ctenidia tissue was obtained from each oyster after six weeks, preserved in 2 mL DNA/RNA Shield (ZymoResearch), then stored at -20ºC.

### DNA Extraction and Library Preparation

DNA was extracted from each sample for Whole Genome Bisulfite Sequencing (WGBS). Samples were centrifuged for one minute at 10,000 RCF to pellet ctenidia tissue. After removing the DNA/RNA shield, ctenidia DNA was extracted using the E.Z.N.A. Mollusc Kit (Omega). Manufacturer’s protocol was followed, with the exception of an overnight incubation at 37ºC after the addition of 350 µL ML1 Buffer and 25 µL Proteinase K Solution. After the MB1 lysis buffer was added to samples, a thick white precipitate formed. Samples were vortexed to emulsify the precipitate, but several samples did not lyse fully. Samples were eluted with 100 µL of Elution Buffer. A Qubit™ 3.0 and dsDNA BR Assay (Invitrogen) was used to confirm that samples contained at least 500 ng of DNA, using 2 µL of DNA for each sample. 

Samples were sent to ZymoResearch for bisulfite conversion and library preparation. Bisulfite conversion was performed with the Pico Methyl-Seq Library Prep Kit (Cat. #D5455). Per ZymoResearch protocol, samples were spiked with a mixture of six unique double-stranded synthetic amplicons (180-200 bp) from the *Escherichia coli* K12 genome. Since each amplicon represents a different percent methylation, the spike-in was used to determine bisulfite conversion efficiency. Libraries were barcoded and pooled into a single lane to generate 150 bp paired-end reads in a single Illumina NovaSeq flowcell. 

### Genome Information and Feature Tracks

Nuclear [@Peñaloza2021] (NCBI GenBank: GCA_902806645.1, RefSeq: GCF_902806645.1) and mitochondrial genome sequences (NCBI Reference Sequence: NC_001276.1) were used in downstream analysis. These sequences were combined for downstream alignment. The `fuzznuc` function within EMBOSS was used to identify CG motifs in the combined nuclear and mitochondrial sequences.

The RefSeq annotation was used to obtain genome feature tracks, and create additional tracks. The annotation included a combination of Gnomon, RefSeq, cmsearch, and tRNAscan-SE predictions. Gene, coding sequence, exon, and lncRNA tracks were extracted from the RefSeq annotation. These tracks were then used to obtain additional genome feature information. To create an intron track, the complement of the exon track was generated with BEDtools `complementBed` v.2.26.0 to create a non-coding sequence track [@Quinlan2010]. The overlap between the genes and the non-coding sequences was found with `intersectBed` and defined as introns. Similarly, untranslated regions of exons were obtained by subtracting coding sequences from exon information with `subtractBed`. Flanking regions were defined as the 1000 bp upstream or downstream of genes. Upstream flanks were generated by adding 1000 bp upstream of genes, taking into account strand, with `flankBed`. Existing genes were removed from flanking sequences using `subtractBed`. A similar process was performed to generate downstream flanking regions. Intergenic regions were isolated by finding the complement of genes with `complementBed`, then using `subtractBed` to remove any flanks. Transposable element locations were obtained from RepeatMasker using the NCBI RefSeq annotation (RefSeq: GCF_902806645.1) [@Smit2013; @Tarailo-Graovac2009]. The number of CG motifs in a genome track was obtained using `intersectBed`.

### Sequence Alignment

Prior to alignment, 150 bp paired-end reads generated for each sample underwent three rounds of trimming. First, Illumina adapter sequences were removed from samples using TrimGalore! v.0.6.6 [@Martin2011]. Additionally, 10 bp were trimmed from both the 5’ (`--clip_R1 10` and `--clip_R2 10`) and 3’ ends of paired reads (`--three_prime_clip_R1 10` and `--three_prime_clip_R2 10`). Remaining adapters were then removed from sequences in the second round of TrimGalore! trimming. Poly-G tails present in data were removed by manually specifying a matching adapter sequence (`--adapter GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG` and `--adapter2 GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG`). After each round of trimming, sequence quality was assessed using FastQC v.0.11.9 [@Andrews2010] and MultiQC reports [@Ewels2016].

Trimmed samples were then aligned to the *C. gigas* genome [@Peñaloza2021]. The full genome was bisulfite converted with Bowtie 2-2.3.4 (Linux x84_64 version [@Langmead2012]) using the `bismark_genome_preparation` function within Bismark v.0.22.3 (Krueger & Andrews, 2011) prior to sequence alignment. Reads were then aligned to each bisulfite-converted genome, specifying non-directional input (`--non_directional`) and allowing for an alignment score no lower than -90 (`-score_min L,0,-0.6`). Aligned BAM files were deduplicated with the `--deduplicate_bismark` function, then sorted and indexed for downstream applications using SAMtools v.1.10 [@Li2009]. Sequence methylation calls were extracted from deduplicated BAM files with `bismark_methylation_extractor`. Coverage files generated from methylation extraction contained CpG information from both strands. To consolidate CpG information between strands and report 1-based chromosomal coordinates for each CpG, `coverage2cytosine` was run on deduplicated coverage files. Using `coverage2cytosine` coverage file output, genomic coordinates for CpGs with at least 5x coverage across all samples were written to BEDgraphs. Finally, `bismark2report` and `bismark2summary` were used to create summary reports with alignment information for each sample that were concatenated with MultiQC.

### SNP Identification

As genomic differences can play a role in methylation, single nucleotide polymorphisms (SNPs) were identified from WGBS data. Two methods were used for SNP identification. First, SNP variants across all samples and in individuals were identified with BS-Snper [@Gao2015]. As `BS-Snper` only takes one input file, deduplicated and sorted BAM alignments generated with Bismark v.0.22.3 and SAMtools v.1.10 [@Krueger2011; @Li2009] were merged (`samtools merge`). Variants were then identified in the merged BAM file or individual deduplicated and sorted BAM files. Default `BS-Snper` settings were used, except for a minimum coverage of 5x to be consistent with methylation analyses. C/T SNPs were filtered by identifying overlaps (`intersectBED`) between each set of SNPs and the CG motif track.

### Global Methylation Characterization

Global methylation patterns were characterized using CpG loci with at least 5x coverage in all samples. BEDgraphs produced with Bismark v.0.22.3 were sorted (`sortBed`), then used to concatenate percent methylation information across samples (`unionBedGraphs`). Percent methylation values were averaged between all samples for each CpG locus. These loci were categorized as highly (≥ 50%), moderately  (10-50%), or lowly methylated (≤10%) based on average percent methylation values. The genomic location of these methylation categories was determined with respect to UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions (`intersectBed`). We tested the null hypothesis that there was no association between the genomic location of CpG loci and methylation status (all CpGs versus highly methylated CpGs) with a chi-squared contingency test (`chisq.test` in R Version 3.5.3). To quantify differences in percent methylation by ploidy status, percent methylation values from the union BEDgraph were averaged for diploid and triploid samples separately.

As methylation detection relies on coverage, we also estimated average coverage for diploid and triploid oysters. Merged coverage files produced with Bismark v.0.22.3 were sorted (`sortBed`), then used to concatenate coverage information across samples (`unionBedGraphs`). Coverage was averaged for diploid and triploid samples separately to quantify if ploidy status influenced coverage.

Methylation patterns were compared between treatments using `methylKit` v.1.17.4 [@Akalin2012] with R Version 3.5.3. Data with at least 2x coverage from merged CpG coverage files produced by `coverage2cytosine` were imported using `methRead.` For each sample, data was filtered further (`filterByCoverage`) by requiring at least 5x coverage for each CpG (`lo.count = 5`), and removing potential PCR duplicates by excluding data in the 99.9th percentile of coverage (`high.perc = 99.9`). Filtered data were then normalized (`normalizeCoverage`) between samples to avoid over-sampling reads from one sample during downstream statistical analysis. Percent methylation (`getMethylationStats`) and CpG coverage (`getCoverageStats`) histograms were generated for each sample to confirm normalization. Filtered and normalized data were combined (unite) for every CpG position with at least 5x coverage with data in at least eight samples per treatment to generate a CpG background. Pairwise Pearson’s correlation coefficients were calculated and a global Principal Component Analysis based on percent methylation profiles was conducted to understand similarities and differences in sample methylation.

### Identification of Differentially Methylated Loci

Differentially methylated loci (DML) were identified using a generalized experimental design with `DSS` v.2.38.0 (Feng et al., 2014). The purpose of this generalized design was to test pH, ploidy, and their interaction simultaneously. For each sample, the corresponding merged coverage file from Bismark v.0.22.3 were modified to meet DSS input formatting requirements. The coverage file contained information for CpG loci with at least 1x coverage in that sample. Only the start position in the coverage files was used, and the number of total reads was calculated by summing the number of methylated and unmethylated reads provided in the coverage files. The final input files included the following columns: chromosome, start position, number of total reads, and number of methylated reads. Data was not pre-filtered based on a coverage threshold, following DSS input specifications. After importing information into R Studio using `list2env`, a BSseq object (`makeBSseqData`, `bsseq` v.1.26.0) was generated with coverage data to use for model fitting. To define the experimental design, a data frame was created with ploidy status (“D” = diploid vs. “T” = triploid), pH treatment (“H” = high pH vs. “L” = low pH), and sample ID. 

Methylation was calculated at each CpG locus using the Bayesian hierarchical model framework provided by `DSS.` Coverage data were modeled using a beta-binomial distribution with an arcsine link function and log-normal priors [@Feng2014]. Unlike the Fisher’s exact test used by `methylKit`, `DSS` assumes that data from each sample do not have identical distributions, retaining biological variation between samples in a treatment. The model was fitted onto transformed data using the generalized least squares method, defining pH, ploidy, and their interaction as coefficients (`DMLfit.multiFactor, formula = ~ ploidy + pH + ploidy:pH`). Smoothing, or using information from nearby CpG sites to inform methylation calculations, was not used, as invertebrate methylation data does not tend to be dense enough to provide multiple CpG dinucleotides in close proximity to each other.

A Wald test (`DMLtest.multiFactor`) was used at each CpG site for statistical testing of differential methylation for each coefficient separately (`ploidyT`, `pHL`, `ploidyT:pHL`). The test itself depends on sequencing depth, and a dispersion parameter defined by a shrinkage estimator from the Bayesian hierarchical model. The DML (ploidy-DML, pH-DML, and interaction-DML) were defined based on a P-value < 0.05 and Benjamini-Hochberg FDR < 0.01. The multiple regression framework used does not provide methylation values, so percent methylation differences were not used to define DML from `DSS.` BEDfiles with DML information were created for downstream analysis and visualized with sample coverage information using IGV.

### DML Characterization

Overlaps between all DML BEDfiles and UTR, CDS, introns, up- and downstream flanks, transposable elements, and intergenic regions (`intersectBed`) were used to describe where DML were located in the *C. gigas* genome. A chi-squared contingency test was used to test the null hypothesis of no association between methylation and specific genome features using all CpGs with 5x data and DML. The number of DML in each chromosome was quantified and divided by chromosome length to discern if DML were distributed uniformly across the genome.

To determine if ploidy and pH treatments elicited similar methylation responses, we identified loci common between pH-DML, ploidy-DML, and interaction-DML. Overlaps between these lists were obtained using intersectBed. Additionally, overlaps between C/T SNPs and DML were also identified to discern if genomic factors other than treatment contributed to methylation responses.

### Functional Enrichment Analysis

A functional enrichment analysis was performed to identify gene ontology (GO) terms significantly overrepresented in genes with ploidy-, pH-, or interaction-DML. Prior to enrichment, a blastx was performed against the Uniprot-SwissProt database (accessed June 01, 2021) [@UniProt2019] to obtain Uniprot Accession information for each RNA nucleotide sequence from the genome (GCF_902806645.1_cgigas_uk_roslin_v1_rna_from_genomic.fna available in the NCBI Annotation Release 102). The Uniprot Accession codes were used to match transcript IDs from the blastx output with GO terms from the Uniprot-Swissprot database. 

Gene enrichment analysis was conducted with `topGO` v.2.34.0 in R [@Alexa2010]. First, a gene ID-to-GO term database (`geneID2GO`) was created for manual GO term annotation. The list of transcript IDs and corresponding GO terms generated during annotation was filtered to only include transcript IDs for genes with at least 1x coverage in the WGBS dataset. These transcript IDs were filtered such that they did not include any transcripts associated with DML-SNP overlaps. Each line of the database contained transcript ID in one column, and all corresponding GO terms in another. The list of transcript IDs from this database was used as the gene universe for enrichment. Genes of interest corresponded to the transcript IDs from individual DML lists. A `topGO` object was generated for each DML list and GO category (biological process, cellular component, or molecular function), with GO term annotation performed using the `geneID2GO` database. A Fisher’s exact test was used to identify GO terms in each DML list significantly enriched with respect to the gene background (P-value < 0.05). Visualization of enriched GO terms was conducted using `simplifyEnrichment` v.1.2.0 default settings [@Gu2021]. The “Relevance” similarity measure and GO tree topology (`GO_similarity`) were used to create a semantic similarity matrix from enriched GO IDs. The matrix was then clustered using the binary cut method and visualized as a word cloud alongside a heatmap of semantic similarity (`simplifyGO`).


##Results
We found no differences between the average male and female pinniped trophic position over the 50-year study period (Figure 3) for the four tested species-region classifications. This finding was consistent for all trophic amino acids-source amino acid pairs (Figure 3). Based on glutamic acid trophic position estimates, both western stock Steller sea lions (2.6 ± 0.5; mean ± sd) and eastern stock Steller sea lions (2.7 ± 0.16) had similar trophic positions. Harbor seals in the Gulf of Alaska foraged higher in the food web than their Steller sea lion counterparts (Figure 3). Harbor seals in the southeast region had a higher trophic position on average than any other pinniped in this study (3.5 ± 0.3) but were similar to harbor seals in the northern region (3.3 ± 0.5). Bristol Bay (3.1 ± 0.4) and Iliamna lake (3.0 ± 0.3) harbor seals had a lower trophic position than their Gulf of Alaska counterparts on average (Figure 3).

###Common trends in Alaskan pinniped trophic position

The best performing model (Table 2, model 6) of pinniped trophic position included both species-region classification and decade as random effects (shared trends) along with an interaction between population and decade (Table 2). Based on the support for decade and classification to be included as group-level effects, these data support consistent differences between classifications over time, as well as differences between trophic position for all classifications. The supported interaction between population and decade (Table 2) indicates distinct decadal changes in trophic position for species-region classifications exist. The model that included decade, classification, and the interaction between decade and classification as fixed effects (model 4) was also supported based on the models LOOIC (Table 2). Therefore, the inclusion of the interaction term was more important for improving model performance than inclusion of decade and classification as fixed versus random effects.

There were consistent differences in trophic position that varied by species and ocean basin for the model with the most support. Harbor seals in the Gulf of Alaska had higher trophic position than their Steller sea lion counterparts. The mean difference of the posterior distributions indicated southeast Gulf of Alaska harbor seals have historically fed at 0.32 [-0.01, 0.61] (highest density 80% credible interval) trophic levels higher than sympatric eastern stock Steller sea lions (Figure 4). Similarly, the mean difference of posterior distributions showed northern Gulf of Alaska harbor seals fed 0.28 [-0.03, 0.50] trophic levels higher than the sympatric western stock Steller sea lions. Within the Gulf of Alaska, the posterior distributions for trophic position overlapped 39% between harbor seals and Steller sea lions in both the eastern and western regions (Figure 4). Iliamna Lake harbor seals have historically fed at a lower trophic level (mean posterior difference 0.16 [-0.11, 0.41]) than harbor seals in Bristol Bay, but these two classifications have 66% overlap of the group-level posterior distributions for trophic position (Figure 4). The 80% credible intervals included 0 for most region-species classifications thus the posterior probabilities support marginal evidence for consistent differences in trophic position between classifications. Regardless, the differences in posterior means were large, although the distributions were wide. 

There were no consistent decadal differences in trophic position across the region-species classification (Figure 5). Pinniped trophic position in the 2000s was slightly higher for all classifications (mean posterior difference 0.03 [-0.09, 0.16]) on average compared to 1990 and the posterior distributions for 1990 and 2000 had an 85% overlap (Figure 5). Similarly, posterior distributions in between 2000 and 2010 had a mean difference of -0.1 [-0.27, 0.08] with a 65% overlap (Figure 4). Overall, decadal differences in pinniped trophic position through time were smaller than the region-species classification effects and were likely ecologically inconsequential.

###Spatial and temporal differences in pinniped trophic structure

Distinct decadal changes in trophic position were observed for each species-region classification and varied more than the shared decadal changes (Figure 6) as indicated by the decade-classification interaction. Most, but not all, pinniped classifications experienced substantial trophic level change in 2000 or 2010 but the magnitude and direction of this change varied by region-species classification based on the combined effects of decade, classification, and the decade-classification interaction (Figure 6). The recent decadal change in trophic position was most prominent for the western stock of Steller sea lions which had a mean trophic level decrease of 0.43 [-0.25, -0.60] from 1990 to 2000 (a percent decrease of 0.15) with only a 21% overlap between the posterior distributions (Figure 6E). This decline in trophic position remained in the 2010s. A similar decline was observed in the southeast Gulf of Alaska harbor seals. This population experienced relatively stable trophic position from 1960-1990, which then declined on average by 0.31 [-0.19, -0.45] trophic levels in 2000 (33% posterior overlap) (Figure 6C). In contrast, harbor seals in the northern Gulf of Alaska had variable trophic position across decades and had the highest trophic position in 2000 in contrast to their southeast Gulf of Alaska harbor seals and Steller sea lion counterparts (Figure 6B). Data were only available for 2000 and 2010 for the eastern stock Steller sea lions, and trophic position was similar for this population during both of these decades (Figure 6F). Both Bristol Bay and Iliamna Lake harbor seals had relatively stable trophic position from 1950s until 2010s (Figure 6A & B). Bristol Bay harbor seals experienced their lowest trophic level in the 1990s with a 0.24 [-0.54, 0.00] trophic level decrease compared to the 1970s and 2000s, but the posterior distribution still overlapped 54% with other decades (Figure 6A). 

##Discussion

Over the past 70 years, Alaskan pinnipeds have exhibited both common and distinct differences in trophic position across region-species classification on decadal scales (Table 2). While potential drivers of change in trophic position were not tested in this study due to data limitations, our results support a combination of local-scale (i.e., vessel traffic, reduction of glacial ice, local foraging) and regional-scale (i.e., environmental condition, basin-wide prey abundance) changes may be influencing pinniped trophic ecology. Furthermore, the largest decadal changes in pinniped trophic position were distinct for each region-species classification and were most apparent during the most recent two decades (2000s and 2010s). These patterns are more pronounced in the Gulf of Alaska compared to Bristol Bay (Figures 5 & 6). 

###Regional and species trends in harbor seal trophic position 

Both Steller sea lions and harbor seals exhibit generalist foraging patterns [@Lance2012; @Geiger2013]. Diets of Alaskan pinnipeds consist of similar prey species but vary between species, population, and local availability of prey [@Iverson1997; @Hirons2001]. Bulk stable isotope studies in the Gulf of Alaska have shown that Steller sea lions feed lower in the food web compared to harbor seals [@Iverson1997]. Our CSIA analysis confirms the interpretation of these previous studies that isotopic differences can be attributed to trophic position changes and not isotopic shift of basal phytoplankton resources. Both western and eastern stock Steller sea lions have lower trophic position compared to sympatric harbor seal populations but have similar trophic position compared to other populations such as Iliamna Lake. However, despite known differences in both diet [@Sinclair2002; @Geiger2013] and nearshore versus offshore foraging [@Merrick1997; @Lowry2001] between the two species, our results also show historical overlap in trophic position, indicating potential trophic redundancy between harbor seals and Steller sea lions in the Gulf of Alaska.

Harbor seals in Bristol Bay and Iliamna Lake are managed as a single population [@Muto2020] despite lack of evidence of migration by the freshwater population and utilization of different resources [@Brennan2019]. A previous study of strontium and carbon stable isotopes showed Iliamna Lake harbor seals utilize freshwater-derived resources (resident lake fishes), particularly early in life, and exhibit an ontogenetic shift to more marine resources (returning sockeye salmon) later in life [@Brennan2019]. Based on CSIA nitrogen data, Iliamna Lake harbor seals also forage lower in the food web compared to Bristol Bay harbor seals. In addition, both classifications exhibited trophic stability, with the Bristol Bay harbor seals only experiencing a trophic shift in the 1990s relative to the 1960s and 1970s. This coincided with the lowest sockeye salmon returns to Iliamna Lake on record [@Hilborn2003]. Interestingly, the decrease in trophic position in the 1990s occurred simultaneously with decreases in basin wide Bristol Bay harbor seal abundance in the late 1990s, which then stabilized and increased in the 2000s and 2010s (Figure 2). Data were not available for the freshwater harbor seals between 1990 and 2000 and thus it is unclear whether the freshwater population also experienced a trophic position change during the 1990s when sockeye salmon returns were low. While quantitative comparisons to salmon abundance were not made in this study, salmon population abundance and harbor seal trophic ecology and population trends are seemingly interrelated. 

###Recent trophic position changes in the Gulf of Alaska

Trophic position changes were observed in all pinniped classifications in the Gulf of Alaska during the past two decades, although the direction of these changes varied on more local scales. During the past two decades (2000-2020), harbor seals in the Gulf of Alaska have experienced stabilization of most monitored populations following long-term declines that persisted from the 1950s through the 1990s (Figure 2, @Muto2020). During this same time period, harbor seals in both southeast and northern Gulf of Alaska experienced a shift in trophic position that was particularly prominent in the 2000s compared to historic estimates of trophic position (Figure 6B & C). It is possible that the observed trophic position shift may have contributed to the population stabilization of Gulf of Alaska harbor seals, either by an increase in prey availability or opportunistically foraging on a novel prey source. @Gagne2018 observed similar trophic position declines in seabird populations, which were attributed to a shift in diet from fish to squid. A similar dietary shift could explain the observed trophic position shift in southeast Gulf of Alaska harbor seals and western stock Steller sea lions. 

Recent regional change in the Gulf of Alaskan food webs has been well documented in other species and primarily attributed to bottom-up effects of climate [@Barbeaux2020; @Litzow2020]. These region-wide trends likely altered prey availability for pinniped populations in the Gulf of Alaska. How pinniped populations have adapted their foraging ecology, however, indicates regional and species trophic divergence, which could be attributed to either local-scale foraging adaptations or differences in prey availability. Pinniped groups that overlap in space (Figure 1) revealed divergent trends in trophic position between Steller sea lions and harbor seals in recent decades (Figure 6B & E). For example, trophic position of northern Gulf of Alaska harbor seals increased in the 2000s while the western stock of Steller sea lions decreased. For western stock Steller sea lions, this shift also persisted into the 2010s (Figure 6E). Posterior distributions of western stock Steller sea lions and northern Gulf of Alaska harbor seals overlapped by 63% in the 1950s but only overlapped by 3% in the 2000s (Figure 6B & E). The recent change in pinniped trophic position within the Gulf of Alaska coincided with population abundance stabilization, albeit at lower than historical abundance for most populations. This trophic divergence indicates there could be increased competition for resources between northern Gulf of Alaska harbor seals and western stock Steller sea lions resulting in diet adaptations. Similar comparisons were challenging to make for the eastern stock of Steller sea lions and southeast Gulf of Alaska harbor seals due to limitations in historical data for the former. However, trophic position in the 2000s showed a 38% overlap (Figure 6C & E) between the two species, indicating any trophic divergence between them may be less pronounced in this region, if existent. 

The observed divergent trends indicate differences in how Alaskan pinnipeds are adapting to environmental and ecological changes. Trophic position changes from stable isotope data can be accounted for by: 1) prey switching between different species, 2) consuming different sizes of the same prey, or 3) consuming different quality prey. These changes can occur at the consumer level (pinnipeds) or lower in the food web and still be reflected in consumer stable isotope signature and thus trophic position. In recent decades, Pacific salmon and halibut in Alaska have both declined in size [@Holsman2019; @Oke2020]. These changes in size distributions of prey have been attributed to changes in marine mammal populations (Groskreutz et al. 2019) and likely contributed to the observed trophic position declines in western Steller sea lion and southeast Gulf of Alaska harbor seals. In contrast, consuming low-quality prey with lower protein content and greater amino acid imbalance between consumer and prey increases the amino acid trophic enrichment factor of nitrogen [@McMahon2015]. If not accounted for in trophic position equations, this increase in trophic enrichment factor can result in erroneously high trophic position estimates. This may explain the observed increase in estimated trophic position in northern Gulf of Alaska harbor seals where this population may be consuming a greater proportion of lower quality prey (i.e., crustaceans, shrimp, cephalopods) in recent decades rather than feeding on prey species that are higher in the food web. 

###Considerations and limitations for CSIA analyses

The data in this study were limited in sample size primarily due to the availability of archived specimens. As a result, we were not able to discern between known fine scale differences in populations or annual trends. For example, harbor seals in the southeast Gulf of Alaska consist of 13 individual stocks. Due to limitations in the number of archived specimens, these stocks were pooled and analyzed as a single classification despite known differences in genetic structure [@Muto2020]. Given the observed broad range in trophic position of these generalist predators, it is unlikely that inclusion of finer spatial dynamics would have changed the supported model, although variation in temporal trends within a classification may have been identified. Similarly, data were only available for eastern stock Steller sea lions for 2000s and 2010s. As a result, no historical comparisons were possible and the conclusions about this population are tentative. Nonetheless, this dataset offers historic documentation of pinniped trophic position that can be updated with future samples or additional archived specimens.

Trophic position estimates in this study were low compared to known foraging strategies of these pinnipeds. For example, Steller sea lions eat primarily walleye pollock and Atka mackerel [@Hobson1997; @Trites2007], which would indicate a trophic position of 3 or higher. Mean trophic position for Steller sea lions was closer to 2.7 in this study, which is lower than expected based on known foraging ecology. It is common for CSIA to underestimate trophic position of marine predators [@Germain2013; @McMahon2019] and the inclusion of multiple amino acid pairs and a multi-trophic enrichment factor framework did not fully resolve this issue. @Nielsen2015 found trophic position estimates can be highly sensitive to the applied $\beta$ values in equation 2. In our trophic position calculation, we assumed a constant $\beta$ represented by marine diatoms. However, $\beta$ values differ by more than 11 per mille between seagrasses and diatoms [@VanderZanden2013] which has been attributed to differences between vascular and nonvascular plants [@Choi2017, @Ramirez2021]. If vascular plants, such as seagrasses, contribute to the food web in addition to non-vascular algae, the applied β would be too high and would result in underestimation of trophic position of marine consumers [@Choi2017, @Ramirez2021]. Even a 10% contribution of vascular plant-derived nitrogen to the food web would result in an underestimation of 0.2 trophic position. It is likely that vascular plants at least partially contribute to the Alaskan food web, as seagrass beds provide essential habitat and food for many fish species and invertebrates. Consideration for variable $\beta$ values may be helpful in resolving trophic position underestimation of future studies, especially in cases where consumer carbon stable isotope data is available and contributions of seagrasses to the food web are well documented.

###Conclusions and Implications

Marine ecosystems in Alaska are experiencing unprecedented environmental change that has altered abundance and size distributions of many fish species consumed by pinnipeds [@Barbeaux2020; @Holsman2019; @Oke2020; @Suryan2021]. Heterogeneity in diet and foraging locations allow top predators to adjust to availability of resources by altering their foraging. Based on the observed region-species specific changes in trophic position over the past two decades, pinnipeds are experiencing different food web conditions than in the past, even those that occur in similar geographic regions. This may be the result of adapting foraging strategies to exploit other prey resources or a change that is occurring lower in the food web and is measurable in predators. While our results cannot discern between these two mechanisms of trophic level change, we can conclude that recent food web dynamics have impacted pinniped trophic ecology in Alaska. Future responses of pinnipeds to food web change will likely be locally variable between species, even those that occur within similar geographic regions. 

\clearpage


##Tables

**Table** \@ref(tab:paramval): Parameter values for trophic discrimination factors between a trophic amino acid (*i*) and phenylalanine (*o*) for harbor seals ($TDF_{(i-o), j}$), for an average consumer (TDF(i-o) ), and for primary producers ($\beta_{(i-o)}$)  derived from previous studies to apply a multi amino acid framework to equation 2.

```{r paramval,echo=FALSE}
supp1 <- read.csv("data/Ch4/Table1.csv")
colnames(supp1) <- c("Trophic Amino Acid (i)",
                           "$\\overline{\\beta}_{(i-o)}$",
                           "$TDF_{(i-o),j}$",
                           "$\\overline{TDF}_{(i-o)}$")
supp1 <- kableExtra:::kable_styling(knitr::kable(supp1, "latex", escape = FALSE,
caption = "Trophic position parameter values for Equation 2",
                  longtable = TRUE, booktabs = TRUE
                  ),font_size = 8)
 
kableExtra::column_spec(supp1, column=2:4, width = "10em")

```

\clearpage



**Table** \@ref(tab:candmodels): Candidate models for identifying spatial and temporal trophic structure of Alaskan pinnipeds. Assumptions define how the model describes trophic structure with regards to decade and classification and LOOIC describes the support of each candidate models. The best model (6) is italicized.

```{r candmodels,echo=FALSE}
supp1 <- read.csv("data/Ch4/Table2.csv")

 supp1<-  kableExtra:::kable_styling(knitr::kable(supp1, "latex", escape = FALSE, col.names = gsub("[.]", " ", names(supp1)), 
                  caption = "Candidate Models",
                  longtable = TRUE, booktabs = TRUE
                  ),font_size = 8)
supp1 <- kableExtra::row_spec(supp1,6,italic=T)
kableExtra::column_spec(supp1, column=2:4, width = "13em")

```

\clearpage