# Polyploidy and environmental stress have distinct impacts on Pacific oyster (*Crassostrea gigas*) ctenidia methylomes

## Abstract

Production of polyploid organisms resilient to environmental change requires a molecular understanding of stress response mechanisms. This is especially important when considering the use of triploid oysters in commercial aquaculture settings. Examination of DNA methylation can elucidate gene expression regulation mechanisms employed by triploid oysters, and how they may differ from their diploid counterparts. In other cultured marine organisms, polyploid induction does not yield methylation variation similar to organisms that undergo natural genome duplication. We examined how artificial genome duplication and environmental variation contribute to stress response phenotypes in the Pacific oyster (*Crassostrea gigas*). We exposed diploid (n = 12) and triploid (n = 12) *C. gigas* to low or high pH stress for six weeks. Ctenidia methylomes were then obtained using whole genome bisulfite sequencing. While there were no global methylation differences between samples, distinct differentially methylated loci (DML) were associated with ploidy status (154 ploidy-DML) and pH condition (178 pH-DML). Functions of genes containing ploidy-DML were not similar to those observed in organisms that undergo natural polyploidization. However, genes with pH-DML were associated with apoptosis, protein ubiquitination, zinc ion binding, and cytoskeletal processes commonly observed in oyster response to ocean acidification. Given the enrichment of both ploidy- and pH-DML in genes with multiple transcripts, it is possible that these methylation changes can influence gene expression and physiological phenotype. Our study — which is the first to provide single base-pair resolution assessment of diploid and triploid methylomes in a marine invertebrate — suggests that while methylation may play different regulatory roles during artificial and natural genome duplication events, environmentally-sensitive methylome responses may be uniform between diploid and triploid *C. gigas*.

## Introduction

As molluscan breeding programs focus on developing oyster lines resilient to environmental stressors associated with climate change, several programs are focusing on culturing triploid organisms. Triploid oysters, which may be generated through chemical induction or by mating diploid and tetraploid parents, contain three copies of the genome [@Nell2002]. This third genome copy renders these oysters partially sterile, with triploids diverting resources that would be for reproduction towards growth [@Nell2002]. As a result, triploids exhibit significantly faster growth rates, improved meat quality, and increased survival, with mated triploids having an additional advantage over chemically-induced oysters [@Wadsworth2019]. The potential advantages of triploidy have been examined with respect to heat stress, salinity, hypoxia, and disease pressure in *Crassostrea spp*. While triploid Pacific oysters (*Crassostrea gigas*) may have an energetic advantage over their diploid counterparts at elevated temperatures due to reduced gonadal investment and corresponding increases in tissue weight, condition index, protein, and carbohydrate levels [@Shpigel1992], triploid eastern oysters (*C. virginica*) are more sensitive to desiccation stress associated with farm rearing, and display higher mortality than their diploid counterparts [@Bodenstein2021]. Triploid hybrids of the edible oyster *Crassostrea hongkongensis* and Suminoe oyster *Crassostrea ariakensis* grew faster as larvae in the hatchery and as juveniles in environmentally-distinct outplant locations, but displayed less salinity tolerance than their diploid hybrid counterparts [@Qin2020]. Low salinity conditions in the Chesapeake Bay removed growth advantages associated with *C. virginica* triploids [@Callam2016]. While triploid *C. virginica* juveniles did not display any survival differences in hypoxic conditions, time to mortality in *C. ariakensis* triploids was three times earlier than diploid oysters [@Lombardi2013]. Active gametogenesis in *C. gigas* lead to increased *Vibrio* infection, irrespective of ploidy status, but triploid oysters had poorer survival after disease challenges in winter conditions [@DeDecker2011]. In the Chesapeake Bay, triploid *C. virginica* had lower mortality and greater shell height, weight, yield, and a higher proportion of market-sized oysters than diploids, even when faced with similar *Perkinsus marinus* disease pressure [@Dégremont2012]. However, triploid *C. virginica* were susceptible to mortality events that are not associated with disease or environmental conditions, suggesting that there are elements of triploid physiology and stress response still unknown [@Matt2020]. To date, there is no documentation of triploid oyster responses to ocean acidification, and how this stress response differs from diploid organisms. Even though triploids are more sensitive to a variety of environmental stressors, without a mechanistic understanding of triploid sensitivity to ocean acidification, it is difficult to implement hatchery techniques such as environmental manipulation to produce more stress-tolerant triploid phenotypes [@Gavery2017].

Epigenetic mechanisms, like DNA methylation, can mediate phenotypic plasticity and responses to environmental stressors, which makes them useful for understanding triploid stress tolerance. Epigenetics refers to changes in gene expression that do not arise from changes in the DNA sequence, with methylation of cytosine bases being the most studied mechanism [@Bird2002; @Deans2015]. Baseline variation in methylation patterns, or standing epigenetic variation, can be associated with genomic structure. Currently, polyploidization impacts on DNA methylation is well-understood in organisms that demonstrate natural ploidy variation. Genome duplication events in plants are subsequently followed by sequence loss, gene duplication, transposon silencing, and changes to genome regulation mediated by epigenetic mechanisms [@Adams2005; @Chen2006; @Henry2005; @Matzke1999; @Salmon2005; @Verhoeven2010; @Wang2004]. These methylation changes promote normal genomic function after polyploidization, and can persist for a few generations after the duplication event [@Wang2004]. Social insects with natural genome duplication also display distinct methylation variation associated predominantly with ploidy status, which may be responsible for reducing transcriptional noise in haploid organisms [@Glastad2014]. Organisms that experience induced triploidy may not display methylation variation associated with genome duplication. For example, examination of diploid and artificially-induced *C. gigas* and brown trout (*Salmo trutta* L.) did not find any global methylation differences [@Covelo-Soto2015; @Jiang2016]. Even so, promoter sequences in triploid *Apostichopus japonicus* sea cucumbers exhibited correlation between methylation and expression [@Han2021]. As standing methylation can still influence gene expression in triploid organisms, investigating these mechanisms can provide insight into triploid oyster gene expression regulation.

Ocean acidification, can also influence methylomes [@Eirin-Lopez2018], with pteropods [@Bogan2020], corals [@Liew2018; @Putnam2016], and urchins [@Strader2020; @Strader2019] demonstrating distinct methylation patterns in response to low pH. Diploid oysters have been well-studied in this respect. Eastern oyster (*C. virginica*) mantle [@Downey-Wall2020] and gonad [@Venkataraman2020], and *C. hongkongensis* mantle [@ChandraRajan2021], and larvae [@Lim2020] all display changes in DNA methylation after experimental ocean acidification exposure. However, no studies to date have examined environmentally-sensitive methylation in triploids. In plants, polyploidization can influence the standing methylation variation that evolves over time and interacts with environmental stressors [@Verhoeven2010]. Transcriptomic investigations have started to unveil differences in diploid and triploid stress responses in molluscan species. For example, triploid abalone exposed to heat stress and hypoxia conditions exhibited significantly different transcriptional changes when compared to diploids [@Kim2021]. Since DNA methylation could influence gene activity and impact stress responses, elucidating genomic and environmental variation contribute to phenotype is crucial for understanding potential acclimatization mechanisms and their application to triploid aquaculture [@Gavery2017].

We present the first single-base pair resolution examination of diploid and triploid *C. gigas* methylomes in response to an environmental stressor. Our comparison of DNA methylation by ploidy status allows us to discern how epigenetic variation differs with genome duplication. By comparing oyster methylomes exposed to low and high pH conditions, we can elucidate how ocean acidification can exert an influence over ctenidia methylation. Finally, we determine if genetic and environmental variation interact to produce a unique methylation phenotype. Our work provides the foundation for understanding the mechanisms behind triploid *C. gigas* stress responses, and the application of knowledge for aquaculture practices that select for oysters resilient to climate stressors.

## Methods

### Experimental Design

Pacific oysters (n = 24) were obtained from the Hawaiian Shellfish LLC hatchery located in Kea`au, Hawaiʻi Island and held at the University of Hawai’i, Hilo. The broodstock used to produce these oysters originated from the Molluscan Broodstock Program at Oregon State University. Triploids were produced by mating diploid female and tetraploid male oysters. The diploids used in the experiment were not used to produce the triploids. Diploid (n = 12) and triploid (n = 12) *C. gigas* were randomly assigned to each of two treatments (high pH and low pH). 

The treatment tanks were 150 gallon conical bottom tanks with flow-through seawater from a deep well. Carbon dioxide was dispersed into both treatment tanks to achieve “non-acidification” conditions (pCO~2~ ranged from 67.37 to 98.61 µatm) to “ocean acidification” conditions (pCO~2~ ranged from 287.88 to 418.20 µatm). Water quality parameters were checked daily. pH probes were placed about 2.5 cm under the surface of the water and connected to pH controllers (American Marine). Dissolved oxygen, salinity, and temperature were monitored daily with an YSI Pro 2030. Ammonia concentration was monitored daily using an aquarium test kit (API). Full water exchanges were performed daily. Since the ctenidia is the primary tissue that interacts with the environment, ctenidia tissue was obtained from each oyster after six weeks, preserved in 2 mL DNA/RNA Shield (ZymoResearch), then stored at -20ºC.

### DNA Extraction and Library Preparation

DNA was extracted from each sample for Whole Genome Bisulfite Sequencing (WGBS). Samples were centrifuged for one minute at 10,000 RCF to pellet ctenidia tissue. After removing the DNA/RNA shield, ctenidia DNA was extracted using the E.Z.N.A. Mollusc Kit (Omega). Manufacturer’s protocol was followed, with the exception of an overnight incubation at 37ºC after the addition of 350 µL ML1 Buffer and 25 µL Proteinase K Solution. After the MB1 lysis buffer was added to samples, a thick white precipitate formed. Samples were vortexed to emulsify the precipitate, but several samples did not lyse fully. Samples were eluted with 100 µL of Elution Buffer. A Qubit™ 3.0 and dsDNA BR Assay (Invitrogen) was used to confirm that samples contained at least 500 ng of DNA, using 2 µL of DNA for each sample. 

Samples were sent to ZymoResearch for bisulfite conversion and library preparation. Bisulfite conversion was performed with the Pico Methyl-Seq Library Prep Kit (Cat. #D5455). Per ZymoResearch protocol, samples were spiked with a mixture of six unique double-stranded synthetic amplicons (180-200 bp) from the *Escherichia coli* K12 genome. Since each amplicon represents a different percent methylation, the spike-in was used to determine bisulfite conversion efficiency. Libraries were barcoded and pooled into a single lane to generate 150 bp paired-end reads in a single Illumina NovaSeq flowcell. 

### Genome Information and Feature Tracks

Nuclear [@Peñaloza2021] (NCBI GenBank: GCA_902806645.1, RefSeq: GCF_902806645.1) and mitochondrial genome sequences (NCBI Reference Sequence: NC_001276.1) were used in downstream analysis. These sequences were combined for downstream alignment. The `fuzznuc` function within EMBOSS was used to identify CG motifs in the combined nuclear and mitochondrial sequences.

The RefSeq annotation was used to obtain genome feature tracks, and create additional tracks. The annotation included a combination of Gnomon, RefSeq, cmsearch, and tRNAscan-SE predictions. Gene, coding sequence, exon, and lncRNA tracks were extracted from the RefSeq annotation. These tracks were then used to obtain additional genome feature information. To create an intron track, the complement of the exon track was generated with BEDtools `complementBed` v.2.26.0 to create a non-coding sequence track [@Quinlan2010]. The overlap between the genes and the non-coding sequences was found with `intersectBed` and defined as introns. Similarly, untranslated regions of exons were obtained by subtracting coding sequences from exon information with `subtractBed`. Flanking regions were defined as the 1000 bp upstream or downstream of genes. Upstream flanks were generated by adding 1000 bp upstream of genes, taking into account strand, with `flankBed`. Existing genes were removed from flanking sequences using `subtractBed`. A similar process was performed to generate downstream flanking regions. Intergenic regions were isolated by finding the complement of genes with `complementBed`, then using `subtractBed` to remove any flanks. Transposable element locations were obtained from RepeatMasker using the NCBI RefSeq annotation (RefSeq: GCF_902806645.1) [@Smit2013; @Tarailo-Graovac2009]. The number of CG motifs in a genome track was obtained using `intersectBed`.

### Sequence Alignment

Prior to alignment, 150 bp paired-end reads generated for each sample underwent three rounds of trimming. First, Illumina adapter sequences were removed from samples using TrimGalore! v.0.6.6 [@Martin2011]. Additionally, 10 bp were trimmed from both the 5’ (`--clip_R1 10` and `--clip_R2 10`) and 3’ ends of paired reads (`--three_prime_clip_R1 10` and `--three_prime_clip_R2 10`). Remaining adapters were then removed from sequences in the second round of TrimGalore! trimming. Poly-G tails present in data were removed by manually specifying a matching adapter sequence (`--adapter GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG` and `--adapter2 GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG`). After each round of trimming, sequence quality was assessed using FastQC v.0.11.9 [@Andrews2010] and MultiQC reports [@Ewels2016].

Trimmed samples were then aligned to the *C. gigas* genome [@Peñaloza2021]. The full genome was bisulfite converted with Bowtie 2-2.3.4 (Linux x84_64 version [@Langmead2012]) using the `bismark_genome_preparation` function within Bismark v.0.22.3 (Krueger & Andrews, 2011) prior to sequence alignment. Reads were then aligned to each bisulfite-converted genome, specifying non-directional input (`--non_directional`) and allowing for an alignment score no lower than -90 (`-score_min L,0,-0.6`). Aligned BAM files were deduplicated with the `--deduplicate_bismark` function, then sorted and indexed for downstream applications using SAMtools v.1.10 [@Li2009]. Sequence methylation calls were extracted from deduplicated BAM files with `bismark_methylation_extractor`. Coverage files generated from methylation extraction contained CpG information from both strands. To consolidate CpG information between strands and report 1-based chromosomal coordinates for each CpG, `coverage2cytosine` was run on deduplicated coverage files. Using `coverage2cytosine` coverage file output, genomic coordinates for CpGs with at least 5x coverage across all samples were written to BEDgraphs. Finally, `bismark2report` and `bismark2summary` were used to create summary reports with alignment information for each sample that were concatenated with MultiQC.

### SNP Identification

As genomic differences can play a role in methylation, single nucleotide polymorphisms (SNPs) were identified from WGBS data. Two methods were used for SNP identification. First, SNP variants across all samples and in individuals were identified with BS-Snper [@Gao2015]. As `BS-Snper` only takes one input file, deduplicated and sorted BAM alignments generated with Bismark v.0.22.3 and SAMtools v.1.10 [@Krueger2011; @Li2009] were merged (`samtools merge`). Variants were then identified in the merged BAM file or individual deduplicated and sorted BAM files. Default `BS-Snper` settings were used, except for a minimum coverage of 5x to be consistent with methylation analyses. C/T SNPs were filtered by identifying overlaps (`intersectBED`) between each set of SNPs and the CG motif track.

### Global Methylation Characterization

Global methylation patterns were characterized using CpG loci with at least 5x coverage in all samples. BEDgraphs produced with Bismark v.0.22.3 were sorted (`sortBed`), then used to concatenate percent methylation information across samples (`unionBedGraphs`). Percent methylation values were averaged between all samples for each CpG locus. These loci were categorized as highly (≥ 50%), moderately  (10-50%), or lowly methylated (≤10%) based on average percent methylation values. The genomic location of these methylation categories was determined with respect to UTR, CDS, introns, up- and downstream flanking regions, transposable elements, and intergenic regions (`intersectBed`). We tested the null hypothesis that there was no association between the genomic location of CpG loci and methylation status (all CpGs versus highly methylated CpGs) with a chi-squared contingency test (`chisq.test` in R Version 3.5.3). To quantify differences in percent methylation by ploidy status, percent methylation values from the union BEDgraph were averaged for diploid and triploid samples separately.

As methylation detection relies on coverage, we also estimated average coverage for diploid and triploid oysters. Merged coverage files produced with Bismark v.0.22.3 were sorted (`sortBed`), then used to concatenate coverage information across samples (`unionBedGraphs`). Coverage was averaged for diploid and triploid samples separately to quantify if ploidy status influenced coverage.

Methylation patterns were compared between treatments using `methylKit` v.1.17.4 [@Akalin2012] with R Version 3.5.3. Data with at least 2x coverage from merged CpG coverage files produced by `coverage2cytosine` were imported using `methRead.` For each sample, data was filtered further (`filterByCoverage`) by requiring at least 5x coverage for each CpG (`lo.count = 5`), and removing potential PCR duplicates by excluding data in the 99.9th percentile of coverage (`high.perc = 99.9`). Filtered data were then normalized (`normalizeCoverage`) between samples to avoid over-sampling reads from one sample during downstream statistical analysis. Percent methylation (`getMethylationStats`) and CpG coverage (`getCoverageStats`) histograms were generated for each sample to confirm normalization. Filtered and normalized data were combined (unite) for every CpG position with at least 5x coverage with data in at least eight samples per treatment to generate a CpG background. Pairwise Pearson’s correlation coefficients were calculated and a global Principal Component Analysis based on percent methylation profiles was conducted to understand similarities and differences in sample methylation.

### Identification of Differentially Methylated Loci

Differentially methylated loci (DML) were identified using a generalized experimental design with `DSS` v.2.38.0 (Feng et al., 2014). The purpose of this generalized design was to test pH, ploidy, and their interaction simultaneously. For each sample, the corresponding merged coverage file from Bismark v.0.22.3 were modified to meet DSS input formatting requirements. The coverage file contained information for CpG loci with at least 1x coverage in that sample. Only the start position in the coverage files was used, and the number of total reads was calculated by summing the number of methylated and unmethylated reads provided in the coverage files. The final input files included the following columns: chromosome, start position, number of total reads, and number of methylated reads. Data was not pre-filtered based on a coverage threshold, following DSS input specifications. After importing information into R Studio using `list2env`, a BSseq object (`makeBSseqData`, `bsseq` v.1.26.0) was generated with coverage data to use for model fitting. To define the experimental design, a data frame was created with ploidy status (“D” = diploid vs. “T” = triploid), pH treatment (“H” = high pH vs. “L” = low pH), and sample ID. 

Methylation was calculated at each CpG locus using the Bayesian hierarchical model framework provided by `DSS.` Coverage data were modeled using a beta-binomial distribution with an arcsine link function and log-normal priors [@Feng2014]. Unlike the Fisher’s exact test used by `methylKit`, `DSS` assumes that data from each sample do not have identical distributions, retaining biological variation between samples in a treatment. The model was fitted onto transformed data using the generalized least squares method, defining pH, ploidy, and their interaction as coefficients (`DMLfit.multiFactor, formula = ~ ploidy + pH + ploidy:pH`). Smoothing, or using information from nearby CpG sites to inform methylation calculations, was not used, as invertebrate methylation data does not tend to be dense enough to provide multiple CpG dinucleotides in close proximity to each other.

A Wald test (`DMLtest.multiFactor`) was used at each CpG site for statistical testing of differential methylation for each coefficient separately (`ploidyT`, `pHL`, `ploidyT:pHL`). The test itself depends on sequencing depth, and a dispersion parameter defined by a shrinkage estimator from the Bayesian hierarchical model. The DML (ploidy-DML, pH-DML, and interaction-DML) were defined based on a P-value < 0.05 and Benjamini-Hochberg FDR < 0.01. The multiple regression framework used does not provide methylation values, so percent methylation differences were not used to define DML from `DSS.` BEDfiles with DML information were created for downstream analysis and visualized with sample coverage information using IGV.

### DML Characterization

Overlaps between all DML BEDfiles and UTR, CDS, introns, up- and downstream flanks, transposable elements, and intergenic regions (`intersectBed`) were used to describe where DML were located in the *C. gigas* genome. A chi-squared contingency test was used to test the null hypothesis of no association between methylation and specific genome features using all CpGs with 5x data and DML. The number of DML in each chromosome was quantified and divided by chromosome length to discern if DML were distributed uniformly across the genome.

To determine if ploidy and pH treatments elicited similar methylation responses, we identified loci common between pH-DML, ploidy-DML, and interaction-DML. Overlaps between these lists were obtained using intersectBed. Additionally, overlaps between C/T SNPs and DML were also identified to discern if genomic factors other than treatment contributed to methylation responses.

### Functional Enrichment Analysis

A functional enrichment analysis was performed to identify gene ontology (GO) terms significantly overrepresented in genes with ploidy-, pH-, or interaction-DML. Prior to enrichment, a blastx was performed against the Uniprot-SwissProt database (accessed June 01, 2021) [@UniProt2019] to obtain Uniprot Accession information for each RNA nucleotide sequence from the genome (GCF_902806645.1_cgigas_uk_roslin_v1_rna_from_genomic.fna available in the NCBI Annotation Release 102). The Uniprot Accession codes were used to match transcript IDs from the blastx output with GO terms from the Uniprot-Swissprot database. 

Gene enrichment analysis was conducted with `topGO` v.2.34.0 in R [@Alexa2010]. First, a gene ID-to-GO term database (`geneID2GO`) was created for manual GO term annotation. The list of transcript IDs and corresponding GO terms generated during annotation was filtered to only include transcript IDs for genes with at least 1x coverage in the WGBS dataset. These transcript IDs were filtered such that they did not include any transcripts associated with DML-SNP overlaps. Each line of the database contained transcript ID in one column, and all corresponding GO terms in another. The list of transcript IDs from this database was used as the gene universe for enrichment. Genes of interest corresponded to the transcript IDs from individual DML lists. A `topGO` object was generated for each DML list and GO category (biological process, cellular component, or molecular function), with GO term annotation performed using the `geneID2GO` database. A Fisher’s exact test was used to identify GO terms in each DML list significantly enriched with respect to the gene background (P-value < 0.05). Clustering of enriched GO terms was conducted using `simplifyEnrichment` v.1.2.0 default settings [@Gu2021]. The “Relevance” similarity measure and GO tree topology (`GO_similarity`) were used to create a semantic similarity matrix from enriched GO IDs. The matrix was then clustered using the binary cut method to identify broad GO term categories impacted by environmentally-responsive methylation.

## Results

### Sequence Alignment

Whole genome bisulfite sequencing produced 2.5 x 10^9^ total 150 bp paired-end reads. After trimming, 2.4 x 10^9^ total reads remained. Poly-G tails were present in the second paired read for all samples, and constituted 1-2% of reads removed. Of the trimmed paired-end reads, 7.4 x 10^8^ (61.0%) were uniquely aligned to the *C. gigas* genome and appended mitochondrial sequence. Across samples, 3.5 x 10^8^ cytosines (11.6%) were methylated in a CpG context. Coverage between diploid and triploid samples were distributed similarly (Figure \@ref(fig:coveragemeth)A-B). Of the 12,794,074 total CpG loci with 1x coverage in at least one sample, 5,395,386 had higher average coverage in triploids, and 6,956,658 had equal or lower average coverage in triploids.

### SNP Identification

Possible SNP variants were identified in WGBS data to determine if genomic differences contributed to differential methylation. A total of 14,857,859 unique SNPs, including 289,826 unique C/T SNPs, were found in WGBS data and used in downstream analyses.

### Global Methylation Characterization

Global methylation patterns were first characterized using 10,497,320 CpG loci (79.2% of 13,246,547 total CpGs in *C. gigas* genome) with at least 5x coverage in each sample from the union BEDgraph. Of these loci, 989,453 (9.4%) were highly methylated (≥ 50%), 848,655 (8.1%) were moderately methylated (10-50%), and 8,659,212 (82.5%) were lowly methylated. Both diploid and triploid samples were predominantly lowly methylated (Figure \@ref(fig:coveragemeth)C-D), with 2,735,188 loci having higher percent methylation in triploids and 6,605,916 loci with equal or lesser percent methylation in triploids. The highly methylated CpGs were found primarily in genes (932,568; 94.3%), with 526,656 (53.2%) genic CpGs in introns, followed by 353,333 (35.7%) in CDS, then 55,067 (5.6%) CpGs in exon UTR (Figure \@ref(fig:cpglocations)). More highly methylated CpGs were found in transposable elements (295,204; 29.8%) than intergenic regions (32,028; 3.2%), downstream flanking regions, (21,839; 2.2%), lncRNA (17,023; 1.7%), or upstream flanking regions (4,683; 0.5%). The proportion of highly methylated CpGs in various genome features was significantly different than the proportion of 5x CpGs in those same features.

Methylation patterns were compared between ploidy and pH treatments using CpG loci with at least 5x coverage and eight samples per ploidy treatment in the CpG background. Pairwise Pearson’s correlation coefficients were consistently between 0.87 and 0.93 for all samples. A global Principal Component Analysis did not separate samples based on pH treatment or ploidy status (Figure \@ref(fig:coveragemeth)E).

### Differential Methylation Characterization

Using a Bayesian hierarchical modeling approach, a total of 178 pH-DML, 154 ploidy-DML, and 53 interaction-DML were identified from 12,794,074 total CpG loci with 1x coverage in at least one sample (Figure \@ref(fig: multiheatmap)A-C). Of these DML, 21 overlapped between pH and ploidy treatments, 11 between pH and ploidy:pH, 17 between ploidy and ploidy:pH. Additionally, 25 DML were common between all lists. A small number of DML — with five (2.8%) pH-DML, three (1.9%) ploidy-DML, and one (1.9%) interaction-DSS — were present in unique C/T SNPs. The DML were identified in the ten main chromosomes, as well as other unplaced scaffolds. The ninth chromosome contained the most DML even though it has the fewest genes (Figure \@ref(fig: multiheatmap)D).

The DML were primarily in genes (Figure \@ref(fig: DMLgenomicloc)). For pH-DML, 123 (69.1%) DML were found in 94 unique genes, the majority of which were in intron sequences (104; 58.4%), followed by CDS (15; 8.4%) and exon UTR (4; 2.2%). Ploidy-DML and interaction-DML were similarly distributed between introns (ploidy: 114; 74.0% and interaction: 41; 77.4%), CDS (ploidy: 26; 16.9% and interaction: 6; 11.3%), and exon UTR (ploidy: 8; 5.2% and interaction: 3; 5.7%). Ploidy-DML and interaction-DML were also predominantly in 109 and 29 unique genes, respectively (ploidy: 145; 94.2% and interaction: 48; 90.1%). Most genes contained only one pH-DML, ploidy-DML, or interaction-DML. However, several genes contained multiple DML, with genes containing up to 8 pH-DML, 8 ploidy-DML, or 23 interaction-DML.

Transposable elements contained the second highest amount of DML (pH: 86; 48.3%, ploidy: 66; 42.9%, interaction: 14; 26.4%), followed by intergenic regions (pH: 27; 15.2%, ploidy: 18; 11.7%, interaction: 5; 9.4%). Few DML were found in lncRNA (pH: 4; 2.2%, ploidy: 1; 0.6%, interaction: 2; 3.8%), upstream flanking regions (pH: 2; 1.1% and ploidy: 8; 5.2%), or downstream flanks (pH: 2; 1.1% and ploidy: 8; 5.2%). No interaction-DML were found in either flanking region. The distribution of DML in the genome differed significantly from that of all CpGs with 5x data: specifically, the number of ploidy-DML in introns and intergenic regions, the number of pH-DML in introns, transposable elements, and intergenic regions, and the number of interaction-DML in introns and intergenic regions.

### Functional Enrichment Analysis

A series of Fisher’s exact tests identified 21 significantly enriched GOterms total in genes that do not overlap with C/T SNPs (Table \@ref(tab:enrichedterms)). A total of 84 genes annotated with GOterms containing 104 ploidy-DML were used to identify three enriched biological process terms associated with dephosphorylation and cell-cell binding. The enriched GOterms were found in eight unique genes and 29 unique transcripts. The remaining enriched GOterms were associated with pH-DML. For pH-DML, 69 genes with GOterm annotations and containing 87 DML were used for the analysis. Ten enriched biological process terms were identified, and were related to apoptotic processes. An additional eight molecular function terms were related to myosin and cytoskeletal binding, ubiquitin activity, and zinc ion binding.

## Discussion

Understanding how ploidy and environmental variation interact to shape phenotype is important for aquaculture operations moving forward with developing polyploid lines resilient to climate stressors like ocean acidification. Our investigation of methylation in diploid and triploid *C. gigas* is the first single-base pair resolution study to examine the influence of ploidy on DNA methylation in a mollusc species. Additionally, we discern the influence of pH and its interaction with ploidy on methylation. Our results provide insight into stress responses in triploid oysters, which may be leveraged to improve triploid aquaculture production. 

### Small-scale methylation differences associated with polyploidization

The lack of global methylation differences observed between diploid and triploid *C. gigas* suggests that genome duplication does not induce broad changes in molluscan methylation. Conversely, organisms that naturally undergo polyploidization may exhibit global methylation changes associated with ploidy status. Methylation patterns in haplodiploid ants (*Solenopsis invicta*) are highly divergent between haploid and diploid individuals, irrespective of morphology or caste behavior [@Glastad2014]. Haploid ants also exhibit consistently higher methylation at sequenced loci. Naturally polyploid loaches (*Misgurnus anguillicaudatus*) exhibit significant differences in methylation levels between diploids and triploids and diploids and tetraploids [@Zhou2016]. In our experiment there were no significant differences between global methylation profiles in diploid and triploid *C. gigas*, which implies methylation regulation may function differently at different ploidy states for naturally polyploid organisms. Since triploidy is not a part of the natural reproductive process in oysters, it is possible that *C. gigas* has not evolved global methylation patterns associated with the triploid state. Methylation profiling with MSAP did not detect global methylation differences in between diploid and artificially-induced triploid *C. gigas* or *S. trutta* L. [@Covelo-Soto2015; @Jiang2016]. While MSAP may only detect a subset of epigenetic variation, our WGBS results corroborate these results. The mating of diploid and tetraploid oysters to produce the triploid *C. gigas* used in our study itself may have introduced a higher level of standing epigenetic variation than what may be expected through natural polyploidization. Future studies should examine how different methods of triploid generation — mating versus chemical induction — influence methylation patterns in molluscs, and investigate genetic resistance to duplication processes.

Although we did not detect global methylation differences associated with polyploidization, we found individual CpG loci differentially methylated between diploid and triploid oysters. The identification of ploidy-DML suggests a role for methylation regulation of genome duplication, irrespective of pH condition. In plant species, DNA methylation regulates duplicated genes and transposable element silencing, and is generally involved in normal function and maintenance of polyploid genomes [@Adams2005; @Chen2006; @Henry2005; @Matzke1999; @Salmon2005; @Verhoeven2010; @Wang2004]. Gene ontology enrichment revealed enrichment of cell-cell adhesion and protein dephosphorylation processes. These processes are not explicitly related to the transcriptional control processes observed during genome duplication in plants, and to our knowledge have not been documented in association with polyploidization in any species. However, DML in these genes may still modify gene expression, which could then influence tissue structure and function.

The concentration of ploidy-DML in genes, specifically introns, implies that these DML may control gene expression. Additionally, enriched GOterms associated with ploidy-DML were found in genes with multiple transcripts, suggesting gene expression regulation may occur through alternative splicing [@Glastad2014; @Roberts2012]. We also found several ploidy-DML in transposable elements, which aligns with epigenetic regulatory pathways observed after genome duplication in plants. However, it is still unclear if methylation silencing of transposable elements occurs in invertebrate species [@Eirin-Lopez2018]. Interestingly, three (1.9%) ploidy-DML overlapped with unique C/T SNPs identified in WGBS data. While SNPs may only have a small influence on methylation patterns associated with ploidy status, additional research is needed to examine how genome duplication and variation interact to produce methylation profiles.

### Role of methylation in stress response elucidated by pH- and interaction-DML

The enriched gene ontologies associated with genes containing pH-DML reveal the potential for plastic responses to low pH stress. Enrichment of apoptotic processes imply a need to regulate the cell cycle in response to ocean acidification. Differential regulation of apoptotic processes in oysters exposed to ocean acidification conditions was also found in *C. virginica* mantle tissue [@Downey-Wall2020] and larval *C. hongkongensis* [@Lim2020], and there is a broader association between DNA methylation and apoptosis pathways [@Meng2011]. Enrichment of molecular functions related to protein ubiquitination complement enrichment of apoptosis functions. Protein ubiquitination is a post-translational protein modification that is involved in protein synthesis and degradation [@Komander2009; @Peng2003] and has been documented in molluscan response to ocean acidification. For example, shotgun proteomic characterization of posterior gill lamellae from adult *C. gigas* exposed to high pCO~2~ revealed increased abundance of proteins involved in ubiquitination and decreased protein degradation [@Timmins-Schiffman2014]. Elevated pCO~2~ levels were also found to upregulate malate dehydrogenase in adult *C. virginica* mantle tissue [@Tomanek2011]. Genes involved in protein ubiquitination also displayed differential methylation in *C. virginica* gonad tissue [@Venkataraman2020], and displayed both differential methylation and gene expression downregulation in *C. hongkongensis* mantle tissue [@ChandraRajan2021]. Degradation processes are also prominent in coral methylome responses to temperature and salinity variation [@Liew2018], urchin responses to temperature and pH [@Strader2020], and oyster responses to disease [@Johnson2020]. Together, these results imply that regulation of molecular and cellular degradation processes are conserved across taxa in response to environmental change, and concurrent gene expression and methylation studies are required to elucidate how changes in these processes impact physiology.

Other overrepresented molecular functions in genes with pH-DML are also consistent with existing research examining molluscan response to ocean acidification. Enrichment of myosin binding functions suggest low pH impacts the methylome associated with cytoskeletal processes. Low pH also induced changes to methylation in cytoskeletal genes in larval *C. hongkongensis* [@Lim2020]. Proteomic analysis of *C. gigas* mantle, *C. virginica* mantle, *C. hongkongensis* larvae, and *C. gigas* larvae found that ocean acidification exposure altered the abundance of various cytoskeletal proteins [@Wei2015; @Tomanek2011; @Dineshram2012; @Dineshram2013; @Dineshram2015; @Dineshram2016]. Additionally, selectively-bred Sydney rock oysters (*Saccostrea glomerata*) with a history of ocean acidification exposure had higher expression of cytoskeletal genes than wild type oysters [@Goncalves2016]. Changes to cytoskeletal proteins, gene expression, and methylation could be reflective of a conserved response to establish homeostasis in response to ocean acidification. The remaining enriched molecular function GOterms were related to zinc ion binding. Larval *C. hongkongensis* exposed to low pH also demonstrated methylation changes to genes associated with this function [@Lim2020]. Since zinc ion binding is necessary for several enzymes and signaling pathways, methylation regulation of this function could represent a generalist response to ocean acidification.

Our finding that low pH impacted *C. gigas* ctenidia methylation at specific loci is consistent with previous studies investigating the influence of ocean acidification on invertebrate methylomes. However, there appear to be species- and tissue-specific methylome responses regarding the exact location of genic responses. The pH-DML and interaction-DML were identified primarily in introns, followed by CDS and untranslated regions of exons. Baseline assessment of *C. gigas* ctenidia methylome found higher levels of methylation in exons, but did note a larger proportion of intronic methylation in comparison to other invertebrate methylomes [@Gavery2013]. Similarly, low pH-induced differential methylation in *C. hongkongensis* mantle tissue was more common in introns [@ChandraRajan2021]. In contrast, adult *C. virginica* mantle and gonad methylomes [@Downey-Wall2020; @Venkataraman2020] and larval *C. honkongensis* demonstrate a higher concentration of DML associated with ocean acidification in exons over introns. If methylation does regulate gene expression through alternative splicing, then the presence of DML in introns versus exons may impact which components are retained in the alternatively-spliced transcript. To date, pH-responsive methylation in *Crassostrea spp*. have not corresponded with changes in gene expression [@ChandraRajan2021; @Downey-Wall2020]. Additional research is needed in all three related species to determine how differential methylation in coding and non-coding sequences influence gene expression, alternative splicing, and protein abundance, and the influence of tissue type or developmental stage on those relationships.

The presence of interaction-DML suggests genomic and environmental variation can shape the ctenidia methylome and subsequent phenotypes. We predicted that the triploid methylome would be more impacted by pH, as triploids can be more sensitive to environmental stressors. This sensitivity would have been associated with distinct gene ontologies involved in stress response. However, we found no significantly enriched gene ontologies associated with these DML. While ploidy and pH may interact, our results signify that this variation does not produce a clear physiological phenotype.

## Conclusion

It is critical to understand the molecular mechanisms underpinning genomic regulation and environmental response in aquaculture species to improve production. Our single base-pair resolution sequencing of *C. gigas* ctenidia methylomes reveals distinct biological processes and molecular functions impacted by ploidy status and low pH exposure. In particular, pH influenced molecular and cellular degradation processes in both diploid and triploid oysters while ploidy had a smaller impact on cell binding. Interestingly, the lack of gene ontology enrichment in methylation responses associated with both ploidy and pH suggests that genomic and environmental variation have a lesser effect on physiology together than separately. Future studies should examine if the differential methylation observed in this study results in persistent changes in gene expression and stress tolerance phenotypes. If these DML can influence phenotype, environmental hardening in the hatchery may increase oyster resilience to low pH stressors and improve triploid production [@Gavery2017].

\clearpage

## Tables

**Table** \@ref(tab:enrichedterms): Significantly enriched GOterms identified using a Fisher’s exact test. There were 3 biological process terms associated with ploidy-DML, 10 biological process terms associated with pH-DML, and 8 molecular function terms associated with pH-DML. Biological process terms for ploidy-DML were broadly associated with cell-cell adhesion and dephosphorylation, while apopotis was enriched in pH-DML. Molecular functions overrepresented in genes with pH-DML were associated with protein ubiquitination, cytoskeletal proteins, and zinc ion binding.

```{r enrichedterms,echo=FALSE}
enrichedterms <- read.csv("data/Ch4/Table4.1.csv")
  kableExtra:::kable_styling(knitr::kable(enrichedterms, "latex",col.names = gsub("[.]", " ", names(enrichedterms)), 
                  caption = "Enriched GO terms for ploidy- and pH-DML",
                  longtable = TRUE, booktabs = TRUE), font_size = 6, bootstrap_options = c("striped"))
```

\clearpage

## Figures

**Figure** \@ref(fig:coveragemeth): Coverage and global methylation information for diploid and triploid samples. Average coverage for A) diploid and B) triploids. Coverage was calculated using information for CpGs with at least 1x coverage. Percent methylation distribution for C) diploids and D) triploids. Percent methylation was calculated using information for CpGs with at least 5x coverage in all samples.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch4/Figure4.1.pdf}
  \caption{Diploid and triploid coverage and global methylation}
  (\#fig:coveragemeth)
\end{figure}

\clearpage

**Figure** \@ref(fig:cpglocations): Percent of total CpGs present in distinct *C. gigas* genome features for various CpG categories (from left to right: all CpGs with at least 5x coverage in all samples, highly methylated CpGs, moderately methylated CpGs, and lowly methylated CpGs).\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch4/Figure4.2.pdf}
  \caption{Genome distribution of various CpG categories}
  (\#fig:cpglocations)
\end{figure}

\clearpage

**Figure** \@ref(fig:multiheatmap): Heatmap of A) ploidy-, B) pH-, and C) interaction-DML identified, and D) distribution in *C. gigas* chromosomes. Density plots associated with heatmaps quantify the number of DML in various methylation bins.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch4/Figure4.3.pdf}
  \caption{DML heatmaps and chromosomal distribution}
  (\#fig:multiheatmap)
\end{figure}

\clearpage

**Figure** \@ref(fig:DMLgenomicloc): Distribution of all 5x CpGs, ploidy-DML, pH-DML, and interaction-DML in various *C. gigas* genome features.\newline

\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch4/Figure4.4.pdf}
  \caption{DML heatmaps and chromosomal distribution}
  (\#fig:DMLgenomicloc)
\end{figure}

\clearpage
