`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

<!--
If you feel it necessary to include an appendix, it goes here.
-->

# Appendix 1

Data accessibility information and additional materials for each dissertation chapter can be found below.

## Chapter 1: Characterization of Pacific oyster (*Crassostrea gigas*) proteomic response to natural environmental differences

Raw data can be accessed in the PeptideAtlas under accession PASS01304 and PASS01305. Skyline documents can be found on Panorama Public. All scripts and workflows can be found in the associated Github repository (https://github.com/RobertsLab/paper-gigas-DNR-proteomics).

## Chapter 2: Larval response to parental low pH exposure in Pacific oysters (*Crassostrea gigas*)

All scripts and workflows can be found in the associated Github repository (https://github.com/RobertsLab/paper-gigas-early-gametogenic-exposure). 

Additional histology can be found in this repository, including microscope images (https://github.com/RobertsLab/paper-gigas-early-gametogenic-exposure/tree/master/images/Gigas-gonad-histology) and specific sex information for each individual sampled (https://github.com/RobertsLab/paper-gigas-early-gametogenic-exposure/blob/master/data/2017-Adult-Gigas-Tissue-Sampling/2018-02-27-Gigas-Histology-Classification.csv).

## Chapter 3: Low pH influences methylation patterns of gonad growth genes in the Pacific oyster (*Crassostrea gigas*)

All data, scripts, workflows, and outputs can be found in the associated Github repository (https://github.com/RobertsLab/project-gigas-oa-meth), including:

- A summary of sequencing and trimming information: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/12-functional-enrichment/all-CC-EnrichedGO-DML-withTranscript.csv
- Chi-squared test output comparing genomic locations of highly methylated CpGs against all CpGs with 5x data: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/09-methylation-landscape/CpG-location-statResults.txt
- Principal Components Analysis visualizing global methylation patterns: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/06-methylKit/figures/all-sample-PCA.pdf
- Plot of pairwise Pearson's Correlations for global methylation: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/06-methylKit/general-stats/Full-Sample-Pearson-Correlation-Plot-FilteredCov5Destrand.jpeg
- Chromosomal distribution of DML, including number of DML in scaffolds not placed in any chromosome: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/06-methylKit/DML/All-DML-by-chr.csv
- Number of DML in genes: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/10_DML-characterization/Number-of-DML-per-Gene.csv
- Chi-squared test output comparing genomic locations of DML against all CpGs with 5x data: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/10_DML-characterization/CpG-location-statResults-All.txt
- `simplifyEnrichment` figure for enriched cellular components: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/12-functional-enrichment/figures/simplifyEnrichment-CC.pdf
- Enriched biological process GO terms and associated gene and transcript information: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/12-functional-enrichment/all-BP-EnrichedGO-DML-withTranscript.csv
- Enriched cellular component GO terms and associated gene and transcript information: https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/12-functional-enrichment/all-CC-EnrichedGO-DML-withTranscript.csv

## Chapter 4: Polyploidy and environmental stress have distinct impacts on Pacific oyster (*Crassostrea gigas*) ctenidia methylomes

All data, scripts, workflows, and outputs can be found in the associated Github repository (https://github.com/RobertsLab/project-oyster-oa), including:

- Chi-squared test output comparing genomic locations of highly methylated CpGs against all CpGs with 5x data: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_06-methylation-landscape/CpG-location-statResults.txt
- Plot of pairwise Pearson's Correlations for global methylation: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_04-methylKit/general-stats/Full-Sample-Pearson-Correlation-Plot-FilteredCov5Destrand.jpeg
- Chromosomal distribution of ploidy-DML, including number of ploidy-DML in scaffolds not placed in any chromosome: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_04-DSS/DML/ploidy-DML-by-chr.csv
- Chromosomal distribution of pH-DML, including number of pH-DML in scaffolds not placed in any chromosome: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_04-DSS/DML/pH-DML-by-chr.csv
- Chromosomal distribution of interaction-DML, including number of interaction-DML in scaffolds not placed in any chromosome: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_04-DSS/DML/ploidypH-DML-by-chr.csv
- Number of ploidy-DML in genes: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/Number-of-ploidy-DML-per-Gene.csv
- Number of pH-DML in genes: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/Number-of-pH-DML-per-Gene.csv
- Number of interaction-DML in genes: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/Number-of-ploidypH-DML-per-Gene.csv
- Chi-squared test output comparing genomic locations of ploidy-DML against all CpGs with 5x data: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/CpG-location-statResults-ploidy.txt
- Chi-squared test output comparing genomic locations of pH-DML against all CpGs with 5x data: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/CpG-location-statResults-pH.txt
- Chi-squared test output comparing genomic locations of interaction-DML against all CpGs with 5x data: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_07-DML-characterization/CpG-location-statResults-ploidypH.txt
- `simplifyEnrichment` figure for enriched biological process terms associated with ploidy-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/figures/simplifyEnrichment-ploidy-BP.pdf
- Enriched biological process GO terms and associated gene and transcript information for ploidy-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/ploidy-BP-FisherTestResults.csv
- `simplifyEnrichment` figure for enriched biological process terms associated with pH-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/figures/simplifyEnrichment-pH-BP.pdf
- Enriched biological process GO terms and associated gene and transcript information for pH-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/pH-BP-FisherTestResults.csv
- `simplifyEnrichment` figure for enriched molecular function terms associated with pH-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/figures/simplifyEnrichment-pH-MF.pdf
- Enriched molecular function GO terms and associated gene and transcript information for pH-DML: https://github.com/RobertsLab/project-oyster-oa/blob/master/analyses/Haws_09-functional-enrichment/pH-MF-FisherTestResults.csv
