---
# UW thesis fields
title: "Molecular techniques for resilient Pacific oyster (*Crassostrea gigas*) aquaculture"
author: "Yaamini R. Venkataraman"
year: "2021"
program: "School of Aquatic and Fishery Sciences"
chair: "Steven B. Roberts"
chairtitle: "Associate Professor"
signature1: "Jonathan P. Davis"
signature2: "Jacqueline Padilla-Gamiño"
abstract: |
  As ocean acidification continues to impact marine ecosystems at unprecedented rates, phenotypic plasticity may allow organisms to withstand more stressful conditions. Genomic methods can elucidate molecular mechanisms that contribute to phenotypic plasticity, allowing for a deeper understanding of how physiological processes will be impacted by low pH. My dissertation examines the effects of ocean acidification on the Pacific oyster (*Crassostrea gigas*) stress response and reproduction; elucidate how exposure history impacts phenotype; and explore the role of functional role DNA methylation in somatic and reproductive tissue. I investigated the effect of regional environmental variation on the molecular physiology of *C. gigas* outplanted at five different estuarine sites (four in Puget Sound, one in Willapa Bay) in Washington, USA using gel-free proteomic methods. While there was no difference in survival, or any protein abundances due to pH differences between sites, *C. gigas* outplanted at the site with the highest temperature had significantly higher abundances of antioxidant enzymes and molecular chaperones, elucidating the molecular underpinnings of thermotolerance. In a hatchery setting, I explored the impact of ocean acidification on reproductive maturity and output. A seven week low pH exposure did not affect sex ratio or maturation stage; however, it did significantly affect survival of larvae. Even though adult oysters spent four months in ambient pH conditions between low pH exposure and strip spawning, larvae from females that experienced low pH conditions had significantly higher mortality. Finally, I conducted the first investigations examining the effect of ocean acidification in *C. gigas* methylomes. To investigate the role of environmentally-responsive methylation in reproductive tissue, I analyzed gonad methylomes of female *C. gigas* exposed to low pH. A total of 1,599 differentially methylated loci (DML) were found in gene bodies. The genic DML were associated with cilium movement, development, and cytoskeletal processes, implying a need to regulate cellular growth in the gonad in response to low pH. I then explored the influence of low pH on the somatic tissue methylome using diploid and triploid oyster ctenidia. Differences in ploidy status yielded 154 DML. These ploidy-DML were associated with cell-cell adhesion and dephosphorlylation processes, which are not commonly associated with methylome changes in organisms that undergo natural polyploidization. The 178 pH-DML were associated with processes commonly observed in oysters exposed to ocean acidification, including apoptosis, protein ubiquitination, zinc ion binding, and cytoskeletal processes. In both reproductive and somatic tissue, the enrichment of DML in genes with multiple transcripts could indicate a role for methylation to regulate gene expression via alternative splicing. Investigating the molecular underpinnings of responses to ocean acidification in *C. gigas* will provide a thorough understanding of this global aquaculture product’s ability to withstand future ocean conditions.
acknowledgments: |
  I am eternally grateful for the community that has supported me over the past five years. I cannot thank my adviser, Dr. Steven B. Roberts, enough for his mentorship. He gave me room to grow and think for myself as a scientist, while constantly supporting my goals and advocating for my best interests. He fostered a welcoming, community-oriented lab environment that I will deeply miss. My other committee members — Drs. Jacqueline Padilla-Gamiño, Jonathan P. Davis, Julian D. Olden, and Lauren Buckley — challenged me to think about my work broadly and were always enthusiastic about my work in a way that refreshed my own interests. Although they were not officially part of my committee, Drs. Hollie M. Putnam at the University of Rhode Island and Kathleen E. Lotterhos at Northeastern University taught me so much in our collaborations, and showed me how new faculty members could pursue engaging research while also advocating for better academic environments. 
  A big thank you to all past and present members of the Roberts Lab for helping me every time my code didn’t run, teaching me proper pipetting technique, and bringing levity and joy to what can feel like a slog-fest. The SAFS graduate student community has been integral to my time in Seattle; it was a joy to learn, protest, and laugh with you all.
  And most importantly, thank you to my family and my parents, Sudha Rajagopalan and Dr. Shankar Venkataraman. I would be nothing without you. Thank you, and I love you.

dedication: |
  To my Appa, Dr. Shankar Venkataraman. Your Appa reminded you not to forget to complete your Ph.D, so that you could do the same for me. 
  
  I completed it.
# End of UW thesis fields
knit: "bookdown::render_book"
site: bookdown::bookdown_site
output: 
  huskydown::thesis_pdf: 
    latex_engine: xelatex
    extra_dependencies: ["flafter"]
#  huskydown::thesis_gitbook: default
#  huskydown::thesis_word: default
#  huskydown::thesis_epub: default
bibliography: bib/thesis.bib
link-citations: yes
# Download your specific bibliography database file and refer to it in the line above.
csl: csl/apa.csl
# Download your specific csl file and refer to it in the line above.
lot: true
lof: true
# space_between_paragraphs: true
# Delete the # at the beginning of the previous line if you'd like
# to have a blank new line between each paragraph
#header-includes:
#- \usepackage{tikz}
---

<!--
Above is the YAML (YAML Ain't Markup Language) header that includes a lot of metadata used to produce the document.  Be careful with spacing in this header!

If you'd like to include a comment that won't be produced in your resulting file enclose it in a block like this.
-->

<!--
If you receive a duplicate label error after knitting, make sure to delete the index.Rmd file and then knit again.
-->

```{r include_packages, include = FALSE}
# This chunk ensures that the huskydown package is
# installed and loaded. This huskydown package includes
# the template files for the thesis.
if(!require(devtools))
  install.packages("devtools", repos = "http://cran.rstudio.com")
if(!require(huskydown))
  devtools::install_github("benmarwick/huskydown")
library(huskydown)
```

<!-- The {.unnumbered} option here means that the introduction will be "Chapter 0." You can also use {-} for no numbers on chapters. -->

# Introduction {.unnumbered}

- The enviornment is changing
-In order to understand how the environment is changing, we need data from the past
-Historical data for species abundance is usually avaialble and how the environment is changing species abundance is well documented across ecosystems globally
-Abundance is not the only important ecosystem component, we know species interactions are just as important for shaping and influencing ecosystems
-But historical indices of ecosystem interactions are rare and can be challenging to reproduce for modern datasets.
-Chemical tracers can be a useful tool for measuring ecological interactions especially on long timescales.
Here we use CSSIA of amino acids and inorganic nitrogen to assess changes in species interactions in teperate marine and riparian ecosystems.
<!--  If you're still on the fence about using _R Markdown_, check out the resource for newbies available at <https://ismayc.github.io/rbasics-book/> .  -->

**Why use it?**

_R Markdown_ creates a simple and straightforward way to interface with the beauty of LaTeX.  Packages have been written in **R** to work directly with LaTeX to produce nicely formatting tables and paragraphs. In addition to creating a user friendly interface to LaTeX, _R Markdown_ also allows you to read in your data, to analyze it and to visualize it using **R** functions, and also to provide the documentation and commentary on the results of your project.  Further, it allows for **R** results to be passed inline to the commentary of your results.  You'll see more on this later.  

<!--  Having your code and commentary all together in one place has a plethora of benefits! -->

**Who should use it?**

Anyone who needs to use data analysis, math, tables, a lot of figures, complex cross-references, or who just cares about the final appearance of their document should use _R Markdown_. Of particular use should be anyone in the sciences, but the user-friendly nature of _Markdown_ and its ability to keep track of and easily include figures, automatically generate a table of contents, index, references, table of figures, etc. should make it of great benefit to nearly anyone writing a thesis project.

