% From https://github.com/UWIT-IAM/UWThesis

\documentclass [11pt, proquest] {uwthesis}[2015/03/03]

% Syntax highlighting #22
  \usepackage{color}
  \usepackage{fancyvrb}
  \newcommand{\VerbBar}{|}
  \newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
  \DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
  % Add ',fontsize=\small' for more characters per line
  \usepackage{framed}
  \definecolor{shadecolor}{RGB}{248,248,248}
  \newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
  \newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
  \newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
  \newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\BuiltInTok}[1]{#1}
  \newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
  \newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
  \newcommand{\ExtensionTok}[1]{#1}
  \newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ImportTok}[1]{#1}
  \newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\NormalTok}[1]{#1}
  \newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
  \newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
  \newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\RegionMarkerTok}[1]{#1}
  \newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}

%% https://github.com/rstudio/rmarkdown/issues/1649
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newenvironment{CSLReferences}%
{\setlength{\parindent}{0pt}%
\everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces}%
{\par}

% fix for pandoc 1.14
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\newtheorem{theorem}{Jibberish}

%% \bibliography{references}

\hyphenation{mar-gin-al-ia}

%
% ----- apply watermark to every page
% ----- change 'stamp' to 'nostamp'
%------ to omit watermark
%
\usepackage[nostamp]{draftwatermark}
% % Use the following to make modification
\SetWatermarkText{DRAFT}
\SetWatermarkLightness{0.95}

%% for the per mil symbol
\usepackage[nointegrals]{wasysym}

%% something about tables, from https://github.com/ismayc/thesisdown/issues/122
\usepackage{calc}

%% for copyright symbol
\usepackage{textcomp}

%% to allow to rotate pages to landscape
\usepackage{lscape}
%% to adjust table column width
\usepackage{tabularx}

% suppress bottom page numbers on first page of each chapter
% because they overlap with text
\usepackage{etoolbox}
\patchcmd{\chapter}{plain}{empty}{}{}

%% for more attractive tables
\usepackage{booktabs}
\usepackage{longtable}


\usepackage{graphicx}


% Double spacing, if you want it.
% \def\dsp{\def\baselinestretch{2.0}\large\normalsize}
% \dsp

% If the Grad. Division insists that the first paragraph of a section
% be indented (like the others), then include this line:
% \usepackage{indentfirst}

%%%%%%%%%%%%%%%%%%
% If you want to use "sections" to partition your thesis
% un-comment the following:
%
% \counterwithout{section}{chapter}
% \setsecnumdepth{subsubsection}
% \def\sectionmark#1{\markboth{#1}{#1}}
% \def\subsectionmark#1{\markboth{#1}{#1}}
% \renewcommand{\thesection}{\arabic{section}}
% \renewcommand{\thesubsection}{\thesection.\arabic{subsection}}
% \makeatletter
% \let\l@subsection\l@section
% \let\l@section\l@chapter
% \makeatother
%
% \renewcommand{\thetable}{\arabic{table}}
% \renewcommand{\thefigure}{\arabic{figure}}
%
%%%%%%%%%%%%%%%%%%


%% Stuff from https://github.com/suchow/Dissertate

% The following line would print the thesis in a postscript font

% \usepackage{natbib}
% \def\bibpreamble{\protect\addcontentsline{toc}{chapter}{Bibliography}}

\setcounter{tocdepth}{1} % Print the chapter and sections to the toc
% controls depth of table of contents (toc): 0 = chapter, 1 = section, 2 = subsection

\usepackage{biblatex}

\prelimpages

%% from thesisdown
% To pass between YAML and LaTeX the dollar signs are added by CII
\Title{Molecular techniques for resilient Pacific oyster (\emph{Crassostrea gigas}) aquaculture}
\Author{Yaamini R. Venkataraman}
\Year{2021}
\Program{School of Aquatic and Fishery Sciences}
\Chair{Steven B. Roberts}{Associate Professor}{School of Aquatic and Fishery Sciences}
\Signature{Jacqueline Padilla-Gamiño}
\Signature{Jonathan P. Davis}
\Signature{}

% commands and environments needed by pandoc snippets
% extracted from the output of `pandoc -s`
%% Make R markdown code chunks work
\usepackage{array}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifxetex
  \usepackage{fontspec,xltxtra,xunicode}
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
\else
  \ifluatex
    \usepackage{fontspec}
    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \else
    \usepackage[utf8]{inputenc}
  \fi
\fi
\usepackage{color}
\usepackage{fancyvrb}


\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\else
  \usepackage[unicode=true,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\fi
\hypersetup{breaklinks=true, pdfborder={0 0 0}}
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{2} %% controls section numbering, e.g. 1 or 1.2, or 1.2.3

\begin{document}
\copyrightpage

\titlepage

\setcounter{page}{-1}
\abstract{As ocean acidification continues to impact marine ecosystems at unprecedented rates, phenotypic plasticity may allow organisms to withstand more stressful conditions. Genomic methods can elucidate molecular mechanisms that contribute to phenotypic plasticity, allowing for a deeper understanding of how physiological processes will be impacted by low pH. My dissertation examines the effects of ocean acidification on the Pacific oyster (\emph{Crassostrea gigas}) stress response and reproduction; elucidate how exposure history impacts phenotype; and explore the role of functional role DNA methylation in somatic and reproductive tissue. I investigated the effect of regional environmental variation on the molecular physiology of \emph{C. gigas} outplanted at five different estuarine sites (four in Puget Sound, one in Willapa Bay) in Washington, USA using gel-free proteomic methods. While there was no difference in survival, or any protein abundances due to pH differences between sites, \emph{C. gigas} outplanted at the site with the highest temperature had significantly higher abundances of antioxidant enzymes and molecular chaperones, elucidating the molecular underpinnings of thermotolerance. In a hatchery setting, I explored the impact of ocean acidification on reproductive maturity and output. A seven week low pH exposure did not affect sex ratio or maturation stage; however, it did significantly affect survival of larvae. Even though adult oysters spent four months in ambient pH conditions between low pH exposure and strip spawning, larvae from females that experienced low pH conditions had significantly higher mortality. Finally, I conducted the first investigations examining the effect of ocean acidification in \emph{C. gigas} methylomes. To investigate the role of environmentally-responsive methylation in reproductive tissue, I analyzed gonad methylomes of female \emph{C. gigas} exposed to low pH. A total of 1,599 differentially methylated loci (DML) were found in gene bodies. The genic DML were associated with cilium movement, development, and cytoskeletal processes, implying a need to regulate cellular growth in the gonad in response to low pH. I then explored the influence of low pH on the somatic tissue methylome using diploid and triploid oyster ctenidia. Differences in ploidy status yielded 154 DML. These ploidy-DML were associated with cell-cell adhesion and dephosphorlylation processes, which are not commonly associated with methylome changes in organisms that undergo natural polyploidization. The 178 pH-DML were associated with processes commonly observed in oysters exposed to ocean acidification, including apoptosis, protein ubiquitination, zinc ion binding, and cytoskeletal processes. In both reproductive and somatic tissue, the enrichment of DML in genes with multiple transcripts could indicate a role for methylation to regulate gene expression via alternative splicing. Investigating the molecular underpinnings of responses to ocean acidification in \emph{C. gigas} will provide a thorough understanding of this global aquaculture product's ability to withstand future ocean conditions.}

\tableofcontents
\listoffigures
\listoftables

\acknowledgments{I am eternally grateful for the community that has supported me over the past five years. I cannot thank my adviser, Dr.~Steven B. Roberts, enough for his mentorship. He gave me room to grow and think for myself as a scientist, while constantly supporting my goals and advocating for my best interests. He fostered a welcoming, community-oriented lab environment that I will deeply miss. My other committee members --- Drs. Jacqueline Padilla-Gamiño, Jonathan P. Davis, Julian D. Olden, and Lauren Buckley --- challenged me to think about my work broadly and were always enthusiastic about my work in a way that refreshed my own interests. Although they were not officially part of my committee, Drs. Hollie M. Putnam at the University of Rhode Island and Kathleen E. Lotterhos at Northeastern University taught me so much in our collaborations, and showed me how new faculty members could pursue engaging research while also advocating for better academic environments.

A big thank you to all past and present members of the Roberts Lab for helping me every time my code didn't run, teaching me proper pipetting technique, and bringing levity and joy to what can feel like a slog-fest. The SAFS graduate student community has been integral to my time in Seattle; it was a joy to learn, protest, and laugh with you all.

And most importantly, thank you to my family and my parents, Sudha Rajagopalan and Dr.~Shankar Venkataraman. I would be nothing without you. Thank you, and I love you.}

\dedication{\begin{center}To my Appa, Dr.~Shankar Venkataraman. Your Appa reminded you not to forget to complete your Ph.D, so that you could do the same for me.

I completed it.\end{center}}

\textpages


\hypertarget{introduction}{%
\chapter*{Introduction}\label{introduction}}
\addcontentsline{toc}{chapter}{Introduction}

Pacific oysters (\emph{Crassostrea gigas}; Thunberg 1793) are a commercially and ecologically relevant species, making them ideal models for studying physiological responses to ocean acidification. Research efforts have already identified consequences of ocean acidification for distinct life stages. Pacific oyster larvae experience developmental delays and reduced shell growth in response to experimental ocean acidification (\protect\hyperlink{ref-Gazeau2011}{Gazeau et al., 2011}; \protect\hyperlink{ref-Kurihara2007}{Kurihara, Kato, \& Ishimatsu, 2007}; \protect\hyperlink{ref-Timmins-Schiffman2013}{Timmins-Schiffman, O'Donnell, Friedman, \& Roberts, 2013}; \protect\hyperlink{ref-Waldbusser2014}{Waldbusser et al., 2014}). Key protein pathways are disrupted during metamorphosis (\protect\hyperlink{ref-Dineshram2016}{Dineshram et al., 2016}) and adulthood (\protect\hyperlink{ref-Timmins-Schiffman2014}{Timmins-Schiffman et al., 2014}). As seawater pCO\textsubscript{2} increases, adult \emph{C. gigas} calcification rates decrease (\protect\hyperlink{ref-Gazeau2007}{Gazeau et al., 2007}), and their shells display significantly lower fracture toughness (\protect\hyperlink{ref-Timmins-Schiffman2014}{Timmins-Schiffman et al., 2014}). Exposure to experimental ocean acidification also negatively impacts growth, sperm motility, and egg viability in adult oysters (\protect\hyperlink{ref-Omoregie2019}{Omoregie, Mwatilifange, \& Liswaniso, 2019}). Although there is extensive research on ocean acidification's impact on \emph{C. gigas}, key uncertainties remain surrounding the mechanisms behind observed physiological responses. Knowing how the environment shapes physiological phenotypes on a mechanistic level could help identify potential pathways for future-proofing \emph{C. gigas} aquaculture operations.

Genomics are the next frontier for understanding how environmental variability affects an organism's physiological response and ability to acclimate to future ocean conditions. Epigenetic analysis can provide a direct link between changes in the environment and gene expression regulation. The epigenome consists of gene expression changes that do not arise from changes in the DNA sequence itself, with methylation of cytosine bases being the most studied mechanism (\protect\hyperlink{ref-Bird2002}{Bird, 2002}; \protect\hyperlink{ref-Deans2015}{Deans \& Maggert, 2015}). Initial characterization of the \emph{C. gigas} methylome has found that methylation occurs in a mosaic pattern and is concentrated in gene regions (\protect\hyperlink{ref-Gavery2013}{Gavery \& Roberts, 2013}). Environmental response genes that are less methylated are likely prone to more spurious transcription and alternative splicing patterns, thereby increasing phenotypic plasticity (\protect\hyperlink{ref-Roberts2012}{Roberts \& Gavery, 2012}). Recent studies have demonstrated that changes in the environment can affect DNA methylation patterns in other marine invertebrates (\protect\hyperlink{ref-Eirin-Lopez2018}{Eirin-Lopez \& Putnam, 2018}), so it is possible genomic regulation by DNA methylation may also be important for adaptation and acclimation in \emph{C. gigas}. Additionally, manipulation of methylation patterns could be a method for ``stress hardening'' cultured species in hatchery settings (\protect\hyperlink{ref-Gavery2017}{Gavery \& Roberts, 2017}). Similarly, examination of the proteome --- all the proteins in a sample --- can shed light on physiological changes on a molecular level, since proteins direct all cellular functions (\protect\hyperlink{ref-Tomanek2014}{Tomanek, 2014}). Since the proteome is dynamic, proteomic analysis can capture organismal response to real-time environmental conditions, similar to those experienced by \emph{C. gigas} outplanted at aquaculture grow-out sites.

My dissertation uses \emph{C. gigas} as a model system to examine molecular mechanisms important for ocean acidification response. I first explored the impacts of natural pH variation on the \emph{C. gigas} proteome. Next, I investigated the effect of ocean acidification on oyster physiology in a controlled hatchery setting. Finally, I elucidated the mechanistic role of DNA methylation in responding to ocean acidification in somatic and reproductive tissues. My work provides a foundation for using molecular tools to promote resilient aquaculture in the face of climate stressors like ocean acidification.

\hypertarget{characterization-of-pacific-oyster-crassostrea-gigas-proteomic-response-to-natural-environmental-differences}{%
\chapter{\texorpdfstring{Characterization of Pacific oyster (\emph{Crassostrea gigas}) proteomic response to natural environmental differences}{Characterization of Pacific oyster (Crassostrea gigas) proteomic response to natural environmental differences}}\label{characterization-of-pacific-oyster-crassostrea-gigas-proteomic-response-to-natural-environmental-differences}}

\hypertarget{abstract}{%
\section{Abstract}\label{abstract}}

Global climate change is rapidly altering coastal marine ecosystems important for food production. A comprehensive understanding of how organisms will respond to these complex environmental changes can come only from observing and studying species within their natural environment. To this end, the effects of environmental drivers --- pH, dissolved oxygen content, salinity, and temperature --- on Pacific oyster (\emph{Crassostrea gigas}) physiology were evaluated in an outplant experiment. Sibling juvenile oysters were outplanted to eelgrass and unvegetated habitat at five different estuarine sites within the Acidification Nearshore Monitoring Network in Washington State, USA to evaluate how regional environmental drivers influence molecular physiology. Within each site, we also determined if eelgrass presence that buffered pH conditions changed the oysters' expressed proteome. A novel, two-step, gel-free proteomic approach was used to identify differences in protein abundance in \emph{C. gigas} ctenidia tissue after a 29 day outplant by 1) identifying proteins in a data independent acquisition survey step and 2) comparing relative quantities of targeted environmental response proteins using selected reaction monitoring. While there was no difference in protein abundance detected between habitats or among sites within Puget Sound, \emph{C. gigas} outplanted at Willapa Bay had significantly higher abundances of antioxidant enzymes and molecular chaperones. Environmental factors at Willapa Bay, such as higher average temperature, may have driven this protein abundance pattern. These findings generate a suite of new hypotheses for lab and field experiments to compare the effects of regional conditions on physiological responses of marine invertebrates.

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

Global climate change will influence estuarine dynamics and impact the organisms that inhabit these environments. Estuaries are already variable across spatial and temporal scales in terms of phytoplankton production (\protect\hyperlink{ref-Pennock1986}{\textbf{Pennock1986?}}), nutrient availability (\protect\hyperlink{ref-Paerl2014}{\textbf{Paerl2014?}}), heavy metal contamination (\protect\hyperlink{ref-Liu2015}{\textbf{Liu2015?}}), salinity (\protect\hyperlink{ref-Banas2004}{\textbf{Banas2004?}}), and carbonate chemistry (\protect\hyperlink{ref-Feely2010}{\textbf{Feely2010?}}; \protect\hyperlink{ref-Pelletier2018}{\textbf{Pelletier2018?}}). Since climate change will affect these parameters, it is important to consider how estuarine organisms will respond.

Proteomics, or the study of protein abundance and expression, can be used to shed light on physiological changes on a molecular level. Proteins direct all major cellular functions, thus examining protein abundance provides direct evidence of an organism's physiological response to the estuarine environment (\protect\hyperlink{ref-Tomanek2014}{Tomanek, 2014}). The proteome is dynamic, as it must rapidly respond to perturbation, providing mechanistic information that standard gene expression and mRNA quantification studies cannot (\protect\hyperlink{ref-Veldhoen2012}{\textbf{Veldhoen2012?}}; \protect\hyperlink{ref-Flores-Nunes2015}{\textbf{Flores-Nunes2015?}}). As a result of the proteome's dynamic nature, proteins analyzed at the time of collection represent an organism's response to the environment in near real-time. Long-term exposure to environmental conditions, as well as natural organismal aging, are also reflected in the proteome (\protect\hyperlink{ref-Hercus2003}{\textbf{Hercus2003?}}). Discovery-based proteomic methods can elucidate responses to environmental drivers (\protect\hyperlink{ref-Flores-Nunes2015}{\textbf{Flores-Nunes2015?}}). Several studies have connected protein abundances with changes in laboratory-simulated environmental conditions, identifying key proteins and mechanisms involved in specific environmental responses (\protect\hyperlink{ref-Dineshram2016}{Dineshram et al., 2016}; \protect\hyperlink{ref-Timmins-Schiffman2014}{Timmins-Schiffman et al., 2014}; \protect\hyperlink{ref-Meng2017}{\textbf{Meng2017?}}). While these studies provide insight into organismal adaptation and physiology, laboratory studies alone cannot fully encapsulate the effects of multiple environmental drivers within an ecosystem context (\protect\hyperlink{ref-Riebesell2014}{\textbf{Riebesell2014?}}).

Although challenging, in situ field studies provide a necessary biological realism when considering variable environments (\protect\hyperlink{ref-Slattery2012}{\textbf{Slattery2012?}}; \protect\hyperlink{ref-Cornwall2016}{\textbf{Cornwall2016?}}). Such experiments can be leveraged to study the effects of multiple environmental drivers on organismal physiology and to incorporate realistic variability, as opposed to examining the effect of a single stressor on an organism (\protect\hyperlink{ref-Riebesell2014}{\textbf{Riebesell2014?}}). Through transcriptomics, (\protect\hyperlink{ref-Chapman2011}{\textbf{Chapman2011?}}) demonstrated the power of an in situ experimental design for examining the impacts of regional environmental conditions on Eastern oyster (\emph{Crassostrea virginica}) physiology. Transcript signatures from \emph{C. virginica} sampled from various locations in southeastern United States revealed that temperature, pH, salinity, dissolved oxygen and pollutant load at each location impacted gene expression. Furthermore, they were able to disentangle the interactions of these environmental factors on gene expression. RNA and protein abundances can be influenced by several environmental factors, and in situ studies can determine which drivers will be more important to consider for organismal physiology.

Marine invertebrates have proven to be informative bioindicators in proteomic studies to examine the effects of in situ conditions on organismal physiological responses to environmental change. When marine invertebrates have been exposed to varying environmental conditions, proteomics have demonstrated changes in cellular defense, immune responses, and genome function (\protect\hyperlink{ref-Veldhoen2012}{\textbf{Veldhoen2012?}}). Changes in protein abundance in bivalves like the Pacific oyster (\emph{Crassostrea gigas}) and blue mussels (\emph{Mytilus edulis spp}.) have been used to develop biomarkers for environmental contaminants (\protect\hyperlink{ref-Slattery2012}{\textbf{Slattery2012?}}; \protect\hyperlink{ref-Beyer2017}{\textbf{Beyer2017?}}). Proteomic responses to natural environmental drivers have also been evaluated in bivalves. For example, shotgun proteomic analysis of \emph{M. edulis} ctenidia from Baltic Sea microcosms revealed that low salinity conditions lead to decreased abundance of cytoskeleton proteins, as well as calcium-binding messenger calmodulin, which plays an important role in signalling and intracellular membrane trafficking pathways (\protect\hyperlink{ref-Campos2016}{\textbf{Campos2016?}}). Using a growing wealth of genomic information to understand how these species fare under differential environmental conditions is critical for monitoring natural populations and commercial aquaculture.

Pacific oyster (\emph{Crassostrea gigas}) rearing in estuarine environments in Washington State, USA (WA) provides an ideal system to examine the effect of in situ environmental conditions on the expressed proteome. \emph{C. gigas} are extensively farmed in two different estuarine systems that show substantial regional variation: Puget Sound and Willapa Bay. Puget Sound is a complex estuarine system with interconnected sub-basins, each with different freshwater inputs, residence times, and stratification levels (\protect\hyperlink{ref-Feely2010}{\textbf{Feely2010?}}; \protect\hyperlink{ref-Bianucci2018}{\textbf{Bianucci2018?}}; \protect\hyperlink{ref-Pelletier2018}{\textbf{Pelletier2018?}}). Willapa Bay is a large shallow estuary on the Pacific coast that exchanges approximately half its water volume with the Pacific Ocean at each tide (\protect\hyperlink{ref-Banas2004}{\textbf{Banas2004?}}; \protect\hyperlink{ref-Banas2007}{\textbf{Banas2007?}}). Seasonality and location within Puget Sound dictates temperature, dissolved oxygen, salinity, and pH conditions, while Willapa Bay conditions are influenced by diurnal fluctuations and proximity to either the ocean or rivers draining into the bay (\protect\hyperlink{ref-Banas2007}{\textbf{Banas2007?}}; \protect\hyperlink{ref-Feely2010}{\textbf{Feely2010?}}; \protect\hyperlink{ref-Ruesink2015}{\textbf{Ruesink2015?}}).

Both Puget Sound and Willapa Bay also host eelgrass beds (\emph{Zostera spp}.) that affect environmental conditions, such as oxygen concentrations, on diurnal time scales. The 2012 Washington State Blue Ribbon Panel on Ocean Acidification outlines key early actions, which include the examination of ``vegetation-based systems of remediation'' (Action 6.1.1) to improve local pH through photosynthetic drawdown of carbon dioxide. This experiment set out to test whether protein abundance patterns reflect reduced stress within vegetation. For example, eelgrass beds may reduce emersion stress relative to unvegetated areas through shading, the retention of water, and increased evaporative cooling at low tide. They can also ameliorate effects of ocean acidification through photosynthetic activity. Reduced pathogen prevalence has also been documented in seagrass beds, but not specifically in eelgrass (\protect\hyperlink{ref-Lamb2017}{\textbf{Lamb2017?}}). In contrast, eelgrass beds may drive more extreme carbonate chemistry conditions (\protect\hyperlink{ref-Pacella2018}{\textbf{Pacella2018?}}). (\protect\hyperlink{ref-Lowe2018}{\textbf{Lowe2018?}}) also found that \emph{C. gigas} shell strength and survival was significantly lower in eelgrass habitats in WA. Understanding how different aquaculture grow-out locations and habitats will affect the oyster's ability to persist through environmental change is crucial for the industry and the ecosystem.

The purpose of this study was to use proteomic techniques to uncover the impacts of environmental drivers on Pacific oyster physiological outcomes in estuarine environments in WA. Naturally existing environmental variation was harnessed by outplanting \emph{C. gigas} in different locations within Puget Sound and Willapa Bay, and habitat effects were taken into consideration by placing oysters in eelgrass and unvegetated habitats. Gel-free proteomic methods were used to examine the effects of outplant conditions on relative quantities of all expressed proteins in a series of in situ experiments in order to identify differentially abundant proteins. We predicted that differences in environmental drivers at each outplant location and within outplant habitats would yield unique protein abundance patterns. Oysters at outplant locations with warmer water temperatures, more variable water temperatures, lower dissolved oxygen content, lower salinity, or lower pH may have higher abundances of proteins related to environmental response. Eelgrass beds were expected to ameliorate stressful conditions, resulting in lower abundances of environmental stress response proteins than oysters in unvegetated habitats.

\hypertarget{methods}{%
\section{Methods}\label{methods}}

\hypertarget{shellfish-deployment}{%
\subsection{Shellfish Deployment}\label{shellfish-deployment}}

Sibling juvenile \emph{C. gigas} (average shell length = 27.2 mm, age = 2 months) were outplanted for 29 days starting June 19, 2016 at five locations: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB) in Washington State, USA (Table \ref{tab:deployment}). These sites were selected for differences in environmental parameters, as well as for the presence of unvegetated areas and eelgrass beds within each site. All sites were part of the Acidification Nearshore Monitoring Network, a network of sensors placed in various WA locations to monitor marine chemistry (ANeMoNe; Washington Department of Natural Resources). Prior to the outplant, oysters were reared in a controlled hatchery setting. At each site and habitat combination, custom-built Durafet-based sensors (Honeywell) were used to monitor pH. Commercially-available MiniDOT loggers (Precision Measurement Engineering) were used to measure dissolved oxygen, and Odyssey loggers were used to measure conductivity. All sensors recorded temperature measurements, and all sensors logged at 10-minute intervals across the outplant period, with the exception of SK, where sensors were installed two days into the outplant period. At each site, juvenile oysters were placed in bags of five oysters each directly onto the substrate at a tidal height of -1.5 MLLW, both inside and outside of eelgrass beds (n=15 per habitat type), for a total of thirty outplanted oysters per site. The animals were uniformly placed less than a lateral distance of 0.5 m from the sensors at the same tidal height as the instruments. Oysters were housed in exclusion cages to prevent predation. Juvenile oysters remained at each site for a 29-day exposure period. Because the ctenidia is the primary site where oysters interact with the environment, ctenidia samples were dissected at the end of the outplant and held on dry ice until storage at -80ºC {[}(\protect\hyperlink{ref-Beyer2017}{\textbf{Beyer2017?}}); (\protect\hyperlink{ref-Meng2017}{\textbf{Meng2017?}})).

Environmental data was treated as follows. Conductivity observations were removed when less than zero, which occurs when the instrument is dry at low tide. Remaining observations were converted to salinity measurements using the \texttt{swSCTp} function in the \texttt{oce} package in R (\protect\hyperlink{ref-Kelley2018}{\textbf{Kelley2018?}}; \protect\hyperlink{ref-R2018}{\textbf{R2018?}}), with temperature at 25ºC and pressure at 10 dbar. For dissolved oxygen, pH, and salinity datasets, data were removed when collected by probes 1) during low tide or 2) when tidal depth was less than one foot to remove readings where the probes may have been exposed. Values collected during low tide or a depths less than one foot were retained for temperature datasets. Outliers were screened using the Tukey method for temperature, dissolved oxygen, pH, and salinity datasets (\protect\hyperlink{ref-Hoaglin1986}{\textbf{Hoaglin1986?}}). Uniform outplant tidal heights were checked using \texttt{prop.test} in R (\protect\hyperlink{ref-R2018}{\textbf{R2018?}}).

A non-metric multidimensional scaling analysis (NMDS) was used to evaluate differences in environmental parameters. First, mean and variance were calculated for each day of the outplant. Values were log-transformed, and a separate Gower's distance matrix was calculated for daily mean and daily variances, accounting for missing data. The NMDS was conducted with the Gower's distance matrix to visually compare means or variances between sites and habitats. Significant differences between site and habitat were identified using a One-way Analysis of Similarities (ANOSIM) for each environmental parameter. Pairwise ANOSIM tests for significant one-way ANOSIM results and two-way ANOSIM tests by site and habitat were not conducted due to lack of replicates within each site-habitat combination. R Scripts are available in the supplementary Github repository (\protect\hyperlink{ref-Venkataraman2018}{\textbf{Venkataraman2018?}}).

\hypertarget{protein-discovery}{%
\subsection{Protein Discovery}\label{protein-discovery}}

To identify select protein targets for characterization across locations and environmental conditions, a subset of tissue samples were analyzed with data independent acquisition (DIA) mass spectrometry analysis. Two tissue samples were used from each site to make a peptide library and maximize the amount of protein abundance data collected from each sample.

\hypertarget{protein-quantification}{%
\subsubsection{Protein Quantification}\label{protein-quantification}}

Tissue samples were homogenized in a solution of 50 mM NH4HCO3 with 6M Urea (500µl). Tissues were then sonicated 3 times (Fisher Scientific Sonic Dismembrator Model 100) for 10 seconds each and cooled between sonications in a bath of ethanol and dry ice. Protein quantities were measured with the Pierce BCA Protein Assay Kit microplate assay with a limited quantity of sonicated sample (11 µL). The protein concentration was measured via spectroscopy at 540 nm in a Labsystems (Waltham, MA) Multiskan MCC/340 and accompanying Ascent Software Version 2.6. Protein concentrations were calculated based on a standard curve with BSA (Pierce) per manufacturer's instructions.

\hypertarget{protein-digestion}{%
\subsubsection{Protein Digestion}\label{protein-digestion}}

Protein digestion followed the protocol outlined in \protect\hyperlink{ref-Timmins-Schiffman2013}{Timmins-Schiffman, O'Donnell, Friedman, \& Roberts} (\protect\hyperlink{ref-Timmins-Schiffman2013}{2013}). To each sample of 30 µg protein,1.5 M Tris pH 8.8 buffer (6.6 µL) and 200 mM TCEP (2.5 µL) were added. After solvent additions, each sample's pH was verified to be basic (pH ≥ 8), and placed on a 37ºC heating block for one hour. Iodoacetamide (200 mM, 20 µL) was then added to each sample to maximize digestion enzyme access to protein cleavage sites. Samples were covered with aluminum foil to incubate in the dark for 1 hour at room temperature. Afterwards, diothiolthreitol (200 mM, 20 µL) was added and samples were incubated at room temperature for one hour. Lysyl Endopeptidase (Wako Chemicals) was then added to each sample in a 1 µg enzyme:30 µg oyster protein ratio, followed by one hour of incubation at room temperature. Urea was diluted with NH\textsubscript{4}HCO\textsubscript{3} (25 mM, 800 µL) and HPLC grade methanol (200 µL). Trypsin (Promega) was added to each sample in a 1 µg trypsin: 30 µg oyster protein ratio for overnight digestion at room temperature.

\hypertarget{peptide-isolation}{%
\subsubsection{Peptide Isolation}\label{peptide-isolation}}

After overnight incubation, samples were evaporated to near dryness at 4ºC with a speedvac (CentriVap ® Refrigerated Centrifugal Concentrator Model 7310021). Samples were then reconstituted in 100 µL of a 5\% Acetonitrile and 0.1\% Trifluoroacetic Acid (Solvent A) to isolate peptides. If samples were not at pH ≤ 2, 10-20 µL aliquots of 10\% Formic Acid were added until this pH was achieved.

Before desalting peptide samples, Macrospin C18 columns (The Nest Group) were prepared by adding 200 µL of a 60\% Acetonitrile with 0.1\% Trifluoroacetic Acid (Solvent B). The columns were spun for three minutes at 2000 rpm, and flow-through liquid from the column was discarded. The spinning and discarding process was completed a total of four times.To wash columns, 200 µL of Solvent A was added to each column. The columns were once again spun for three minutes at 2000 rpm and liquid was discarded afterwards; the solvent addition, spinning, and discarding process was completed a total of three times.

To bind peptides to the columns, digested peptides were added to prepared columns, then the columns were spun at 3000 rpm for three minutes. The filtrate was pipetted back onto the column and spun again at 3000 rpm for three minutes. Solvent A (200 µL) was added to each column three separate times, then the column was spun for three minutes at 3000 rpm to wash salts off the column.

Peptides were eluted with two additions of 100 µL Solvent B to each column. Columns were spun at 3000 rpm for three minutes and the peptide fraction (filtrate) was reserved. Samples were placed in a speed vacuum at 4ºC until they were nearly dry (approximately two hours) to dry peptides. Peptides were reconstituted with 60 µL of 3\% Acetonitrile + 0.1\% Formic Acid, and stored at -80ºC.

\hypertarget{internal-standard-addition}{%
\subsubsection{Internal Standard Addition}\label{internal-standard-addition}}

Peptide Retention Time Calibration (PRTC; Pierce) is used as an internal standard to ensure consistency of peptides detected and measured throughout a mass spectrometry run. The stock solution of PRTC was diluted to 0.2 pmol/µL using 3\% Acetonitrile with 0.1\% Formic Acid. In a clean centrifuge tube, 6 µg of oyster protein and 0.376 pmol of PRTC were mixed together as per the PRTC user guide. Sample volume was brought up to 15 µL using a 3\% acetonitrile and 0.1\% formic acid solution. A quality control solution was also prepared (1 µL PRTC + BSA:3 µL 3\% Acetonitrile and 0.1\% Formic Acid solution).

\hypertarget{data-independent-acquisition-mass-spectrometry}{%
\subsection{Data Independent Acquisition Mass Spectrometry}\label{data-independent-acquisition-mass-spectrometry}}

Peptides were analyzed on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) using Data Independent Acquisition Mass Spectrometry (DIA). DIA analyses were completed as a comprehensive, non-random analytical method for detecting peptide ions present within a sample to create a peptide library. The peptide library was then leveraged to develop a targeted proteomics assay for quantification (see \emph{Selected Reaction Monitoring Assay}). A 30 cm analytical column and 3 cm pre-column were packed in-house with 3 µm C18 beads (Dr.~Maisch). Samples were run in a randomized order. A blank injection followed each sample, with procedural blanks run at the very end. Every injection was 3 µL, which included 1 µg of oyster protein and 0.0752 pmol of PRTC. Peptides were analyzed in MS1 over the m/z range of 450-950 with 12 m/z wide windows with 5 m/z overlaps (\protect\hyperlink{ref-Egertson2013}{\textbf{Egertson2013?}}). MS1 resolution was 60000 and AGC target was 400000 with a three second cycle time. The MS2 loop count was set to 20 and MS2 data was collected with a resolution of 15000 on charge state of 2 with an AGC target of 50000. No dynamic exclusion was used.

\hypertarget{peptide-centric-proteomic-analyses}{%
\subsection{Peptide-Centric Proteomic Analyses}\label{peptide-centric-proteomic-analyses}}

Unknown peptide spectra from mass spectrometry samples were matched with known peptides using Peptide-Centric Analysis in the PECAN software (\protect\hyperlink{ref-Ting2015}{\textbf{Ting2015?}}). Raw mass spectrometry files were converted to mzML files, then demultiplexed using MSConvert (\protect\hyperlink{ref-Chambers2012}{\textbf{Chambers2012?}}). The C. gigas proteome was digested with in silico tryptic digest using Protein Digestion Simulator (\protect\hyperlink{ref-Riviere2015}{\textbf{Riviere2015?}}). All known peptides from the mzML files were identified in comparison to the digested \emph{C. gigas} proteome (\protect\hyperlink{ref-Riviere2015}{\textbf{Riviere2015?}}).

The PECAN-generated spectral library (.blib) file was used to detect peptides of interest in raw DIA files in Skyline (\protect\hyperlink{ref-MacLean2010}{\textbf{MacLean2010?}}). Skyline identified peptides using chromatogram peak picking, where ions that elute at the same time and mass are detected as a peptide (file available at Panorama Public). All PRTC peptides and approximately 100 different oyster proteins and their peptide transitions were manually checked for retention time and peak area ratio consistency to determine a Skyline auto peak picker error rate (24.3\% ± 25\%, range: 0\% to 100\%).

Proteins had to satisfy four criteria to be considered appropriate targets for the study. 1) After an extensive literature search, functions related to oxidative stress, hypoxia, heat shock, immune resistance, shell formation, growth, and cellular maintenance were determined useful for evaluating environmental response. Proteins with annotations matching these functions were considered potential targets. 2) Protein data was then evaluated in Skyline to ensure there was no missing data for any peptide or sample. 3) Peaks with significant interference from other peptides were not considered. 4) Proteins needed at least two peptides with three transitions per peptide to quality as a potential target for downstream assays. The fifteen proteins (41 peptides and 123 transitions) that matched all of these criteria were selected as targets (Table 1.2).

\hypertarget{selected-reaction-monitoring-assay}{%
\subsection{Selected Reaction Monitoring Assay}\label{selected-reaction-monitoring-assay}}

Following the protein discovery phase (DIA), proteins were isolated as described above from an additional five randomly selected samples per site and habitat combination (for a total of 5 oysters per group) and analyzed with Selected Reaction Monitoring (SRM). Samples were prepared as described for DIA, except tissue samples were homogenized in 100 µL, and peptide samples were evaporated at 25ºC after peptide isolation.
Proteins of interest identified from the DIA analysis were used as targets in a SRM assay following the workflow and informatic pipeline of (\protect\hyperlink{ref-Timmins-Schiffman2016}{\textbf{Timmins-Schiffman2016?}}). Target peptide transitions were monitored using SRM on a Thermo TSQ Vantage. SRM data were collected during a gradient of 2-60\% acetonitrile over 40 minutes. All samples were run in technical duplicates in a randomized order with a 1 µg oyster peptide and 0.0752 pmol PRTC injection. A quality control injection and blank injection were run after every five sample injections, and PRTC peptides were monitored throughout the experiment.

\hypertarget{target-peptide-specificity}{%
\subsubsection{Target Peptide Specificity}\label{target-peptide-specificity}}

To ensure SRM assay specificity to oyster peptides of interest, oyster peptides were diluted in a background matrix of similar complexity (Pacific geoduck --- \emph{Panopea generosa} --- peptides), then analyzed using the oyster SRM assay. An oyster-specific SRM target would decrease in abundance with a decreasing abundance of oyster peptides in a mixture. Non-specific peptides --- more likely to be found in background matrix of similar complexity --- or peptides susceptible to interference would not correlate with oyster peptide abundance, and therefore, would be uninformative. Five \emph{C. gigas} samples used for SRM were randomly selected and pooled in equal quantities. A ten-sample oyster:geoduck dilution series was prepared and run using the same methods as other SRM samples.

\hypertarget{target-analysis}{%
\subsubsection{Target Analysis}\label{target-analysis}}

Raw SRM files, a background \emph{C. gigas} proteome, and the PECAN spectral library file from DIA were used to create a Skyline document (file available at Panorama Public). Correct transition peaks were selected based on predicted retention times from DIA results by comparing the relative retention times between identical PRTC peptides in the DIA and SRM datasets (R\textsuperscript{2} = 0.99431). Based on peptide specificity analyses, heat shock protein 70 B2 and one constituent peptide of
glucose-6-phosphate 1-dehydrogenase were removed from analyses.

Further filters were applied to the data to maintain only high quality peptides and transitions in the analysis. Coefficients of variation were calculated between technical replicates for each peptide transition. Peptides were removed from the dataset if CV \textgreater{} 20\%. To maintain high sample quality, any sample missing data for more than 50\% of peptide transitions was deemed poor quality for downstream analyses and excluded. Abundance data was normalized using total ion current (TIC) values from the mass spectrometer. Consistency between technical replicates was verified in remaining samples using a NMDS with TIC-normalized data and a euclidean dissimilarity matrix. Technical replicates were consistent if replicates lay closer together than to other samples in the NMDS. These replicates were then averaged for multivariate analytical methods.

Averaged technical replicate data was used to determine if peptides were differentially abundant between outplant sites and habitats. Before proceeding with analysis, peptide abundances were subjected to a Hellinger transformation to give low weights to any peptides with low counts. A NMDS was used to visually compare relative peptide abundance. One-way ANOSIM tests by site, region (Puget Sound vs.~Willapa Bay), and habitat, as well as a two-way ANOSIM test by site and habitat, were used to determine significant differences. Pairwise ANOSIM tests and post-hoc similarity percentage (SIMPER) analyses were conducted for each one- or two-way ANOSIM result significant at the 0.05 level. The first ten SIMPER entries were deemed influential peptides for each significant comparison.

The importance of environmental variables for explaining peptide abundance was evaluated with a redundancy analysis (RDA). For each site and habitat combination, mean and variance were calculated for pH, dissolved oxygen, salinity, and temperature over the course of the entire outplant. Environmental variables were then used as predictors to constrain peptide abundance. Predictors with missing values were not included. A triplot was used to visually assess differences in peptide abundance by site and habitat and the influence of individual peptides and environmental parameters. Analysis of Variance (ANOVA) was used to calculate significance of the RDA and environmental variables, with predictors deemed significant at p \textless{} 0.05. Since estuarine sites are highly variable, a second RDA was conducted constraining peptide abundance by environmental conditions on the day of collection to evaluate robustness of proteomic methods. R Scripts are available in the supplementary Github repository (\protect\hyperlink{ref-Venkataraman2018}{\textbf{Venkataraman2018?}}).

\#\#Results

Bulk soil stable isotope analysis indicated that salmon carcasses enriched the N isotope pools (Table 1). \(\delta^{15}N\) values peaked between 3 and 6 m from the stream edge, which was the distance salmon were typically relocated to during the experiment and declined at distances greater than 6 m. Maximum \(\delta^{15}N\) of bulk soils was 11.8‰ for the salmon enhanced bank and 11.6‰ for the salmon depleted bank and no observations exceeded the sockeye salmon end-member value of 12.6‰ (Figure \ref{fig:modIsotope}a). \(\delta^{13}C\) was more enriched at greater distances from the bank and on average was highest at 20 m (Figure \ref{fig:modIsotope}b). \(\delta^{13}C\) was primarily governed by distance, with some evidence {[}N\textsubscript{tot}{]} and bank also had an effect (\ref{tab:suppmod1}).

Salmon carcass manipulation also enriched \(\delta^{15}N\) of soil NH\textsubscript{4}\textsuperscript{+}. Stable isotope values were enriched at 3 m from the stream edge on the salmon enhanced bank, and declined at distances \textgreater{} 3 m. On the salmon depleted bank, \(\delta^{15}N\) of soil NH\textsubscript{4}\textsuperscript{+} was most enriched at 1 m and declined with distance (Figure @ref(fig:modsupp1.2)C). The only model with support contained a quadratic interaction of distance and bank, which provides strong evidence that \(\delta^{15}N\) of NH4+ was affected by salmon (Table \ref{tab:suppmod1}). In contrast to bulk soil N, \(\delta^{15}N\) values of NH\textsubscript{4}\textsuperscript{+} exceeded the salmon endmember of 12.6‰ for 23\% of all observations (n=21).

Inorganic nitrogen concentrations were primarily governed by bank and GW (Table \ref{tab:suppmod1}). The salmon enhanced bank had a higher mean {[}NH\textsubscript{4}\textsuperscript{+}{]} and {[}NO\textsubscript{3}\textsuperscript{-}{]} compared to the salmon depleted bank (Figure \ref{fig:modConc}d, e). The most supported models for both {[}NH\textsubscript{4}\textsuperscript{+}{]} and {[}NO\textsubscript{3}\textsuperscript{-}{]} showed evidence for H1, that observed differences were not caused by salmon. For {[}NH\textsubscript{4}\textsuperscript{+}{]} there was substantial model uncertainty, with six competing models receiving relative support (ΔAIC \textless{} 2) (Table \ref{tab:suppmod1}) but none of the competing models supported a salmon effect. Two competing models for {[}NO\textsubscript{3}\textsuperscript{-}{]} supported a site variability effect and one competing model supported a salmon effect (Tabe \ref{tab:suppmod1}) and all three contained gravimetric water content as a covariate. This indicates {[}NH\textsubscript{4}\textsuperscript{+}{]} was driven by site factors unrelated to salmon while {[}NO\textsubscript{3}\textsuperscript{-}{]} was driven by gravimetric water content and with some support for salmon enhancement.

Nitrogen transformation rates were unaffected by salmon carcass manipulation. Both net nitrification and net mineralization models with relative support contained N substrate ({[}NH\textsubscript{4}\textsuperscript{+}{]} and {[}N\textsubscript{org}{]} respectively), and the models with the most support did not include distance or bank. Net mineralization had some model uncertainty, with four models receiving relative support; however, all of the competing models supported either H1 or H3 with no support for a salmon effect. {[}N\textsubscript{org}{]} was the only covariate included in all of the competing models, indicating {[}N\textsubscript{org}{]} was the most important covariate tested for determining net mineralization. Net nitrification had greater model certainty and both models that received relative support contained {[}NH\textsubscript{4}\textsuperscript{+}{]} and gravimetric water content. Similar to net mineralization, these models supported H1 and H3 with no support for H2, the salmon effect, though net nitrification was slightly higher on average between 3 -- 6 m on the salmon enhanced bank (Table \ref{tab:suppmod1}). Overall, these results demonstrated the manipulation of salmon carcasses did not have clearly detectable effects on N transformation rates.

Both {[}N\textsubscript{org}{]} and GW indicated there are site differences caused by distance and bank unrelated to salmon carcass manipulation. On average {[}N\textsubscript{org}{]} was higher on the salmon depleted bank than the salmon enhanced bank. There was model support of H1 for both GW and {[}N\textsubscript{org}{]}, indicating these variables decrease with distance (Table \ref{tab:suppmod1}, Figure \ref{fig:modConc} h, i). While there was some evidence that there was both a distance and bank effect on GW, it was not caused by salmon as the salmon enhanced bank does not show a peak GW at 3 - 6 m from the stream, which was where there was the highest observed isotopic enrichment and expected MDN. However, one competing model for {[}N\textsubscript{org}{]} did support H2, indicating site factors and salmon may both affect {[}N\textsubscript{org}{]}. However, the mean {[}N\textsubscript{org}{]} for the salmon enhanced bank was 18.42 mg/g and 18.97 mg/g for the salmon depleted bank indicating salmon decrease {[}N\textsubscript{org}{]}, if they affect it at all.\newline 

C:N, percent nitrification, and percent carbon indicate relatively high nitrogen availability across sampling sites in the Hansen Creek system. Mean percent carbon was 24.2 and 24.9 on the enhanced and depleted banks respectively (S3). Soil C:N of bulk isotopes was less than 20 for all sites, with a mean of 15.8 (enhanced) and 14.2 (depleted). These values are well below the critical microbial C:N threshold of 29, demonstrating N is more available to meet microbial metabolic demands relative to C (Figure \ref{fig:modConc}j). In contrast, percent nitrification was relatively high with a mean of 64\% and 62\% on the enhanced and depleted banks (S3).

\#\#Discussion

This study confirmed that MDN was both present in soils and increased on the bank enhanced with salmon carcasses for 20 years. However, plant-available inorganic N pools and N transformation rates measured in soil during the peak growing season immediately prior to the annual return of salmon were largely unaffected by salmon enhancement. Even though the salmon enhanced bank had increased net nitrification compared to the salmon depleted bank, our analysis found no pattern with distance from the stream, suggesting that elevated nitrification was caused by bank characteristics unrelated to salmon carcass density. Given numerous conventional long-term fertilization experiments worldwide have shown a consistent pattern of elevated soil inorganic N pools and N transformations, (\protect\hyperlink{ref-Hogberg2006}{\textbf{Hogberg2006?}}; \protect\hyperlink{ref-Lu2011}{\textbf{Lu2011?}}), it was surprising that 20 years of MDN inputs did not clearly accelerate soil N cycling in our study. Soils are the dominant (\textgreater{} 70\%) sink for added N in forests worldwide (\protect\hyperlink{ref-Templer2012}{\textbf{Templer2012?}}) and tree growth in high latitude conifer forests is often strongly N-limited (\protect\hyperlink{ref-Nordin2001}{\textbf{Nordin2001?}}), both of which should have fostered retention of salmon N inputs to our site. Indeed, the 20 years of cumulative salmon N additions in the zone near the stream in our study (\textasciitilde{} 6,690 kg N/ha) greatly exceeded typical riparian surface soil N pools (500 to 2500 kg N/ha) (\protect\hyperlink{ref-Walker1989}{\textbf{Walker1989?}}; \protect\hyperlink{ref-Morris2011}{\textbf{Morris2011?}}; \protect\hyperlink{ref-Perry2017}{\textbf{Perry2017?}}), suggesting that even partial retention of salmon N inputs in soils should have increased soil {[}N\textsubscript{org}{]}. The lack of increase in soil {[}N\textsubscript{org}{]} due to salmon that we observed is consistent with the lack of increase in N availability, because soil {[}N\textsubscript{org}{]} fuels long-term changes in N availability and recycling via plant uptake, litterfall, and decomposition (\protect\hyperlink{ref-Chapell1999}{\textbf{Chapell1999?}}; \protect\hyperlink{ref-Perakis2011}{\textbf{Perakis2011?}}; \protect\hyperlink{ref-Perakis2012}{\textbf{Perakis2012?}}). Combined with observations of low C:N and high percent nitrification, this indicates N from salmon subsidies is not being retained in this system. Overall, the lack of increase in soil organic and inorganic N concentrations and N transformations that we observed following 20-year salmon manipulation raises questions of whether plant growth responses should be expected at our site.

Prior work at Hansen Creek inferred that MDN stimulated white spruce growth based on tree ring analyses (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}). However, substantial salmon enhancement corresponding to approximately 669 g/m\textsuperscript{2} (6,690 kg/ ha) of N and 113 g/m\textsuperscript{2} (1,130 kg/ha) of P over the past 20 years was unable to overcome pre-treatment differences in forest growth between banks. For reference, it is estimated white spruce in floodplain stands require approximately 1.35 g/m\textsuperscript{2}/y of N (\protect\hyperlink{ref-Chapin2006}{\textbf{Chapin2006?}}), which was far exceeded by the mean change of 33.45 g/m\textsuperscript{2}/y of N added from this manipulation. Additionally, fertilization experiments apply N on the order of 100 - 1,000 kg/ha with clear results (\protect\hyperlink{ref-Chapell1999}{\textbf{Chapell1999?}}), a much lower application rate than in this study. Factors such as climate, stand demography, and site and landscape variability also affect tree growth in this system. Indeed, white spruce growth response to recent warming across southwest Alaska depends strongly on tree density (\protect\hyperlink{ref-Wright2018}{\textbf{Wright2018?}}). Basal area density is highly variable across our site, differing on average 40\% between salmon-enhanced and salmon-depleted banks, although the difference was not statistically significant (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}). Ultimately, the hierarchy of drivers of tree growth in this ecosystem appears to be landscape position (and associated forest demography) followed by climate and thirdly, nutrients. All told, a lack of long-term changes in soil nutrient dynamics and only marginal response in tree growth indicates that salmon nutrients are not a strong bottom-up force in northern riparian forest dynamics.

Our \textsuperscript{15}N/\textsuperscript{14}N stable isotope data raise further questions of assessing MDN subsidies to tree growth. Vegetation typically takes up only 17\% of added N to forests, with soils instead being the dominant N sink (\protect\hyperlink{ref-Templer2012}{\textbf{Templer2012?}}). Thus, elevated bulk soil \textsuperscript{15}N/\textsuperscript{14}N in our study suggests a potentially significant MDN sink in soil. On the other hand, elevated bulk soil \textsuperscript{15}N/\textsuperscript{14}N may also reflect increases in soil N fractionation during N cycling and loss under salmon. Highly localized N pulses (as occur with MDN and other N subsidies) temporarily exceed plant and soil N sinks, leading to accelerated N loss via ammonia volatilization, nitrification and nitrate leaching, and/or denitrification (\protect\hyperlink{ref-Perakis2002}{\textbf{Perakis2002?}}). All of these N loss pathways favor 14N and discriminate against 15N (in some cases with a fractionation up to 30‰), and effects are strongest at high N availability, leading to high values of residual soil 15N (\protect\hyperlink{ref-Hogberg1998}{\textbf{Hogberg1998?}}). Prior work has shown that MDN inputs accelerate N losses from soil, particularly gaseous N losses (\protect\hyperlink{ref-Holtgrieve2009}{\textbf{Holtgrieve2009?}}) that are associated with large isotope fractionation (\protect\hyperlink{ref-Hogberg1998}{\textbf{Hogberg1998?}}). Our finding that \(\delta^{15}N\) of soil NH\textsubscript{4}\textsuperscript{+} was greater than bulk soil \(\delta^{15}N\) for 95\% of observations on the salmon enhanced bank and 84\% of observations on the salmon depleted bank, further confirms that isotopic fractionation is important at Hansen Creek and likely elsewhere.

There is a global trend for higher foliar \(\delta^{15}N\) with increased soil N supply (\protect\hyperlink{ref-Craine2009}{\textbf{Craine2009?}}) indicating accelerated soil N cycling and \(\delta^{15}N\) fractionation due to exogenous N (from salmon or elsewhere) will alter plant foliar \(\delta^{15}N\). This has important implications for using two-source mixing models to assess salmon N subsidies to riparian forests. Typical MDN mixing models assume 1) the isotopic signature of salmon is unchanged in the soils prior to plant uptake, and 2) reference sites are biogeochemically similar to salmon sites. However, our data suggest that both of these assumptions are violated at Hansen Creek, and are likely violated at all salmon-influenced riparian ecosystems. First, we observed that \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+}, the dominant form of inorganic N in our soils, exceeded the 12.6‰ salmon end-member for 26\% of our observations from the salmon enriched bank and 9\% of observations from the salmon depleted bank, thus violating assumption (1) above. Our soil N data indicate Hansen Creek is a site of intermediate fertility relative to other boreal forests, so that soil NH\textsubscript{4}\textsuperscript{+} (rather than organic N or NO\textsubscript{3}\textsuperscript{-}) is most likely the dominant N source taken up by plants (\protect\hyperlink{ref-Chapin2011}{\textbf{Chapin2011?}}). Second, {[}N\textsubscript{Org}{]}, C:N, \(\delta^{13}C\), and GW varied with distance from the stream independent of salmon enhancement indicating site variability is a dominant driver of N cycling in this system. This presents a challenge for selecting control sites to calculate terrestrial end members, as key N cycling factors vary longitudinally away from streams and simply selecting reference sites that are beyond the reach of salmon would likely violate the mixing model assumption of biogeochemical similarity. Additionally, observations of \(\delta^{13}C\) increasing and GW decreasing from the creek edge are consistent with higher water use efficiency and less 13C discrimination by vegetation, resulting in higher \(\delta^{13}C\) in soil due to litterfall (\protect\hyperlink{ref-Oltean2016}{\textbf{Oltean2016?}}). These data identify systematic differences between salmon-enhanced vs.~salmon-depleted banks that cannot be attributed to salmon, and which likely reflect landscape or soil differences. Previous studies examining contributions of MDN to riparian vegetation have not tested biogeochemical similarity across sites, an assumption that is likely violated beyond Hansen Creek specifically.

Violation of mixing model assumptions can lead to significant bias in calculations of MDN sources. To illustrate this point, we applied a typical mixing model framework to our maximum observed \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+} values to calculate the percent MDN contribution of salmon to NH\textsubscript{4}\textsuperscript{+} for the most extreme observation, representing the greatest possible bias in calculations. Assuming soil processes have no effect on the isotopic signature yielded impossible result of 298\% MDN contribution. To account for isotopic fractionation in soils, we applied our mean observed \(\delta^{15}N\) of soil NH\textsubscript{4}\textsuperscript{+} at the 3 m distance (19.25‰) as the marine endmember to mean foliar 15N data at the same site from (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}) and estimate 59.24\% MDN on the salmon bank, which is 27.6\% lower than the original estimate of 86.8\% using salmon \(\delta^{15}N\) as the marine endmember. Repeating this with our maximum observed value for \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+} (41.2‰), we estimate only 28.9\% of foliar N on salmon enhanced bank was MDN (a 57.9\% reduction from (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}) estimates). Thus, failure to account for isotopic enrichment associated with soil N transformations can lead to overestimates of MDN contributions to plants, and observed variability in \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+} can produce a wide range of MDN estimates not previously considered. Given that our elevated \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+} values are consistent with expected changes during soil N transformation (\protect\hyperlink{ref-Hogberg1998}{\textbf{Hogberg1998?}}), there is a distinct possibility that previous MDN studies have overestimated the amount of MDN by not considering the effects of \textsuperscript{15}N/\textsuperscript{14}N fractionation in mixing model calculations. (\protect\hyperlink{ref-Wheeler2017}{\textbf{Wheeler2017?}}) found similar results in a semi-arid ecosystem of central Idaho, where accounting for fractionation from decomposition resulted in a 16\% reduction in estimated N deposition rates from salmon carcasses. The effects of fractionation on soil N pools is occurring in both of these systems, and likely elsewhere, and needs to be considered when applying mixing models to MDN data to avoid overestimations of salmon N contributions to riparian systems.

Our study is comprehensive in terms of the number of ecosystems factors considered but limited in that it includes only one seasonal timeframe. As much as 40\% of the annual inorganic N flux is released during the eight-month dormant season (September-May) and it has been posited spring and fall may be important for many biogeochemical processes in boreal forests (\protect\hyperlink{ref-Hobbie1996}{\textbf{Hobbie1996?}}; \protect\hyperlink{ref-Chapin2006}{\textbf{Chapin2006?}}; \protect\hyperlink{ref-Drake2006}{\textbf{Drake2006?}}). While MDN inputs do not affect the N pools and transformation rates during the summer growth period based on our results, N concentrations and transformations may be elevated in this system on shorter timescales (weeks to months after salmon return). The objective of this study was to identify the long-term legacy of salmon subsidies; short-term effects were both beyond the scope of this study and have been previously investigated in this system (\protect\hyperlink{ref-Holtgrieve2009}{\textbf{Holtgrieve2009?}}). Considering long-term effect of N subsidies as opposed to short-term provides new information on sustained N use and retention in the ecosystem and whether these salmon nutrients have lasting impacts on ecosystem function meaningful in a restoration context.

While this study is limited to one system, the results that N transformations cause a fractionation that can bias MDN mixing model estimates and that landscape factors are the primary driver of long-term N retention and use, are relevant to other systems where anadromous, semelparous salmon are abundant (Pacific, Atlantic, Great Lakes) (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}). This result also agrees with related research examining fractionation of mineralization and nitrification (\protect\hyperlink{ref-Hogberg1998}{\textbf{Hogberg1998?}}), and fertilization studies (\protect\hyperlink{ref-Lu2011}{\textbf{Lu2011?}}). Additionally, it demonstrates salmon N subsidies may have a short-term and likely small spatial scale (\protect\hyperlink{ref-Drake2006}{\textbf{Drake2006?}}) legacy in soils. While the importance of site variability relative to salmon subsidies may vary by system, this work demonstrates the importance of considering site variability and demonstrating biogeochemical similarity when selecting control sites for riparian MDN studies.

Salmon provide critical food resources to many of terrestrial and aquatic consumers (\protect\hyperlink{ref-Cederholm1999}{\textbf{Cederholm1999?}}; \protect\hyperlink{ref-Gende2002}{\textbf{Gende2002?}}; \protect\hyperlink{ref-Schindler2003}{\textbf{Schindler2003?}}), but the evidence that MDN stimulate terrestrial primary production is less certain. The salmon carcass manipulation experiment described here and in (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}) represents an extreme case of carcass addition and depletion to riparian areas, as measured by bulk \(\delta^{15}N\) and estimated percent contribution was approximately twice previous studies for both trees and soils (\protect\hyperlink{ref-Helfield2002}{\textbf{Helfield2002?}}; \protect\hyperlink{ref-Bartz2005}{\textbf{Bartz2005?}}). Generally, results of this manipulation were equivocal for soils and a statistically significant but ecologically small effect on trees (\protect\hyperlink{ref-Quinn2018}{\textbf{Quinn2018?}}). Simultaneously, other recent changes to boreal forest systems, such as moisture and temperature, appear to have a greater potential than MDN to alter biogeochemical pathways and primary production in these systems (\protect\hyperlink{ref-Chapin2006}{\textbf{Chapin2006?}}; \protect\hyperlink{ref-Yarie2008}{\textbf{Yarie2008?}}; \protect\hyperlink{ref-Lloyd2013}{\textbf{Lloyd2013?}}; \protect\hyperlink{ref-Wright2018}{\textbf{Wright2018?}}). This study also demonstrates the importance of testing biogeochemical and site similarity between experimental and control sites in nutrient subsidy studies, as even banks on the same creek can have landscape and soil variability that alter N concentration, transformations, and vegetative growth. Altogether, while salmon have clear benefits for consumers, management of salmon populations or application of compensatory restoration strategies based on terrestrial productivity response to salmon inputs may be unfounded for some systems, and at least, hard to predict.

\hypertarget{tables}{%
\section{Tables}\label{tables}}

\textbf{Table} \ref{tab:deployment}: Latitude and longitude of \emph{C. gigas} outplants, as well as time of day and tidal height at collection. Oysters were placed at five locations sites: Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), and Willapa Bay (WB).

\begingroup\fontsize{10}{12}\selectfont
\begin{longtable}[t]{lrrlr}
\caption{\label{tab:deployment}Outplant locations and time of day and tidal height at collection}\\
\toprule
Location & Latitude & Longitude & Time at Collection & Tidal Height  MLLW  feet  at Collection\\
\midrule
CI & 47.35794 & -122.7958 & 12:15 & -1.8\\
FB & 48.48169 & -122.5835 & 12:12 & -1.7\\
PG & 47.84268 & -122.5838 & 11:11 & -1.6\\
SK & 47.35523 & -123.1572 & 11:51 & -1.8\\
WB & 46.49448 & -124.0261 & 9:28 & -1.7\\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage
\begin{landscape}

Table 1.2: Proteins used as targets for a Selected Reaction Monitoring Assay (SRM). Targets were identified based on differential abundance and stress-related annotations. At least two peptides and six transitions were included in the assay for each protein. The protein Catalase had two isoforms under separate proteome IDs and target peptides were chosen from both IDs. A total of four peptides and twelve associated transitions were used as SRM targets.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{llrr}
\caption{\label{tab:targets}Target peptides used for Selected Reaction Monitoring Assay (SRM)}\\
\toprule
Protein & Proteome ID & Number of Peptides & Number of Transitions\\
\midrule
3-ketoacyl-CoA thiolase & CHOYP\_ACAA2.1.1|m.30666 & 3 & 9\\
Peroxiredoxin-5 & CHOYP\_BRAFLDRAFT\_119799.1.1|m.23765 & 2 & 6\\
Thioredoxin reductase 3 & CHOYP\_BRAFLDRAFT\_122807.1.1|m.3729 & 3 & 9\\
Protein phosphatase 1B & CHOYP\_BRAFLDRAFT\_275870.1.1|m.12895 & 3 & 9\\
Carbonic anyhdrase 2 & CHOYP\_CAH2.1.1|m.42306 & 3 & 9\\
\addlinespace
Catalase 1 & CHOYP\_CATA.1.3|m.11120 & 3 & 9\\
Catalase 2 & CHOYP\_CATA.3.3|m.21642 & 1 & 3\\
Glucose-6-phosphate 1-dehydrogenase & CHOYP\_G6PD.2.2|m.46923 & 3 & 9\\
Heat shock 70 kDa protein & CHOYP\_HS12A.25.33|m.60352 & 2 & 6\\
Heat shock protein 70 B2 & CHOYP\_HSP74.1.1|m.13095 & 2 & 6\\
\addlinespace
NAD(P) transhydrogenase & CHOYP\_LOC100633041.1.1|m.35428 & 2 & 6\\
Glycogen phosphorylase & CHOYP\_LOC100883864.1.1|m.41791 & 3 & 9\\
Multidrug resistance-associated protein & CHOYP\_MRP1.5.10|m.34368 & 2 & 6\\
Protein disulfide-isomerase 1 & CHOYP\_PDIA1.1.1|m.5297 & 3 & 9\\
Protein disulfide-isomerase 2 & CHOYP\_PDIA3.1.1|m.60223 & 3 & 9\\
\addlinespace
Puromysin-sensitive amirase & CHOYP\_PSA.1.1|m.27259 & 3 & 9\\
\bottomrule
\end{longtable}
\endgroup{}

\end{landscape}
\clearpage

\textbf{Table} \ref{tab:outplantenv}: Environmental conditions (mean ± standard deviation) at outplant locations (Case Inlet (CI), Fidalgo Bay (FB), Port Gamble Bay (PG), Skokomish River Delta (SK), Willapa Bay (WB).
\begin{verbatim}
Warning in read.table(file = file, header = header, sep = sep, quote = quote, :
incomplete final line found by readTableHeader on 'data/Ch1/Table1.3.csv'
\end{verbatim}
\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{llllll}
\caption{\label{tab:outplantenv}Outplant conditions}\\
\toprule
Metric & CI & FB & PG & SK & WB\\
\midrule
Mean Temperature (ºC) & 16.1 ± 1.7 & 14.8 ± 1.8 & 15.1 ± 2.9 & 15.1± 2.2 & 18.0 ± 1.3\\
Mean Salinity (PSU) & 24.47 ± 1.35 & 30.20 ± 0.38 & 23.25 ± 1.56 & 13.43 ± 1.04 & 27.28 ± 0.73\\
Mean Dissolved Oxygen Content (mg/L) & 8.27 ± 1.91 & 9.63 ± 4.59 & 11.70 ± 3.70 & 9.99 ± 4.58 & 8.76 ± 1.78\\
Mean pH & 7.63 ± 0.19 & 7.54 ± 0.23 & 7.33 ± 0.25 & 7.37 ± 0.24 & 7.56 ± 0.18\\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage

\textbf{Table} \ref{tab:envmeans}: Analysis of Similarities (ANOSIM) R statistic and p-value for mean values of environmental variables by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{llll}
\caption{\label{tab:envmeans}ANOSIM for mean environmental conditions}\\
\toprule
Environmental Variable & Contrast & R statistic & p value\\
\midrule
pH & Site & 0.3281 & 0.135\\
Habitat & 0.2125 & 0.151 & \\
Dissolved oxygen & Site & 0.4063 * & 0.0530 *\\
Habitat & -0.14375 & 0.922 & \\
Salinity & Site & -0.0133 & 0.482\\
\addlinespace
Habitat & -0.1667 & 0.971 & \\
Temperature & Site & 1.0000 ** & 0.0020 **\\
Habitat & -0.192 & 0.905 & \\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage

\textbf{Table} \ref{tab:envvariance}: Analysis of Similarities (ANOSIM) R statistic and p-value for environmental variable variances by site and habitat. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{llll}
\caption{\label{tab:envvariance}ANOSIM for variance of environmental conditions}\\
\toprule
Environmental Variable & Contrast & R statistic & p value\\
\midrule
pH & Site & 0.5313 ** & 0.0180 **\\
Habitat & -0.1813 & 0.934 & \\
Dissolved oxygen & Site & 0.6800 ** & 0.0030 **\\
Habitat & -0.124 & 0.848 & \\
Salinity & Site & 0.8400 ** & 0.0130 **\\
\addlinespace
Habitat & -0.125 & 0.708 & \\
Temperature & Site & 0.9400 ** & 0.0010 **\\
Habitat & -0.22 & 0.987 & \\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage

\textbf{Table} \ref{tab:protANOSIM}: Results of pairwise ANOSIM tests between sites for protein abundance data. R values are listed above the diagonal, and p values are listed below the diagonal. Values significant at the 0.05 level are marked by a double asterisk, and values significant at the 0.10 level are marked by a single asterisk.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{llllll}
\caption{\label{tab:protANOSIM}Pairwise ANOSIM for protein abundance}\\
\toprule
X & CI & FB & PG & SK & FB 1\\
\midrule
CI & - & 0.0197 & -0.0707 & -0.0583 & 0.0926\\
FB & 0.331 & - & -0.0313 & 0.0949 & 0.2568\\
PG & 0.78 & 0.5992 & - & -0.0061 & 0.0727\\
SK & 0.733 & 0.086 & 0.382 & - & 0.1547\\
WB & 0.16 & 0.0350 ** & 0.192 & 0.0790 * & -\\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage

\textbf{Table} \ref{tab:RDA}: Significance of each predictor in the RDA. Salinity mean and variance were not included due to missing values. Predictors significant at the 0.10 level are marked by a single asterisk.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{lrrll}
\caption{\label{tab:RDA}RDA Results}\\
\toprule
Environmental Variable & Df & Variance & F & Pr  F \\
\midrule
Temperature Mean & 1 & 0.0014 & 2.3275 * & 0.0650 *\\
Temperature Variance & 1 & 0.0013 & 2.1627 * & 0.0670 *\\
pH Mean & 1 & 0.0003 & 0.5264 & 0.781\\
pH Variance & 1 & 0.0007 & 1.2171 & 0.301\\
Dissolved Oxygen Mean & 1 & 0.0006 & 1.0354 & 0.349\\
\addlinespace
Dissolved Oxygen Variance & 1 & 0.0003 & 0.5692 & 0.725\\
Residual & 19 & NA &  & \\
\bottomrule
\end{longtable}
\endgroup{}
\clearpage
\begin{landscape}

Table 1.8: Functions of differentially abundant proteins.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{ll}
\caption{\label{tab:protfunctions}Protein functions}\\
\toprule
Protein & Function\\
\midrule
Peroxiredoxin-5 (PRX) & Scavenges ROS located in mitochondria using cysteine residuals to reduce substrates\\
Catalase (CAT) & Degrades hydrogen peroxide into water and oxygen\\
NAD(P) transhydrogenase (NADPt) & Involved in maintenance of cellular redox state\\
Carbonic anhydrase (CA) & Maintenance of acid-base balance by catalyzing CO2 conversion to HCO3- \\
Glucose-6-phosphate 1-dehydrogenase (G6PD) & Catalyzes first step in the pentose phosphate pathway and controls rate of pathway’s oxidative portion\\
\addlinespace
Protein disulfide-isomerase 1 (PDI1) and Protein disulfide-isomerase 2 (PDI2) & General chaperone protein that aids in protein disulfide bond formation and isomerization\\
\bottomrule
\end{longtable}
\endgroup{}
\end{landscape}
\clearpage

\#\#Figures

\textbf{Figure} \ref{fig:npathways}: Nitrogen pathways in soil where MDN enters terrestrial systems via decay of salmon organic tissues or excretion from direct salmon consumers such as bears. Arrows represent conversion pathways with the potential to impart isotopic fractionations on plant available nitrogen (NH\textsubscript{4}\textsuperscript{+} or NO\textsubscript{3}\textsuperscript{-}).\newline
\begin{figure}[h]
  \includegraphics{figure/Ch1/fig1.1.pdf}
  \caption{Nitrogen pathways in soil}
  \label{fig:npathways}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:modIsotope}: Data (closed circles) and predicted values (open circles) for the model with the most support (Table \ref{tab:suppmod1}) for soil organic \(\delta^{15}N\) and \(\delta^{13}C\), \(\delta^{15}N\) of NH\textsubscript{4}\textsuperscript{+}, and C:N for both the salmon-enhanced and the salmondepleted banks of Hansen Creek at 1, 3, 6, 10, and 20 m from the edge of the creek bed with 95\% confidence intervals (dashed line) for predicted values. Blue (a and c) denotes measures of marine-derived nitrogen, and green (b and d) denotes site variable factors.\newline
\begin{figure}[h]
  \includegraphics[width=0.85\textwidth]{figure/Ch1/Figure2_Feddernetal.pdf}
  \caption{Data and predicted values for the model with the most support: Stable Isotopes}
  \label{fig:modIsotope}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:modConc}: Data (closed circles) and predicted values (open circles) for the model with the most support (Table \ref{tab:suppmod1}) for NH\textsubscript{4}\textsuperscript{+} and NO\textsubscript{3}\textsuperscript{-}, net mineralization and nitrification, {[}N\textsubscript{org}{]}, and gravimetric water content for both the salmon-enhanced and the salmon-depleted banks of Hansen Creek at 1, 3, 6, 10, and 20 m from the edge of the creek bed with 95\% confidence intervals (dashed line) for predicted values. Red (a, b, c, d) denotes measures of soil productivity, and green (e and f) denotes site variable factors.\newline 
\begin{figure}[h]
  \includegraphics[width=0.85\textwidth]{figure/Ch1/Figure3_Feddernetal.pdf}
  \caption{Data and predicted values for the model with the most support: Concentrations and Transformations}
  \label{fig:modConc}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:ch1resid}: Predicted verse observed values and predicted verse residuals for the model with the most support (Table \ref{tab:suppmod1}, Figure \ref{fig:modIsotope}, \ref{fig:modConc}) for each the response variables. \newline 
\begin{figure}[h]
  \includegraphics[width=1\textwidth]{figure/Ch1/ch1residuals.pdf}
  \caption{Residual Plots for Best Models}
  \label{fig:ch1resid}
\end{figure}
\hypertarget{stable-isotope-signatures-in-historic-harbor-seal-bone-link-food-web-assimilated-carbon-and-nitrogen-resources-to-a-century-of-environmental-change}{%
\chapter{Stable isotope signatures in historic harbor seal bone link food web-assimilated carbon and nitrogen resources to a century of environmental change}\label{stable-isotope-signatures-in-historic-harbor-seal-bone-link-food-web-assimilated-carbon-and-nitrogen-resources-to-a-century-of-environmental-change}}

\#\#Abstract

Anthropogenic climate change will impact nutrient cycles, primary production, and ecosystem structure in the world's oceans, although considerable uncertainty exists regarding the magnitude and spatial variability of these changes. Understanding how regional-scale ocean conditions control nutrient availability and ultimately nutrient assimilation into food webs will inform how marine resources will change in response to climate. To evaluate how ocean conditions influence the assimilation of nitrogen and carbon into coastal marine food webs, we applied a novel dimension reduction analysis to a century of newly acquired molecular isotope data derived from historic harbor seal bone specimens. By measuring bulk \(\delta^{13}C\) and \(\delta^{15}N\) values of source amino acids of these top predators from 1928-2014, we derive indices of primary production and nitrogen resources that are assimilated into food webs. We determined coastal food webs responded to climate regimes, coastal upwelling, and freshwater discharge, yet the strength of responses to individual drivers varied across the northeast Pacific. Indices of primary production and nitrogen availability in the Gulf of Alaska were dependent on regional climate indices (i.e., North Pacific Gyre Oscillation) and upwelling. In contrast, the coastal Washington and Salish Sea food webs were associated with local indices of freshwater discharge. For some regions (eastern Bering Sea, northern Gulf of Alaska) food web assimilated production was coupled with nitrogen sources, however other regions demonstrated no production-nitrogen coupling (Salish Sea). Temporal patterns of environmental indices and isotopic data from Washington state varied about the long-term mean with no directional trend. Data from the Gulf of Alaska, however, showed below average harbor seal \(\delta^{13}C\) values and above average ocean conditions since 1975, indicating a change in primary production in recent decades. Altogether, these findings demonstrate stable isotope data can provide useful indices of nitrogen resources and phytoplankton dynamics specific to what is assimilated by food webs.

\#\#Introduction

Changing ocean conditions are reshaping the structure and function of marine food webs on regional scales. Ocean temperature (\protect\hyperlink{ref-Hoegh2010}{\textbf{Hoegh2010?}}), oxygen availability (\protect\hyperlink{ref-Brietburg2018}{\textbf{Brietburg2018?}}), and climatic regimes such as El Niño Southern Oscillation (ENSO) (\protect\hyperlink{ref-Vecchi2010}{\textbf{Vecchi2010?}}) alter nutrient availability and cycling, and thus, the ecological structure of marine systems. Projected global redistribution of nutrients suggests net primary production in the ocean is likely to change both spatially and temporally. Yet, substantial uncertainty remains, with predictions suggesting both increases and decreases in global net primary productivity of up to 20\% by 2100 (\protect\hyperlink{ref-Bopp2013}{\textbf{Bopp2013?}}; \protect\hyperlink{ref-Kwiatkowski2017}{\textbf{Kwiatkowski2017?}}; \protect\hyperlink{ref-Gregg2003}{\textbf{Gregg2003?}}). An important contributor to this uncertainty is regional variability in phytoplankton response to ocean conditions and how that variability will impact other trophic levels and dependent fisheries (\protect\hyperlink{ref-Moore2018}{\textbf{Moore2018?}}; \protect\hyperlink{ref-Brander2010}{\textbf{Brander2010?}}). Ocean conditions (i.e., sea surface temperature, freshwater discharge, wind, and ice cover) have been associated with abundance and recruitment of many fish species in the Northeast Pacific (\protect\hyperlink{ref-Cunningham2018}{\textbf{Cunningham2018?}}; \protect\hyperlink{ref-Puerta2019}{\textbf{Puerta2019?}}; \protect\hyperlink{ref-Stachura2014}{\textbf{Stachura2014?}}). Nonetheless, these studies rarely include indicators of nutrient availability or primary production linking the ecosystem response to its environment. Understanding how regional and local scale physical drivers control nutrient availability and ultimately nutrient assimilation into food webs will be important for predicting the future availability of marine resources.

A strong empirical understanding of food web response to changing ocean conditions and nutrient constraints requires time series data that span multiple climate regimes to decouple natural variability with long-term anthropogenic changes. Currently, quantitative methods are also limited in their ability to scale primary production trends to ecosystem-level responses. Stable isotope measures of \(\delta^{15}N\) (\textsuperscript{15}N/\textsuperscript{14}N) of individual amino acids is an emerging tool for reconstructing trends in nitrogen sources from historic specimens (\protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}; \protect\hyperlink{ref-Sherwood2011}{\textbf{Sherwood2011?}}; \protect\hyperlink{ref-Sherwood2014}{\textbf{Sherwood2014?}}; \protect\hyperlink{ref-Whitney2019}{\textbf{Whitney2019?}}). The \(\delta^{15}N\) signature at the base of the food web is primarily controlled by utilization and the isotopic signatures of different nitrogen sources, particularly urea, nitrate, and ammonium, by primary producers (\protect\hyperlink{ref-Graham2010}{\textbf{Graham2010?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}). Measurements of bulk \(\delta^{15}N\) values from consumers can be difficult to attribute to changes at the base of the food web because trophic level shifts also effect the isotopic composition of bulk nitrogen (\protect\hyperlink{ref-Fry2006}{\textbf{Fry2006?}}). Amino acid specific \(\delta^{15}N\) data addresses this challenge, as amino acids exhibit two distinct patterns in isotopic enrichment: trophic amino acids (i.e., glutamic acid, alanine, proline) become enriched in \(\delta^{15}N\) with each trophic transfer and source amino acids (i.e., phenylalanine, lysine, methionine) show minimal change and thus are reflective of the base of the food web (\protect\hyperlink{ref-McClelland2002}{\textbf{McClelland2002?}}; \protect\hyperlink{ref-Chikaraishi2009}{\textbf{Chikaraishi2009?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}).

Similar to the nitrogen stable isotope composition of amino acids as a proxy for nitrogen sources, carbon isotopic composition has emerged as a useful tool for assessing historic changes in phytoplankton (\protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}; \protect\hyperlink{ref-Lorrain2020}{\textbf{Lorrain2020?}}). However, cellular growth rates, phytoplankton community composition, the isotopic composition of carbon in CO\textsubscript{2}, and CO\textsubscript{2} concentration all affect the \(\delta^{13}C\) (\textsuperscript{13}C/\textsuperscript{12}C) values of phytoplankton in tandem (\protect\hyperlink{ref-Lorrain2020}{\textbf{Lorrain2020?}}; \protect\hyperlink{ref-Burkhardt1999}{\textbf{Burkhardt1999?}}). The relative effects of these factors remain difficult to discern from carbon isotope data alone. Nonetheless, carbon stable isotope data is highly correlated with copepod biomass in the northeast Pacific and thus can be a useful combined index of ocean productivity (\protect\hyperlink{ref-Espinasse2020}{\textbf{Espinasse2020?}}). While both source amino acid \(\delta^{15}N\) and bulk \(\delta^{13}C\) values can be influenced by a number of biogeochemical and physiological processes (Figure 2.1), they are useful indicators of nitrogen utilization (source amino acid \(\delta^{15}N\)) and phytoplankton dynamics (bulk \(\delta^{13}C\)), despite the difficulty in identifying specific mechanisms of fractionation.

Here we use source amino acid \(\delta^{15}N\) and bulk \(\delta^{13}C\) values of consumer bone collagen as indicators of change in food web-assimilated nitrogen (nitrogen utilization and isotopic composition at the base of the food web) and food web-assimilated production (phytoplankton composition, {[}CO\textsubscript{2}{]}, cellular growth, and physiology). These definitions assume major changes in nitrogen utilization and phytoplankton dynamics are recorded in the stable isotope composition of nitrogen and carbon in phytoplankton (\protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}; \protect\hyperlink{ref-Sherwood2011}{\textbf{Sherwood2011?}}; \protect\hyperlink{ref-delaVega2021}{\textbf{delaVega2021?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}), scaled to the spatial and temporal resource use of consumers, and conserved with minimal trophic fractionation (\protect\hyperlink{ref-Chikaraishi2009}{\textbf{Chikaraishi2009?}}). Bulk \(\delta^{13}C\) and \(\delta^{15}N\) values of source amino acids such as phenylalanine (\(\delta^{15}N_{Phe}\)) from long-lived, generalist consumers provide ecosystem-level information of carbon and nitrogen dynamics that are integrated over space, time, and multiple energy pathways in the food web (\protect\hyperlink{ref-McCann2005}{\textbf{McCann2005?}}; \protect\hyperlink{ref-delaVega2021}{\textbf{delaVega2021?}}). As a result, these data sources are more relevant to questions of food web responses to large-scale environmental forcing than discrete measurements of inorganic nutrients or phytoplankton. Ultimately these data can be used to understand how ecosystems have responded to environmental variability in the past and glean insights into food web responses to oceanic conditions in the future.

Harbor seals (\emph{Phoca vitulina}) are a particularly well-suited predator to understand food web shifts through time because of their primarily piscivorous diet, generalist foraging strategies, high site fidelity, and frequent occurrence in museum specimen collections. Adult harbor seals typically forage 5 - 10 km from haul out sites and at depths \textless{} 200 m and are opportunistic feeders (\protect\hyperlink{ref-Lance2012}{\textbf{Lance2012?}}). Therefore, the nitrogen and carbon stable isotope composition of harbor seals offer a robust representation of the isotopic composition of carbon and nitrogen assimilated into coastal food webs. Harbor seal specific trophic enrichment factors for nitrogen have been quantified in controlled feeding studies, confirming minimal trophic enrichment for phenylalanine between seals and their prey (\protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}}). Environmentally induced shifts in foraging patterns, specifically nearshore verse offshore feeding, has the potential to affect the carbon isotope composition in harbor seal tissues (Figure 2.1). We assume these behavioral effects are minimal on annual time scales compared to changes in the carbon and nitrogen isotope composition at the base of the food web given their restricted foraging ranges.

We aim to identify how archived \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values vary regionally across the northeast Pacific on ecologically relevant scales (integrated annually and regionally) and through time using museum harbor seal specimens from 1928-2014 (Figure \ref{fig:map}). Additionally, we characterize abiotic factors that influence harbor seal \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values to identify ocean conditions important for food web assimilation of nitrogen and carbon. The effect of regional ocean condition on the stable isotope signature of source amino acids limits the application of short-term datasets for productivity studies, as short-term environmental perturbations are difficult to decouple from longer term trends such as climate regimes (\protect\hyperlink{ref-Vokshoori2014}{\textbf{Vokshoori2014?}}). We therefore identify long-term environmental drivers that are important for interpreting reconstructed isotope data.

\#\#Methods
\#\#\#Sample Collection and Analysis

Harbor seal bone samples were obtained from specimens curated at the Burke Museum (University of Washington), the Slater Museum (University of Puget Sound), the Museum of the North (University of Alaska Fairbanks), the Royal British Columbia Museum, the Smithsonian Institute, and the National Marine Mammal Laboratory (NOAA). Specimens were either treated by maceration in warm water or cleaned by beetles and soaked in a dilute ammonia solution then stored in acid free boxes. Adult specimens were sampled from three regions: eastern Bering Sea, the Gulf of Alaska, and Washington state, which also included 18 specimens from the southern British Columbia coast (Figure \ref{fig:map}). We further stratified samples from the Gulf of Alaska into two subregions (northern and southeast) and Washington state into two subregions (coastal and Salish Sea) for a total of five subregions. Sampling prioritized long-term temporal coverage, specifically focusing on climate regimes shifts (i.e., PDO). Additionally, samples with sex and size metadata were prioritized, although it was not available for most specimens. Metadata was accessed through \href{http://www.vertnet.org/index.html}{VertNet} using catalogue numbers and institution codes.

Bone samples were decalcified with the resulting collagen acid hydrolyzed, derivatized, and analyzed for compound-specific nitrogen stable isotope analysis (CSIA) of 11 individual amino acids, including one source amino acid, phenylalanine (phe). Of the 11 amino acids, phenylalanine was the only discernable source amino acid and phenylalanine is the only amino acid data are reported in this manuscript (Appendix S1). CSIA samples were analyzed by GC-C-irMS at the University of Washington Facility for Compound-Specific Stable Isotope Analysis of Environmental Samples using a Thermo Scientific Trace GC + GC IsoLink coupled to a Delta V irMS following the procedures developed by (\protect\hyperlink{ref-Chikaraishi2007}{\textbf{Chikaraishi2007?}}) and protocols by Rachel Jeffrey's lab at University of Liverpool UK (full analytical details are provided in Appendix 1). Individual collagen samples were analyzed in triplicate along with a mixed amino acid standard of known isotopic composition (Sigma-Aldrich Co.) (mean precision of analytical standard for phenylalanine = 0.3‰). Internal and external standards were used and data processing included a drift correction. A total of 215 specimens were sampled from the time period of 1928-2014 for CSIA, making this the largest CSIA dataset of a mammal to date. Decalcified collagen of 190 specimens was analyzed for bulk \textsuperscript{13}C/\textsuperscript{12}C and bulk \textsuperscript{15}N/\textsuperscript{14}N at the University of Washington's IsoLab using a Costech ElementalAnalyzer, ConFlo III, MAT253 for continuous flow-based measurements. \textsuperscript{15}N/\textsuperscript{14}N and \textsuperscript{13}C/\textsuperscript{12}C are reported in standard delta notation:
\begin{equation} 
  \delta^{15}N_ ( \textperthousand vs. air) =   
  [(\frac{^{15}N/^{14}N_{Sample}}{^{15}N/^{14}N_{Air}} -1)*1000]
  \label{eq:deltN}
\end{equation}
\begin{equation} 
  \delta^{13}C_ ( \textperthousand vs. VPBD) =   
  [(\frac{^{13}C/^{13}C_{Sample}}{^{13}C/^{13}C_{VPBD}} -1)*1000]
  \label{eq:deltC}
\end{equation}
Internal laboratory standards (Bristol Bay salmon and glutamic acid) were interspersed with samples for a two-point calibration and blank correction (mean standard precision 0.09‰ for \(\delta^{15}N\) and 0.04‰ for \(\delta^{13}C\)). A linear drift correction was also applied using IsoDat software. The collagen C:N ratio was used to verify the integrity of collagen for stable isotope analysis following specimen treatment and storage (\protect\hyperlink{ref-vanKlinken1999}{\textbf{vanKlinken1999?}}).

The isotopic composition of marine dissolved organic carbon has been steadily depleted in \textsuperscript{13}C over the past 100 years due to increases in anthropogenic CO\_2 in the atmosphere (referred to as the Oceanic Seuss Effect) (\protect\hyperlink{ref-Quay1992}{\textbf{Quay1992?}}). \(\delta^{13}C\) data were therefore corrected for the Seuss Effect using the following equation (\protect\hyperlink{ref-Misarti2009}{\textbf{Misarti2009?}}):
\begin{equation} 
 \mbox {Seuss Effect Correction Factor} =   
  d * e^{0.027*(t-1850)}
  \label{eq:seuss}
\end{equation}
Where \emph{d} is the maximum annual rate of \(\delta^{13}C\) decrease specific to the North Pacific (-0.014 derived from (\protect\hyperlink{ref-Quay1992}{\textbf{Quay1992?}})), \emph{t} is the year represented by the year of specimen collection with a one-year lag. The Seuss effect varies regionally (\protect\hyperlink{ref-Tagliabue2008}{\textbf{Tagliabue2008?}}) and we applied a northeast Pacific parameterization (\protect\hyperlink{ref-Misarti2009}{\textbf{Misarti2009?}}).

Standard linear models were used to identify whether size (standard length, cm), sex, and subregion of the harbor seals sampled were related to isotopic composition and to test whether these parameters needed to be standardized in environmental models. \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values were modelled independently as univariate continuous response variables using the following equation:
\begin{equation} 
 y_i \sim N(\boldsymbol{\alpha} + \boldsymbol{\beta X}_i, \sigma^2_y)
  \label{eq:linmods}
\end{equation}
where \emph{y} is the mean triplicate value for each individual \emph{i} for either \(\delta^{15}N_{Phe}\) or \(\delta^{13}C\) values. \textbf{X} represents the matrix of predictors (sex, length, subregion), \(\boldsymbol{\alpha}\) is a scalar and \(\boldsymbol{\beta}\) is a vector of coefficients for the predictors. Length (n = 116) was modelled as a continuous variable and was natural log transformed; subregion and sex (n = 190) were modelled as factors. Individual models were used to test whether a predictor was significant as opposed to a multivariate framework because, 1) sample sizes for \(\delta^{15}N_{Phe}\) (n = 215) and \(\delta^{13}C\) (n = 190) data varied, and 2) predictor metadata was incomplete for specimens. A pairwise t-test using the Bonferroni correction and non-pooled standard deviation was also used to compare differences in mean isotope signature between subregions and sex (Figure \ref{fig:dist}, Tables \ref{tab:ttestN} \& \ref{tab:ttestC}).

To understand the extent of coupling between indices of food web assimilated production and nitrogen resources, a linear model representing the basin wide relationship was fit to \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values as continuous variables assuming normal errors. To understand spatial variation in this relationship, a hierarchical model was fit to the same dataset with varying slope and varying intercept based on subregion as a random effect. This model took the following form:
\begin{equation} 
 y_i \sim N(\boldsymbol{\alpha}_{j[i]} + \boldsymbol{\beta}_{j[i]}\boldsymbol{x}_i, \sigma^2_y)
  \label{eq:hiermods}
\end{equation}
Where \emph{y} represents \(\delta^{13}C\) values as a continuous variable and x represents \(\delta^{15}N_{Phe}\) values as a continuous variable and j represents the group level predictor, subregion. \(\boldsymbol{\alpha}\) and \(\boldsymbol{\beta}\) are each vectors of coefficients that vary by subregion.

\#\#\#Quantifying effects of ocean condition on food web isotope indices

Linear models were used to identify environmental drivers of \(\delta^{13}C\) and \(\delta^{15}N_{Phe}\) values using a suite of environmental indices as covariates. A total of 42 environmental time series were compiled as potential predictor variables (Table 2.2) based on previous evidence for food web importance in the northeast Pacific (\protect\hyperlink{ref-Stachura2014}{\textbf{Stachura2014?}}; \protect\hyperlink{ref-DiLorenzo2008}{\textbf{DiLorenzo2008?}}). Each environmental time series was standardized around a mean of 0 and standard deviation of 1 and discharge data was also natural log transformed. We divided these environmental covariates a priori into four main mechanistic properties based on the expected effect on nutrient assimilation into the food web: climate regime, freshwater discharge, circulation (wind and upwelling), and sea surface temperature (Figure 2.1). Given the three regions in our analysis, each of these hypotheses were also divided according to our regional geographic breaks (eastern Bering Sea, Gulf of Alaska, and Washington). To reduce collinearity between environmental time series and reduce the total number of candidate models, a subset of 7 environmental times series were selected for each region based on the temporal overlap with stable isotope data. Each subset contained at least one time series for each of the four mechanistic properties and all possible combinations of predictors were tested (Table 2.3). While reduction of the number of times series provides analytical benefits, it comes at the cost of potentially conservative estimates of which covariates are important, meaning important components of ocean condition to the food webs may be missed.

\(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values were independently considered as response variables to evaluate relationships between predictors (environmental indices and location) and stable isotope data using Figure \eqref{eq:linmods} where X is a matrix of predictors using the 7 standardized environmental time series (continuous) and subregion (factor) as covariates. We treated carbon and nitrogen isotopes as response variables separately in linear models, rather than in a combined multivariate model due to differences in sample size and differences in the strength of correlation between for \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values for each subregion. Time series data prior to 1950 and after 2014 was excluded from this analysis as data for some covariates did not extend beyond 1950. Candidate models (n = 53) were compared using Akaike Information Criteria with a small sample size correction {[}AICc{]} and included all combinations of the environmental indices. In addition, a subregion factor was included with two levels for Washington (Salish Sea and coastal Washington) and the Gulf of Alaska (southcentral and southeast) and a null model (intercept only) was also tested. Tissue turnover time of bone collagen has not been measured in mammals of this size to our knowledge but is approximately 173 days for birds (\protect\hyperlink{ref-Hobson1992}{\textbf{Hobson1992?}}). Thus, a lag of one year was applied to the stable isotope datasets to account for the timing of tissue turnover in bone collagen. To validate this approach of applying a 1-year lag, 0- and 2-year lags were also applied to the best models of each region an compared to the 1-year lag using AIC. Additionally, month was tested as a smoothed predictor with 12 knots for stable isotope data in Washington samples using a generalized additive model (GAM). Support for a significant smoothing term would identify and seasonality in the data, which would be expected if tissue turnover time is less than a year. A one-year tissue turnover time was confirmed as a suitable assumption for harbor seal bone collagen, as 0- and 2- year lags had similar or less model support. There was no support for a smoothing effect by month in generalized additive models of \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) which would have indicated any seasonal variability in isotope composition and thus a turnover time of less than or greater than a year (p \textless{} 0.05; Figure \ref{fig:Length}) and thus a 1-year lag was applied to isotope data for all temporal analyses.

For each model with relatively high support (ΔAICc \textless{} 2) the AICc weight and the coefficient for each covariate is reported (Figure \ref{fig:dist}. To confirm collinearity was not problematic in the candidate models that included more than one environmental covariate, matrix scatterplots and variance inflation factors (vif) were used from the car package (Fox et al.~2019) in R (R Development Core Team, 2020).

\#\#\#Gaussian Process Dynamic Factor Analysis (GPDFA)
To further understand how the environment, \(\delta^{13}C\), and \(\delta^{15}N_{Phe}\) values covary through time in the Northeast Pacific, we developed a novel extension of conventional Dynamic Factor Analysis (DFA). DFA is a dimension reduction technique that identifies common processes underlying a set of multivariate time series. This technique has been applied to multivariate time series problems in fisheries and ecology to identify patterns of oceanographic variability that drive Pacific salmon stocks (\protect\hyperlink{ref-Stachura2014}{\textbf{Stachura2014?}}; \protect\hyperlink{ref-Jorgensen2016}{\textbf{Jorgensen2016?}}; \protect\hyperlink{ref-Ohlberger2016}{\textbf{Ohlberger2016?}}).

DFA models identify common trends across multiple time series (``latent trends'') and estimates the importance of that trend for each individual time series as a coefficient (``factor loading''). The two equations describing DFA take on the following form:
\begin{equation} 
 \boldsymbol{y}_t = \boldsymbol{Zx}_t + \boldsymbol{v}_t,\mbox{ where }\boldsymbol{v}_t \sim MVN(0,\boldsymbol{R})
  \label{eq:gdfa1}
\end{equation}
\begin{equation} 
 \boldsymbol{x}_t = \boldsymbol{x}_{t-1} + \boldsymbol{w}_t,\mbox{ where }\boldsymbol{w}_t \sim MVN(0,\boldsymbol{I})
  \label{eq:gdfa2}
\end{equation}
The observed data \textbf{y}\textsubscript{t} are modeled as combinations of latent trends \textbf{x}\textsubscript{t} at time \emph{t} (the dimensions of \textbf{x}\textsubscript{t} matching the number of trends which are also referred to as states) and factor loadings (\textbf{Z}) (a coefficient for each time series for each trend) at time \emph{t}, which are modeled as a random walk (Zuur et al.~2003). In addition there is an optional random observation error (\textbf{v}\textsubscript{t}) and process error (\textbf{w}\textsubscript{t}) which are multivariate normal

Our extension of DFA adopts an alternative model of the latent trends, modeling them with Gaussian Processes rather than random walks.Gaussian Processes (GP) have been widely used in fisheries and other fields (\protect\hyperlink{ref-Munch2018}{\textbf{Munch2018?}}). Instead of modeling a time series as an autoregressive process, GPs model a time series via a mean and variance function, \emph{x\textasciitilde{}}\(MVN(u,\Sigma)\) where \emph{u} represents and optional mean vector and \textbf{Σ} a covariance matrix. For GPDFA, we assume the mean to be zero, letting just the covariance function determine the GP smoothing. GPs are flexible in that the covariance matrix can be described by a wide range of flexible functions; for this application we use a Gaussian kernel (squared exponential) so that \(\Sigma_{i,j}=\sigma^2 exp(-d_{i,j}/\sigma)\), where \(\sigma^2\) is a variance parameter controlling the magnitude, θ is a shape parameter controlling how quickly covariance declines, and \(d_{i,j}\) is the known distance between time points \emph{i} and \emph{j}. A benefit of modeling \textbf{Σ} with a covariance function is that regardless of the dimensionality, all elements of \textbf{Σ} can be described by a small number of parameters. For GPDFA, we choose to use a GP predictive process model, because the number of time points may be large (\protect\hyperlink{ref-Latimer2009}{\textbf{Latimer2009?}}). This predictive model estimates the function values at a subset of locations (knots), and combines these estimates with the distance to locations at which data are observed to make predictions. More specifically, the values of the time series at the knot locations are \(x^*\)\textasciitilde{}\(MVN(0,\Sigma^*)\). Given the known distances between the locations of knots and locations of data, the covariance matrix between the two can be calculated, \(\Sigma_{(x,x^*)}\). Finally, the predictions of the time series at the observed data can be calculated as \(\hat{x}=\Sigma'_{(x,x^*)}\Sigma^{*-1}x^*\). In this extension of DFA, all other model components are identical to the conventional time series version with latent trends modeled as a random walk.

With the Gaussian Process DFA model, a decision needs to be made a priori about selecting the number and location of knots, where the function parameters are estimated at. There are multiple approaches for doing this; we adopted a model with 15 knots (more knots resulting in a smoother function), and estimated the knot location by performing a clustering approach of the years corresponding to the raw observations (partitioning around medoids, using the `pamk' function in the fpc library in R).

With conventional DFA using an autoregressive model, long gaps in time series data result in large, overestimations of the variance of the latent trends. Gaussian Processes model time series as a multivariate normal distribution, with estimated mean vector \textbf{u} and covariance matrix \textbf{Σ} (\protect\hyperlink{ref-Munch2018}{\textbf{Munch2018?}}). To constrain the number of estimated parameters, elements of \textbf{Σ} were modeled with a Gaussian or squared covariance exponential function such that \(\Sigma_{(i,j)}= \sigma^2 exp(-(t_i-t_j )^2/\theta)\). In this parameterization, \(\sigma^2\) controls the variability of the stochastic process, \(\theta\) controls the rate of decay in correlation between time steps, and t\textsubscript{i} and t\textsubscript{j} are the time variables (e.g.~years) for locations \emph{i} and \emph{j}.

We considered models with 1- 4 underlying trends. Each trend was modelled separately (different means) but models with multiple trends to have a shared covariance matrix amongst trends. The GPDFA approach was applied to time series from each region and the best model was selected using leave-one-out cross-validation (LOOIC) from the loo package in R (\protect\hyperlink{ref-Vehtari2017}{\textbf{Vehtari2017?}}). The choice of knots affects the degree of smoothness, with more knots creating more smooth functions. We tested several different numbers of knots and found results to be qualitatively similar. Similar to the previous analysis, time series data prior to 1948 for Washington state and prior to 1940 and after 2008 for the Gulf of Alaska was excluded from this analysis. We fit GPDFA to data from each region including all of the initial 42 identified environmental time series for that region (Table 2.2), \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values, with location as a factor. We implemented GPDFA using the Stan language (Stan Development Team 2019, \protect\hyperlink{ref-Carpenter2017}{\textbf{Carpenter2017?}}), and R (R Core Development Team 2019, version 3.6.2) via R package rstan (Stan Development Team 2019, version 2.21.2). Code to implement GPDFA is available \href{https://github.com/mfeddern/CSIA-AA/blob/master/SourceData/Src/Analysis/gpdfa.stan}{here}

\#\#Results

\(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values did not vary by sex (p \textgreater{} 0.05, Figure \ref{fig:dist} or size for the individuals sampled (p \textgreater{} 0 .05; Figure \ref{fig:Length}). Spatial variation in harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values were observed on subregional scales. \(\delta^{15}N_{Phe}\) values were similar for harbor seals in the northern Gulf of Alaska (11.9 ± 2.9, mean ± 1SD), southeast Gulf of Alaska (10.8 ± 1.7), and coastal Washington (11.3 ± 1.9). The eastern Bering Sea had significantly higher \(\delta^{15}N_{Phe}\) values compared to other subregions (15.2 ± 1.8) followed by the Salish Sea (12.2 ± 2.3) which had similar \(\delta^{15}N_{Phe}\) values compared to the northern Gulf of Alaska (Figure \ref{fig:dist}, Table \ref{tab:ttestN}). \(\delta^{13}C\) values varied by subregion (p \textless{} 0.05) with the exception of the Gulf of Alaska, where the northern (-14.6 ± 0.9) and southeast (-14.4 ± 1.1) subregions were not significantly different, and the eastern Bering Sea (-13.4 ± 0.9) and coastal Washington (-13.6 ± 0.9) were not significantly different (Figure \ref{fig:dist}, Table \ref{tab:ttestC}). The variation between subregions appeared to follow a latitudinal gradient, where harbor seal mean \(\delta^{13}C\) values were most enriched in \(^{13}C\) in the Salish Sea (-12.2 ± 1.5), became more depleted from coastal Washington and into the Gulf of Alaska (Table \ref{tab:ranges}).

The relationship between harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values also varied on subregional scales. There was positive linear association between harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values in the combined northeast Pacific basin and Bering Sea model with a slope of 0.12 (Figure \ref{fig:hiermod}A). For the hierarchical subregion model, the eastern Bering Sea and coastal Washington demonstrated similar relationship, with slopes of 0.08 (95\% CI {[}0.05, 0.11{]}) and 0.07 (95\% CI {[}0.05, 0.09{]}) respectively. Similarly, harbor seals in both Gulf of Alaska subregions demonstrated comparable coupling of \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\), with slopes of 0.13 (95\% CI {[}0.11, 0.14{]}) for the northern subregion and 0.12 (95\% CI {[}0.10, 0.14{]}) for the southeastern subregion. Salish Sea harbor seals had a distinct relationship between \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values relative to other subregions with a slope of only 0.02 that was not significantly different from 0 (95\% CI {[}0.0, 0.04{]}) (Figure \ref{fig:hiermod}B).

For both \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values there was substantial support for models including environmental indices rather than null or subregion only models. The relationship between environmental indices and harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values in the northeast Pacific varied on regional scales. For Washington, the best model to predict harbor seal \(\delta^{15}N_{Phe}\) values included Columbia River discharge in high flow months, summer upwelling, and subregion. There was substantial model uncertainty for \(\delta^{15}N_{Phe}\) values in the Washington region, however 90\% of model weight supported the inclusion of Columbia River discharge (Figure \ref{fig:coefres}A). The model for harbor seal \(\delta^{13}C\) values with the most support indicated a positive association between PDO, spring upwelling, and freshwater discharge in the Washington region (Figure \ref{fig:coefres}B). In the Gulf of Alaska, the summer upwelling model had the most support as a predictor of harbor seal \(\delta^{15}N_{Phe}\) values with some model support for inclusion of the NPGO (North Pacific Gyre Oscillation), although the coefficients for this covariate did not differ substantially from 0 (Figure \ref{fig:coefres}C). The best model for harbor seal \(\delta^{13}C\) values for the Gulf of Alaska included subregion, PDO (Pacific Decadal Oscillation), and NPGO (Figure \ref{fig:coefres}D). In contrast to Washington, the Gulf of Alaska models supported a negative association between δ13C values and PDO. The null model for \(\delta^{15}N_{Phe}\) values in the eastern Bering Sea had the most support (Figure \ref{fig:coefres}E). Lack of model support for environmental covariates in the eastern Bering Sea may have been a result of the small sample size in the region. Cross-shelf wind was included as a predictor in the best model (Figure \ref{fig:coefres}F) for \(\delta^{13}C\) values in the eastern Bering Sea and was supported by 76\% of the model weight.

PDO and Kuskokwim river discharge during high flow months were found to be highly collinear (VIF \textgreater{} 10) and PDO was omitted from the candidate model set for the eastern Bering Sea analysis. All other models containing multiple environmental predictors with relative support had variance inflation factors of less than 2 indicating only moderate collinearity across covariates. Model residuals for the best models did not show trends through time (Figure \ref{fig:linresid}). This indicates that there were no trends associated with other potential ecosystem changes, such as harbor seal foraging strategy for example, after accounting for ocean condition. Model results did not change when using \(\delta^{13}C\) data that were not corrected for the regional Seuss Effect.

The GPDFA analysis showed temporal synchronies and shared trends across environmental conditions and stable isotope values in the northeast Pacific. In the Gulf of Alaska, the data supported three latent trends (Figure \ref{fig:GDFAAK}). Both \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values had the highest loadings for trend 1, which showed an increase starting in 1965 until 1980 followed by the trend oscillating at approximately 25\% above the long-term average. The harbor seal \(\delta^{15}N_{Phe}\) values for the southeast subregion, harbor seal \(\delta^{13}C\) values, and spring upwelling had negative loadings on trend 1; loadings of \(\delta^{15}N_{Phe}\) values were generally weaker relative to loadings of \(\delta^{13}C\) values. For the other two trends (2-3), loadings were clustered by environmental driver category. Latent trend 2 oscillated around the long-term average and was uninformative. Trend 3 was below average starting in 1985 with strong loadings for climate time series, spring and summer upwelling, and discharge in high flow months (Figure \ref{fig:GDFAAK}). Annual discharge, autumn upwelling, Oceanic Niño Index and Northern Oscillation Index did not demonstrate strong loadings for any trend. In Washington, there was support for two latent trends. Latent trend 1 shows a rapid increase in the 1940's to 25\% above the long-term mean then a gradual decline until 1986 to approximately 40\% below the long-term mean, with values below the mean starting in 1977 (Figure \ref{fig:GDFAWA}). Trend loadings for harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values were stronger for coastal seals and trend 1 had stronger loadings for freshwater discharge than trend 2. Trend 2 had strong loadings for \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values for both Salish Sea and coastal Washington harbor seals. Trend 2 oscillated above and below the long-term mean and had large loadings for sea surface temperature, summer upwelling, Fraser River discharge and climate indices (Figure \ref{fig:GDFAWA}).

To validate this approach of applying a 1-year lag, 0- and 2-year lags were also applied to the best models of each region an compared to the 1-year lag using AIC. Additionally, month was tested as a smoothed predictor with 12 knots for stable isotope data in Washington samples using a generalized additive model (GAM). Support for a significant smoothing term would identify and seasonality in the data, which would be expected if tissue turnover time is less than a year. A one-year tissue turnover time was confirmed as a suitable assumption for harbor seal bone collagen, as 0- and 2- year lags had similar or less model support. There was no support for a smoothing effect by month in generalized additive models of \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) which would have indicated any seasonal variability in isotope composition and thus a turnover time of less than or greater than a year (p \textless{} 0.05; Figure \ref{fig:Length}) and thus a 1-year lag was applied to isotope data for all temporal analyses.

\#\#Discussion

We analyzed bone collagen \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values from harbor seal museum specimens collected between 1928 and 2014 as indices of change in food web assimilated nitrogen and carbon. Based on previous research (\protect\hyperlink{ref-Sherwood2014}{\textbf{Sherwood2014?}}; \protect\hyperlink{ref-delaVega2019}{\textbf{delaVega2019?}}; \protect\hyperlink{ref-Lorrain2020}{\textbf{Lorrain2020?}}), we interpret \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values as primarily representing nitrogen and carbon resource utilization, and growth and community composition of primary producers at the base of the food web. Our data show the relationship between indices of primary production and nitrogen resources assimilated into food webs varies regionally across the northeast Pacific. By pairing these data with environmental time series data, we provide new insights into large scale environmental forcing that impacts the base of the food web and is transferred to higher trophic levels. Specifically, oceanic conditions associated with climate regimes and upwelling explain significant temporal variation in \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values of coastal predators in northeast Pacific (Figure \ref{fig:coefres}; Figure \ref{fig:linresid}). This analysis demonstrates \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values are useful indicators of resources assimilated by coastal food webs.

\#\#\#Spatial variation in stable isotope indices

The geographically widespread association between harbor seal \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values indicates food web assimilated primary production is coupled with nitrogen resources in most regions of the northeast Pacific, with the Salish Sea as a notable exception (Figure \ref{fig:hiermod}). Short-term studies in coastal Washington showed phytoplankton respond considerably to nitrogen inputs and are frequently nitrogen limited (\protect\hyperlink{ref-Dortch1989}{\textbf{Dortch1989?}}; \protect\hyperlink{ref-Kudela2009}{\textbf{Kudela2009?}}). Similarly, short term studies of the inner Gulf of Alaska shelf demonstrated primary production is generally nitrogen limited, and size, growth rates, and community composition are all tightly coupled with nutrient availability (\protect\hyperlink{ref-Strom2006}{\textbf{Strom2006?}}). A significant relationship between bulk \(\delta^{15}N\) and \(\delta^{13}C\) values was also observed in the tissues of some gorgonian corals over the same time period in coastal Gulf of Alaska (\protect\hyperlink{ref-Williams2007}{\textbf{Williams2007?}}). Given the evidence of nitrogen limitations and its relationship with phytoplankton growth and community composition in these coastal environments, the association between \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values could be the result of nitrogen limiting growth at the base of the food web. Alternatively, the \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) coupling could be driven by covariance with an untested environmental variable that impacts most of the northeast Pacific but not the Salish Sea.

The coastal Washington and the Salish Sea food webs assimilate different nitrogen and carbon sources (Figure \ref{fig:coefres} A \& B). Salish Sea harbor seals have higher \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values compared to individuals on the outer coast, which is likely due to significant contributions of intertidal producers and the legacy of anthropogenic N in the Salish Sea food web. Intertidal macrophytes (seagrass and algae) have similar δ13C values (\textasciitilde{} -10‰) compared to harbor seals in the Salish Sea, while other potential sources are much lower (i.e., marine derived sources \textasciitilde{} -20‰, terrestrial derived sources \textasciitilde{} -30‰) (\protect\hyperlink{ref-Conway2015}{\textbf{Conway2015?}}; \protect\hyperlink{ref-Howe2015}{\textbf{Howe2015?}}). Incorporation of intertidal producers into the Salish Sea food web explains the difference in carbon stable isotope signatures between Salish Sea and coastal Washington harbor seals (\textasciitilde1.4‰, Figure \ref{fig:coefres}A). However, it does not explain the higher \(\delta^{15}N_{Phe}\) values (Figure \ref{fig:coefres}B, Table \ref{tab:ranges}). Surface nitrate was observed to be 8‰ -- 12‰ off the coast of Washington in spring 1993 (\protect\hyperlink{ref-Wu1997}{\textbf{Wu1997?}}) which was exceeded by harbor seals in both coastal Washington and the Salish Sea (Table \ref{tab:ranges}). It is likely anthropogenically derived nitrogen sources contribute to the higher observed \(\delta^{15}N_{Phe}\) values both directly and indirectly, particularly in the Salish Sea where harbor seal \(\delta^{15}N_{Phe}\) values were up to 2.4‰ higher than coastal Washington seals. Wastewater treatment facilities and agriculture runoff contribute substantial amounts (\textasciitilde32\%) of nitrogen in the Salish Sea (Mohamedali et al.~2011) and are enriched in \(^{15}N\). In recent decades, Salish Sea waters have also been characterized by low dissolved oxygen and hypoxic events (PSEMP 2019) from human derived nitrogen loading. Anoxic conditions are conducive to denitrification, another potential indirect source of \(^{15}N\) from human activities in the region.

\#\#\#Ocean condition and stable isotope indices

Washington state food webs exhibit environmentally induced changes in assimilated primary production and nitrogen sources. The isotope-ocean condition relationship in the region can be explained by introduction of terrestrial derived nutrients and climatically induced changes in phytoplankton community structure observed in previous studies (\protect\hyperlink{ref-Du2015}{\textbf{Du2015?}}; \protect\hyperlink{ref-Du2014}{\textbf{Du2014?}}; \protect\hyperlink{ref-Kudela2008}{\textbf{Kudela2008?}}). For example, the PDO has been associated with phytoplankton community shifts between dinoflagellates and diatoms in the northern California Current (\protect\hyperlink{ref-Du2015}{\textbf{Du2015?}}). Similarly, the phytoplankton community composition is distinct in the early (spring) upwelling season compared to the late (summer) upwelling season (\protect\hyperlink{ref-Du2014}{\textbf{Du2014?}}). This could explain the inversely related associations between bulk \(\delta^{13}C\) values and summer and spring upwelling (Figure \ref{fig:coefres}B). Shifts in phytoplankton community structure are therefore a mechanism to explain the relationship between harbor seal bulk \(\delta^{13}C\) values and ocean condition. In addition, freshwater discharge explains 16\% of variation observed in both \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values in Washington. The Columbia River Plume introduces terrestrial derived nutrients, including nitrogen, and has been associated with increased primary production and fish production (\protect\hyperlink{ref-Kudela2008}{\textbf{Kudela2008?}}). The covariation between \(\delta^{15}N_{Phe}\), bulk \(\delta^{13}C\), and discharge indicates isotopically distinct nitrogen resources introduced by freshwater discharge alters primary production which is then assimilated into the Washington food web, and ultimately harbor seals.

In the eastern Bering Sea, our results suggest ice-born algae and \(^{15}N\) enriched nitrogen from the inner shelf are important for supporting the coastal food web. Recent evidence supports that consumer \(\delta^{15}N_{Phe}\) values reflect nitrate \(\delta^{15}N\) values in the arctic (de la Vega et al.~2020). However, our \(\delta^{15}N_{Phe}\) values from harbor seals of the eastern Bering Sea were high relative to previous studies of summer nitrate (5 to 9‰) (\protect\hyperlink{ref-Lehmann2005}{\textbf{Lehmann2005?}}) and plankton nitrogen isotope signatures (6-12‰) (\protect\hyperlink{ref-Smith2002}{\textbf{Smith2002?}}) from the outer and mid Bering Sea shelf. (\protect\hyperlink{ref-Morales2014}{\textbf{Morales2014?}}) subsequently found the stable isotope composition of nitrogen in diatoms ranged from 5-21‰ in late winter and early spring. These values also increased in association to sea ice with a positive shoreward gradient (\protect\hyperlink{ref-Morales2014}{\textbf{Morales2014?}}). The range of sea-ice algae \(\delta^{15}N\) values observed by (\protect\hyperlink{ref-Morales2014}{\textbf{Morales2014?}}) are consistent with our observed \(\delta^{15}N_{Phe}\) values in harbors seals (\ref{tab:ranges}). Furthermore, the harbor seals in this study were located near the inner shelf in an area that has been partially covered by sea ice from January to May during the past century (\protect\hyperlink{ref-Stabeno2007}{\textbf{Stabeno2007?}}). Together this indicates ice algae as a significant contributor to the coastal food web. The disconnect between the \(\delta^{15}N\) values of offshore nitrate (\protect\hyperlink{ref-Lehmann2005}{\textbf{Lehmann2005?}}) and harbor seals also highlights the problem in assuming spatially and temporally discrete nitrate or phytoplankton measurements are representative of resources utilized by, and assimilated into, coastal food webs. Consumer \(\delta^{15}N_{Phe}\) measurements by their nature represent the N assimilated into the food web and integrated over relatively long time scales, while discrete measurements of nitrate may be spatially or temporally biased.

A short term (1998-2011) study of abiotic drivers in the Gulf of Alaska found chlorophyll-a anomalies were positive when downwelling favorable winds were low and had a negative relationship with sea level (\protect\hyperlink{ref-Waite2013}{\textbf{Waite2013?}}). Similarly, (\protect\hyperlink{ref-Espinasse2020}{\textbf{Espinasse2020?}}) found chlorophyll-a, SST, and sea level anomalies were the best predictors of carbon and nitrogen isotope data for secondary consumers over the past two decades. Our results agree with these studies as NPGO (an index of sea level) is negatively associated with both harbor seal \(\delta^{15}N_{Phe}\) (Figure \ref{fig:coefres}C) and \(\delta^{13}C\) values (Figure \ref{fig:coefres} D) in the Gulf of Alaska. Similarly, summer upwelling is positively associated with our \(\delta^{15}N_{Phe}\) values (Figure \ref{fig:coefres}C). Based on our results, these environmentally induced changes represent long-term ecosystem dynamics that extend beyond merely the base of the food web and ultimately impact resources assimilated by top predators. In addition, regional climate indices characterize nutrient and primary production assimilated annually into the food web better than sea surface temperature data alone. It is possible that other untested abiotic factors such as cross-shelf exchanges via eddy propagation or local wind stress (\protect\hyperlink{ref-Waite2013}{\textbf{Waite2013?}}) may be important to food web assimilated nitrogen and primary production in the Gulf of Alaska. Regardless, local variability in upwelling and basin scale indices of sea surface height and temperature (i.e., NPGO) ultimately determine resource assimilation in Gulf of Alaska food web in which harbor seals forage.

By comparing consumer stable isotope values against environmental covariates across multiple sub basins we show environmental forcing on coastal food webs is regionally distinct. For example, climate indices (i.e., PDO) in the Gulf of Alaska were inversely associated with food web-assimilated primary production (Figure \ref{fig:coefres}D, Figure \ref{fig:GDFAAK} Trend 1-2) and positively associated in Washington (Figure \ref{fig:coefres}B, Figure \ref{fig:GDFAWA} Trends 1-2). This agrees with previous studies where the Pacific Decadal Oscillation has been associated with alternating salmon production in the northeast Pacific (\protect\hyperlink{ref-Mantua1997}{\textbf{Mantua1997?}}). In cool phase years (i.e., 1947-1977), Washington stocks experience above average production and Alaska stocks experience below average production. Our results show that \(\delta^{13}C\) values for Washington and Gulf of Alaska also indicate alternating primary production between the two regions in association with PDO. Surprisingly, \(\delta^{13}C\) values are higher in cool phase years for the Gulf of Alaska (Figure \ref{fig:coefres}D) and lower in cool phase years for Washington (Figure \ref{fig:coefres}B). This suggests there is lower phytoplankton growth in Washington and higher in Gulf of Alaska in cool phase years. This is contrary to results of previous studies, assuming 1) higher \(\delta^{13}C\) values represent higher growth rates and 2) PDO is inhibiting growth at the base of the food web and indirectly constraining higher trophic levels such as salmon (\protect\hyperlink{ref-Mantua1997}{\textbf{Mantua1997?}}). It is likely the relationship between PDO, salmon production, and \(\delta^{13}C\) values of harbors seals is instead caused by phytoplankton community structure constraining higher trophic levels rather than growth.

Common temporal trends in harbor seal stable isotopes and ocean condition empirically derived from the GPDFA analysis (Figures 6 \& 7) show changes in biogeochemical cycling and food web-assimilated production in recent decades that are associated with climatic variables. Since 1975, shared trends in environmental time series and stable isotope data in the Gulf of Alaska are above average for temperature, discharge, and NPGO and below average for assimilated \(\delta^{13}C\) values (as indicated by its negative loadings; Figure \ref{fig:GDFAAK}). Trends 2 and 3 in the Gulf of Alaska (Figure \ref{fig:GDFAAK}) show a distinct change in environmental indices starting in 1988. Loadings on these trends were higher for environmental indices than stable isotope data, suggesting a decoupling of environment-food web relationship in the region starting around 1988, which has also been observed between climate regimes and fish species (\protect\hyperlink{ref-Litzow2020}{\textbf{Litzow2020?}}). This environmental-food web decoupling was not observed in Washington (Figure \ref{fig:GDFAWA}) in our study or others (\protect\hyperlink{ref-Litzow2020}{\textbf{Litzow2020?}}).

\#\#\#Using stable isotopes as food web indicators

Previous research has shown lower trophic levels are sensitive to environmental variation in bottom-up drivers of productivity (\protect\hyperlink{ref-Frank2015}{\textbf{Frank2015?}}; \protect\hyperlink{ref-Jennings2010}{\textbf{Jennings2010?}}), but few studies have demonstrated how the impact of these changes span entire food webs on long time scales. By applying CSSIA to museum specimens of a generalist predators, we provide a novel piece of the ecological puzzle not previously available. First, these data provide a measure of changing nitrogen resources and phytoplankton dynamics that are spatially and temporally integrated for food web resource assimilation, rather than measuring the availability of inorganic nutrients or lower trophic level biomass and assuming an associated food web response. Dominant species of marine zooplankton exhibit selective foraging, particularly when resources are highly available (\protect\hyperlink{ref-Bi2020}{\textbf{Bi2020?}}; \protect\hyperlink{ref-Jungbluth2017}{\textbf{Jungbluth2017?}}; \protect\hyperlink{ref-Boersma2015}{\textbf{Boersma2015?}}) thus discrete measures of resources are not necessarily representative of what is utilized by the food web. Second, studies directly measuring primary production are often temporally limited to short time scales and recent decades. CSSIA of historic specimens allows for retrospective analyses that span long time scales (\protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}; \protect\hyperlink{ref-Sherwood2011}{\textbf{Sherwood2011?}}) and thus identify long-term environmental forcing on food webs.

Despite these benefits, CSSIA (and stable isotope analysis data more generally) is limited in its ability to discern different mechanistic processes for isotopic enrichment in observational studies. Multiple mechanisms of fractionation often operate in tandem (Figure 2.1) and can be both additive and subtractive. For example, both the isotopic composition of dissolved inorganic nitrogen sources (primarily \(NO_3^-\), but also urea and \(NH_4^+\)) and the relative uptake of these sources impact the isotopic composition of nitrogen in primary producers (\protect\hyperlink{ref-Graham2010}{\textbf{Graham2010?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}). As a result, these data on their own are limited in their ability to track exact mechanisms of fractionation and specific biogeochemical changes through time or space. Regardless, stable isotope signatures of nitrogen from source amino acids and bulk carbon can be used to trace variations in nitrogen sources at the base of the food web (\protect\hyperlink{ref-Sherwood2014}{\textbf{Sherwood2014?}}; \protect\hyperlink{ref-delaVega2019}{\textbf{delaVega2019?}}) and changes in phytoplankton dynamics (i.e., production) (\protect\hyperlink{ref-delaVega2019}{\textbf{delaVega2019?}}; \protect\hyperlink{ref-Lorrain2020}{\textbf{Lorrain2020?}}) broadly. In addition, CSSIA of carbon is also emerging as reliable proxy for phytoplankton community composition (\protect\hyperlink{ref-Larsen2013}{\textbf{Larsen2013?}}). We also assume a constant and small trophic enrichment factor for both bulk \(\delta^{13}C\) and \(\delta^{15}N_{Phe}\) values. While trophic enrichment in \(\delta^{13}C\) and \(\delta^{15}N_{Phe}\) values is minimal (\protect\hyperlink{ref-Hobson1996}{\textbf{Hobson1996?}}; \protect\hyperlink{ref-Bocherens2003}{\textbf{Bocherens2003?}}; \protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}), and thus unlikely to impact overall correlations between datasets, it can produce enriched absolute isotope values and increased variation between observations, which was not accounted for in this study. Nonetheless, ours is among a number of supporting studies that show food webs are impacted by changing environmental conditions in the northeast Pacific (\protect\hyperlink{ref-Cunningham2018}{\textbf{Cunningham2018?}}; \protect\hyperlink{ref-Puerta2019}{\textbf{Puerta2019?}}; \protect\hyperlink{ref-Stachura2014}{\textbf{Stachura2014?}}).

Climate change will alter nutrient distributions and primary production throughout the worlds' oceans (\protect\hyperlink{ref-Marinov2010}{\textbf{Marinov2010?}}; \protect\hyperlink{ref-Kwiatkowski2017}{\textbf{Kwiatkowski2017?}}). Based on analysis of historical patterns of consumer isotopic variation with environmental forcing, we anticipate there will be region-specific spatial variability in how primary production and its dependent food webs respond to environmental change throughout the northeast Pacific over the next century. As environmental conditions (i.e., sea surface temperature, discharge, anthropogenic nitrogen) continue to change, so will resources available to and assimilated by food webs. Given both resource availability and community composition of resources impact the function and stability of food webs (\protect\hyperlink{ref-Narwani2012}{\textbf{Narwani2012?}}) it is likely that ecosystem interactions will change in response to environmentally induced shifts in resources. Understanding dynamics influencing food web responses to their environment is important, as it provides information useful for predicting climate change impacts to aquatic resources and the communities and economies that depend on them.

\clearpage

\#\#Tables

\textbf{Table} \ref{tab:ranges}: Range of \(\delta^{13}C\) and \(\delta^{15}N_{Phe}\) values observed in harbor seals for each of the five northeast Pacific subregions.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}>{}p{10em}}
\caption{\label{tab:ranges}Ranges of stable isotope data}\\
\toprule
X & δ15NPhe     & δ13C    \\
\midrule
Coastal WA & 6.0 – 15.8 & -15.6 – -11.8\\
Salish Sea & 5.9 – 18.2 & -16.6 – -6.8\\
Northern Gulf of Alaska & 6.2  – 21.5 & -16.7 – -12.5\\
Southeast Gulf of Alaska & 8.0 – 15.2 & -17.3 – -12.1\\
Eastern Bering Sea & 12.4 – 18.9 & -15.0 – -12.1\\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage
\begin{landscape}
Table 2.2: Environmental datasets. SST data was obtained from NOAA ERSST V5 data provided by the NOAA/OAR/ESRL PSD, Boulder, Colorado, USA, from their Web site at https://www.esrl.noaa.gov/psd/ 


\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{20em}>{\raggedright\arraybackslash}p{20em}>{\raggedright\arraybackslash}p{20em}}
\caption{\label{tab:envvar}Environmental Datasets}\\
\toprule
Environmental Driver Category & Eastern Bering Sea & Gulf of Alaska & Washington\\
\midrule
Discharge & Total discharge from the Kuskokwim River at Crooked Creek, AK during the winter months of low discharge (Nov-Apr) and summer months of high discharge (May-Oct) from monthly U.S. Geological Survey discharge data. 1951-2018. N = 3 & Estimates of total freshwater discharge for a location near Seward, Alaska during winter months of low discharge (Jan-Jul) and summer months of high discharge (Aug-Dec) from monthly data. 1931-2011. N= 3. & Total discharge from the Columbia River at Dalles, WA and Fraser River at Hope during the winter months of low discharge (Nov-Apr) and summer months of high discharge (May-Oct) from monthly U.S. Geological Survey discharge data. 1879-2018 and 1913-2016. N= 6.\\
 & Data Source: USGS 15304000 & Data Source: Tom Royer, Royer and Grosch 2007 & Data Source: USGS 14105700; BC Fraser 08MF005\\
Sea Surface Temperature (SST) & Average of monthly NOAA Extended Reconstructed SST for winter (Jan-Mar), spring (Apr-Jun), summer (Jul-Sep), and fall (Oct-Dec) and annually at 60°N, 170°W. 1854-2019. N = 5 & Average of monthly NOAA Extended Reconstructed SST for winter (Jan-Mar), spring (Apr-Jun), summer (Jul-Sep), and fall (Oct-Dec) and annually in southcentral AK (60°N 149°W). 1854-2019. N = 5 & Average of monthly NOAA Extended Reconstructed SST for winter (Jan-Mar), spring (Apr-Jun), summer (Jul-Sep), and fall (Oct-Dec) and annually in coastal Washington (48°N, 125°W). 1854-2019. N=5\\
 & Data Source: NOAA ERSST V5 & Data Source: NOAA ERSST V5 & Data Source: NOAA ERSST V5\\
Upwelling/Circulation & Average winter (Oct-Apr) cross-shelf and along-shelf wind at 60°N, 170°W from monthly NCEP/NCAR reanalysis data. 1949-2011. N = 2 & Mean coastal upwelling index (CUI) the Gulf of AK (60°N, 149°W and 60°N, 147°W) using Bakun upwelling calculation based on Ekman's theory of mass transport due to wind stress, for spring and  summer. 1946-2019.  N = 4 & Mean coastal upwelling index (CUI) coastal Washington (45°N, 125°W) using Bakun upwelling calculation based on Ekman's theory of mass transport due to wind stress, for spring, summer, winter and annual. 1946-2019.\\
\addlinespace
 & Data Source: Megan Stachura, Stachura et al. 2014 from NOAA ESRL & Data Source: NOAA ERD SWFSC & Data Source: NOAA ERD SWFSC\\
 &  &  & Total upwelling magnitude Index (TUMI, 45°N). 1965-2019.\\
 &  &  & Data Source: NOAA CCIEA\\
Climate Regime & Multivariate ENSO Index (1950-2019), Oceanic Nino Index (1950-2019), Pacific Decadal Oscillation Index (1900-2018), the Northern Oscillation Index (1928-2019), North Pacific Gyre Oscillation (1950-2019). N = 5 & Same as eastern Bering Sea & Same as eastern Bering Sea\\
 & Data Sources: PDO; NPGO; NOI; MEI; ONI &  & \\
\bottomrule
\end{longtable}
\endgroup{}
\end{landscape}
\textbf{Table} \ref{tab:reduced}: Reduced time series for linear models by regions.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{15em}>{\raggedright\arraybackslash}p{15em}>{\raggedright\arraybackslash}p{15em}}
\caption{\label{tab:reduced}Reduced Time Series}\\
\toprule
Mechanism & Washington & Gulf of Alaska & Eastern Bering Sea\\
\midrule
Climate Regime & Pacific Decadal Oscillation & Pacific Decadal Oscillation & Pacific Decadal Oscillation\\
Climate Regime & Multivariate ENSO Index & Multivariate ENSO Index & Multivariate ENSO Index\\
Climate Regime & North Pacific Gyre Oscillation & North Pacific Gyre Oscillation & North Pacific Gyre Oscillation\\
Temperature & Mean summer sea surface temperature (Jul-Sep) at 48°N, 125°W & Mean summer sea surface temperature (Jul-Sep) at 60°N 149°W & Mean summer sea surface temperature (Jul-Sep) at 60°N, 170°W\\
Circulation & Mean summer coastal upwelling (Jul-Sep) at 48°N, 125°W & Mean summer coastal upwelling (Jul-Sep) at 60°N 149°W & Mean winter (Oct-Apr) cross-shelf wind vector at 60°N, 170°W\\
\addlinespace
Circulation & Mean Coastal Upwelling (Spring) & Mean Coastal Upwelling (Spring) & Average winter (Oct-Apr) along-shelf wind vector at 60°N, 170°W\\
Discharge & Columbia River Discharge during summer months of high discharge (May-Oct) & Total freshwater discharge for a location near Seward during summer months of high discharge (May-Oct) & Total discharge from the Kuskokwim River at Crooked Creek during summer months of high discharge (May-Oct)\\
 &  &  & \\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage

\textbf{Table} \ref{tab:ttestN}: Pairwise t-test by sub region with bonferroni correction and pooled standard deviation for \(\delta^{15}N_{Phe}\).
\begin{verbatim}
Warning in read.table(file = file, header = header, sep = sep, quote = quote, :
incomplete final line found by readTableHeader on 'data/Ch2/Table4.csv'
\end{verbatim}
\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}l}
\caption{\label{tab:ttestN}Nitrogen Phenylalanine T-Test}\\
\toprule
X & Eastern Bering Sea & Coastal WA & Salish Sea & Southcentral GoA\\
\midrule
Coastal WA & p < 0.05* & - & - & -\\
Salish Sea & p < 0.05* & p < 0.05* & - & -\\
Southcentral GoA & p < 0.05* & 0.93 & 1 & -\\
Southeast GoA & p < 0.05* & 1 & p < 0.05* & 0.28\\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage

\textbf{Table} \ref{tab:ttestC}: Pairwise t-test by sub region with bonferroni correction and pooled standard deviation for \(\delta^{13}C\).
\begin{verbatim}
Warning in read.table(file = file, header = header, sep = sep, quote = quote, :
incomplete final line found by readTableHeader on 'data/Ch2/Table5.csv'
\end{verbatim}
\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}l}
\caption{\label{tab:ttestC}Bulk Carbon T-Test}\\
\toprule
X & Eastern Bering Sea & Coastal WA & Salish Sea & Southcentral GoA\\
\midrule
Coastal WA & 1 & - & - & -\\
Salish Sea & p < 0.05* & p < 0.05* & - & -\\
Southcentral GoA & p < 0.05* & p < 0.05* & p < 0.05* & -\\
Southeast GoA & p < 0.05* & p < 0.05* & p < 0.05* & 1\\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage

\#\#Figures
\begin{landscape}

Figure 2.1: Mechanisms of environmentally induced changes in resources (A-D) assimilated into stable isotope ratios of primary producers (1-2), which are conserved when assimilated into higher trophic levels in the food web (3).\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=1.1\textwidth]{figure/Ch2/Figure1.pdf}
  \caption{Mechanisms of stable isotope change}
  \label{fig:theo}
\end{figure}
\end{landscape}
\clearpage

\textbf{Figure} \ref{fig:map}: Spatial and temporal distributions of northeast Pacific harbor seal specimens by subregion analyzed for \(\delta^{15}N_{Phe}\) and bulk \(\delta^{13}C\) values. Subplot colors correspond to map locations and x-axis (years) is the same for each subplot.\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=1.2\textwidth]{figure/Ch2/Fig2_Map.pdf}
  \caption{Distribution of harbor seal specimens}
  \label{fig:map}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:dist}: Variability in \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values based on sub region and sex. * denotes a significant difference in isotopic signature between males and females for that region (colors correspond to \ref{fig:map}).
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch2/Figure3_SexLocation.pdf}
  \caption{Distribution of harbor seal specimens}
  \label{fig:dist}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:hiermod}: Relationship between nitrogen sources (\(\delta^{15}N_{Phe}\)) and primary production (\(\delta^{13}C\)) assimilated into the food web for A. a single linear model for the combined data across the northeast Pacific and eastern Bering Sea and B. a mixed effects model with random slope and intercept by sub region (colors correspond to \ref{fig:map}).
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=1\textwidth]{figure/Ch2/hiermod.pdf}
  \caption{Hierarchical $\delta^{15}N_{Phe}$ and $\delta^{13}C$ Models}
  \label{fig:hiermod}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:coefres}: Coefficients of environmental covariates for models with relative support (\(\Delta AIC_c\) \textless{} 2) for harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values in three regions of the northeast Pacific: Washington, Gulf of Alaska, and the eastern Bering Sea. Color indicates model support based on AIC\textsubscript{c} weight, points are the coefficient estimates for each environmental covariate included in an individual model, and bars show two standard deviations from the coefficient estimate.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=0.9\textwidth]{figure/Ch2/Figure5_CoefPlot.pdf}
  \caption{Coefficients of Evironmental Covariates}
  \label{fig:coefres}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:GDFAAK}: Common trends in environmental condition and food web assimilated stable isotope values for the regional Gulf of Alaska gaussian-dynamic factor analysis model. The solid lines represent the modelled trends, where 0 is the long-term average and 1 and -1 represent the maximum and minimum possible values respectively; the dash line is the 90\% credible interval. Factor loadings can be interpreted as coefficients, representing the strength of association between the modelled trend and each observed environmental time series (colors represent a priori driver category). Values close to 0 mean the observed time series did not correlate to the corresponding trend, while values close to 1 show the observed time series closely matched the modelled trend. Negative loadings indicate an inverse relationship between the observed time series and modelled trend. Stable isotope times series are modelled separately for the northern (N. \(\delta^{15}N_{Phe}\); N. \(\delta^{13}C\)) and southeast (S. \(\delta^{15}N_{Phe}\); S. \(\delta^{13}C\)) subregions.\\
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=0.9\textwidth]{figure/Ch2/Figure6_AK.GDFA.pdf}
  \caption{Alaska GDFA Results}
  \label{fig:GDFAAK}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:GDFAWA}: Common trends in environmental condition and food web assimilated stable isotope values for the regional Washington gaussian-dynamic factor analysis model. Stable isotope times series are modelled separately for the coastal (C. \(\delta^{15}N_{Phe}\); C. \(\delta^{13}C\)) and Salish Sea (S.S. \(\delta^{15}N_{Phe}\); S.S. \(\delta^{13}C\)) subregions. See \ref{fig:GDFAAK} caption for further interpretation.\\
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=0.9\textwidth]{figure/Ch2/Figure7_WA.GDFA.pdf}
  \caption{Washington GDFA Results}
  \label{fig:GDFAWA}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:Month}: Analysis of a) \(\delta^{15}N_{Phe}\) and b) \(\delta^{13}C\) values by month. For both models, s(month) p \textgreater{} 0.1 indicating no seasonality of harbor seal bone collagen stable isotope signature.\\
\newline 
\begin{figure}[h]
  \centering
  \includegraphics[width=0.8\textwidth]{figure/Ch2/figures2.png}
  \caption{Bone collagen stable isotope seasonality analysis}
  \label{fig:Month}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:Length}: Analysis of a) \(\delta^{15}N_{Phe}\) and b) \(\delta^{13}C\) values by length. For both models, there was no significant slope (p\textgreater0.1)
\newline 
\begin{figure}[h]
\centering
  \includegraphics[width=0.8\textwidth]{figure/Ch2/FigureS2.pdf}
  \caption{Bone collagen stable isotope length analysis}
  \label{fig:Length}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:TSphe}: Bone collagen \(\delta^{15}N\) values of phenylalanine from archival harbor seal specimens collected in the northeastern Pacific in five subregions.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.8\textwidth]{figure/Ch2/FigureS3.pdf}
  \caption{Time series of $\delta^{15}N_{Phe}$ data}
  \label{fig:TSphe}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:nonSuess}: Bone collagen bulk \(\delta^{13}C\) values of archival harbor seal specimens collected in the northeastern Pacific in five subregions.These values are not corrected for the Suess effect.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.8\textwidth]{figure/Ch2/FigureS4.pdf}
  \caption{Time series of $\delta^{13}C$ data}
  \label{fig:nonSuess}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:Suess}: Bone collagen bulk \(\delta^{13}C\) values of archival harbor seal specimens collected in the northeastern Pacific in five subregions.These values are corrected for the Seuss effect.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.8\textwidth]{figure/Ch2/FigureS5.pdf}
  \caption{Time series of $\delta^{13}C$ data corrected for Suess effect}
  \label{fig:Suess}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:bulkN}: Bone collagen bulk \(\delta^{15}N\) values of archival harbor seal specimens collected in the
northeastern Pacific in five subregions.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.8\textwidth]{figure/Ch2/FigureS6.pdf}
  \caption{Time series of bulk $\delta^{15}N$ data}
  \label{fig:bulkN}
\end{figure}
\clearpage

\textbf{Figure} \ref{fig:linresid}: Residuals for the model with the most support (\ref{fig:coefres}) plotted by year. A trend in model residuals would indicate environmental variables do not account for all temporal variation in harbor seal \(\delta^{15}N_{Phe}\) and \(\delta^{13}C\) values.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.85\textwidth]{figure/Ch2/FigureS7.pdf}
  \caption{Residuals trends for linear models with the most support}
  \label{fig:linresid}
\end{figure}
\clearpage

Figure 2.15: Model residual plots for the models with the most support from the candidate model set. Note: EBS phenylalanine is an intercept only model, GOA phenylalanine only contains a 2-factor location as a covariate.
\newline 
\begin{figure}[h]
\centering
  \includegraphics[height=0.8\textwidth]{figure/Ch2/FigureS8.pdf}
  \caption{Residuals for linear models with the most support}
  \label{fig:linresid2}
\end{figure}
\hypertarget{ref-labels}{%
\chapter{Tables, Graphics, References, and Labels}\label{ref-labels}}

\hypertarget{tables}{%
\section{Tables}\label{tables}}

By far the easiest way to present tables in your thesis is to store the contents of the table in a CSV or Excel file, then read that file in to your R Markdown document as a data frame. Then you can style the table with the \texttt{kable} function, or functions in the \href{https://cran.r-project.org/web/packages/kableExtra/index.html}{kableExtra} pacakge.

In addition to the tables that can be automatically generated from a data frame in \textbf{R} that you saw in {[}R Markdown Basics{]} using the \texttt{kable} function, you can also create tables using \emph{pandoc}. (More information is available at \url{http://pandoc.org/README.html\#tables}.) This might be useful if you don't have values specifically stored in \textbf{R}, but you'd like to display them in table form. Below is an example. Pay careful attention to the alignment in the table and hyphens to create the rows and columns. Generally I don't recommend this approach of typing the table directly into your R Markdown document.
\begin{longtable}[]{@{}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.31}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.51}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.18}}@{}}
\caption{\label{tab:inher} Correlation of Inheritance Factors for Parents and Child}\tabularnewline
\toprule
Factors & Correlation between Parents \& Child & Inherited \\
\midrule
\endfirsthead
\toprule
Factors & Correlation between Parents \& Child & Inherited \\
\midrule
\endhead
Education & -0.49 & Yes \\
Socio-Economic Status & 0.28 & Slight \\
Income & 0.08 & No \\
Family Size & 0.18 & Slight \\
Occupational Prestige & 0.21 & Slight \\
\bottomrule
\end{longtable}
We can also create a link to the table by doing the following: Table \ref{tab:inher}. If you go back to {[}Loading and exploring data{]} and look at the \texttt{kable} table, we can create a reference to this max delays table too: Table \ref{tab:maxdelays}. The addition of the \texttt{(\textbackslash{}\#tab:inher)} option to the end of the table caption allows us to then make a reference to Table \texttt{\textbackslash{}@ref(tab:label)}. Note that this reference could appear anywhere throughout the document after the table has appeared.

\clearpage

\hypertarget{figures}{%
\section{Figures}\label{figures}}

If your thesis has a lot of figures, \emph{R Markdown} might behave better for you than that other word processor. One perk is that it will automatically number the figures accordingly in each chapter. You'll also be able to create a label for each figure, add a caption, and then reference the figure in a way similar to what we saw with tables earlier. If you label your figures, you can move the figures around and \emph{R Markdown} will automatically adjust the numbering for you. No need for you to remember! So that you don't have to get too far into LaTeX to do this, a couple \textbf{R} functions have been created for you to assist. You'll see their use below.

In the \textbf{R} chunk below, we will load in a picture stored as \texttt{uw.png} in our main directory. We then give it the caption of ``UW logo,'' the label of ``uwlogo,'' and specify that this is a figure. Make note of the different \textbf{R} chunk options that are given in the R Markdown file (not shown in the knitted document).
\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{knitr}\SpecialCharTok{::}\FunctionTok{include\_graphics}\NormalTok{(}\AttributeTok{path =} \StringTok{"figure/uw.png"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}
\begin{figure}
\includegraphics[width=6.25in]{figure/uw} \caption{UW logo}\label{fig:uwlogo}
\end{figure}
Here is a reference to the UW logo: Figure \ref{fig:uwlogo}. Note the use of the \texttt{fig:} code here. By naming the \textbf{R} chunk that contains the figure, we can then reference that figure later as done in the first sentence here. We can also specify the caption for the figure via the R chunk option \texttt{fig.cap}.

\clearpage

Below we will investigate how to save the output of an \textbf{R} plot and label it in a way similar to that done above. Recall the \texttt{flights} dataset from Chapter \ref{rmd-basics}. (Note that we've shown a different way to reference a section or chapter here.) We will next explore a bar graph with the mean flight departure delays by airline from Portland for 2014. Note also the use of the \texttt{scale} parameter which is discussed on the next page.
\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{flights }\SpecialCharTok{\%\textgreater{}\%} \FunctionTok{group\_by}\NormalTok{(carrier) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{summarize}\NormalTok{(}\AttributeTok{mean\_dep\_delay =} \FunctionTok{mean}\NormalTok{(dep\_delay)) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{ggplot}\NormalTok{(}\FunctionTok{aes}\NormalTok{(}\AttributeTok{x =}\NormalTok{ carrier, }\AttributeTok{y =}\NormalTok{ mean\_dep\_delay)) }\SpecialCharTok{+}
  \FunctionTok{geom\_bar}\NormalTok{(}\AttributeTok{position =} \StringTok{"identity"}\NormalTok{, }\AttributeTok{stat =} \StringTok{"identity"}\NormalTok{, }\AttributeTok{fill =} \StringTok{"red"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}
\begin{figure}
\centering
\includegraphics{thesis_files/figure-latex/delaysboxplot-1.pdf}
\caption{\label{fig:delaysboxplot}Mean Delays by Airline}
\end{figure}
Here is a reference to this image: Figure \ref{fig:delaysboxplot}.

A table linking these carrier codes to airline names is available at \url{https://github.com/ismayc/pnwflights14/blob/master/data/airlines.csv}.

\clearpage

Next, we will explore the use of the \texttt{out.extra} chunk option, which can be used to shrink or expand an image loaded from a file by specifying \texttt{"scale=\ "}. Here we use the mathematical graph stored in the ``subdivision.pdf'' file.
\begin{figure}
\includegraphics[scale=0.75]{figure/subdivision} \caption{Subdiv. graph}\label{fig:subd}
\end{figure}
Here is a reference to this image: Figure \ref{fig:subd}. Note that \texttt{echo=FALSE} is specified so that the \textbf{R} code is hidden in the document.

\textbf{More Figure Stuff}

Lastly, we will explore how to rotate and enlarge figures using the \texttt{out.extra} chunk option. (Currently this only works in the PDF version of the book.)
\begin{figure}
\includegraphics[angle=180, scale=1.1]{figure/subdivision} \caption{A Larger Figure, Flipped Upside Down}\label{fig:subd2}
\end{figure}
As another example, here is a reference: Figure \ref{fig:subd2}.

\hypertarget{footnotes-and-endnotes}{%
\section{Footnotes and Endnotes}\label{footnotes-and-endnotes}}

You might want to footnote something.\footnote{footnote text} The footnote will be in a smaller font and placed appropriately. Endnotes work in much the same way.

\hypertarget{cross-referencing-chapters-and-sections}{%
\section{Cross-referencing chapters and sections}\label{cross-referencing-chapters-and-sections}}

The \href{https://bookdown.org/yihui/bookdown/cross-references.html}{bookdown documentation} is an excellent source for learning how to cross-reference in a bookdown project such as a huskydown document. Here we only cover the most common uses for a typical thesis. If you want something more complex or fancy, please refer to the bookdown documentation and seek help from the developers of that package.

By default, all of your chapter and section headers will get an auto-generated ID label For example, e.g., \texttt{\#\ Chapter\ 1} will have an auto-generated ID \texttt{chapter-1}. Note that the ID label is all lower case, and has no spaces. If you have any kind of punctuation in your header, such as a colon (:), it will not appear in the ID label. Then in your text you can reference chapter one in your Rmd file like this: `as discussed in Chapter \texttt{\textbackslash{}@ref(chapter-1)},' which will print as `as discussed in Chapter 1'

We strongly recommend that you to manually assign ID labels to your chapter header to make it easy to cross-reference. For example, at the top of the Rmd file for this chapter, you can see:

\texttt{\#\ Tables,\ Graphics,\ References,\ and\ Labels\ \{\#ref-labels\}}

The \texttt{\{\#ref-labels\}} part of this header is the ID label. It doesn't show in the output, but is there for us to use for easy cross-referencing, because it can be short, and we don't need to change it elsewhere our document when we update the chapter header. We can use this custom ID label in our Rmd document like this: `as discussed in Chapter \texttt{\textbackslash{}@ref(ref-labels)},' which will print as `as discussed in Chapter \ref{ref-labels}.' If you need to show custom text instead of the chapter number, you use this syntax in your Rmd document: \texttt{see\ {[}my\ chapter\ about\ labels{]}(\#ref-labels)\ for\ more\ details} which will appear as `see \protect\hyperlink{ref-labels}{my chapter about labels} for more details'

To cross-reference a specific section in the same chapter, we recommend adding a custom ID label to the section header, and using that to cross-reference. For example, earlier in this chapter we have a section on tables and in the Rmd file we see \texttt{\#\#\ Tables\ \{\#tables\}}. We can cross-reference that in the text like this `as discussed in the section on \texttt{{[}tables{]}(\#tables)}' which will appear as `as discussed in the above section on \protect\hyperlink{tables}{tables}'

To cross-reference a section in a different chapter we can use the ID label from that section directly. For example, we can write in our Rmd document \texttt{as\ discussed\ in\ the\ section\ on\ {[}R\ code\ chunks{]}(\#r-chunks)\ in\ Chapter\ \textbackslash{}@ref(rmd-basics)} which will appear as `as discussed in the section on \protect\hyperlink{r-chunks}{R code chunks} in Chapter \ref{rmd-basics}.'

If you prefer to cross-reference by the section number, we can use custom ID labels in our Rmd document. For example, to refer to a section in our first chapter, we can write in the Rmd document: \texttt{as\ discussed\ in\ section\ \textbackslash{}@ref(r-chunks)\ in\ Chapter\ \textbackslash{}@ref(rmd-basics)}. This will appear with section and chapter numbers like so: as `as discussed in section \ref{r-chunks} in Chapter \ref{rmd-basics}.'

\hypertarget{bibliographies}{%
\section{Bibliographies}\label{bibliographies}}

Of course you will need to cite things, and you will probably accumulate an armful of sources. There are a variety of tools available for creating a bibliography database (stored with the .bib extension). In addition to BibTeX suggested below, you may want to consider using the free and easy-to-use tool called Zotero. Some Zotero documentation is at \url{http://libguides.reed.edu/citation/zotero}. In addition, a tutorial is available from Middlebury College at \url{http://sites.middlebury.edu/zoteromiddlebury/}.

\emph{R Markdown} uses \emph{pandoc} (\url{http://pandoc.org/}) to build its bibliographies. One nice caveat of this is that you won't have to do a second compile to load in references as standard LaTeX requires. To cite references in your thesis (after creating your bibliography database), place the reference name inside square brackets and precede it by the ``at'' symbol. For example, here's a reference to a book about worrying: (\protect\hyperlink{ref-Molina1994}{\textbf{Molina1994?}}). This \texttt{Molina1994} entry appears in a file called \texttt{thesis.bib} in the \texttt{bib} folder. This bibliography database file was created by a program called BibTeX. You can call this file something else if you like (look at the YAML header in the main .Rmd file) and, by default, is to placed in the \texttt{bib} folder.

For more information about BibTeX and bibliographies, see (\url{http://web.reed.edu/cis/help/latex/index.html})\footnote{(\protect\hyperlink{ref-reedweb2007}{\textbf{reedweb2007?}})}. There are three pages on this topic: \emph{bibtex} (which talks about using BibTeX, at \url{http://web.reed.edu/cis/help/latex/bibtex.html}), \emph{bibtexstyles} (about how to find and use the bibliography style that best suits your needs, at \url{http://web.reed.edu/cis/help/latex/bibtexstyles.html}) and \emph{bibman} (which covers how to make and maintain a bibliography by hand, without BibTeX, at \url{http://web.reed.edu/cis/help/latex/bibman.html}). The last page will not be useful unless you have only a few sources.

If you look at the YAML header at the top of the main .Rmd file you can see that we can specify the style of the bibliography by referencing the appropriate csl file. You can download a variety of different style files at \url{https://www.zotero.org/styles}. Make sure to download the file into the csl folder.

\textbf{Tips for Bibliographies}
\begin{itemize}
\tightlist
\item
  Like with thesis formatting, the sooner you start compiling your bibliography for something as large as thesis, the better.
\item
  The cite key (a citation's label) needs to be unique from the other entries.
\item
  When you have more than one author or editor, you need to separate each author's name by the word ``and'' e.g.~\texttt{Author\ =\ \{Noble,\ Sam\ and\ Youngberg,\ Jessica\},}.
\item
  Bibliographies made using BibTeX (whether manually or using a manager) accept LaTeX markup, so you can italicize and add symbols as necessary.
\item
  To force capitalization in an article title or where all lowercase is generally used, bracket the capital letter in curly braces.
\end{itemize}
\hypertarget{anything-else}{%
\section{Anything else?}\label{anything-else}}

If you'd like to see examples of other things in this template, please \href{https://github.com/benmarwick/huskydown/issues/new}{contact us} (email \href{mailto:bmarwick@uw.edu}{\nolinkurl{bmarwick@uw.edu}}) with your suggestions. We love to see people using \emph{R Markdown} for their theses, and are happy to help.

\hypertarget{recent-divergent-changes-in-alaskan-pinniped-trophic-position-detected-using-compound-specific-stable-isotope-analysis}{%
\chapter{Recent divergent changes in Alaskan pinniped trophic position detected using compound-specific stable isotope analysis}\label{recent-divergent-changes-in-alaskan-pinniped-trophic-position-detected-using-compound-specific-stable-isotope-analysis}}

\#\#Abstract

Over the past century Alaskan pinnipeds have experienced dramatic changes in abundance, but these changes have been highly variable across species and region. In recent decades, changes in atmospheric forcing and sea surface temperature have been particularly pronounced in the Gulf of Alaska and eastern Bering Sea, impacting the food webs in which Alaskan pinnipeds forage. We used compound-specific stable isotope analysis of nitrogen in amino acids to estimate historic and modern trophic position of harbor seals and Steller sea lions in the Gulf of Alaska and Bristol Bay. We applied a Bayesian hierarchical framework to determine whether shared trends through time exist across pinnipeds (classified by species and region) on decadal scales. Model results identified both shared trends through time and classification-specific decadal changes in pinniped trophic position. The largest change in trophic position occurred in the 2000s and 2010s and was observed in both Steller sea lions and harbor seals in the Gulf of Alaska, but not harbor seals in Bristol Bay or Iliamna Lake. Divergent trophic position patterns in the 2000s were identified in the western stock of Steller sea lions, which increased in trophic position, and sympatric harbor seals in the northern Gulf of Alaska, which decreased in trophic position. Our results indicate that these species have begun exploiting distinct trophic niches or experiencing unique food web conditions in recent decades in the Gulf of Alaska, likely in response to recent climate-induced ecological change in the region.

\#\#Introduction
Over the past century, pinniped populations in the northeast Pacific Ocean have experienced changes in adult and pup abundances (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}). Understanding specific drivers of these population trends is important for management, as multiple stocks have been listed as threatened or endangered over the past two decades (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}). The observed population dynamics have also corresponded with shifts in both the physical and ecological marine environment, which frequently occur simultaneously. As a result, disentangling drivers of population trends is complex, as multiple factors (environmental conditions, prey availability, anthropogenic disturbances) can change in tandem and potentially act synergistically on pinniped populations.

Data on long-term trends in trophic position across regions, species, and populations is one potential way to assess how food web changes have impacted pinnipeds in Alaska. This approach can identify how broad shifts in foraging ecology correspond to changes in abundance and population dynamics. More specifically, examining trophic position during periods of declining versus increasing predator abundance can provide insight into whether foraging behavior and prey availability are important drivers of population dynamics. In this study, we aim to identify whether common temporal trends in trophic ecology exist across harbor seals (\emph{Phoca vitulina}) and Steller sea lions (\emph{Eumetopias jubatus}) and their locations by deriving 70-years of trophic position data from compound-specific stable isotope analysis (CSIA) of museum specimens.

Following climatic changes in the 1970s that altered ocean currents and sea surface temperature (\protect\hyperlink{ref-Hare2000}{\textbf{Hare2000?}}), most Gulf of Alaska and Bering Sea pinniped populations experienced declines that persisted through the 1990s (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}). However, these responses differed across populations and species. For example, the western stock of Steller sea lions (located west of 144°W) decreased from approximately 240,000 animals in the late 1970s to 50,000 in 2000 (\protect\hyperlink{ref-Burkanov2005}{\textbf{Burkanov2005?}}). Similarly, harbor seal populations in Prince William Sound and Glacier Bay declined by approximately 60\% between the 1980s and 2000 (\protect\hyperlink{ref-Frost1999}{\textbf{Frost1999?}}; \protect\hyperlink{ref-Womble2010}{\textbf{Womble2010?}}). In contrast, the eastern stock of Steller sea lions (located east of 144°W) increased by 3-4\% per year over the same time period (Figure 2) (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}; \protect\hyperlink{ref-Pitcher2007}{\textbf{Pitcher2007?}}). More recently, atmospheric circulation anomalies in the northeast Pacific Ocean have resulted in unprecedently warm sea surface temperatures during the past decade (\protect\hyperlink{ref-Walsh2018}{\textbf{Walsh2018?}}) and this environmental shift has altered fish abundances (\protect\hyperlink{ref-Bond2015}{\textbf{Bond2015?}}; \protect\hyperlink{ref-Litzow2020}{\textbf{Litzow2020?}}). For example, the unprecedented marine heatwave that occurred in 2014 - 2016 triggered dramatic ecosystem change, including a 71\% decline in Pacific cod in the Gulf of Alaska (\protect\hyperlink{ref-Barbeaux2020}{\textbf{Barbeaux2020?}}). Declines in phytoplankton biomass, forage fish abundance, and changes in community structure as a whole were also observed (\protect\hyperlink{ref-Suryan2021}{\textbf{Suryan2021?}}). During this recent period of environmental change, many pinniped populations have experienced increases or stabilization of population abundance (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}) (Figure 2), although declines in some Gulf of Alaska Steller sea lion populations were observed following the marine heat wave (\protect\hyperlink{ref-Suryan2021}{\textbf{Suryan2021?}}).

These variable changes in Alaskan pinniped populations over the past 50 years cannot be attributed to a single cause, as multiple environmental, anthropogenic, and ecological factors have changed simultaneously. For example, the rapid decline of the western stock of Steller sea lions between the 1970s and 1990s has been attributed to myriad factors, including change to the physical environment, competition with fisheries for common prey, predation, disease, and human-caused mortality (\protect\hyperlink{ref-Atkinson2008}{\textbf{Atkinson2008?}}). Glacier Bay harbor seal populations have primarily, but not exclusively, been impacted by the decline of sea ice, which provides a majority of their haulout sites (\protect\hyperlink{ref-Womble2010}{\textbf{Womble2010?}}). Population declines have also been associated with increased numbers of tour vessels, particularly in glacier fjords that provide important nursing and whelping habitat (\protect\hyperlink{ref-Jansen2015}{\textbf{Jansen2015?}}; \protect\hyperlink{ref-Matthews2016}{\textbf{Matthews2016?}}). The differences in pinniped population trends across the Gulf of Alaska and Bering Sea suggest varied environmental and ecological drivers underlying these dynamics. Interestingly, harbor seals and Steller sea lions that occur in the same geographic region (sympatric) have experienced different population trends over similar time period (Figure 2). Identifying trophic position trends through time that are shared, compared to changes that only impact a specific species or region, can elucidate how widescale ecological forcing versus localized change influence top predators and potentially explain variable population abundance trends.

Both harbor seals and Steller sea lions exhibit generalist, piscivorous foraging strategies, although differences in foraging range, body size, and diet exist. Adult harbor seals have high site fidelity, opportunistically forage 5 - 10 km from haulout sites and at depths \textless{} 200 m (\protect\hyperlink{ref-Lance2012}{\textbf{Lance2012?}}; \protect\hyperlink{ref-Lowry2001}{\textbf{Lowry2001?}}), and weigh up to 300 pounds. Steller sea lions are central place foragers known to migrate to prey aggregations on the continental shelf and oceanographic boundary zones (\protect\hyperlink{ref-Sinclair2002}{\textbf{Sinclair2002?}}; \protect\hyperlink{ref-Womble2006}{\textbf{Womble2006?}}). Foraging trips can last 1-3 days (\protect\hyperlink{ref-Maniscalco2006}{\textbf{Maniscalco2006?}}) with average distances of 133 km for adult females (\protect\hyperlink{ref-Merrick1997}{\textbf{Merrick1997?}}), although foraging trips are shorter in the breeding season (\protect\hyperlink{ref-Maniscalco2006}{\textbf{Maniscalco2006?}}). Adult females can weigh up to 800 pounds whereas adult males can exceed 2,500 pounds, indicating a higher energetic demand compared to harbor seals. Diet studies of Steller sea lions and harbor seals are spatially and temporally limited, and primarily utilize scat samples. In the Gulf of Alaska, gadids, cephalopods, and forage fishes are prevalent in both harbor seal and Steller sea lion diet (\protect\hyperlink{ref-Sinclair2002}{\textbf{Sinclair2002?}}; \protect\hyperlink{ref-Geiger2013}{\textbf{Geiger2013?}}), whereas salmonids are also important for harbors seals in Bristol Bay and Iliamna lake (\protect\hyperlink{ref-Hauser2008}{\textbf{Hauser2008?}}).

Stable isotopes have been used to reconstruct historical differences in diet and trophic position in Alaskan pinnipeds (\protect\hyperlink{ref-Hobson1997}{\textbf{Hobson1997?}}; \protect\hyperlink{ref-Hirons2001}{\textbf{Hirons2001?}}; \protect\hyperlink{ref-Brennan2019}{\textbf{Brennan2019?}}). These previous studies utilized bulk stable isotope analysis exclusively and were therefore limited in their inferential strength. Differences in the bulk \textsuperscript{15}N/\textsuperscript{14}N of consumer tissues can indicate either a trophic level change of the consumer or a change in nitrogen resources at the base of the food web. The specific cause of the isotopic variation cannot be ascertained from consumer bulk stable isotope values unless the data are paired with temporal information on \textsuperscript{15}N/\textsuperscript{14}N in primary producers. Lack of consistent, concurrent sampling of nitrogen stable isotope composition of primary producers therefore presents a challenge for previous long-term studies of the trophic dynamics of consumers from bulk stable isotope data. CSIA data address this challenge, as amino acids exhibit two distinct patterns in isotopic enrichment: trophic amino acids (i.e., glutamic acid, alanine, proline) become enriched in \textsuperscript{15}N with each trophic transfer and source amino acids (i.e., phenylalanine) show minimal change and thus are reflective of the base of the food web (\protect\hyperlink{ref-McClelland2002}{\textbf{McClelland2002?}}; \protect\hyperlink{ref-Chikaraishi2009}{\textbf{Chikaraishi2009?}}; \protect\hyperlink{ref-Ohkouchi2017}{\textbf{Ohkouchi2017?}}). With the ability to internally correct for expected changes in \textsuperscript{15}N/\textsuperscript{14}N at the base of the food web (\protect\hyperlink{ref-Feddern2021}{\textbf{Feddern2021?}}; \protect\hyperlink{ref-McMahon2021}{\textbf{McMahon2021?}}), CSIA allows for a more robust retrospective analysis of consumer trophic dynamics on decadal and century scales.

The objective of this work is to describe and compare changes in trophic ecology for Alaskan pinnipeds throughout the past century and investigate trophic position differences for sympatric populations. We apply hierarchical Bayesian analyses to 70 years of trophic position data derived from CSIA from pinnipeds (harbor seal and Steller sea lion) in the Gulf of Alaska, Bristol Bay, and a small population of freshwater harbor seals in Iliamna Lake, Alaska which is adjacent to Bristol Bay. We build on previous research examining pinniped nitrogen stable isotope composition (\protect\hyperlink{ref-Hobson1997}{\textbf{Hobson1997?}}; \protect\hyperlink{ref-Hirons2001}{\textbf{Hirons2001?}}; \protect\hyperlink{ref-Misarti2009}{\textbf{Misarti2009?}}; \protect\hyperlink{ref-Brennan2019}{\textbf{Brennan2019?}}) by adding two decades of data to the record (2000s and 2010s) and incorporating a broad spatial scope (Bristol Bay, Iliamna Lake, Gulf of Alaska). Additionally, by analyzing nitrogen stable isotopes derived from amino acids, we were able to control for known changes in nitrogen resources and phytoplankton composition at the base of the food web that can confound trophic position interpretations from bulk stable isotope data collected over decadal scales (\protect\hyperlink{ref-Feddern2021}{\textbf{Feddern2021?}}). Furthermore, by comparing trophic position dynamics across species and region through time, regional and location-specific ecological responses to a changing ecosystem can be identified.

\#\#Methods

\#\#\#Sample collection and analysis

Samples were obtained using methods described in (\protect\hyperlink{ref-Feddern2021}{\textbf{Feddern2021?}}). Briefly, harbor seal and Steller sea lion bones were sampled from specimens curated at the University of Alaska Museum of the North (Supplementary Information Table S1). Specimens were treated by maceration in warm water and soaked in a dilute ammonia solution then stored in acid free boxes. Adult specimens were sampled exclusively to avoid dietary differences between adults and juveniles. Specimens were classified based on species and region. We prioritized long-term temporal coverage in four regional classifications of harbor seals (Iliamna Lake, southeast Gulf of Alaska, northern Gulf of Alaska, eastern Bering Sea) and two regional classifications of Steller sea lions (eastern and western stocks) for a total of 6 species x region classifications. Specimens were extremely limited for the eastern Steller sea lion stock (n = 2) and Iliamna Lake harbor seas (n = 3). We also prioritized specimens with sex and age identifications, but these data were not available for some specimens. A total of 106 harbor seal and 21 Steller sea lion specimens were sampled representing the 1950s to 2010s (Figure 1).

Steller sea lions were classified according to the National Oceanic and Atmospheric Administration's (NOAA) distinct population segments, where Steller sea lions east of 144°W are considered the eastern stock and west of 144°W are considered the western stock (Figure 1). NOAA has identified twelve stocks of harbor seals in Alaska and, due to limitations of archived specimens, harbor seals were not able to be classified according to NOAA stocks. Instead, they were classified based on their range relative to the Steller sea lion stocks and utilization of marine versus freshwater habitats. Harbor seals that were west of 144°W, which included samples from the Prince William Sound and Cook Inlet/Shelikof Strait stocks (Figure 1), were classified as northern Gulf of Alaska harbor seals. Harbor seals that were located east of 144°W, which included samples from the Glacier Bay/Icy Strait, Sitka/Chatham Strait, Lynn Canal/Stephens Passage, Dixon/Capes Decisions, and Clarence Strait stocks (Figure 1), were classified as southeast Gulf of Alaska harbor seals. The Bristol Bay harbor seal stock was divided into two classifications, Bristol Bay referring to marine harbor seals, and Iliamna Lake referring to freshwater harbor seals (Figure 1). This allowed for comparison of three pairs of geographically overlapping classifications: western stock of Steller sea lions and northern Gulf of Alaska harbors seals, eastern stock of Steller sea lions and southeast Gulf of Alaska harbor seals, and Bristol Bay and Iliamna Lake harbor seals.

\#\#\#Trophic position calculation

Bone collagen within the samples was decalcified, acid hydrolyzed, derivatized and analyzed for compound-specific stable isotope analysis (CSIA) of nitrogen (\(\delta^{15}N\)) for 12 individual amino acids following the protocol described in (\protect\hyperlink{ref-Feddern2021}{\textbf{Feddern2021?}}). \(\delta^{15}N\) was measured as:
\begin{equation} 
  \delta^{15}N ( \textperthousand vs. air) =   
  [(\frac{^{15}N/^{14}N_{Sample}}{^{15}N/^{14}N_{Air}} -1)*1000]
  \label{eq:deltN2}
\end{equation}
Collagen samples were measured in triplicate with a laboratory standard containing a 12 amino acid mixture of known isotopic composition. Full analytical details are described in Appendix S1.

Trophic position was calculated using a harbor seal-specific trophic discrimination factor (difference in \textsuperscript{15}N/\textsuperscript{14}N between trophic and source amino acids in consumers for a trophic transfer; (\protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}})). This approach assumed trophic discrimination factors (TDF) derived from controlled feeding studies of harbor seals were similar to Steller sea lions. Applying a ``multi-TDF'' approach that combines both average and taxa-specific TDF can improve trophic position estimates in marine predators including pinnipeds (\protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}}; \protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}). The following equation was used to determine the trophic position of each sampled individual:
\begin{equation} 
Trophic Position =   
  \frac{\delta^{15}N_i - \delta^{15}N_o - TDF_{(i-o),j} - \overline{\beta}_{(i-o)}}{\overline{TDF}_{(i-o)}}+2
  \label{eq:TP}
\end{equation}
where, \(\delta^{15}N_i\) is the measured stable isotope composition of a trophic amino acid i in a sample and \(\delta^{15}N_o\) is the stable isotope composition of a source amino acid o in a sample. \(\overline{TDF}_{(i-o)}\) is the mean difference between given trophic amino acid \emph{i} and source amino acid \emph{o} across all consumers described in (\protect\hyperlink{ref-Nielsen2015}{\textbf{Nielsen2015?}}). \(TDF_{(i-o), j}\) is the trophic discrimination factor between trophic amino acid \emph{i} and source amino acid \emph{o} from a controlled feeding study of a specific consumer \emph{j}; here we use harbor seals from (\protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}}) (Table 1). \(\overline\beta_{(i-o)}\) is the mean difference in \(\delta^{15}N\) across aquatic phytoplankton between a specific trophic amino acid \emph{i} and source amino acid \emph{o} {[}(\protect\hyperlink{ref-Nielsen2015}{\textbf{Nielsen2015?}}); Table 1{]}. (\protect\hyperlink{ref-Nielsen2015}{\textbf{Nielsen2015?}}) also determined using multiple amino acids to estimate trophic position improves precision. Therefore, we used multiple trophic amino acids \emph{i} (alanine, glutamic acid, aspartic acid and proline) and one source amino acid \emph{o} (phenyalanine) to calculate trophic position (Table 1). These amino acids were chosen based on their prevalence in previous studies to derive parameters for equation 2, and their concentrations in bone collagen (see Appendix S1).

\#\#\#Model framework
Sex was considered as a predictor for trophic position, however, sex metadata were not available for all specimens. In order to evaluate difference in trophic position by sex, we fit linear statistical models to each individual trophic amino acid, by classification (species x region). These models took the following form:
\begin{equation} 
y_i = \alpha + \boldsymbol\beta x_i + \epsilon_i, \epsilon \sim N(0,\sigma)
  \label{eq:linsex}
\end{equation}
where, \(y_i\) is trophic position for an individual amino acid and \(\boldsymbol\beta\) is a vector of coefficients for the predictor, in this case sex, and \(\epsilon\) are residual errors assumed to be normally distributed with mean 0 and standard deviation \(\sigma\). There was not sufficient metadata for the eastern stock Steller sea lion population or the Iliamna Lake population and these two classifications were omitted from this analysis.

A Bayesian hierarchical mixed effects model was used to identify decadal change across pinniped classifications (species x region), and the degree to which these changes were shared by testing the effects of classification, decade, and a classification-decade interaction as either population-level (fixed) or group-level (random) effects (see candidate models in Table 2). Hierarchical models share information across `groups' to identify common responses, which refers to both decade and classification in this study. The interaction term allows for increased flexibility, letting each classification have slight departures from the group-level means. The mean and variance of pinniped trophic position for each region-species classification and decade were estimated using a generalized linear Bayesian hierarchical model with decade, population, and trophic amino acid as predictors:
\begin{equation} 
y_i = \boldsymbol\alpha + \boldsymbol\beta x_i + \epsilon_i, \epsilon \sim N(0,\sigma_y)
  \label{eq:linsex}
\end{equation}
\begin{equation} 
\boldsymbol\alpha_{k=1:k} \sim N(\mu_{\alpha,k},\sigma_{\alpha,k})
  \label{eq:linsex}
\end{equation}
where, for data point \emph{i}, \(\boldsymbol\beta\) is a vector of coefficients for the unpooled predictors (fixed effects, Table 2) and \(\boldsymbol\alpha\) is a vector of coefficients for the partially pooled group-level predictors (random effects, Table 2) for group \emph{k} (amino acid, decade or classification). At minimum, the \(\boldsymbol\alpha\) included a random term for the amino acid corresponding to data point \emph{i}, and depending on the model included up to a total of 4 random effects (also effects of decade, classification, and their interaction, Model 6 in Table 2). For each random effect included, \(\mu_{α,k}\) and \(\sigma_{α,k}\) are hyperparameters representing the mean and standard deviation of group-level effects on trophic position, for random effect \emph{k}. For models with more than one random effect, we assumed the deviations to be independent and uncorrelated. We considered models that included decade, classification, and the interaction between decade and classification either as fixed or random effects (e.g.~Model 4 v Model 6, Table 2), but did not consider models that included both as fixed and random (Table 2). Parameter estimates were obtained using the brms package ((\protect\hyperlink{ref-Burkner2017}{\textbf{Burkner2017?}}), version 2.14.4) in R (R Core Development Team 2021, version 3.6.2), which implements a Hamiltonian Monte Carlo sampler and its extension no-U-turn sampler (\protect\hyperlink{ref-Hoffman2014}{\textbf{Hoffman2014?}}) through Stan (Stan Development Team 2020). Minimally informative priors were used for random effects (normal distributions with a mean of 0 and variance of 10) and fixed effects (Student's t-distribution with a mean of 0, standard deviation of 2.5 and 3 degrees of freedom). Trophic amino acid was included as a random effect for all models (Table 2). Selection of the best models (Table 2) given the data was based on approximate leave-one-out cross-validation (LOOIC) using the loo package ((\protect\hyperlink{ref-Veharti2017}{\textbf{Veharti2017?}}), version 2.4.1).

\#\#Results
We found no differences between the average male and female pinniped trophic position over the 50-year study period (Figure 3) for the four tested species-region classifications. This finding was consistent for all trophic amino acids-source amino acid pairs (Figure 3). Based on glutamic acid trophic position estimates, both western stock Steller sea lions (2.6 ± 0.5; mean ± sd) and eastern stock Steller sea lions (2.7 ± 0.16) had similar trophic positions. Harbor seals in the Gulf of Alaska foraged higher in the food web than their Steller sea lion counterparts (Figure 3). Harbor seals in the southeast region had a higher trophic position on average than any other pinniped in this study (3.5 ± 0.3) but were similar to harbor seals in the northern region (3.3 ± 0.5). Bristol Bay (3.1 ± 0.4) and Iliamna lake (3.0 ± 0.3) harbor seals had a lower trophic position than their Gulf of Alaska counterparts on average (Figure 3).

\#\#\#Common trends in Alaskan pinniped trophic position

The best performing model (Table 2, model 6) of pinniped trophic position included both species-region classification and decade as random effects (shared trends) along with an interaction between population and decade (Table 2). Based on the support for decade and classification to be included as group-level effects, these data support consistent differences between classifications over time, as well as differences between trophic position for all classifications. The supported interaction between population and decade (Table 2) indicates distinct decadal changes in trophic position for species-region classifications exist. The model that included decade, classification, and the interaction between decade and classification as fixed effects (model 4) was also supported based on the models LOOIC (Table 2). Therefore, the inclusion of the interaction term was more important for improving model performance than inclusion of decade and classification as fixed versus random effects.

There were consistent differences in trophic position that varied by species and ocean basin for the model with the most support. Harbor seals in the Gulf of Alaska had higher trophic position than their Steller sea lion counterparts. The mean difference of the posterior distributions indicated southeast Gulf of Alaska harbor seals have historically fed at 0.32 {[}-0.01, 0.61{]} (highest density 80\% credible interval) trophic levels higher than sympatric eastern stock Steller sea lions (Figure 4). Similarly, the mean difference of posterior distributions showed northern Gulf of Alaska harbor seals fed 0.28 {[}-0.03, 0.50{]} trophic levels higher than the sympatric western stock Steller sea lions. Within the Gulf of Alaska, the posterior distributions for trophic position overlapped 39\% between harbor seals and Steller sea lions in both the eastern and western regions (Figure 4). Iliamna Lake harbor seals have historically fed at a lower trophic level (mean posterior difference 0.16 {[}-0.11, 0.41{]}) than harbor seals in Bristol Bay, but these two classifications have 66\% overlap of the group-level posterior distributions for trophic position (Figure 4). The 80\% credible intervals included 0 for most region-species classifications thus the posterior probabilities support marginal evidence for consistent differences in trophic position between classifications. Regardless, the differences in posterior means were large, although the distributions were wide.

There were no consistent decadal differences in trophic position across the region-species classification (Figure 5). Pinniped trophic position in the 2000s was slightly higher for all classifications (mean posterior difference 0.03 {[}-0.09, 0.16{]}) on average compared to 1990 and the posterior distributions for 1990 and 2000 had an 85\% overlap (Figure 5). Similarly, posterior distributions in between 2000 and 2010 had a mean difference of -0.1 {[}-0.27, 0.08{]} with a 65\% overlap (Figure 4). Overall, decadal differences in pinniped trophic position through time were smaller than the region-species classification effects and were likely ecologically inconsequential.

\#\#\#Spatial and temporal differences in pinniped trophic structure

Distinct decadal changes in trophic position were observed for each species-region classification and varied more than the shared decadal changes (Figure 6) as indicated by the decade-classification interaction. Most, but not all, pinniped classifications experienced substantial trophic level change in 2000 or 2010 but the magnitude and direction of this change varied by region-species classification based on the combined effects of decade, classification, and the decade-classification interaction (Figure 6). The recent decadal change in trophic position was most prominent for the western stock of Steller sea lions which had a mean trophic level decrease of 0.43 {[}-0.25, -0.60{]} from 1990 to 2000 (a percent decrease of 0.15) with only a 21\% overlap between the posterior distributions (Figure 6E). This decline in trophic position remained in the 2010s. A similar decline was observed in the southeast Gulf of Alaska harbor seals. This population experienced relatively stable trophic position from 1960-1990, which then declined on average by 0.31 {[}-0.19, -0.45{]} trophic levels in 2000 (33\% posterior overlap) (Figure 6C). In contrast, harbor seals in the northern Gulf of Alaska had variable trophic position across decades and had the highest trophic position in 2000 in contrast to their southeast Gulf of Alaska harbor seals and Steller sea lion counterparts (Figure 6B). Data were only available for 2000 and 2010 for the eastern stock Steller sea lions, and trophic position was similar for this population during both of these decades (Figure 6F). Both Bristol Bay and Iliamna Lake harbor seals had relatively stable trophic position from 1950s until 2010s (Figure 6A \& B). Bristol Bay harbor seals experienced their lowest trophic level in the 1990s with a 0.24 {[}-0.54, 0.00{]} trophic level decrease compared to the 1970s and 2000s, but the posterior distribution still overlapped 54\% with other decades (Figure 6A).

\#\#Discussion

Over the past 70 years, Alaskan pinnipeds have exhibited both common and distinct differences in trophic position across region-species classification on decadal scales (Table 2). While potential drivers of change in trophic position were not tested in this study due to data limitations, our results support a combination of local-scale (i.e., vessel traffic, reduction of glacial ice, local foraging) and regional-scale (i.e., environmental condition, basin-wide prey abundance) changes may be influencing pinniped trophic ecology. Furthermore, the largest decadal changes in pinniped trophic position were distinct for each region-species classification and were most apparent during the most recent two decades (2000s and 2010s). These patterns are more pronounced in the Gulf of Alaska compared to Bristol Bay (Figures 5 \& 6).

\#\#\#Regional and species trends in harbor seal trophic position

Both Steller sea lions and harbor seals exhibit generalist foraging patterns (\protect\hyperlink{ref-Lance2012}{\textbf{Lance2012?}}; \protect\hyperlink{ref-Geiger2013}{\textbf{Geiger2013?}}). Diets of Alaskan pinnipeds consist of similar prey species but vary between species, population, and local availability of prey (\protect\hyperlink{ref-Iverson1997}{\textbf{Iverson1997?}}; \protect\hyperlink{ref-Hirons2001}{\textbf{Hirons2001?}}). Bulk stable isotope studies in the Gulf of Alaska have shown that Steller sea lions feed lower in the food web compared to harbor seals (\protect\hyperlink{ref-Iverson1997}{\textbf{Iverson1997?}}). Our CSIA analysis confirms the interpretation of these previous studies that isotopic differences can be attributed to trophic position changes and not isotopic shift of basal phytoplankton resources. Both western and eastern stock Steller sea lions have lower trophic position compared to sympatric harbor seal populations but have similar trophic position compared to other populations such as Iliamna Lake. However, despite known differences in both diet (\protect\hyperlink{ref-Sinclair2002}{\textbf{Sinclair2002?}}; \protect\hyperlink{ref-Geiger2013}{\textbf{Geiger2013?}}) and nearshore versus offshore foraging (\protect\hyperlink{ref-Merrick1997}{\textbf{Merrick1997?}}; \protect\hyperlink{ref-Lowry2001}{\textbf{Lowry2001?}}) between the two species, our results also show historical overlap in trophic position, indicating potential trophic redundancy between harbor seals and Steller sea lions in the Gulf of Alaska.

Harbor seals in Bristol Bay and Iliamna Lake are managed as a single population (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}) despite lack of evidence of migration by the freshwater population and utilization of different resources (\protect\hyperlink{ref-Brennan2019}{\textbf{Brennan2019?}}). A previous study of strontium and carbon stable isotopes showed Iliamna Lake harbor seals utilize freshwater-derived resources (resident lake fishes), particularly early in life, and exhibit an ontogenetic shift to more marine resources (returning sockeye salmon) later in life (\protect\hyperlink{ref-Brennan2019}{\textbf{Brennan2019?}}). Based on CSIA nitrogen data, Iliamna Lake harbor seals also forage lower in the food web compared to Bristol Bay harbor seals. In addition, both classifications exhibited trophic stability, with the Bristol Bay harbor seals only experiencing a trophic shift in the 1990s relative to the 1960s and 1970s. This coincided with the lowest sockeye salmon returns to Iliamna Lake on record (\protect\hyperlink{ref-Hilborn2003}{\textbf{Hilborn2003?}}). Interestingly, the decrease in trophic position in the 1990s occurred simultaneously with decreases in basin wide Bristol Bay harbor seal abundance in the late 1990s, which then stabilized and increased in the 2000s and 2010s (Figure 2). Data were not available for the freshwater harbor seals between 1990 and 2000 and thus it is unclear whether the freshwater population also experienced a trophic position change during the 1990s when sockeye salmon returns were low. While quantitative comparisons to salmon abundance were not made in this study, salmon population abundance and harbor seal trophic ecology and population trends are seemingly interrelated.

\#\#\#Recent trophic position changes in the Gulf of Alaska

Trophic position changes were observed in all pinniped classifications in the Gulf of Alaska during the past two decades, although the direction of these changes varied on more local scales. During the past two decades (2000-2020), harbor seals in the Gulf of Alaska have experienced stabilization of most monitored populations following long-term declines that persisted from the 1950s through the 1990s (Figure 2, (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}})). During this same time period, harbor seals in both southeast and northern Gulf of Alaska experienced a shift in trophic position that was particularly prominent in the 2000s compared to historic estimates of trophic position (Figure 6B \& C). It is possible that the observed trophic position shift may have contributed to the population stabilization of Gulf of Alaska harbor seals, either by an increase in prey availability or opportunistically foraging on a novel prey source. (\protect\hyperlink{ref-Gagne2018}{\textbf{Gagne2018?}}) observed similar trophic position declines in seabird populations, which were attributed to a shift in diet from fish to squid. A similar dietary shift could explain the observed trophic position shift in southeast Gulf of Alaska harbor seals and western stock Steller sea lions.

Recent regional change in the Gulf of Alaskan food webs has been well documented in other species and primarily attributed to bottom-up effects of climate (\protect\hyperlink{ref-Barbeaux2020}{\textbf{Barbeaux2020?}}; \protect\hyperlink{ref-Litzow2020}{\textbf{Litzow2020?}}). These region-wide trends likely altered prey availability for pinniped populations in the Gulf of Alaska. How pinniped populations have adapted their foraging ecology, however, indicates regional and species trophic divergence, which could be attributed to either local-scale foraging adaptations or differences in prey availability. Pinniped groups that overlap in space (Figure 1) revealed divergent trends in trophic position between Steller sea lions and harbor seals in recent decades (Figure 6B \& E). For example, trophic position of northern Gulf of Alaska harbor seals increased in the 2000s while the western stock of Steller sea lions decreased. For western stock Steller sea lions, this shift also persisted into the 2010s (Figure 6E). Posterior distributions of western stock Steller sea lions and northern Gulf of Alaska harbor seals overlapped by 63\% in the 1950s but only overlapped by 3\% in the 2000s (Figure 6B \& E). The recent change in pinniped trophic position within the Gulf of Alaska coincided with population abundance stabilization, albeit at lower than historical abundance for most populations. This trophic divergence indicates there could be increased competition for resources between northern Gulf of Alaska harbor seals and western stock Steller sea lions resulting in diet adaptations. Similar comparisons were challenging to make for the eastern stock of Steller sea lions and southeast Gulf of Alaska harbor seals due to limitations in historical data for the former. However, trophic position in the 2000s showed a 38\% overlap (Figure 6C \& E) between the two species, indicating any trophic divergence between them may be less pronounced in this region, if existent.

The observed divergent trends indicate differences in how Alaskan pinnipeds are adapting to environmental and ecological changes. Trophic position changes from stable isotope data can be accounted for by: 1) prey switching between different species, 2) consuming different sizes of the same prey, or 3) consuming different quality prey. These changes can occur at the consumer level (pinnipeds) or lower in the food web and still be reflected in consumer stable isotope signature and thus trophic position. In recent decades, Pacific salmon and halibut in Alaska have both declined in size (\protect\hyperlink{ref-Holsman2019}{\textbf{Holsman2019?}}; \protect\hyperlink{ref-Oke2020}{\textbf{Oke2020?}}). These changes in size distributions of prey have been attributed to changes in marine mammal populations (Groskreutz et al.~2019) and likely contributed to the observed trophic position declines in western Steller sea lion and southeast Gulf of Alaska harbor seals. In contrast, consuming low-quality prey with lower protein content and greater amino acid imbalance between consumer and prey increases the amino acid trophic enrichment factor of nitrogen (\protect\hyperlink{ref-McMahon2015}{\textbf{McMahon2015?}}). If not accounted for in trophic position equations, this increase in trophic enrichment factor can result in erroneously high trophic position estimates. This may explain the observed increase in estimated trophic position in northern Gulf of Alaska harbor seals where this population may be consuming a greater proportion of lower quality prey (i.e., crustaceans, shrimp, cephalopods) in recent decades rather than feeding on prey species that are higher in the food web.

\#\#\#Considerations and limitations for CSIA analyses

The data in this study were limited in sample size primarily due to the availability of archived specimens. As a result, we were not able to discern between known fine scale differences in populations or annual trends. For example, harbor seals in the southeast Gulf of Alaska consist of 13 individual stocks. Due to limitations in the number of archived specimens, these stocks were pooled and analyzed as a single classification despite known differences in genetic structure (\protect\hyperlink{ref-Muto2020}{\textbf{Muto2020?}}). Given the observed broad range in trophic position of these generalist predators, it is unlikely that inclusion of finer spatial dynamics would have changed the supported model, although variation in temporal trends within a classification may have been identified. Similarly, data were only available for eastern stock Steller sea lions for 2000s and 2010s. As a result, no historical comparisons were possible and the conclusions about this population are tentative. Nonetheless, this dataset offers historic documentation of pinniped trophic position that can be updated with future samples or additional archived specimens.

Trophic position estimates in this study were low compared to known foraging strategies of these pinnipeds. For example, Steller sea lions eat primarily walleye pollock and Atka mackerel (\protect\hyperlink{ref-Hobson1997}{\textbf{Hobson1997?}}; \protect\hyperlink{ref-Trites2007}{\textbf{Trites2007?}}), which would indicate a trophic position of 3 or higher. Mean trophic position for Steller sea lions was closer to 2.7 in this study, which is lower than expected based on known foraging ecology. It is common for CSIA to underestimate trophic position of marine predators (\protect\hyperlink{ref-Germain2013}{\textbf{Germain2013?}}; \protect\hyperlink{ref-McMahon2019}{\textbf{McMahon2019?}}) and the inclusion of multiple amino acid pairs and a multi-trophic enrichment factor framework did not fully resolve this issue. (\protect\hyperlink{ref-Nielsen2015}{\textbf{Nielsen2015?}}) found trophic position estimates can be highly sensitive to the applied \(\beta\) values in equation 2. In our trophic position calculation, we assumed a constant \(\beta\) represented by marine diatoms. However, \(\beta\) values differ by more than 11 per mille between seagrasses and diatoms (\protect\hyperlink{ref-VanderZanden2013}{\textbf{VanderZanden2013?}}) which has been attributed to differences between vascular and nonvascular plants (\protect\hyperlink{ref-Ramirez2021}{\textbf{Ramirez2021?}}). If vascular plants, such as seagrasses, contribute to the food web in addition to non-vascular algae, the applied β would be too high and would result in underestimation of trophic position of marine consumers (\protect\hyperlink{ref-Ramirez2021}{\textbf{Ramirez2021?}}). Even a 10\% contribution of vascular plant-derived nitrogen to the food web would result in an underestimation of 0.2 trophic position. It is likely that vascular plants at least partially contribute to the Alaskan food web, as seagrass beds provide essential habitat and food for many fish species and invertebrates. Consideration for variable \(\beta\) values may be helpful in resolving trophic position underestimation of future studies, especially in cases where consumer carbon stable isotope data is available and contributions of seagrasses to the food web are well documented.

\#\#\#Conclusions and Implications

Marine ecosystems in Alaska are experiencing unprecedented environmental change that has altered abundance and size distributions of many fish species consumed by pinnipeds (\protect\hyperlink{ref-Barbeaux2020}{\textbf{Barbeaux2020?}}; \protect\hyperlink{ref-Holsman2019}{\textbf{Holsman2019?}}; \protect\hyperlink{ref-Oke2020}{\textbf{Oke2020?}}; \protect\hyperlink{ref-Suryan2021}{\textbf{Suryan2021?}}). Heterogeneity in diet and foraging locations allow top predators to adjust to availability of resources by altering their foraging. Based on the observed region-species specific changes in trophic position over the past two decades, pinnipeds are experiencing different food web conditions than in the past, even those that occur in similar geographic regions. This may be the result of adapting foraging strategies to exploit other prey resources or a change that is occurring lower in the food web and is measurable in predators. While our results cannot discern between these two mechanisms of trophic level change, we can conclude that recent food web dynamics have impacted pinniped trophic ecology in Alaska. Future responses of pinnipeds to food web change will likely be locally variable between species, even those that occur within similar geographic regions.

\clearpage

\#\#Tables

\textbf{Table} \ref{tab:paramval}: Parameter values for trophic discrimination factors between a trophic amino acid (\emph{i}) and phenylalanine (\emph{o}) for harbor seals (\(TDF_{(i-o), j}\)), for an average consumer (TDF(i-o) ), and for primary producers (\(\beta_{(i-o)}\)) derived from previous studies to apply a multi amino acid framework to equation 2.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{l>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}>{\raggedright\arraybackslash}p{10em}}
\caption{\label{tab:paramval}Trophic position parameter values for Equation 2}\\
\toprule
Trophic Amino Acid (i) & $\overline{\beta}_{(i-o)}$ & $TDF_{(i-o),j}$ & $\overline{TDF}_{(i-o)}$\\
\midrule
Glutamic acid (Glu) & 2.9 & 3.4 & 6.6\\
Alanine (Ala) & 2.8 & 2.5 & 6.8\\
Aspartic Acid (Asp) & 1.8 & 3.5 & 5.4*\\
Proline (Pro) & 2.7 & 5.5 & 5\\
Data Sources & Nielsen et al. 2015 & Germain et al. 2013 & Nielsen et al. 2015\\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage

\textbf{Table} \ref{tab:candmodels}: Candidate models for identifying spatial and temporal trophic structure of Alaskan pinnipeds. Assumptions define how the model describes trophic structure with regards to decade and classification and LOOIC describes the support of each candidate models. The best model (6) is italicized.

\begingroup\fontsize{8}{10}\selectfont
\begin{longtable}[t]{r>{\raggedright\arraybackslash}p{13em}>{\raggedright\arraybackslash}p{13em}>{\raggedright\arraybackslash}p{13em}l}
\caption{\label{tab:candmodels}Candidate Models}\\
\toprule
Model & Fixed Effects & Random Effects & Assumption & LOOIC  Standard error \\
\midrule
1 & Decade & Trophic Amino Acid & Trophic position varies by decade but not classification & 878.8 (-52.3)\\
2 & Classification & Trophic Amino Acid & Trophic position varies by classification but not decade & 816.5 (-52.3)\\
3 & Classification, Decade & Trophic Amino Acid & Trophic position varies by both classification and decade & 816.6 (-52.1)\\
4 & Classification*Decade & Trophic Amino Acid & Trophic position varies by classification and decade; decadal change is distinct for each classification & 797.9 (-53.1)\\
5 & - & Classification, Decade, Trophic Amino Acid & Trophic position varies with classification and decade but common trends exist across classification and decade & 813.7 (-52.6)\\
\addlinespace
\em{6} & \em{-} & \em{Classification*Decade, Trophic Amino Acid} & \em{Trophic position varies by classification and decade; decadal change is distinct for each classification. Common trends exist across classification and decade} & \em{771.4 (-53.1)}\\
\bottomrule
\end{longtable}
\endgroup{}

\clearpage

\hypertarget{conclusion}{%
\chapter*{Conclusion}\label{conclusion}}
\addcontentsline{toc}{chapter}{Conclusion}

If we don't want Conclusion to have a chapter number next to it, we can add the \texttt{\{-\}} attribute.

\textbf{More info}

And here's some other random info: the first paragraph after a chapter title or section head \emph{shouldn't be} indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there.

\appendix

\hypertarget{appendix-1}{%
\chapter{Appendix 1}\label{appendix-1}}

\hypertarget{full-analytical-details-for-bulk-stable-isotopes}{%
\section{Full analytical details for bulk stable isotopes}\label{full-analytical-details-for-bulk-stable-isotopes}}

Collagen samples have been analyzed for both CSSIA and bulk \(\delta^{15}N\) which require 10 mg of purified collagen (100 mg of bone). Preliminary analyses were conducted to determine the highest rate of collagen return from bone sampled from different parts of the skull to minimize destruction. Samples were taken from the internal occipital shelf to maintain external integrity. Bone was decalcified using 0.2 M HCl for 24-72 hours depending on bone thickness, followed by centrifugation and nanopure water rinse. Removal of humic acids was conducted using 0.125 M NaOH for 20 hours. Samples were washed to a neutral pH, then solubilized in 0.01N HCl. Once solubilized samples were blown down under N2 to prevent isotopic fractionation, and freeze dried. Freeze dried collagen was be analyzed for bulk isotopic composition of nitrogen by the UW IsoLab (isolab.ess.washington.edu) using a coupled elemental analyzer-isotope ratio mass spectrometer following the standard protocols of the laboratory. C:N ratios were calculated from this data, which is a measure of the quality for carbon and nitrogen analyses of bone collagen for isotopic analysis. Only three observations were outside of the acceptable rang of 2.7-3.6; indicating there was no substantial loss of glycine or addition of nitrogen due to microbial processing from mortality, decay, curation, and analysis.

\(\delta^{15}N\) of eleven amino acids were measured in the UW Facility for Compound-Specific Isotope Analysis of Environmental Samples. Samples were prepared following the procedures developed by Popp Marine Lab at University of Hawaii Manoa. Briefly, proteins were hydrolyzed in 6N HCl and purified using a cation exchange column. Amino acids were esterified using isopropanol acetyl chloride, and derivatized via acylation with 4:1 toluene: pivaloyl chloride. Samples were brought up in ethyl acetate and analyzed using a coupled gas chromatography-combustion-isotope ratio mass spectrometer system (GC-C-irMA; Thermo Scientific Trace GC + GC IsoLink coupled to a Delta V irMS) in continuous flow mode monitoring masses (m/z) 28 and 29 using a db-35 column. For each run a 12 amino acid external standard with known isotopic composition was injected three times followed by sample injections. Samples were injected in triplicate, with the 12 amino acid standard injected every two samples (or six injections). A two-hour column oxidation was performed after 6 samples (25 injections). Samples and standards included norleucine as an internal standard.

For each machine run, a linear model was fit for each individual amino acid using the following equation:
\begin{equation} 
  Std_{aa} = m_{aa}t + b_{aa}
  \label{eq:std}
\end{equation}
Where m represents the slope of the precision drift, \emph{t} represents the injection number since last column oxidation, and \emph{Std} represents the \(\delta^{15}N\) of an individual amino acid for a standard observation. The data was then corrected using the following equations:
\begin{equation} 
  D_{aa, t} = Std_{aa,t} - True
  \label{eq:diff}
\end{equation}
Where \(D_{aa,t}\) is the difference between an observed standard \(\delta^{15}N\) of \(Std_{aa,t}\) for a given amino acid at a given injection number and the true \(\delta^{15}N\) for that standard. Then:
\begin{equation} 
  Sample_{corrected,aa,t} = Sample_{obs,aa,t} - D_{aa,t}
  \label{eq:sampcorr}
\end{equation}
Where the drift value, \(D_{aa,t}\), is subtracted from the sample value for a given amino acid and a given injection to correct the observed sample values for precision drift since last column oxidation. Mean sample corrected values for the triplicate injections were used for all amino acid \(\delta^{15}N\).

\hypertarget{the-second-appendix-for-fun}{%
\chapter{The Second Appendix, for Fun}\label{the-second-appendix-for-fun}}

\hypertarget{colophon}{%
\chapter*{Colophon}\label{colophon}}
\addcontentsline{toc}{chapter}{Colophon}

This document is set in \href{https://github.com/georgd/EB-Garamond}{EB Garamond}, \href{https://github.com/adobe-fonts/source-code-pro/}{Source Code Pro} and \href{http://www.latofonts.com/lato-free-fonts/}{Lato}. The body text is set at 11pt with \(\familydefault\).

It was written in R Markdown and \(\LaTeX\), and rendered into PDF using \href{https://github.com/benmarwick/huskydown}{huskydown} and \href{https://github.com/rstudio/bookdown}{bookdown}.

This document was typeset using the XeTeX typesetting system, and the \href{http://staff.washington.edu/fox/tex/}{University of Washington Thesis class} class created by Jim Fox. Under the hood, the \href{https://github.com/UWIT-IAM/UWThesis}{University of Washington Thesis LaTeX template} is used to ensure that documents conform precisely to submission standards. Other elements of the document formatting source code have been taken from the \href{https://github.com/stevenpollack/ucbthesis}{Latex, Knitr, and RMarkdown templates for UC Berkeley's graduate thesis}, and \href{https://github.com/suchow/Dissertate}{Dissertate: a LaTeX dissertation template to support the production and typesetting of a PhD dissertation at Harvard, Princeton, and NYU}

The source files for this thesis, along with all the data files, have been organised into an R package, xxx, which is available at \url{https://github.com/xxx/xxx}. A hard copy of the thesis can be found in the University of Washington library.

This version of the thesis was generated on 2021-08-19 01:12:49. The repository is currently at this commit:

The computational environment that was used to generate this version is as follows:
\begin{verbatim}
- Session info ---------------------------------------------------------------
 setting  value                       
 version  R version 4.1.0 (2021-05-18)
 os       macOS Big Sur 10.16         
 system   x86_64, darwin17.0          
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2021-08-19                  

- Packages -------------------------------------------------------------------
 package     * version date       lib source                               
 assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.1.0)                       
 bookdown      0.23.1  2021-08-18 [1] Github (rstudio/bookdown@6643bb9)    
 cachem        1.0.5   2021-05-15 [1] CRAN (R 4.1.0)                       
 callr         3.7.0   2021-04-20 [1] CRAN (R 4.1.0)                       
 cli           3.0.1   2021-07-17 [1] CRAN (R 4.1.0)                       
 colorspace    2.0-2   2021-06-24 [1] CRAN (R 4.1.0)                       
 crayon        1.4.1   2021-02-08 [1] CRAN (R 4.1.0)                       
 DBI           1.1.1   2021-01-15 [1] CRAN (R 4.1.0)                       
 desc          1.3.0   2021-03-05 [1] CRAN (R 4.1.0)                       
 devtools    * 2.4.2   2021-06-07 [1] CRAN (R 4.1.0)                       
 digest        0.6.27  2020-10-24 [1] CRAN (R 4.1.0)                       
 dplyr       * 1.0.7   2021-06-18 [1] CRAN (R 4.1.0)                       
 ellipsis      0.3.2   2021-04-29 [1] CRAN (R 4.1.0)                       
 evaluate      0.14    2019-05-28 [1] CRAN (R 4.1.0)                       
 fansi         0.5.0   2021-05-25 [1] CRAN (R 4.1.0)                       
 farver        2.1.0   2021-02-28 [1] CRAN (R 4.1.0)                       
 fastmap       1.1.0   2021-01-25 [1] CRAN (R 4.1.0)                       
 fs            1.5.0   2020-07-31 [1] CRAN (R 4.1.0)                       
 generics      0.1.0   2020-10-31 [1] CRAN (R 4.1.0)                       
 ggplot2     * 3.3.5   2021-06-25 [1] CRAN (R 4.1.0)                       
 git2r         0.28.0  2021-01-10 [1] CRAN (R 4.1.0)                       
 glue          1.4.2   2020-08-27 [1] CRAN (R 4.1.0)                       
 gtable        0.3.0   2019-03-25 [1] CRAN (R 4.1.0)                       
 highr         0.9     2021-04-16 [1] CRAN (R 4.1.0)                       
 htmltools     0.5.1.1 2021-01-22 [1] CRAN (R 4.1.0)                       
 httr          1.4.2   2020-07-20 [1] CRAN (R 4.1.0)                       
 huskydown   * 0.0.5   2021-08-18 [1] Github (benmarwick/huskydown@addb48e)
 kableExtra    1.3.4   2021-02-20 [1] CRAN (R 4.1.0)                       
 knitr         1.33    2021-04-24 [1] CRAN (R 4.1.0)                       
 labeling      0.4.2   2020-10-20 [1] CRAN (R 4.1.0)                       
 lifecycle     1.0.0   2021-02-15 [1] CRAN (R 4.1.0)                       
 magrittr      2.0.1   2020-11-17 [1] CRAN (R 4.1.0)                       
 memoise       2.0.0   2021-01-26 [1] CRAN (R 4.1.0)                       
 munsell       0.5.0   2018-06-12 [1] CRAN (R 4.1.0)                       
 pillar        1.6.2   2021-07-29 [1] CRAN (R 4.1.0)                       
 pkgbuild      1.2.0   2020-12-15 [1] CRAN (R 4.1.0)                       
 pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 4.1.0)                       
 pkgload       1.2.1   2021-04-06 [1] CRAN (R 4.1.0)                       
 png           0.1-7   2013-12-03 [1] CRAN (R 4.1.0)                       
 prettyunits   1.1.1   2020-01-24 [1] CRAN (R 4.1.0)                       
 processx      3.5.2   2021-04-30 [1] CRAN (R 4.1.0)                       
 ps            1.6.0   2021-02-28 [1] CRAN (R 4.1.0)                       
 purrr         0.3.4   2020-04-17 [1] CRAN (R 4.1.0)                       
 R6            2.5.0   2020-10-28 [1] CRAN (R 4.1.0)                       
 remotes       2.4.0   2021-06-02 [1] CRAN (R 4.1.0)                       
 rlang         0.4.11  2021-04-30 [1] CRAN (R 4.1.0)                       
 rmarkdown     2.10    2021-08-06 [1] CRAN (R 4.1.0)                       
 rprojroot     2.0.2   2020-11-15 [1] CRAN (R 4.1.0)                       
 rstudioapi    0.13    2020-11-12 [1] CRAN (R 4.1.0)                       
 rvest         1.0.0   2021-03-09 [1] CRAN (R 4.1.0)                       
 scales        1.1.1   2020-05-11 [1] CRAN (R 4.1.0)                       
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 4.1.0)                       
 stringi       1.7.3   2021-07-16 [1] CRAN (R 4.1.0)                       
 stringr       1.4.0   2019-02-10 [1] CRAN (R 4.1.0)                       
 svglite       2.0.0   2021-02-20 [1] CRAN (R 4.1.0)                       
 systemfonts   1.0.2   2021-05-11 [1] CRAN (R 4.1.0)                       
 testthat      3.0.4   2021-07-01 [1] CRAN (R 4.1.0)                       
 tibble        3.1.3   2021-07-23 [1] CRAN (R 4.1.0)                       
 tidyselect    1.1.1   2021-04-30 [1] CRAN (R 4.1.0)                       
 usethis     * 2.0.1   2021-02-10 [1] CRAN (R 4.1.0)                       
 utf8          1.2.2   2021-07-24 [1] CRAN (R 4.1.0)                       
 vctrs         0.3.8   2021-04-29 [1] CRAN (R 4.1.0)                       
 viridisLite   0.4.0   2021-04-13 [1] CRAN (R 4.1.0)                       
 webshot       0.5.2   2019-11-22 [1] CRAN (R 4.1.0)                       
 withr         2.4.2   2021-04-18 [1] CRAN (R 4.1.0)                       
 xfun          0.25    2021-08-06 [1] CRAN (R 4.1.0)                       
 xml2          1.3.2   2020-04-23 [1] CRAN (R 4.1.0)                       
 yaml          2.2.1   2020-02-01 [1] CRAN (R 4.1.0)                       

[1] /Library/Frameworks/R.framework/Versions/4.1/Resources/library
\end{verbatim}
\backmatter

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\markboth{References}{References}

\noindent

\setlength{\parindent}{-0.20in}
\setlength{\leftskip}{0.20in}
\setlength{\parskip}{8pt}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\hypertarget{ref-Bird2002}{}%
Bird, A. (2002). {DNA} methylation patterns and epigenetic memory. \emph{Genes Dev.}, \emph{16}(1), 6--21.

\leavevmode\hypertarget{ref-Deans2015}{}%
Deans, C., \& Maggert, K. A. (2015). What do you mean, {``epigenetic?''} \emph{Genetics}, \emph{199}(4), 887--896.

\leavevmode\hypertarget{ref-Dineshram2016}{}%
Dineshram, R., Chandramouli, K., Ko, G. W. K., Zhang, H., Qian, P.-Y., Ravasi, T., \& Thiyagarajan, V. (2016). Quantitative analysis of oyster larval proteome provides new insights into the effects of multiple climate change stressors. \emph{Glob. Chang. Biol.}, \emph{22}(6), 2054--2068.

\leavevmode\hypertarget{ref-Eirin-Lopez2018}{}%
Eirin-Lopez, J. M., \& Putnam, H. M. (2018). Marine environmental epigenetics. \emph{Ann. Rev. Mar. Sci.}

\leavevmode\hypertarget{ref-Gavery2013}{}%
Gavery, M. R., \& Roberts, S. B. (2013). Predominant intragenic methylation is associated with gene expression characteristics in a bivalve mollusc. \emph{PeerJ}, \emph{1}, e215.

\leavevmode\hypertarget{ref-Gavery2017}{}%
Gavery, M. R., \& Roberts, S. B. (2017). Epigenetic considerations in aquaculture. \emph{PeerJ}, \emph{5}, e4147.

\leavevmode\hypertarget{ref-Gazeau2011}{}%
Gazeau, F., Gattuso, J.-P., Greaves, M., Elderfield, H., Peene, J., Heip, C. H. R., \& Middelburg, J. J. (2011). Effect of carbonate chemistry alteration on the early embryonic development of the pacific oyster (crassostrea gigas). \emph{PLoS One}, \emph{6}(8), e23010.

\leavevmode\hypertarget{ref-Gazeau2007}{}%
Gazeau, F., Quiblier, C., Jansen, J. M., Gattuso, J.-P., Middelburg, J. J., \& Heip, C. H. R. (2007). Impact of elevated {CO2} on shellfish calcification. \emph{Geophys. Res. Lett.}, \emph{34}(7), L07603.

\leavevmode\hypertarget{ref-Kurihara2007}{}%
Kurihara, H., Kato, S., \& Ishimatsu, A. (2007). Effects of increased seawater {pCO2} on early development of the oyster crassostrea gigas. \emph{Aquat. Biol.}, \emph{1}, 91--98.

\leavevmode\hypertarget{ref-Omoregie2019}{}%
Omoregie, E., Mwatilifange, N. S. I., \& Liswaniso, G. (2019). Futuristic ocean acidification levels reduce growth and reproductive viability in the pacific oyster (crassostrea gigas). \emph{J. Appl. Sci. Environ. Manage.}, \emph{23}(9), 1747--1754.

\leavevmode\hypertarget{ref-Roberts2012}{}%
Roberts, S. B., \& Gavery, M. R. (2012). Is there a relationship between {DNA} methylation and phenotypic plasticity in invertebrates? \emph{Front. Physiol.}, \emph{2}.

\leavevmode\hypertarget{ref-Timmins-Schiffman2014}{}%
Timmins-Schiffman, E., Coffey, W. D., Hua, W., Nunn, B. L., Dickinson, G. H., \& Roberts, S. B. (2014). Shotgun proteomics reveals physiological response to ocean acidification in crassostrea gigas. \emph{BMC Genomics}, \emph{15}, 951.

\leavevmode\hypertarget{ref-Timmins-Schiffman2013}{}%
Timmins-Schiffman, E., O'Donnell, M. J., Friedman, C. S., \& Roberts, S. B. (2013). Elevated {pCO2} causes developmental delay in early larval pacific oysters, crassostrea gigas. \emph{Mar. Biol.}, \emph{160}(8), 1973--1982.

\leavevmode\hypertarget{ref-Tomanek2014}{}%
Tomanek, L. (2014). Proteomics to study adaptations in marine organisms to environmental stress. \emph{J. Proteomics}, \emph{105}, 92--106.

\leavevmode\hypertarget{ref-Waldbusser2014}{}%
Waldbusser, G. G., Hales, B., Langdon, C. J., Haley, B. A., Schrader, P., Brunner, E. L., \ldots{} Gimenez, I. (2014). Saturation-state sensitivity of marine bivalve larvae to ocean acidification. \emph{Nat. Clim. Chang.}, \emph{5}, 273.

\end{CSLReferences}
\end{document}
